#Tangled Bank #45
Date: January 18, 2006

Everyone else it advertising it, I might as well:



[<img alt="The Tangled Bank" height="31" src="http://pharyngula.org/images/tbbadge.gif" width="88"/>](http://tangledbank.net/)<br/>
[Tangled Bank #45](http://www.greythumb.org/blog/index.php?/archives/65-Tangled-Bank-45.html) has been posted at GreyThumb.  Nothing else to see here, move along . . .

#We’re All Uncle Leo
Date: January 20, 2006

Just don’t let Elaine draw in your eyebrows.


The [New York Times has an article](http://www.nytimes.com/2006/01/20/business/media/20adco.html) profiling the advertising side of ScienceBlogs (you know, the thing that allows Seed to provide us with bandwidth so that we can spew our ivory tower propaganda far and wide).  Anyway, they refer to the type of people who like to read about science in their spare time as “Leonardos”.  And all this time I thought we were just dorks, dweebs, and nerds — but not [geeks](http://www.m-w.com/cgi-bin/dictionary?book=Dictionary&amp;va=geek) (they bite the heads off of live chickens).  To learn more about your bad-ass-Renaissance-man selves, check out the [Seed guide to Leonardos](http://www.seedmediagroup.com/renaissance/).


I also noticed that [P. Zed Myers has his own take](http://scienceblogs.com/pharyngula/2006/01/hiya_leonardo.php) on it — apparently he doesn’t like being a pawn in the money game.  Don’t hate the playa, P. Zed, hate the game.

#Does Interspecific Competition Mean Anything in an Evolutionary Context?
Date: January 20, 2006

The first thing I think when I read the title to this post is, “Man, that’s a long friggen title, and it’s not at all catchy.”  The next thing I think is, “I thought I said I wasn’t going to write about ecology.”  Well, I actually [wrote](http://scienceblogs.com/evolgen/2006/01/welcome_to_evolgen_at_scienceb.php):


> “I am in no way, shape, or form an organismal biologist nor am I an ecologist, and if you catch me out of my element, by all means, please set me straight.”


So, I invite you to put me in place if I’m talking out of my ass.  But why would I write about an ecological concept like interspecific competition?  This actually stems from a course that I am TAing this semester, entitled “Populations and Communities”.  The class is a sort of primer for ecology and evolution, introducing the students to behavioral ecology, population ecology, population genetics, and community ecology, among the general ecology and evolution topics.  For one of the laboratory experiments, the students study plant populations grown with and without another species present to examine the effects of interspecific competition (it is designed as an application of the [Lotka-Volterra model](http://www.blackwellpublishing.com/Townsend/Model/model05_l.html)).  This triggered a thought in my mind: “What does interspecific competition mean in an evolutionary sense?”  My mental meanderings can be found below the fold . . .





Natural selection, one of our favorite evolutionary forces, operates on individuals — certain genotypes are more fit than others and leave more offspring who, in turn, leave more offspring *ad infinitum*.  As a self proclaimed “reductionist”, I like to think of selection operating on the actual genotypes or, even better, the alleles (with the genotype merely part of the environment).  The reductionist within me (tiny little fellow) likes to think of it as selection operating on nucleotides, and the rest of the genome becomes part of the environment.  So, I asked myself, where can between species competition fit into this framework?  What difference does it make if an individual, or an allele, or a nucleotide must compete with a member of its own species or one from a different species?


Before I can let my mind wander any further down this road, I must define what I mean by interspecific competition.  For the purpose of this discussion, we will consider two seperate species that are diverged enough such that there is not dispute that they are different species.  These two species occupy the same (or very similar) ecological niches.  By definition, two species cannot occupy the same niche in the same space at the same time — one must shift its niche, move to a different location, or go extinct.  I will assume that the two species recently came into contact, and that they occupy extremely similar niches.


The one difference that stands out to me between intra- and inter-specific competition is that in the former there is a single gene pool, whereas in the latter there are two separate gene pools.  If we assume random mating within each species, we should expect a decrease in population size if there is interspecific competition.  Population size plays an important role in evolution.  Smaller populations experience more stochastic fluctuations in allele frequencies; this is analogous to flipping a fair coin (no matter how many times you flip it, you *expect* it come up heads half of the time, and tails the other half, but the more times you flip it, the higher the probability that the frequency of heads will be 0.50).  Population size also affects the ability of natural selection to fix beneficial mutations and purge deleterious ones.  The strength of selection is proportional to the fitness effect conferred by a particular allele (*s*) and the population size (*N*).  An allele with drastically beneficial or deleterious effects will be fixed or lost regardless of population size, but those alleles that are considered “weakly deleterious” or “weakly beneficial” will only experience significant selective pressures in larger populations.  This means that deleterious alleles are more likely to remain at appreciable frequencies and beneficial mutations have a lower probability of fixation in smaller populations.


At this point, I guess we can hazard a guess at the original question: yes, inter-specific competition does have an evolutionary meaning because it reduces population sizes.  We would expect that a species involved in a fight for limited resources with another species will have a lower population size than one not in such a competitive environment.  The end result would be more fluctuations in allele frequencies due to drift.  Additionally, these populations would have more mildly deleterious alleles segregating in their gene pool, and a beneficial mutation will have a harder time reaching fixation.


Let us dwell on natural selection for bit.  With a new competitor in their midst, each species faces a new pressure . . . or do they?  If we think of selection operating on individuals, it should be no different if an individual is competing with its own species or another species.  An individual (or new mutation) will be under selective pressures to exploit a new niche in either case, so does this selective pressure differ under interspecific competition?  The one situation I can think of in which it would, is selection for a larger [effective population](http://en.wikipedia.org/wiki/Effective_population_size) size.  The effective population size (the size of an idealized Wright-Fisher population that models our actual population) is affected by census population size, mating patterns (whether they are random or not), ancestral population sizes, and other demographic causes.  To maximize its effective population size, a population could undergo random mating, but this would come at a cost to those individuals who are reaping the benefits of non-random mating (ie, the dudes who are getting all of the chicks).  This makes me think switching the mating scheme is probably unlikely (does anyone know of an example where this has happened?).  Alternatively, if the species explored a new niche, it could increase its census population size which would lead to an increase in effective population size.  This pressure to explore a new niche would be greater under interspecific competition, as compared to intraspecific competition, because the initial population size would be smaller in the former scenario.  Here is where my limited knowledge of ecology forces me to pose the question, do populations that are competing with other species explore more niches than populations without such competition?


I have thought about how the [genetic relatedness](http://evolgen.blogspot.com/2005/10/can-you-marry-your-half-sisters_16.html) of individuals in the population affects interspecific competition, but I couldn’t figure out how to work it in to my framework.  For example, what role does kin-selection play in the relative effects of intra- and inter-specific competition?  Is interspecific competition fundamentally different from intraspecific competition because in the latter alleles/genotypes are competing with more closely related alleles/genotypes?  Any suggestions would be welcome.  Also, please point out any egregious mistakes or erroneous assumptions on my part.

#When Medical Doctors Take All of the Fun Out of Genetics
Date: January 20, 2006

According to Ken Maclean in a letter to Nature, patients don’t like it when clever geneticists name genes after cartoon characters, video game characters, and Monty Python characters (see [here](http://evolgen.blogspot.com/2005/12/competitive-world-of-gene-naming.html) if you don’t follow).  I’m not sure if I can reproduce the letter or if it’s under copyright protection (now that I’m [working for the man](http://scienceblogs.com/evolgen/2006/01/were_all_uncle_leo.php), I don’t wanna be breaking no laws), but it’s available [here](http://www.nature.com/nature/journal/v439/n7074/full/439266d.html).  The gist of it is, if you’re ill ([not like the Beastie Boys](http://www.amazon.com/gp/product/B0000024JN/qid=1137798650/sr=8-2/ref=pd_bbs_2/002-3473940-3958414?n=507846&amp;s=music&amp;v=glance), but like dying from cancer), you don’t wanna be told that your *Pokemon* or *Sonic hedgehog* gene is all fucked up.


---



**Update:** [Here](http://evol.mcmaster.ca/~brian/netevoldir/Other/Gene.names.Greek.mythology.answers) is some more on gene names inspired by mythology.

#There is no Controversy
Date: January 22, 2006

**Are There Disagreements Between the Fossil Record and Molecular Data?**


Molecular biologists have a tradition of reworking a lot of the evolutionary relationships and timescales that morphologists and paleontologists worked so hard to figure out.  This can really piss off the non-molecular folks, but I prefer to think of it as a cooperative relationship.  The [molecular clock](http://en.wikipedia.org/wiki/Molecular_clock), for example, would not be possible without calibration from the fossil record.  It is important to note that molecular and morphological data tell two different stories, which I outline below the fold . . .





When I wrote that [the Cambrian Explosion never happened](http://evolgen.blogspot.com/2005/02/cambrian-explosion-never-happened.html), I did not mean that the Cambrian Explosion never happened — I meant, the Cambrian Explosion never happened.  Confused?  Well, it all depends on how we define the Cambrian Explosion.  My preferred definition of the Cambrian Explosion is “The sudden appearance of many animal body plans in the fossil record.”  (Of, course, by sudden we mean over the span of millions of years.)  If we use that definition, then the Cambrian Explosion is a reality — there are some forms that did fossilize prior to the explosion.  But this definition is unsatisfying because it only tells us about the fossil record, not the underlying evolutionary process.


The Cambrian Explosion is just one example of an apparent disagreement between molecular and morphological data (and I’ll get back to it in a moment).  Before we return to it, we must outline what the fossil record and what the molecular data tell us.  The fossil record shows a preserved body plan — in the case of earliest body plans, it shows the first discovered body plan.  The molecular data, on the other hand, reveals the relationship of evolutionary lineages.  It is possible to reconstruct evolutionary relationships with morphological data (both extant morphologies and fossilized ones), but the molecular data does this [far more objectively](http://evolgen.blogspot.com/2005/07/genotype-and-phenotype-and-how-to.html).  The molecular data does not contain direct information regarding the actual time two lineages diverged (the molecular clock must be calibrated with the fossil record), nor does it inform us about the morphologies of the organisms in a particular lineage.


Even a perfect fossil record will give us different information than molecular data.  The fossil record documents the evolution of morphologies; the molecular data shows evolutionary relationships.  Getting back to the Cambrian Explosion, it may be possible that both the fossil record and molecular data are correct.  Many of those body plans may have first emerged during Cambrian.  The molecular data tells us that many of the lineages that gave rise to those body plans were around prior to the Cambrian, but it says nothing about what the animals looked like prior to the Cambrian.  Those lineages may have been present but in their ancestral body plans — some worm-like animal.  In that case, the Cambrian Explosion is a rapid diversification of body plans, but not a rapid evolutionary radiation.


Another example of disagreements between morphological and molecular data is the radiation of mammals.  The fossil record reveals many new mammalian body plans following the extinction of the dinosaurs.  The molecular data, however, suggests that the lineages that gave rise to the different mammalian body plans were present prior to the phenotypic diversification.  Both the Cambrian Explosion and mammalian radiation are examples of animals exploiting new niches that appeared after a major change in the earth’s climate.  In the case of the Cambrian Explosion, a [snowball earth](http://en.wikipedia.org/wiki/Snowball_earth#Cambrian_explosion) may have predated the appearance of new body plans.  When the earth thawed, new niches appeared, and the drastic morphological changes occurred.  The lineages that gave rise to those new morphologies were just “chilling” in the ocean.  In the case of the mammals, they were restricted to nocturnal critters while the dinosaurs filled all of the day-time niches.  After the catastrophic extinction of the dinosaurs, all of those mammalian lineages that were little shrew-like critters during the reign of the dinosaurs, evolved so as to take advantage of the newly vacated niches.


This logic can be extended to any other disagreement between molecular and morphological data.  It is important to realize that these are not conflicting sources of information, but complementary.  The fossil record informs regarding morphologies and absolute dates.  The molecular data reveals evolutionary relationships, but it is restricted to evolutionary lineages, not morphology or absolute time.  If we combine the fossil record and molecular data with geological information regarding major environmental changes, we can gain a clearer picture of historical evolution.

#Doctoring Your Way to a Doctorate
Date: January 24, 2006

If you have not read it, go check out [Nicholas Wade’s article](http://www.nytimes.com/2006/01/24/science/24frau.html?pagewanted=print) on doctored images in scientific publications.  This is especially pertinent given the recent Hwang Woo Suk stem cell debacle.  There is nothing all that revolutionary, but Wade gives a nice review and introduces us to some of the editors who are trying to catch the cheaters.  In commenting on the article, [John Hawks](http://johnhawks.net/weblog/topics/meta/photoshopping_science_wade_2006.html) brings up a good point regarding Photoshop:


> 
“I don’t worry too much about Photoshopping illustrations of fossils. Instead I worry about two things.
“One is picture selection. It is easy to choose pictures that support your argument and hide pictures that don’t. Sometimes these are really obvious, like when two bones are shown side by side in different orientations. When that happens, you just know that something isn’t what the paper is saying.”



I’m not too concerned with fossils — I think I’ve only seen [one scientific talk](http://evolgen.blogspot.com/2004/12/dinosaurs.html) that involved fossils — but I do see a lot of talks with pretty pictures of cells and tissues (one of the side effects of being an evolutionary geneticist is that you end up getting exposed to developmental and cellular genetics as well).  Keep in mind, I’m not talking about doctoring pictures of gels (where I think the biggest concern lies).  More on images in publications after the jump . . .





About a year ago, I was talking with another graduate student, and the topic of statistical rigor in different areas of genetics came up.  He also studies population geneticists, and we commented how in evolutionary genetics it’s impossible to not be [exposed to statistics that go over your head](http://evolgen.blogspot.com/2005/04/my-own-little-hell.html), and you have to do your best to tread water and make sense of all the math.  We also agreed that it was aggravating that in other areas of research, a lot of information is presented as a “sample size of one” (I’ll get to what I mean by that in a moment).


Now, I am not arguing that the mathematics in other areas of genetics is not important.  In fact, I think it’s often overlooked how important statistics are in all of science.  Researchers that use genetics as a tool to study development and cell biology do perform statistical tests that show significant differences in some quality between two genotypes.  I’m often surprised, though, by how limited the understanding of statistics is in the non-quantitative areas of genetics.


What bothers me most, however, is that aforementioned sample size of one.  Anyone who has seen a talk in which mutant and wild-type phenotypes are shown knows that there is usually one representative for each phenotype.  This is often the best picture the researcher could find of what they consider the archetypical mutant.  There could be different [penetrance](http://evolgen.blogspot.com/2005/04/penetrance.html), depending on environmental conditions which lead to variations in the expression of the mutant phenotype, but we never know because we are only shown a single picture.  Oftentimes, it’s not clear whether the mutant phenotype shown is found in all individuals with a particular genotype or just a few, and we’re seeing a picture of one of them.  I got so frustrated with this sample size of one that I began asking anyone who showed a picture of a mutant phenotype whether all of the individuals have the phenotype or if they just picked out a particularly good picture.  I think I stopped going to these types of talks before anyone admitted cheating (and I really don’t think it’s a very big deal, but it bothered me none the less).


But why do people manipulate images?  There’s the obvious reason that they needed to cheat in order to get their paper published (a major concern).  But there is also a lot of competition for who can produce the prettiest pictures.  If you want your paper to make the cover of an issue, it doesn’t hurt if you have some eye-catching illustration.  Being on the cover equals more exposure for your research, and every scientist wants exposure.  The *Drosophila* community also gives out awards for the [best published image](http://genetics.faseb.org/genetics/g-gsa/dros-image.shtml) of the year (I bet other research communities do this too), which is great for exposure.  I don’t know how big of a problem cheating is in these areas, but it’s not as big of a concern if you doctored your image to make it look better than if you did it to change your results.  It’s not like these people are adding or removing bands from gels, but there is the pressure to manipulate your images.

#Evolution, Development, and a Misphrased Question
Date: January 26, 2006

To be filed under: [You keep using that word. I do not think it means what you think it means.](http://www.imdb.com/title/tt0093779/quotes)


My name is not Inigo Montoya.  You did not kill my father.  And I couldn’t care less if you died or not.  But give me a god damn break here people.  If you want to ask a question, then ask the question.  There is no reason to not ask the question.


What question am I talking about?  As [Chris](http://scienceblogs.com/intersection/2006/01/brits_on_evolution.php) and [John](http://scienceblogs.com/strangerfruit/2006/01/surprising_numbers_from_britai.php) have pointed out, the Brits ain’t down with evolution.  [According to the BBC](http://news.bbc.co.uk/1/hi/sci/tech/4648598.stm):


> 
“More than half the British population does not accept the theory of evolution, according to a survey.
“Furthermore, more than 40% of those questioned believe that creationism or intelligent design should be taught in school science lessons.”



Find out what I think below the fold . . .





They reached this conclusion by asking people “what best described their view of the origin and development of life.”  The available answers were creationism, intelligent design, evolution, and no freakin’ idea.  Ignoring the fact that intelligent design is creationism (cheap tuxedo or not), this survey is still a load of crap.  As I pointed out in the comments on Chris’s blog, evolution is not a theory on the origin of life; evolution requires an imperfect replicator (ie, heritability).  Biological evolution does not mean anything without the biology — or something resembling biology, like self-replicating RNA.


Adding to the utter nonsense is their choice of the word “development”.  To a biologist, this word conjurs up [images of cells dividing and differentiating](http://scienceblogs.com/pharyngula/2006/01/textbooks_and_haeckel_again.php).  Combining *origin* with *development* makes me think of fertilization, cleavage, gastrulation, and suffering through developmental biology my junior year of college.  You know what best describes my view of the origin and development of life?  [Xenopus embryos](http://www-cbd.ups-tlse.fr/organismes/bles/bles.html).


Here is what I think the question should have looked like:


What best describes your understanding/belief regarding the current biodiversity of life on Earth?


- Evolution
- Intelligent design
- Biblical creationism
- I got no idea, man


I find it much easier to answer that question.  It’s still not perfect, because creationism and science aren’t comparable — one is taken on faith, the other on evidence.  But I doubt they would ask what I would really like to see them ask:


Where did all of these living things around us come from?


- Mutation, natural selection, and other natural forces led to the evolution of modern life.
- Goddidit.

#20th Anniversary of Challenger Disaster
Date: January 27, 2006

I may not be the oldest of the ScienceBlogs cohort (that’s an understatement, by the way), but I do remember the *Challenger* disaster.  I don’t remember watching it on TV.  I don’t remember seeing the explosion on the news.  I don’t think I even knew what news was or how I could watch it.  I was in kindergarten.  Among the few things I remember from that first year of school were wetting my pants and hearing that a spaceship exploded and a teacher was on board.  I didn’t realize that there were other teachers besides mine.  I knew my teacher was not dead, and this confused me.  This is among the ten earliest memories of my life.


Seed will be running a collection of responses to the question, “Where were you when the Challenger exploded?” on their [website](http://seedmagazine.com/).

#Is it just me, or is this just wrong?
Date: January 29, 2006

I am a [big](http://evolgen.blogspot.com/2005/12/toilet-seat-debate-settled.html) [fan](http://evolgen.blogspot.com/2006/01/tips-for-purchasing-llama_26.html) of [The Science Creative Quarterly](http://www.bioteach.ubc.ca/quarterly/).  I especially like how they integrate science with humor — sort of like [the Onion](http://www.theonion.com/content/index), but focused on science.  Now they are getting into science education.


If you know nothing about phylogenetics and systematics, [this introduction to phylogenetics](http://www.bioteach.ubc.ca/quarterly/?p=140) from the SCQ will be quite informative.  Of course, if you’re reading evolgen, you probably know something about phylogenetics — and if you don’t, read it and evolgen will make more sense.


The treatment of phylogenetics isn’t wrong, it’s just not quite right.  Take, for example, their distinction between taxonomy and systematics.  Systematics does not take “taxonomy one step further by elucidating new methods and theories that can be used to classify species.”  Systematics is the process of determining the evolutionary relationships between species.  Taxonomy, on the other hand, is the process of coming up with arbitrary categories and names for taxa (that’s my passive aggressive way of saying what I think about taxonomy).


Elsewhere in the science blogosphere, Razib offers us a [great account of the early days of population genetics](http://scienceblogs.com/gnxp/2006/01/every_ratio_31_1.php).  It also reminds me that I still need to read Will Provine’s *[The Origins of Theoretical Population Genetics](http://www.amazon.com/gp/product/0226684644/002-3473940-3958414)*.  For more about Provine, check out [this post](http://evolgen.blogspot.com/2005/09/william-provine-and-biological-meaning.html) a wrote a while back.

#Chad Is My Muse
Date: January 31, 2006

This blog doesn’t seem to want to write itself.  I’ve got a few posts in the pipeline (including the next on [detecting natural selection](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php)), but I can’t seem to finish them.  I’m in this writing funk where I start to lay some words onto paper (well, text editor, actually), and then I can’t organize all my thoughts or just can’t finish writing the post (do I have an undiagnosed case of ADD?).


Luckily for me, I have Chad at [Uncertain Principles](http://scienceblogs.com/principles/) to inspire me, as he’s already [done once before](http://scienceblogs.com/evolgen/2006/01/best_biology_experimentdiscove.php).  This time he’s asking people about their [least favorite misconception in their field](http://scienceblogs.com/principles/2006/01/cfp_least_favorite_misconcepti_1.php).  One [commentor](http://scienceblogs.com/principles/2006/01/cfp_least_favorite_misconcepti_1.php#comment-5991) brings up the misconception that evolution is random — something I’ve [blogged on before](http://evolgen.blogspot.com/2005/06/random-mutation-and-natural-selection.html).  I prefer to think of evolution as a combination of stochastic (mutation and drift) and deterministic (selection) processes.  The stochastic aspects consist of random draws from different probability distributions.  Random means something different to a statistician than it does to a layman — just like theory has a very specific meaning in science (it’s not just a guess) — and therein lies the misconception regarding the random nature of evolution.


Seeing as how I’ve already written about randomness, I won’t dwell on it as my favorite misconception.  Rather, I’ll focus on something that bothers the hell out of me: [the idea](http://www.nih.gov/news/NIH-Record/11_30_99/story02.htm) [that by sequencing](http://pharmabiz.com/article/detnews.asp?articleid=11275&amp;sectionid=46) [a genome](http://www.lanl.gov/orgs/pa/newsbulletin/2004/03/26/text05.shtml), [we are](http://www.earthvision.net/ColdFusion/News_Page1.cfm?NewsID=22584) [“decoding” it](http://usunrome.usmission.gov/UNIssues/HIV/docs/a4110101.htm) ([as you can see, I did not have a very hard time locating articles in the popular press or from official news releases that used that wording](http://www.google.com/search?hs=UwL&amp;hl=en&amp;lr=&amp;newwindow=1&amp;safe=off&amp;client=firefox-a&amp;rls=org.mozilla%3Aen-US%3Aofficial&amp;q=%22decode+genome%22&amp;btnG=Search)).


Genome sequencing consists of determining the primary structure (ie, linear sequence) of a few long molecules (chromosomes) found in the nuclei of cells.  Decoding implies translating from one language to another or determining some hidden meaning.  While we do have symbols for the four nucleotides found in DNA molecules (A, T, C, and G), this is not what we mean by the “[genetic code](http://molbio.info.nih.gov/molbio/gcode.html)“.  And even if we identify all hypothetical protein coding genes in a genome (thereby mapping all of the genetic codes), have we really decoded it?  I’d argue that we have not — I’m not saying that this is an arbitrary procedure, but it’s one thing to predict genes, and an entirely different thing to determine function.


Therein lies the problem; decoding a genome is a long and tedious (possibly endless) process.  To decode a genome, we would have to figure out the function for every gene product and how those gene products interact.  And even if we simplify decoding to merely identify genes, there is more to a gene that its protein coding sequence.  We still have very few algorithms to identify *cis* regulatory regions (and the ones we do have are nowhere near where they need to be for *de novo* identification).  The amount of transcriptional regulatory regions that are characterized by a genome sequencing project pales in comparison to the total number in a genome.  Not to mention that there are structural elements that receive very little treatment in the first few runs of analysis.


So, let’s abandon the idea of “decoding a genome” and refer to the process for what it is: sequencing euchromatin and preliminary analysis of protein coding sequences.  That’s my least favorite misconception (number two is the idea that [*Drosophila* are fruit flies](http://www.genetics.org/cgi/content/full/162/1/1)), what’s yours?

#It Beats Pairs Figure Skating
Date: February 16, 2006

Apparently, you can [fall down](http://www.nbcolympics.com/figureskating/5105184/detail.html) in figure skating and still win a silver medal.  Imagine if this happened in a real sport.  Say, for example, one of the [Italian hockey players](http://scienceblogs.com/evolgen/2006/02/news_from_the_olympics.php) falls to the ice and a Canadian player scores a goal.  Does the Italian team get a do over?  I don’t know what I hate more, figure skating or tape delay.


My favorite drunk skier, Bode Miller, was DQ’d after [straddling a post](http://).  I’m guessing that post won’t be the only think straddled in the [Olympic village](http://www.kegz.net/archives/2004/07/20/hard_bodies_in_the_olympic_village.html).  My second favorite drunk skier is [this guy](http://www.noobsports.com/2006/02/bode-miller-experiment-part-ii.html).  I’m especially fond of this description of his experiment on drunk skiing:


> During the timed portion, the test subject had a couple near misses with skiers on the mountain, and almost ate shit a couple times going around turns. Going off a couple of the rollers, the test subject broke his tuck position, and started “rolling down the windows” to keep from falling.


Rolling down the windows is sweet description (one I’ve never heard), and I’m gonna have to start using it.  The so called “Bode Miller Experiment” ended with the drunken skier failing on a 360 off of a catwalk (only making it 210 degrees around) and crashing in a magnificent yard sale.

#Now This is Intelligent Design
Date: February 24, 2006

My apologies for the utter lack of posting over the past week.  I’ve got stuff sitting around waiting to be written about, and I just haven’t been writing.  I’m not going to make excuses; I just haven’t been managing my time well.  While you wait for me to post again (soon, I promise), I give you [this](http://www.eurekalert.org/pub_releases/2006-02/dbnl-abi022306.php) article on “intelligently designing” promiscuous enzymes to perform specific functions.  Here’s a quote from the write up:


> 
According to the theory of divergent molecular evolution, primordial enzymes and other proteins started out as “promiscuous” so that primitive organisms would be better able to adapt to their environment. Driven by selective pressures, these promiscuous enzymes and other proteins evolved along divergent lines to acquire the specialized functions needed by a host organism to survive.
…
The Berkeley researchers identified the plasticity residues for the Grand fir sesquiterpene synthase, then systematically recombined mutations of these residues through site-directed mutagenesis, based on a mathematical model developed by Yoshikuni. Construction of the seven sesquiterpene synthases was accomplished with the screening of fewer than 2,500 mutants. An alterative approach, called directed evolution or molecular breeding, that is currently being tested at other laboratories, requires the screening of tens of thousands to a million or more mutants.



I wonder if this is how the Flying Spaghetti Monster works its magic.

#Evolution Wiki
Date: February 25, 2006

I have always been disappointed by the [EvoWiki](http://www.evowiki.org/index.php/Main_Page) — I found that I could get better information on evolutionary biology from the regular [Wikipedia](http://en.wikipedia.org/wiki/Main_Page).  Now some folks have organized the evolution content on Wikipedia into navigation templates.  I have not examined the content of the entries listed in these templates, but this seems like a cool idea.


> 
Evolution: http://en.wikipedia.org/wiki/Template:Evolution
Population genetics: http://en.wikipedia.org/wiki/Template:Popgen
Development of phenotype (or “genetic architecture”, its original name): http://en.wikipedia.org/wiki/Template:Genarch
Phylogenetics: http://en.wikipedia.org/wiki/Template:Phylo
Quantitative genetics: http://en.wikipedia.org/wiki/Template:Qg
Speciation: http://en.wikipedia.org/wiki/Template:Speciation
Evolution of chromosomes: http://en.wikipedia.org/wiki/Template:Chromo
Origin of life: http://en.wikipedia.org/wiki/Template:Origin_of_life



(Via [EvolDir](http://evol.mcmaster.ca/~brian/netevoldir/Other/WikipediaResources.EvolBiolTeaching).)

#Commenting on evolgen
Date: February 26, 2006

When ScienceBlogs first started out, it was sending out some bad cookies.  If you’re having trouble commenting on evolgen, delete all of your ScienceBlogs cookies so that you can get the new ones.

#What to do with Your Negative Results
Date: February 26, 2006

My advisor received an email from a fairly prominent geneticist regarding some results published by Dobzhansky over fifty years ago.  The geneticist had done some back of the envelope calculations and noticed some trends that had been overlooked for a half of a century.  We happened to have the animals to replicate the experiments (and I was planning on doing some similar experiments) so my advisor had me perform the crosses.  I ended up with a negative result — I did not see the same trends that Dobzhanksy and colleagues observed.  I guess you could say my negative result was a positive because I rejected a previous positive result.


I’m planning to publish my results (along with some additional statistical analysis) in a regular journal, but I could always submit it [here](http://www.jnr-eeb.org/) if I can’t get anyone to publish it.  From the Journal of Negative Results’ website:


> <p>
The primary intention of Journal of Negative Results is to provide an online-medium for the publication of peer-reviewed, sound scientific work in ecology and evolutionary biology that may otherwise remain unknown. In recent years, the trend has been to publish only studies with ‘significant’ results and to ignore studies that seem uneventful. This may lead to a biased, perhaps untrue, representation of what exists in nature. By counter-balancing such selective reporting, JNR aims to expand the capacity for formulating generalizations. The work to be published in JNR will include studies that 1) test novel or established hypotheses/theories that yield negative or dissenting results, or 2) replicate work published previously (in either cognate or different systems). Short notes on studies in which the data are biologically interesting but lack statistical power are also welcome. JNR also intends to present the results of studies in a format suitable for formal meta-analysis. Research quality is of highest importance for JNR. Manuscripts will be assessed for publication on this basis – positive results or support for current scientific dogma are not essential.</p>


(Via [Nature](http://www.nature.com/nature/journal/v439/n7078/full/439782a.html).)

#A Shout-Out to the New Peeps
Date: February 26, 2006

The [Frink Tank](http://scienceblogs.com/frinktank/) has [jumped the shark](http://evolgen.blogspot.com/2006/01/evolgen-jumps-shark.html) and joined the ScienceBlogs [evil empire](http://www.frinktank.com/blog/?p=145).  If you like your science with a dash of humor (and dick jokes) check them out.


Another blog, [Stoat](http://scienceblogs.com/stoat/), has come on board.  This one’s new to me, but it looks like it deals with climate sciences (and no dick jokes).

#Copy Number Polymorphism
Date: February 27, 2006

[I have](http://evolgen.blogspot.com/2005/10/human-genome-is-mess.html) a [little bit](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-1.html) of an [infatuation](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-2.html) with [copy number polymorphism](http://scienceblogs.com/evolgen/2006/01/are_deletions_deleterious_part.php) (CNP), which describes the fact that individuals within a population can differ from each other in gene content.  Some genes, such as [olfactory receptors](http://en.wikipedia.org/wiki/Olfactory_receptor) (ORs), have many different related variants in any animal genome.  New copies spring up via duplication events (a type of mutation), so one could imagine that individuals from a single population differ in the number of copies of these genes.  In fact, this is the case with any gene or gene family (a group of related genes) in the genome — there may be duplications segregating as CNPs.  A few recent publications deal with this phenomenon, and I summarize some of the findings below the fold.





<li>
**Nadeau and Lee [review](http://www.nature.com/nature/journal/v439/n7078/full/439798a.html) CNP (subscription required):** They point out that the number of copies of particular genes can be used to determine predisposition to certain diseases. In the [example](http://www.nature.com/nature/journal/v439/n7078/abs/nature04489.html) they use, a deletion of a second copy of the *Fcgr3* in rats makes them more susceptible to [glomerulonephritis](http://www.nlm.nih.gov/medlineplus/ency/article/000484.htm). Humans with fewer copies of the gene are also at a higher risk for the disease.  Interestingly, existing duplications can encourage new duplications because of the repeat mediated [molecular mechanism](http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=genomes.figgrp.8722) responsible for generating segmental duplications.  This means that genes that are duplicated have a higher probability of being duplicated again than those that have never been duplicated.  
</li>
<li>
**George Zhang’s group shows that [gene loss along the human lineage may be under positive selection](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0040052):** They focus on the CASPASE12 gene and argue that a null allele has nearly fixed in humans due to selection for resistance to [sepsis](http://www.nlm.nih.gov/medlineplus/ency/article/000666.htm).  After identifying pseudogenes (genes that are no longer functional) in the human genome, they used [SNP data](http://snp.cshl.org/) to determine whether the pseudogenes are common to all humans or unique to the few individuals who contributed their DNA to the genome projects.  They identified 36 OR pseudogenes out of 67 total pseudogenes despite the fact that OR genes only make up 2% of all functional genes in the human genome (ie, these genes get duplicated and encourage more duplicates, many of which are non-functional).  The CASPASE12 gene is part of a gene family that contains 11 functional genes in the human genome.  A null allele of CASPASE12 has fixed in non-African populations and is at a frequency of 89% in individuals of African descent.  The amount of [nucleotide sequence polymorphism](http://biology.plosjournals.org/perlserv/?request=slideshow&amp;type=figure&amp;doi=10.1371/journal.pbio.0040052&amp;id=44544) in the region flanking the mutation that makes the protein non-functional is decreased in null alleles relative to functional alleles, suggesting that [directional selection](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php) has favored the recent null allele.  Most summaries of CNP focus on the evolutionary importance of gaining extra copies of genes, but gene loss may also serve a purpose in adaptation to the environment.
</li>
<li>
**Chris Ponting’s group presents an analysis of [selection of human CNPs](http://genetics.plosjournals.org/perlserv/?request=get-document&amp;doi=10%2E1371%2Fjournal%2Epgen%2E0020020):** They found that [telomeres](http://en.wikipedia.org/wiki/Telomere) and [centromeres](http://en.wikipedia.org/wiki/Centromere) are enriched for CNPs — not surprising considering that these regions tend to be loaded with repeat sequences which encourage duplications.  Their data set included the entire genome (both protein coding and non-coding sequences), but they found that CNPs had more genes within them than expected if they were duplications of random sequences.  Genes associated with Mendelian diseases are underrepresented in CNPs, whereas genes involved in immunity and olfaction are overrepresented (the OR genes pop up again).  Genes segregating as CNPs have accumulated more non-synonymous mutations relative to synonymous mutations than other genes, indicating that they are either under [relaxed selective constraint or positive selection](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-3.html).  Additionally, CNPs that are the result of a deletion (rather than a duplication) do not show this pattern.  They also analyzed CNPs in mouse and found none of the trends I just described.  
</li>


We have known that [gene duplication](http://evolgen.blogspot.com/2005/10/name-that-tune.html) plays an important role in evolution for some time now.  It’s neat to see people examining the population genetics of duplicated genes and CNPs to understand how they evolve.

#Long Branch Attraction and the Branching Order of Deuterostomes
Date: February 27, 2006

You, me, your pet dog, and any other animal with a backbone are [deuterostomes](http://en.wikipedia.org/wiki/Deuterostomes).  So are [sea stars](http://www.octopusisland.com/images/shells/sugar_sea_star.jpg), [sea urchins](http://www.alaskaunderseatours.com/photos/Red-Sea-Urchin.jpg), and [sea cucumbers](http://www.afsc.noaa.gov/kodiak/images/photo/miscuke03.jpg).  During early development you, me, and echinoderms (sea stars et al) are a round ball of cells.  The ball of cells invaginates and that opening becomes our anus.  This differs from other animals like flies, worms, and snails whose first opening becomes a mouth.  A second opening forms later in development, and it becomes our mouth (hence the name “deuterostome”, or mouth second).  The deuterostomes can be broken into two groups: the chordates and the echinoderms.  The chordates include vertebrates and some less appreciated animals like [tunicates](http://www.auburn.edu/academic/classes/zy/0301/Topic3/tunicate.jpeg) and [lancelets](http://www.anselm.edu/homepage/jpitocch/genbios/34-04a-LanceletAnatomy-L.gif).


Why am I telling you all of this?  Well, a [recent paper](http://www.nature.com/nature/journal/v439/n7079/abs/nature04336.html) examines the evolutionary relationship of the deuterostomes, and comes to a surprising conclusion — the tunicates and vertebrates are close relatives, as are the lancelets and echinoderms.  In case you didn’t click the links and look at the pictures, the three groups of chordates seem pretty different from echinoderms.  Is this a case of conclusions being made using [subjective morphological data](http://evolgen.blogspot.com/2005/07/genotype-and-phenotype-and-how-to.html) being overturned by objective molecular data?  Of course I’m going to advocate the [awesomeness of molecular data](http://scienceblogs.com/evolgen/2006/02/excuse_me_while_i_repeat_mysel.php), right? Wrong.  Molecular data is only better than phenotypic data when the molecular data is good.  As [Martin](http://lancelet.blogspot.com/2006/02/old-icons-will-die-hard.html) at the appropriately named Lancelet has pointed out, there are some [long external branches](http://en.wikipedia.org/wiki/Long_branch_attraction) on that tree.  Long branches separated by short internal branches are a hallmark of unreliability in phylogenetics.  I just want to point out that I’m not a dogmatic molecular phylogenetics supporter — if you don’t understand what you are doing with the data, you can come to some erroneous premature conclusions.  I’m not saying that the authors of this study are wrong, simply that I would take their findings with a grain of salt.


This goes in the Nature loves publishing [evolutionary genetics papers](http://evolgen.blogspot.com/2005/10/evolution-of-drosophila-non-coding-dna_31.html) that [seem a bit off](http://evolgen.blogspot.com/2005/03/ripping-someone-new-asshole-using-only.html) file.

#What Makes a Conference Good?
Date: February 28, 2006

There’s a fun [article](http://www.nature.com/nrm/journal/v7/n3/abs/nrm1832.html) in *Nature Reviews Molecular Cell Biology* on what distinguishes a good scientific meeting from a not so good one.  The author advocates attending small meetings or workshops (under 100 people), which is tough for a young scientist.  Small workshops are usually either not well advertised or difficult to get to.  The only small meetings that I attend are local meetings, and the only workshops I go to are the workshops that are hosted at larger meetings.  For a young scientist, large nation/international meetings allow for the most interaction with the most people in your field.  Networking is important, and a young scientist needs to make as many connections as possible.  Granted, small meeting meetings are more intimate, but they also have less people.


As for my meetings, I try to go to at least one ‘far away’ meeting every year — last year it was [San Diego](http://www.drosophila-conf.org/genetics/gsa/dros/drosmenu2005.htm), this year I’m going to [Arizona](http://www.smbe.org/geb/).  I also try to make it to as many ‘local’ meetings that I can; whenever my university sponsors a meeting in a relevant field, I try my best to attend.  Also, if the meeting is a short drive away, I’ll usually go.  I once traveled to a [small meeting](http://evolgen.blogspot.com/2005/01/evolution-of-phenotype-and-genotype.html), but that was because I happened to be in the area for other reasons.  I always try to present either a poster or talk at any meeting I attend (it gives you something to talk about), gunning for a talk whenever possible.


I have reproduced the author’s tips for conference organizers and conference goers below the fold.





### [Ten tips for conference organizers:](http://www.nature.com/nrm/journal/v7/n3/box/nrm1832_BX1.html)


- The best part of any conference is the human interactions, and the best conferences are those that facilitate them.
- The ideal length for most meetings is less than 3 days — for big ones, maybe 4, no more.
- There’s no such thing as too much background, no matter how ‘expert’ the audience is supposed to be. Clarity of thought is the most important quality in a speaker. Meetings that recognize these things and choose their speakers accordingly will be successful.
- Gender and ethnic diversity in speakers and the audience helps in many ways.
- Even specialized conferences benefit from a few talks that are outside the box.
- The world can be divided into morning people and evening people. Meetings with a schedule that only caters to one type are making a mistake.
- Meetings with no free time aren’t just exhausting, they’re inefficient.
- Scheduling important talks opposite one another annoys the conference goers as well as the speakers. This is true even if the subjects are different.
- Nothing ruins a conference faster than inaudibility. (You can help: if you ask a question, make yourself heard; and if you’re a speaker, repeat the question so the whole audience can hear it — regardless of how good you think the sound system is.) And, everyone needs to be able to see the screen clearly.
- The setting matters. So does the food. Ambience is underrated.


### [Ten tips for conference goers:](http://www.nature.com/nrm/journal/v7/n3/box/nrm1832_BX2.html)


- In a developing country, don’t drink the water. In a developed country, don’t drink the beer.
- Don’t hesitate to give feedback: organizers need it. And don’t neglect positive feedback: organizers deserve it.
- Try not to hide, even if you’re shy. (I’ve met some of the best friends of my life at meetings.)
- Don’t monopolize poster presenters. And if you’re presenting a poster, don’t let one or two people monopolize you.
- If you only go to talks and meetings in your own field, you will never grow. Take a chance.
- You are what you eat. (Given this point, you might want to skip the rump roast.)
- Never underestimate the importance of a good night’s sleep.
- Life is too short to only go to meetings close to home.
- Life is too short to spend a week at a meeting.
- Have fun. Remember, science beats working for a living.

#Circus of the Spineless Numero 6
Date: February 28, 2006

[<img alt="i-0c0c3d352469dd15ba00a62422558e7c-spineless.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-0c0c3d352469dd15ba00a62422558e7c-spineless.jpg"/>](http://invertebrates.blogspot.com/)


The sixth edition of the [Circus of the Spineless](http://sciencepolitics.blogspot.com/2006/02/circus-of-spineless-6.html) is up at Science and Politics.  Get your fix of inverts now…

#Junk DNA is Junk
Date: March 23, 2006

The term “Junk DNA” is [bullshit](http://evolgen.blogspot.com/2005/10/evolution-of-drosophila-non-coding-dna_31.html).  There, I said it.  The moment I hear someone utter that phrase, I immediately lose respect for him or her.  No one whose opinion is worth anything will refer to non-coding DNA as junk.  That’s why [this article](http://www.eurekalert.org/pub_releases/2006-03/jhmi-jdm032306.php) bothers me.  The title, “Junk DNA may not be so junky after all” has nothing to do with the content of the article.  The research being described is not about showing that non-coding DNA has a function.  The function has already been determined.  The researchers have used zebrafish to identify enhancers for human genes.  The subtitle to the article (Researchers develop new tool to find gene control regions) actually describes the research pretty well.  The actual title is a load of crap.

#A Taxonomy of Evolutionary Geneticists
Date: March 23, 2006

[Alex Palazzo](http://scienceblogs.com/transcript/2006/03/listen_up_all_you.php) managed to piss off some people with his taxonomy of biomedical disciplines.  We have also learned that there are different types of [physics geeks](http://scienceblogs.com/principles/2006/03/know_your_geeks.php) and [anthropologists](http://scienceblogs.com/afarensis/2006/03/21/taxonomy_of_anthropology/). (By the way, don’t ever call me a geek; [geeks](http://www.m-w.com/cgi-bin/dictionary?va=geek) bite the heads off of chickens.  I’m a nerd.)  I previously attempted to [classify evolutionary biologists](http://evolgen.blogspot.com/2005/05/more-on-darwinists.html) and named them after the important names in their particular field.  It was actually a satire of the creationist ploy to call people Darwinists, so laugh.  Now I’m going to further divide up the evolutionary geneticists (already a sub-set of biologists) into a bunch of other groups.  Let’s call it an exercise in [over-splitting](http://evolgen.blogspot.com/2005/05/split-up.html), which really makes me feel like a taxonomist.  [SNAP!](http://www.urbandictionary.com/define.php?term=snap)  Find out where you fit in below the fold.





- **Classical population geneticists:** These people are loyal to the early theory of Wright, Fisher, et al, and [they](http://drrob.typepad.com/hpb_etc/) still argue about it.  They probably never stepped foot in a wet lab, or if they did it was to run an allozyme gel.  They still have not caught wind of the molecular revolution.  Chances are they are either emeritus, dead, or a historian/philosopher of biology.  Talk to these guys if you want to know all of the theory that allows you to study evolutionary genetics.
- **Molecular ecologists:** These guys are really just ecologists (aka, people that like to count things) who have discovered that if they run a few PCRs they can infer population structure in their species of choice. They can easily be distinguished from other molecular population geneticists by their interest in non-model organisms and crappy molecular markers (microsatellites, mtDNA, rRNA).
- **Molecular population geneticists:** They have sequenced multiple alleles of some gene (probably Adh) from a single species (of *Drosophila*).  They then ran it through the [same stable](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php) of [tests for selection](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php) that everyone else does.  You can recognize the higher ranking members of this group by their knowledge of [coalescent theory](http://scienceblogs.com/evolgen/2006/02/the_coalescent.php).
- **Human population geneticists**: They use the same data analysis techniques as the molecular population geneticists, but they don’t do any of the sequencing themselves.  Instead, they mine [HapMap](http://www.hapmap.org/) for SNPs to find out how humans migrated around the world and answer important questions like what genes make Scandinavians blonde.
- **Systematicists:** These guys are the taxonomists of evolutionary geneticists, only they study systematics and not taxonomy.  Anyway, they basically try to find out [how different species are related](http://tolweb.org/tree/) and they come up with [phylogenies for a bunch of species](http://scienceblogs.com/evolgen/2006/03/a_very_pretty_picture.php).  They may work with the molecular ecologists to do some phylogeography.
- **Evolutionary developmental biologists:** These guys study evo-devo, but calling them evo-devo-ers sounds like they’re being labeled [evildoers](http://www.cafepress.com/votevotevote/415636) or they like guys in [funny hats](http://images.amazon.com/images/P/0946719497.01.LZZZZZZZ.jpg).  They are characterized by their love of [taking *cis* regulatory regions from one species and attaching them to a gene from another species](http://dx.doi.org/10.1038/nature03235).  They are really no different from developmental biologists, except they incorporate something about evolution in the abstract of their paper.
- **Data miners:** Realizing that all of the genes they’ll ever need have already been sequenced and deposited in the [NCBI](http://ncbi.nlm.nih.gov/) database, these guys’ grant applications include requests for faster computers, bigger servers, and grad students from the computer science department. They have taken over many biology departments because they don’t require any wet-lab space.  Using their fancy computer programs, they compare genomes searching for highly conserved sequences, rapidly evolving sequences, and other things that some biologist told them are of interest.  Chances are, they have never actually seen the organism they are studying.
- **Molecular evolutionists:** Every taxonomic group needs its own grab bag of species that do not fit in any other taxon.  These guys may sequence genes from multiple species, but they aren’t all that interested in systematics.  They usually don’t sequence enough individuals from a single species to be considered population geneticists.  They employ many of the same analytical techniques that the data miners do, but they use their own data.  They really like to compare [synonymous and non-synonymous substitutions](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-3.html) and [rates of evolution along lineages](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-4.html).


Now it’s your turn to tell me how I misclassified your discipline, or, even worse, left your important area of study off of my list.  Go ahead and bitch away in the comments or add a trackback to a post on your blog.  Allow me to point out, however, that I probably fall into at least half of these (paraphyletic) taxa.


---



**Added about 10 minutes after I posted this entry:** I totally forgot about all of the people that study the [genetics of speciation](http://scienceblogs.com/evolgen/evolution/speciation/), like [Coyne and Orr](http://www.amazon.com/gp/product/0878930892/).  They kind of fit in with the molecular population geneticists, molecular ecologists systematicists, and classical population geneticists, but they definitely deserve their own taxon.

#If You Study Genomics, John Hawks Hates You
Date: March 24, 2006

The title is, obviously, an overstatement, exaggeration, and blatant misrepresentation.  But it gets your attention, don’t it?  Anyway, Hawks has been questioning whether genomics is really any different from genetics.  In [this post](http://johnhawks.net/weblog/2006/03/23#genomics_etymology_2006), he says something that I’ve been thinking for a while:


> “Actually, the thing that bothers me about genomics is not the renaming — I’m totally fine with coming up with clever new names, even if they carry little information content . . . What bothers me is that so many people are being trained in genomics without learning any genetics!”


I agree.  Entirely.  I know a lot of people studying genomics who never took a course in either classical genetics or molecular genetics.  Some have taken one, but not the other.  Furthermore, there are a lot of people studying evolutionary genomics who have never taken a course in population genetics.  I know one faculty member who will no longer sit on someone’s graduate committee unless they take his population genetics course (unless he’s the outside faculty member).  I’m running into more an more people who have come from a non-biology background and don’t even know the basic biology behind their system of interest.  You’d think that if your background is in the computational sciences and you made the jump into the life-science, you would enjoy taking a more mathematically oriented course like popgen.

#Double Entendre Friday – 24 March 2006
Date: March 24, 2006

I’m running out of good biology related double entendres, and I want to wait a few more weeks before I post those that people suggested in the comments or that I stole from other blogs (don’t worry, I’ll give you mad props if I jones something off you).  So, for this week’s evolgen double entendre, I give you an essay that I came across in high school.  To give some context, we were required to write a timed essay on the essay during class.  I spent the first five to ten minutes (of the allotted fifty) trying to compose myself before I could even begin to think about what I would write.  This week’s evolgen double entendre comes courtesy of [E.M. Forster](http://en.wikipedia.org/wiki/E._M._Forster)‘s essay, [“My Wood”](http://www.moonstar.com/~acpjr/Blackboard/Common/Glossary/Allusions.html).  The (pardon the pun) money quote (I’ve taken the artistic liberty to crop it to enhance the sexual innuendo):


> What’s the effect on me of my wood? In the first place, it makes me feel heavy . . . and it was a man of weight who failed to get into the Kingdom of Heaven . . . he stuck out in front . . . and as he wedged himself this way and that in the crystalline entrance . . . he saw beneath him a comparatively slim camel passing through the eye of a needle . . . My wood makes me feel heavy. In the second place, it makes me feel it ought to be larger . . . Pray, does my wood belong to me or doesn’t it? . . . There is a wood near Lyme Regis . . . where the owner has not hesitated on this point . . . He really does own his wood, this able chap.


The creative creationist style quote mining doesn’t do this essay justice.  It’s really about a man’s struggle to understand what the ownership of property means too him.  Calling it “my property” or “my land”, however, would not have the same effect as calling it “my wood” (hehehe, wood).

#My Blog Don’t Own Me
Date: March 27, 2006

[Alright, already!](http://scienceblogs.com/grrlscientist/2006/03/how_much_of_you_does_your_blog.php)  I took [the quiz](http://wannabegirl.org/quiz/owned/).  12.5% is like the equivalent of one leg, right?  That’s not too bad.


<center></center>

<td bgcolor="#FF6800" width="40"><center><font color="black" face="ocr a extended" size="2">12.5 %</font></center></td><td bgcolor="#003060"></td>




#Junk DNA Redux
Date: March 27, 2006

My little [screed on junk DNA](http://scienceblogs.com/evolgen/2006/03/junk_dna_is_junk.php) elicited some good feedback, including a comment from [Dan Graur](http://nsm.uh.edu/~dgraur/).  In a somewhat ill-thought out rant, I implied that anyone who uses the term ‘junk DNA’ should be ostracized from the scientific community (or something along those lines).  I restated my opinion in a far more diplomatic manner in the discussion that followed in the comments: junk DNA is an appropriate term for DNA that serves no function (non-transcribed, non-regulatory, and non-structural), but we should refrain from using that term for all non-coding DNA.  I elaborate my opinion and reference a tangentially related finding below the fold.





One problem I have with describing DNA as non-functional is that it is a null hypothesis; we can disprove non-function, but we cannot prove that a particular sequence is non-functional.  As Dr. Graur mentioned in the comments:


> Finally, it should be noted that “junk DNA” is a scientific theory (sensu Popper) because the theory can be refuted by finding a function for a sequence that has been previously assumed to lack function. The theory “there is no junk in the genome” is a religious statement, because it can never be refuted. It can always be said that one has not looked hard enough for a function.


We can think of junk DNA like we do the neutral theory.  The neutral theory is used as the null hypothesis in tests for detecting natural selection.  Junk DNA could be used in a similar manner in detecting functional untranscribed sequences.  However, I’m not entirely sure that transcription is a great predictor of function.  Not all transcribed RNAs are translated.  The ones that are untranslated may be rRNAs, tRNA, regulatory RNAs, and enzymatic RNAs, but others may just be transcriptional ‘mistakes’.


I don’t mind if someone uses the term junk DNA so long as it is clear what is meant.  As I pointed out in the comments, the press surrounding the paper in question (including [this article](http://www.seedmagazine.com/news/2006/03/junk_dna_in_the_trunk.php) from Seed) implies that geneticists think that all non-coding DNA is junk.  Junk DNA would only represent a subset of all non-coding DNA in the genome.  It’s safe to assume that most microsatellites are junk.  Incomplete pseudogenes are probably junk, although some [new work](http://compbiol.plosjournals.org/perlserv/?request=get-document&amp;doi=10%2E1371%2Fjournal%2Epcbi%2E0020046%2Eeor) suggests that some may be functional.  Unless you are fairly confident that a sequence is non-functional, you should refrain from referring to it as junk.


As for the tangentially related research that I mentioned above, David Begun and colleagues have identified [novel genes in Drosophila genomes](http://www.genetics.org/cgi/content/abstract/172/3/1675).  They were interested accessory gland protein genes (Acp’s; proteins associated with seminal fluids).  Previous research has shown that these genes evolve rapidly and are under positive selection.  In searching for Acp’s in multiple genomes, they identified genes that were unique to some species.  From the abstract:


> A genomic analysis of previously unknown genes isolated from cDNA libraries of these species revealed several cases of genes present in one or both species, yet absent from ingroup and outgroup species. We found no evidence that these novel genes are attributable primarily to duplication and divergence, which suggests the possibility that Acp’s or other genes coding for small proteins may originate from ancestrally noncoding DNA.


New genes have appeared in previously non-coding sequences.  Would you characterize these non-coding regions as junk?  It seems that one genome’s trash is another genome’s treasure.

#Contact Your US Representative about the NIH Budget
Date: March 29, 2006

As I [mentioned previously](http://scienceblogs.com/evolgen/2006/03/the_first_step_in_increasing_t_1.php), the Senate passed the Specter-Harkin Amendment.  The House of Representatives is our next target to ensure the Bush’s anti-science budget is improved upon.  The [Genetics Society of America](http://www.genetics-gsa.org/) has done an excellent job keeping its members up to date on these political goings on.  I have included the most recent request to contact your US Representative below the fold.  Follow the instructions to send an email, letter, or to get your representative’s office phone number to request he or she support the budget amendment.





ACTION:  Call or E-Mail Your US Representative! Ask your Member of Congress to support the NIH by increasing health and education funding in the House budget resolution. Ask them to support the efforts to include an additional $7 billion for health and education programs to the FY 2007 budget.


1) If your Representative is a Member of the Budget Committee, ask him/her to support Rep. Rosa DeLauro’s (D-CT) Amendment when it is offered in the Budget Committee.  A list of Budget Committee Members of Congress is posted at the end of this message.


2) If your Member of Congress is not a Member of the Budget Committee, ask him/her to Support Rep. Michael Castle’s (R-DE) plan to amend the budget resolution on the House Floor to include $7 billion the Senate added for health and education programs to the FY 2007 budget. 


ACTION IS NEEDED NOW.  The Congressional schedule has not been finalized, but the House Budget Committee is expected to consider the resolution this week with floor consideration of the measure likely to take place during the week of April 3.


How to Contact Your Member of Congress


— Your voice matters – phone calls as well as emails will make an impact on the way your Representative will vote.  Remember, this is a matter of urgency.


— Simply visit [http://capwiz.com/jscpp/home](http://capwiz.com/jscpp/home) .  At this site, you should be to follow the step-by-step instructions for calling/emailing your Congressional representative.   A sample letter will be provided for you. 


— At the above website, by simply entering your zip code, you will be given the name, address, phone number, fax, website, and email address of your Member of Congress. 


— If you call, don’t hesitate to leave a message with the receptionist or on voice mail.  You may also ask to speak to the Representatives Health Care Legislative Assistant.  Be sure to leave your message, you name, your phone number and if you have time your address.  Keep your message brief and to the point. 


Background


The House of Representatives is taking up the President’s Budget request.  As you know, the Senate added money to the section of the budget that funds the NIH thanks to Senators Specter (R-PA) and Harkin (D-IA).


Rep. Rosa DeLauro (D-Conn.) plans to offer the Specter-Harkin amendment during the House Budget Committee consideration of the resolution.  Rep. Mike Castle (R-Del.) has agreed to take the lead on supporting the Specter-Harkin amendment when the budget is debated on the floor of the House.


Remember As a community we were able to come together to urge a cross section of Republican Senators to vote in favor of providing additional funding for health care and education programs.  We need to exert the same all out effort in contacting all House Members – Republicans and Democrats – to deliver the message that the FY 2007 budget must include an additional $7 billion for health and education programs.


House Budget Committee


Republicans


- Jim Nussle, IA (Chairman) 
- Jim Ryun, KS
- Ander Crenshaw, FL  
- Adam Putnam, FL 
- Roger Wicker, MS 
- Kenny Hulshof, MO 
- Jo Bonner, AL  
- Scott Garrett, NJ
- J. Gresham Barrett, SC 
- Thaddeus McCotter, MI  
- Mario Diaz-Balart, FL 
- Jeb Hensarling, TX  
- Ileana Ros-Lehtinen, FL
- Dan Lungren, CA 
- Pete Sessions, TX
- Paul Ryan, WI  
- Mike Simpson, ID 
- Jeb Bradley, NH
- Patrick McHenry, NC
- Connie Mack, FL
- Mike Conway, TX
- Chris Chocola, IN


Democrats


- John Spratt, SC (Ranking Member)
- Dennis Moore, KS
- Richard Neal, MA
- Rosa DeLauro, CT
- Chet Edwards, TX
- Harold Ford, Jr., TN
- Lois Capps, CA
- Brian Baird, WA
- Jim Cooper, TN
- Artur Davis, AL
- William Jefferson, LA
- Thomas Allen, ME
- Ed Case, HI
- Cynthia McKinney, GA
- Henry Cuellar, TX
- Ron Kind, WI
- Allyson Schwartz, PA

#Activism for Activism’s Sake
Date: March 30, 2006

<img alt="i-85d55c0118cdc4cc75f6175581f447e9-biopiracy.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-85d55c0118cdc4cc75f6175581f447e9-biopiracy.JPG"/>


The Coalition Against Biopiracy has announced their winners for the [2006 Captain Hook Awards for Biopiracy](http://www.captainhookawards.org/winners/2006_pirates), and they’re a hoot.  We already knew that [Darwin was a pirate](http://scienceblogs.com/evolgen/2006/03/black_pirates_hate_darwin_too.php), but now we learn that so are [Craig Venter](http://evolgen.blogspot.com/2005/03/sequencing-genome-of-nyc-air.html) and [Google](http://www.google.com).  What are their crimes?  Venter is accused of being the “Greediest Biopirate”, and Google is accused of being the “Biggest Threat to Genetic Privacy”.  I have some more details on these charges below the fold, in addition to showing why the Coalition Against Biopiracy needs to walk the plank.





Venter is accused of:


> “undertaking, with flagrant disregard for national sovereignty over biodiversity, a US-funded global biopiracy expedition on his yacht, Sorcerer II, to collect and sequence microbial diversity from the world’s oceans and soils. The genetic material will play a role in his most ambitious project to date: building an entirely new artificial organism.”


Venter is [sailing around the world](http://www.sorcerer2expedition.org) scooping up water from the oceans.  He and his collaborators are then sequencing DNA from whatever micro-organisms they happen to scoop up.  They are **NOT** using those sequences to build an artificial organism.  Is this biopiracy?  The Coalition claims, “Biopiracy refers to the monopolization (usually through intellectual property) of genetic resources and traditional knowledge or culture taken from peoples or farming communities that developed and nurtured those resources.”  I don’t see how you can interpret what Venter is doing as taking things from any peoples.  He’s not monopolizing intellectual property, as he will make all of the sequences freely available online.  I see this as a great contribution to biology, not an act of piracy.


<img alt="i-4c2cab23138c87de2207014d1d38fb87-pirate.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-4c2cab23138c87de2207014d1d38fb87-pirate.JPG"/>Venter is also accused, along with Google, of threatening genetic privacy by creating a database of human DNA sequences.  Venter has been using Google’s computing powers to aid in creating a database of genetic information, with data on function and evolutionary relationships.  This service, I imagine, will be akin to the [NCBI Entrez database](http://ncbi.nlm.nih.gov/Entrez/index.html).  Centrally located information is a boon to all researchers in the life sciences, including those doing both pure and applied research on human diseases.  Additionally, many people use these resources to study organisms that have suffered the negative consequences of interactions with industrialized nations.  I’d say that a biological database is antagonistic to the idea of biopiracy.  The details of [the collaboration](http://www.infoworld.com/article/06/01/18/74294_HNgoogleventermum_1.html) are real hush-hush, but that does not stop the Coalition from speculating that they will be creating a database of sequences from 6 billion humans.  Forget about the ethical problems with publishing such a database online.  What about the logistical nightmare of collecting all of that data?  It’s totally not worth it, even for Big Brother.


[Say what you will about Venter](http://www.ghastlyfop.com/blog/2006/01/lights-camera-venter.html), his intentions are far from evil.  Maybe he’s a little bit megalomaniacal, and his human genome sequencing project may have been [bullshit](http://www.pnas.org/cgi/content/extract/100/6/3022), but he has pushed the envelope when it comes to high throughput sequencing.  He was a big reason that shotgun sequencing caught on, and he greatly encouraged the development of new sequencing technology.  He may sail the high seas, but he ain’t no pirate.


(Via [Slashdot](http://science.slashdot.org/article.pl?sid=06/03/30/1336221).)

#Double Entendre Friday – 31 March 2006
Date: March 31, 2006

**Help! I need some more double entendres. Help! Not just any double entendres. Help! You know I need some biology double entendres.**


When I first proposed this wacky idea I figured I’d add funny biology terms to my list as the weeks went by.  Well, I exhausted my original list quite rapidly, and now I’m scraping the dregs of funny terms to bring you something to make your Friday go by a little bit faster.  This week I’m stealing from a list of [molecules with silly names](http://www.chm.bris.ac.uk/sillymolecules/sillymols.htm) (via the recently expunged [Frink Tank](http://www.frinktank.com/)).  Here’s an awkward exchange that may have occurred between a student and her advisor:


> A geology professor and his new female graduate student are camping out in western Massachusetts for the weekend with the intent of cataloging the different rocks in a particular area.  They are sharing a tent and both felt uncomfortable during the first night; the student is fairly attractive and the professor is a handsome man with a reputation for taking care of business.  It’s late Saturday afternoon, and the conversation that day has been quite flirtatious.  As the sun is going down the student comes across a rock that she is not familiar with.  “What is this?” she asks the professor.  He walks over and inspects the rock. After a few moments of contemplation, he exclaims, “Well, I’ll be! [Cummingtonite](http://webmineral.com/data/Cummingtonite.shtml).”  The student replies, “Well that’s a bit presumptuous.”


<img alt="i-635dadb78623a48d8760490bccae4604-cumming_cologne.GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-635dadb78623a48d8760490bccae4604-cumming_cologne.GIF"/>This also gives me an excuse to post a picture of the greatest cologne bottle on the face of the Earth.  [Alan Cummings](http://imdb.com/name/nm0001086/?fr=c2l0ZT1kZnx0dD0xfGZiPXV8cG49MHxrdz0xfHE9YWxhbiBjdW1taW5nc3xmdD0xfG14PTIwfGxtPTUwMHxjbz0xfGh0bWw9MXxubT0x;fc=1;ft=20;fm=1) probably got ragged on quite a bit when he was growing up because of his name.  Now he has taken advantage of the sexual innuendo with [his own scent](http://www.sephora.com/browse/product.jhtml;jsessionid=F24WRQVKZIL11LAUCK5BXCQ?id=P100604&amp;_requestid=180251%20), named Cumming.  It’s pure genius, even if it’s not all that creative.

#One Last Chance for the House to Show They Support Science
Date: March 31, 2006

Our [plan](http://scienceblogs.com/evolgen/2006/03/contact_your_us_representative.php) to have the House Budget Committee approve an amendment to increase the NIH budget failed.  Our next chance to ensure the NIH budget is increased will occur on the House floor.  If you have yet to contact your US Representative about supporting life-sciences research, please do so by visiting [this site](http://capwiz.com/jscpp/home/).  The following comes from an email from the Genetics Society of America.


Your Voice Still Needs to Be Heard


The House Budget Committee voted down the amendment offered by Rep. Rosa  DeLauro  that would have increased health and education funding in the House Budget Resolution by $7 billion.  All 22 Republicans voted no and the 14 Democrats present voted aye.


ACTION:  It’s time to focus all attention on all Members of the House of Representatives.  Call or E-Mail Your US Representative! Ask your Member of Congress to support the NIH by increasing health and education funding in the House budget resolution. Ask them to support the efforts to include an additional $7 billion for health and education programs to the FY 2007 budget. 


More below the fold…<br/>



Ask your representative to Support Rep. Michael Castle’s (R-DE) plan to amend the budget resolution on the House Floor to include $7 billion the Senate added for health and education programs to the FY 2007 budget. 


ACTION IS NEEDED NOW.  The Congressional schedule has not been finalized, but the House is expected to consider the budget during the week of April 3.


How to Contact Your Member of Congress


- Your voice matters – phone calls as well as emails will make an impact on the way your Representative will vote.  Remember, this is a matter of urgency.
- Simply visit [http://capwiz.com/jscpp/home](http://capwiz.com/jscpp/home).  At this site, you should be to follow the step-by-step instructions for calling/emailing your Congressional representative.   A sample letter will be provided for you.
- At the above website, by simply entering your zip code, you will be given the name, address, phone number, fax, website, and email address of your Member of Congress.
- If you call, don’t hesitate to leave a message with the receptionist or on voice mail.  You may also ask to speak to the Representatives Health Care Legislative Assistant.  Be sure to leave your message, you name, your phone number and if you have time your address.  Keep your message brief and to the point.


Background


The House of Representatives is taking up the President’s Budget request.  As you know, the Senate added money to the section of the budget that funds the NIH thanks to Senators Specter (R-PA) and Harkin (D-IA).


Rep. Mike Castle (R-Del.) has agreed to take the lead on supporting the Specter-Harkin amendment when the budget is debated on the floor of the House.


Remember As a community we were able to come together to urge a cross section of Republican Senators to vote in favor of providing additional funding for health care and education programs.  We need to exert the same all out effort in contacting all House Members – Republicans and Democrats – to deliver the message that the FY 2007 budget must include an additional $7 billion for health and education programs.

#I am an endosymbiotic cyanobacterium
Date: April 23, 2006

Otherwise known as a choloroplast:

<td align="center"> <font size="5">**The Chloroplast**</font><br/> You scored 46 Industriousness, 54 Centrality,  and 23 Causticity! </td>
| You’re the Chloroplast!

#Race or Whatever
Date: April 24, 2006

If you read evolgen, you’ve probably been following the race riots that [Wilkins started](http://evolvethought.blogspot.com/2006/04/races-geography-and-genetic-clusters.html).  It’s pretty much died down now, and it was more a debate about semantics rather than an actual scientific disagreement.  This is usually the case in evolutionary biology — take, for example, the [neutralist-selection](http://www.ishpssb.org/ocs/viewabstract.php?id=86) [debate](http://www.blackwellpublishing.com/ridley/a-z/Neutralist_controversy.asp) or the [recent](http://scienceblogs.com/evolgen/2006/03/junk_dna_is_junk.php) [junk DNA](http://scienceblogs.com/evolgen/2006/03/junk_dna_redux.php) [fun](http://scienceblogs.com/evolgen/2006/04/junkity_junk_mcjunkerson_1.php) we had here at evolgen.  I have refrained from offering my opinion on Wilkins’s post due to my poor understanding of human population genetics (as evidenced by my [attempt](http://evolgen.blogspot.com/2005/06/racist-drug-development.html) to [discuss](http://evolgen.blogspot.com/2005/07/more-bidil-news.html) marketing BiDil to African Americans), but I have a few comments I would like to make.  You can find them below the fold.





Before my opinion, a quick recap.  It all started when John Wilkins wrote his opinion about [race](http://evolvethought.blogspot.com/2006/04/races-geography-and-genetic-clusters.html) in humans, arguing that it is not biologically meaningful (defending the Lewontin thesis).  [PZ Myers](http://scienceblogs.com/pharyngula/2006/04/that_question_of_race.php) then posted in agreement with Wilkins, and [Jason Malloy](http://scienceblogs.com/pharyngula/2006/04/that_question_of_race.php#comment-77790) disagreed in the comments.  Razib, who knows more about human population genetics than any armchair scientist should, provided a nice [argument](http://scienceblogs.com/gnxp/2006/04/race.php) for why Lewontin is wrong, and the [Contingency Table](http://contingencytable.blogspot.com/2006/04/race-useful-concept.html) took a look at some of the articles Wilkins cited to show they don’t actually support his argument.  My conclusion (based on something [Razib](http://scienceblogs.com/gnxp/2006/04/race_ii.php) wrote): what one person is willing to call race, another wants to label as population structure.  Oh, and some people think Lewontin is an idealist who believes in a homogenous population of all humans.  I’m going to try to stay as neutral as possible.  This post is mostly an attempt to request feedback from my readers and encourage a discussion of the data, rather than opinions.


First of all, I’m going to refer to populations rather than races.  If you want, you can think of them as metapopulations or some other nested population structure.  Either way, we’re using the word population.  Let’s assume we have no *a priori* assumptions regarding our populations.  Now, it’s not very practical to identify populations based on fixed alleles between the populations because there won’t be very many of those.  Instead, we should employ a probabilistic approach where certain alleles tend to be found in one population versus another.  If we genotype individuals at multiple loci, we can construct our populations using an assignment test.  This is what Jonathan Pritchard’s [Structure](http://pritch.bsd.uchicago.edu/software/structure2_1.html) program does.  In this algorithm, the most probable model tends to be over-split into many populations, but we can constrain the method to find no more than 3, 4, 5 or any other number of populations.


I think everyone involved will agree that we can recover our predefined “races” using an assignment test.  The races have been reproductively isolated (to some extent) allowing for the neutral alleles to reach different frequencies in the different populations.  Determining whether these differences are meaningful beyond population structure requires a bit more work.  First off, we must define what we mean by meaningful.  I would argue that meaningful requires some sort of physiological, anatomical, or other phenotypic difference between the populations.


Are there such differences?  Well, duh.  Human populations have different skin colors, tolerance to types of food, resistance to pathogens, facial characteristics, body types, etc.  None of these differences seem contentious, but when we start dealing with behavior or intelligence, supremacist undertones permeate the conversation.  Most of the aforementioned differences are due to natural selection, and that makes them poor markers for recovering population structure — if selection regimes are similar in multiple populations, allele frequencies may be similar due to convergent or parallel evolution rather than recent common ancestry.  But these loci are what make our populations biologically different.


So, we must first determine our populations using neutral loci, and then examine non-neutral alleles to see which ones differ between the populations.  We can examine the [polymorphism](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php) around a locus to determine whether alleles differ between populations because of selection or population structure.  Also, genes that suggest different population structure than most of the other loci in the genome may have their pattern due to selection.


That’s what I think; sadly, I’m not familiar enough with the data from humans to tell you what they reveal.  My (limited) understanding is that the majority of human genetic diversity can be found in African populations.  African populations differ from each other at both neutral alleles (revealing population structure within Africa) and non-neutral loci (revealing biologically meaningful differences).  My argument has always been that the current races (as defined in the United States) are inappropriate because they marginalize the diversity found within Africa.  If each race were to contain the same amount of diversity, we would need to split Africans into multiple races.  Those who are more familiar with these areas of research, please correct me where I am wrong.

#An Alignment Tree
Date: April 24, 2006

Via [nodalpoint](http://www.nodalpoint.org/2006/04/15/algorithms_algorithms_everywhere_0) comes [this](http://nar.oxfordjournals.org/content/vol34/issue6/images/large/gkl091f1.jpeg) UPGMA tree of sequence alignment algorithms from [this paper](http://nar.oxfordjournals.org/cgi/content/full/34/6/1692).  The first thing that comes to mind is that there are way too many sequence alignment methods.  The second, it’s kinda cool to see one method used to understand another.  And [Nucleic Acids Research](http://nar.oxfordjournals.org/) gets mad props for being entirely open access while still being published by the evil empire that is [Oxford University Press](http://www.oxfordjournals.org/).

#Publishing Original Research on Blogs – Part 1
Date: April 24, 2006

Bora has been [pushing](http://sciencepolitics.blogspot.com/2006/04/publishing-hypotheses-and-data-on-blog.html) the idea of [publishing](http://sciencepolitics.blogspot.com/2006/04/even-more-on-science-online-publishing.html) original research (hypotheses, data, etc) on science blogs.  As a responsible researcher, I would need to obtain permission from any collaborators (including my advisor) before published anything we have been working on together.  But what about small side projects or minor findings that I don’t expect to publish elsewhere?  As it turns out, such a project has been laying dormant since I first started working on it at a class project a few years ago.  I will reveal more information about this project in subsequent posts, but suffice it to say this research is far from earth shattering.  My primary objective is not to present any important findings, but rather to give my readers insight into how easy it is to study evolution.  After reading this series (I hope) someone will be able to download some sequences from the [NCBI database](http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi) and perform their own analyses.  That being the case, all of the software I use will be freely available and easy to run on a PC (sorry Mac users, but that’s my environment of choice).

#Publishing Original Research on Blogs – Part 2
Date: April 25, 2006

*Bora has been [pushing](http://sciencepolitics.blogspot.com/2006/04/publishing-hypotheses-and-data-on-blog.html) the idea of [publishing](http://sciencepolitics.blogspot.com/2006/04/even-more-on-science-online-publishing.html) original research (hypotheses, data, etc) on science blogs.  This post is part of a series exploring the evolution of a duplicated gene in the genus *Drosophila*.  Links to the previous posts can be found below.  Part 2 of this series (The Backstory) can be found after the jump.*


**Previous entries:**


[Part 1 – Introduction](http://scienceblogs.com/evolgen/2006/04/publishing_original_research_o.php)





**The Backstory**


<img alt="i-b99f00f2d745899b10a8cfb0a0541ad8-vert_aldolase.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b99f00f2d745899b10a8cfb0a0541ad8-vert_aldolase.JPG"/>


The enzyme [fructose-1,6-bisphosphate aldolase](http://www.expasy.org/enzyme/4.1.2.13) (hereafter referred to as aldolase) is responsible for splitting fructose-1,6-bisphosphate into glyceraldehyde-3-phosphate and dihydroxyacetone-phosphate during [glycolysis](http://www.indstate.edu/thcme/mwking/glycolysis.html).  Multicellular animals have developed different ways to encode specialized aldolase enzymes for different tissues.  Most vertebrates, for example, have separate genes for muscle tissue (Ald A), liver (Ald B), and brain (Ald C) (Rutter 1964; Horecker et al. 1972), while *Drosophila melanogaster* has one primary copy of aldolase which undergoes [variable splicing](http://en.wikipedia.org/wiki/Alternative_splicing) of the initial transcript into three different mRNAs (Shaw-Lee et al. 1992).  The vertebrate genes have undergone two major duplication events since the [divergence of jawless fish (e.g., lampreys) and jawed vertebrates](http://tolweb.org/Vertebrata/14829) (Merritt and Quattro 2002).  The first duplication event gave rise to the Ald B and Ald A/C lineage, and the second event led to separate Ald A and Ald C genes.  Jawless fishes only have two aldolase genes which have protein products that are similar in function to the vertebrate aldolase A and C (Zhang et al. 1997), but arose due to a duplication event separate from that along the jawed vertebrate lineage.


The vertebrate aldolase gene family also contains [pseudogenes](http://www.pseudogene.org/main.html) (duplications that appear to be non-functional).  In mammals, these pseudogenes tend to occur via [retrotransposition](http://www.rochester.edu/College/BIO/thelab/Main%20pages/Mech.%20of%20retro_.html) of the Ald A transcript, and have been studied in humans (Serero et al. 1988), rabbits (Amsden et al. 1992), and mice (Cortinas and Lessa 2001).


With the complete sequence of the *D. melanogaster* genome, came the discovery of a duplicate copy of Aldolase ([CG5432](http://flybase.org/.bin/fbidq.html?FBgn0039425)) located approximately 150kb from the previously characterized copy ([Ald](http://flybase.org/.bin/fbidq.html?FBgn0000064)).  Whether or not this gene encodes a functional protein is yet to be determined, but it does possess all the necessary flanking sequences for transcription and translation.  In this analysis we will examine the evolution of these [paralogs](http://www.ncbi.nlm.nih.gov/Education/BLASTinfo/Orthology.html) within the *Drosophila* genus.  Both [coding and non-coding sequences](http://evolgen.blogspot.com/2005/11/detecting-natural-selection-part-1.html) will be included, and the coding sequences will be studied at the [nucleotide and amino acid levels](http://evolgen.blogspot.com/2005/11/detecting-natural-selection-part-2.html). Our analysis will consist of comparisons of [synonymous and non-synonymous substitutions](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-3.html), [rates of evolution along the lineages](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-4.html), and phylogenies retrieved using different subsets of the data.  As we progress through the study, suggestions regarding the data analysis may be left in the comments.


---



**References Cited:**


Amsden, A.B., E.E. Penhoet, and D.R. Tolan. 1992. A rabbit Ald A pseudogene derived from a partially spliced primary Aldolase A transcript. Gene 120: 323-324.


Cortinas, M.N., and E.P. Lessa. 2001. Molecular evolution of Aldolase A pseudogenes in mice: multiple origins, subsequent duplications, and heterogeneity of evolutionary rates. Mol. Biol. Evol. 18: 1643-1653.


Horecker, B.L., O. Tsola, C.Y. Lai. 1972. Pp. 213-258 in The Enzymes, Vol. 7. P.D. Boyer, ed. Academic Press, New York.


Merritt, T.J.S. and J.M. Quattro. 2002. Negative Charge Correlates with Neural Expression in Vertebrate Aldolase Isozymes. J. Mol. Evol. 55: 674 – 683.


Rutter, W.J. 1964. Metabolic control mechanisms in animal cells. Fed. Proc. 23: 1248-1257.


Serero, S., P. Maire, V.C. Nguyen, O. Cohen-Haguenauer, M.S. Gross, C. Jegou-Foubert, M.F. de Tand, A. Kahn, J. Frezal. 1988. Localization of the active gene of aldolase on chromosome 16, and two aldolase A pseudogenes on chromosomes 3 and 10. Hum. Genet. 78: 167-174.


Shaw-Lee, R., J.L. Lissemore, D.T. Sullivan, and D.R. Tolan. 1992. Alternative splicing of fructose 1,6-bisphosphate aldolase transcripts in Drosophila melanogaster predicts three isozymes. J. Biol. Chem. 267:3959-3967.


Zhang, R., T. Kusakabe, N. Iwanaga, Y. Sugimoto, K. Kondo, Y. Takasaki, T. Imai, M. Yoshida, and K. Hori. 1997. Lamprey fructose-1,6-bisphosphate aldolase: characterization of the muscle-type and non-muscle-type isozymes. Arch. Biochem. Biophys. 341: 170-176.

#What I Think You Should Know
Date: April 25, 2006

Via [BioCurious](http://biocurious.com/facts-you-should-know) comes [this article](http://www.startribune.com/389/story/369290.html) on ten “science question[s] every high school graduate should be able to answer.”  Read the list — most of the questions are bullshit.  Ok, they aren’t bullshit, but they are trivia.  “What percentage of the earth is covered by water?” “What sorts of signals does the brain use to communicate sensations, thoughts and actions?” “Why is the sky blue?” “How old are the oldest fossils on earth?” I’ll take potpourri for $400, Alex.  Here are some questions every high school graduate should be able to answer:


- What is a hypothesis?
- What is a theory?
- What is probability?
- What is an atom? What is a neutron? What is a proton? What is an electron?
- What is a molecule?
- What is a cell?
- What is a gene? What is a genome? What is the central dogma of molecular biology?


You get the point.  Let’s focus on two things: the hypothetical deductive method and essential information that you must know to be able to read the science section of a newspaper.

#Kids Email the Darndest Things
Date: April 26, 2006

If you know anything about bioinformatics, you know that programmers love to come up with clever names for their applications.  NCBI’s [BLAST](http://www.ncbi.nlm.nih.gov/blast/) (Basic Local Alignment Search Tool) programs are some of the most used tools, but behind every successful program is a stupid acronym or cute name.  Do the people who name these programs realize the confusion this may cause?


Many of these bioinformaticians provide list serves for updates on their software and public discussions.  [TIGR](http://www.tigr.org/) (The Institute for Genomic Research, another acronym) developed a genome annotation tool and named it [Manatee](http://manatee.sourceforge.net/).  A confused kid sent an email to the users’ mailing list (via [Neil Saunders](http://psychro.bioinformatics.unsw.edu.au/neil/index.php?option=com_jd-wp&amp;Itemid=70&amp;p=90)):


> i am doing a school report and i need to ask some questions about manatees please. How can we help save these wonderful mammals. how do you like manatees. do they react good with people. how many times a day do they need to eat. how long can the survive above the water.


Let’s see [Dr. Freeride](http://scienceblogs.com/ethicsandscience/kids_and_science/) try to match that sproginess!

#From the Junk Rise the ORFans
Date: April 26, 2006

Ian Musgrave has a good [summary of genes appearing from non-coding DNA](http://www.pandasthumb.org/archives/2006/04/an_argument_is.html) (ORFans) on the Panda’s Thumb.  I have written about ORFans [here](http://scienceblogs.com/evolgen/2006/03/junk_dna_redux.php) and [here](http://scienceblogs.com/evolgen/2006/04/junkity_junk_mcjunkerson_1.php) (dude’s gotta link to himself sometimes).  Ian’s post is targeted at some claims made by the creationist Paul Nelson, but he focuses more on the science that on the ignorant blowhard.

#In Case You Wanted to Know a Little More About Me
Date: April 28, 2006

Via [Janet](http://scienceblogs.com/ethicsandscience/2006/04/a_meme_between_stacks_of_paper.php) comes the ABC meme.  To learn a bit about me, click through to below the fold.





- **Accent:** None that I can detect.  Of course, no one thinks they have an accent.
- **Booze:** It depends on my mood — I’ll drink pretty much anything.  Sometimes I’m in the mood for a good IPA, sometimes a thick stout, and sometimes a Miller Light.  If it’s wine, it’s best if it’s from Burgundy.  As for liquor, gin is my perfect poison.
- **Chore I Hate:** I agree with Janet; grading sucks.
- **Dog or Cat:** I currently have neither.  I have had both.  Either one is good.
- **Essential Electronics:** TV + cable box.  Computer + router.
- **Favorite Cologne:** I hate that shit.
- **Gold or Silver:** White gold.
- **Hometown:** Los Angeles, CA.
- **Insomnia:** If I’m not tired.
- **Job Title:** <strike>Indentured servant</strike> Graduate student.
- **Kids:** Do I count? If not, none (that I know of).
- **Living arrangements:** Small apartment.
- **Most admirable traits:** My ego, intelligence, and mad skillz. Oh, and my modesty.
- **Not going to cop to:** I have no idea what that means.  Can we get a meme to English dictionary in here?
- **Overnight hospital stays:** Three times: when I was born, when I had major surgery in my second month, and when I ate some bad egg rolls my junior year of college.
- **Phobias:** I’m afraid of fear. Roosevelt was right.
- **Quote:** Um, yeah, I can’t think of any right now.  Sorry.
- **Religion:** Jew by ethnicity, atheist by belief.
- **Siblings:** One sister.
- **Time I wake up:** 6 or 7am.
- **Unusual talent or skill:** Like I said above, I have mad skillz.  I also throw a mean [outside-in flick](http://www3.upa.org/index.php).
- **Vegetable I love:** [Killer tomatoes](http://www.imdb.com/title/tt0080391/).
- **Worst habit:** Why you gotta go raggin’ on nuns?
- **X-rays:** Teeth, shoulder, foot, stomach, knee, maybe others I’ve forgotten.
- **Yummy foods I make:** Everything I cook tastes good.  Pizza, burgers on the grill, and I can bake.
- **Zodiac sign:** Virgo.

#Phylogeny Friday – 28 April 2006
Date: April 28, 2006

Over at my old site, I [lamented](http://evolgen.blogspot.com/2005/05/what-about-distance-based-methods.html) the apparent death of distance based tree building algorithms.  Just as all of life on earth can be divided into [three domains](http://tolweb.org/Life_on_Earth/1), phylogenetic methods can be split into three groups: distance based, maximum parsimony, and maximum likelihood.  Distance and parsimony based approaches have been around for a while (and were used prior to the availability of molecular data).  The combination of molecular data and more powerful computers allowed large molecular datasets to be analyzed using parsimony methods.  Our great computing power has also allowed for the advent of maximum likelihood methods to be applied to solving phylogenies.  [Bayesian](http://en.wikipedia.org/wiki/Bayes'_theorem) likelihood algorithms are the *en vogue* tree building methods and they can be tuned to the specific parameters observed in your data.  But, as I asked in [the pos](http://evolgen.blogspot.com/2005/05/what-about-distance-based-methods.html)t, what about distance based methods?


More below the fold…





The discussion above is far from comprehensive, and I don’t spend a lot of time building trees so I’m not qualified to judge which method is best.  That said, the appropriate method definitely depends on your data, and it’s always good to confirm your phylogeny using multiple methods.  Despite being published nearly twenty years ago, the [neighbor joining](http://mbe.oxfordjournals.org/cgi/content/abstract/4/4/406) method remains one of the most popular tree building algorithms.  The article has been cited an amazing 9,820 times (according to [Google Scholar](http://scholar.google.com/scholar?hl=en&amp;lr=&amp;newwindow=1&amp;safe=off&amp;q=link:t-4gZOn6ULYJ:scholar.google.com/)).  That may be an underestimate, as ISI lists it as having 13,353 citations.


<img alt="i-958c6f5656e9ff42f04841dd6bd4a10b-nei_join.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-958c6f5656e9ff42f04841dd6bd4a10b-nei_join.JPG"/>The token phylogeny is shown to the left.  This is the first ever neighbor joining phylogeny constructed using real data.  The evolutionary distance between these frog species (from the genus *Rana*) were measured using [allozyme loci and biochemical interactions](http://links.jstor.org/sici?sici=0039-7989(197809)27%3A3%3C299%3ABSOMOT%3E2.0.CO%3B2-I) — not exactly DNA sequences, but the original data were published in 1978.  The numbers represent the evolutionary distance along each branch. DNA sequencing was still quite difficult in the 1980s, but technological advances made in the 1990s lead to a [rapid increase](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-681c7c79a007407c8ad9b7ec995789e0-NCBI_seqs.gif) of DNA sequences in public databases.  The neighbor joining algorithm was used to construct many of the early phylogenies using molecular data (some of these may appear in Phylogeny Friday in the coming weeks).

#Phylogeny Friday – 19 May 2006
Date: May 19, 2006

As I [mentioned previously](http://scienceblogs.com/evolgen/2006/05/hibernation_and_such_1.php), I’m busy preparing some data for a [meeting next week](http://www.smbe.org/geb/).  I don’t have much time to devote to Phylogeny Friday, so I’ll be sharing some of my own data with you.  This data is nothing special; it just happens to be the data I finished analyzing a few minutes ago. It’s open, I could build a tree, and I’ll show you that tree below the fold.





<img alt="i-8f1232a7350f5384330f9a866c2f92d6-phyl_fri(19May2006).JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-8f1232a7350f5384330f9a866c2f92d6-phyl_fri(19May2006).JPG"/>


This tree contains a scant three sequences. The two closely related sequences both come from *Drosophila pseudoobscura* and the outgroup is a gene from *D. melanogaster*. This gene was duplicated along the *D. pseudoobscura* lineage. In the time since the duplication event, the two copies in the *D. pseudoobscura* lineage have diverged from each other. One copy has accumulated two amino acid substitutions, and the other copy has accumulated one amino acid substitution. Remember how we discussed how [relative rates of evolution](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-4.html) can be used to detect changes in selective constraint? Well, these paralogs diverged too recently for us to have enough power to detect any differences in rates along the two lineages.


I know it’s not a very interesting tree, but it’s all I have time for today. Sorry. This next week will be hectic, so don’t expect much from Phylogeny Friday for another couple of weeks.

#Get it? Got it. Good.
Date: May 20, 2006

This week, the [SEED kings](http://scienceblogs.com/seed/) are asking us:


> ****“If you could shake the public and make them understand one scientific idea, what would it be?”****





[This](http://scienceblogs.com/strangerfruit/2006/05/question_of_the_week_1.php) is the first thing that comes to mind, and the [scientific method](http://scienceblogs.com/grrlscientist/2006/05/mothership_question_3.php) is a good answer as well. I’m going to take a step back and say the public should understand **what science is**.  That way they can tell the difference between [this](http://www.discovery.org/csc/) and [science](http://nsf.gov/). From there, an understanding of evolution should come naturally.


Answers to last weeks question can be found [here](http://scienceblogs.com/seed/2006/05/ask_a_scienceblogger_round_two.php).

#I am a Gen-X, Soccer Playing, Super Model
Date: May 23, 2006

While procrastinating (I should be analyzing data for a talk I’m giving on [Friday](http://www.smbe.org/geb/schedule.htm#fri)), I stumbled upon [this site](http://www.myheritage.com/FP/Company/tryFaceRecognition.php?s=1&amp;u=g0) (via [EDSBS](http://www.everydayshouldbesaturday.com/?p=2075), not StumbleUpon). I tried to bypass registration (using [BugMeNot](http://www.bugmenot.com/view/myheritage.com)), but none of the cheat passwords worked.  If you want to play around with the gadget, you may need to register, but you can use a fake email address (they don’t require you confirm your registration).


What’s this fun website? Well, you upload a headshot, and it returns a list of celebrities you resemble. I uploaded three different photos. To find out who I look like, click through below the fold.





Photo number one scored a 67% match to [Elle McPherson](http://en.wikipedia.org/wiki/Elle_mcpherson), and matched [Edward Norton](http://en.wikipedia.org/wiki/Edward_Norton) (67%) and [Michael Owen](http://en.wikipedia.org/wiki/Michael_Owen) (62%), as well.  I also matched with [Demi Moore](http://en.wikipedia.org/wiki/Demi_Moore), [Anna Kournikova](http://en.wikipedia.org/wiki/Anna_kournikova), and [Rivaldo](http://en.wikipedia.org/wiki/Rivaldo).


My second photo scored a 62% match with [Kenny Dalglish](http://www.liverpoolfc.tv/team/past_players/players/dalglish/), a 61% match to Michael Owen, and a 58% match to [Shakira](http://en.wikipedia.org/wiki/Shakira) (although her ass shakes much better than mine).


Finally, picture number three scored a 64% match to [Matt Dillon](http://en.wikipedia.org/wiki/Matt_Dillon), a 62% match to [Christy Turlington](http://christy.turlington.com/index2.htm), and a 61% match to Rivaldo.


The recurrent themes that I see are soccer players (Rivaldo, Michael Owen, and Kenny Dalglish are all footballers) and models (Elle McPherson, Christy Turlington, and Anna Kournikova). This application does a great job of stroking egos. I wonder if [Tiny Tim](http://www.tinytim.org/), [Gheorghe Muresan](http://en.wikipedia.org/wiki/Gheorghe_Muresan), or [Sandra Oh](http://en.wikipedia.org/wiki/Sandra_Oh) are in the database. It’s one thing for someone to say you look like a celebrity; it’s another thing to be told you look like an ugly famous person.


I also got repeat matches to [Debra Winger](http://en.wikipedia.org/wiki/Debra_Winger), and I look like a [Bollywood Celebrity](http://www.indianceleb.com/celina/).  Given my symmetrical facial features, does [Razib](http://scienceblogs.com/gnxp/2006/05/jolie_pitt_baby_with_load_muta.php) think my children will have a low [genetic load](http://evolgen.blogspot.com/2005/12/mean-fitness-genetic-load-and.html)?

#Zimmer on Speciation
Date: May 23, 2006

Carl Zimmer has an [article](http://www.nytimes.com/2006/05/23/science/23evol.html?pagewanted=1&amp;ei=5088&amp;en=3e84a72f8d16c9b3&amp;ex=1306036800&amp;partner=rssnyt&amp;emc=rss) in the NYTimes Science section on how humans can interfere with diverging populations, increasing the frequency of hybrids and preventing speciation. He give two examples: three-spine sticklebacks in British Columbia, Canada and ground finches in the Galapagos. The sticklebacks colonized lakes on Victoria Island from the Pacific Ocean and became specialized populations in the new environment. Hybrids between different ecomorphs were selected against because of their intermediate phenotype. Zimmer reports that there are an excess of hybrids in recent samples taken from the lakes, and that human introduction of crayfish may be to blame.  The original study can be found [here](http://dx.doi.org/10.1111/j.1365-294X.2005.02794.x).


The other example Zimmer presents are the ground finches Darwin described in the Galapagos. We all know the story about the different sized beaks and how each is adapted to a specific diet.  If you don’t know the story, read [this book](http://www.amazon.com/gp/product/067973337X/002-3473940-3958414). On Santa Cruz Island, the birds come in two different flavors (large beaks and small beaks), but a population near a human settlement has way too many hybrids. This may be because humans feed the birds rice, removing the selection pressure for beak specialization. The original article can be found [here](http://dx.doi.org/10.1098/rspb.2006.3534).


Zimmer has [more](http://loom.corante.com/archives/2006/05/22/manimals_sticklebacks_and_finches.php) at the Loom.

#Once, Twice, Three Times a Name Change
Date: May 23, 2006

Given the inability of [certain](http://mikethemadbiologist.blogspot.com/2006/05/interesting-perspective-on-science.html) [people](http://jgrr.blogspot.com/2006/05/emerging-answer-to-seeds-question.html) to differentiate between evolgen and [Evolution Blog](http://scienceblogs.com/evolutionblog/), we’re changing our name. This isn’t like the [last time](http://scienceblogs.com/evolgen/2006/01/welcome_to_evolgen_at_scienceb.php) we tried to rename ourselves. This is for real . . . and [4 eva](http://www.imdb.com/title/tt0116287/).  We are no longer evolgen. We aren’t Evolution Blog, either, but we never claimed to be Evolution Blog (and we never claimed to know anything about evolution or blogging for that matter). I’m not sure who “we” are. It’s always just been little old me. This name change is giving us quite a [complex](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5d9a45f4e89de0458fbf83f321d4b3f7-queen.jpg).


What’s our new name? Well, if you’re reading this through a newsreader or some other computer program type device (yup, we’re still computer illiterate) you’re going to have to, as the kids say, click through. The new name is below the fold, on the banner, and (damn, we just realized this) listed as the new title of this feed. Well, click through and see the newly renamed evolgen.





Hey, hey, hey! Thanks for clicking through you awesome person. The ****NEW NAME****, in case you can’t read it because you’re red-green colorblind or something, is….


****Big Happy Fun Time****


Our new catchphrase: who’s your daddy??? Although we did consider going with, “We’re Your Momma!” A close third was, “Call me the soccer mom ’cause I just drove yo’ ass to practice.”


I’m off to the A to the Z for the rest of the week. Enjoy <strike>all of</strike> the scheduled post<strike>s</strike> we have waiting in the wings.

#From the Arctic to Hades
Date: May 24, 2006

If all has gone well, this post should appear as I’m on my way across the country to [John Lynch](http://scienceblogs.com/strangerfruit/)‘s neck of the woods.  The [warmth of the desert](http://www.weather.com/weather/local/USAZ0233?from=search_city) should cause quite a shock to my system coming from the frigid north east.  I may not have much access to the internets until next week, and I haven’t scheduled any other posts.  Evolgen is going dark for the next few days. If you have the urge to post a comment (on one of the old posts, I guess), log in with Type Key so that it can appear without my moderation.


While I’m gone, make sure [the Frinkers](http://www.frinktank.com/blog/?p=273) don’t steal any of my [my jokes](http://scienceblogs.com/evolgen/2006/05/more_opression_of_the_white_ma.php).

#Detroit Rawk Sit on My Ass
Date: May 29, 2006

The [meeting](http://www.smbe.org/geb/) went pretty well (I may have more to say later once I’ve had time to digest everything). When I got to the airport on Sunday, I found out my flight was one hour late. I live in the middle of nowhere, so it’s impossible to get a direct flight from the western or southern US. Long story short, I missed my connecting flight by about 5 minutes. [These assholes](http://www.nwa.com/) wouldn’t hold my plane for me — they figured it was better to waste their money to put me up in a hotel and give me money for food than go a few minutes behind schedule. They also gave me a coupon for $100 dollars off my next flight with them. How nice. I guess that means I’ll be flying with them again. Damn.


I also had my first blog-person encounter. You know, when two people that only know each other from blogging get together in real life. I met [John Lynch](http://scienceblogs.com/strangerfruit/) on Wednesday. John, I take back all the bitching about the heat in Arizona. Eighty degrees and humid is much worse than 100 degrees and dry. And those temps are in fahrenheit. Don’t ask me to convert them to celsius. I can’t.


And my rescheduled flight from Detroit (pronounced “day-twah”) to Pennsylvania goes through Cincinnati. Rock on! Flying sucks.

#Spending Other People’s Money
Date: May 30, 2006

Another week, another question from [the Seeders](http://scienceblogs.com/seed/). This week they ask us:


> ****Since they’re funded by taxpayer dollars (through the NIH, NSF, and so on), should scientists have to justify their research agendas to the public, rather than just grant-making bodies?****


My answer is below the fold.





It all depends on how one defines “the public”. If you’re asking me whether a school teacher, a construction worker, and a street vendor should be evaluating grant proposals and making funding decisions, I say no. It’s not that the general public shouldn’t have an influence on what research gets funded. They simply do not have enough training to adequately evaluate research grants.


But I don’t think that’s what the question is getting at. The public already has an indirect role in determining what research gets funded. The budgets of US governmental funding agencies are determined by Congress — [not always with satisfactory results](http://scienceblogs.com/evolgen/2006/05/good_news_about_the_nih_budget.php) for the scientists begging for cash.  In my <strike>many</strike> one experience getting a grant funded, I had to write a short description of my project so that my Congressman could understand it (or some staffer in his office in charge of science and technology). Apparently, legislators read these abstracts to determine whether the funding agencies are doling out money in a satisfactory manner. And, yes, my description was all about how important my research is in the study of evolution, so I hope [Dicky S](http://en.wikipedia.org/wiki/Santorum_Amendment) reads it.


All of this assumes that the public cares enough about scientific research to want input into the process. They care enough about [stem-cell research](http://www.msnbc.msn.com/id/6384390/) to campaign for bond measures, but I don’t think population genetics would get them as riled up. Most people probably don’t understand what scientific research is and how funding dollars get spent. I think it’s more important for us to explain our research to the public rather than justify our grants. That way they understand how the money is being spent, so that they can support legislators who fund agencies they would like to see receive money.


So, yes, scientists should (and currently do) have to justify their research to the public (I’m not sure about “research agendas” as I have no idea what a research agenda is).

#4 eva = one week
Date: May 30, 2006

I should be working instead of dicking around on the ole’ blog. But, give me a break, I just washed 600 [vials](https://www1.fishersci.com/Coupon;jsessionid=E8Q64AVDJrVtn22iR2fBScradjYFEHgPouBCaSuI6W4R4x6znSHe!1019346352?cid=1340&amp;gid=33426&amp;details=Y). I needed a break.


After letting the [new ugly banner and stupid name](http://scienceblogs.com/evolgen/2006/05/once_twice_three_times_a_name.php) hang around for a week, I’ve decided it’s time for an update. We’re going back to being called evolgen (although you can still refer to us as [Throatwarbler Mangrove](http://evolgen.blogspot.com/2005/07/friday-random-ten-15-july-2005.html)). The slogan (or nickname, or catchphrase, or stupid saying that goes under the blog title), however, is…


You’re gonna have to click through below the fold for that. This type of announcement doesn’t get sent out over the RSS feed.





> AT THE CONVERGENCE OF EVOLUTION BLOG AND GENOMICS


A right bit clever, we are. Aren’t we? Eh? We were never good at the British accent. Or the Canadian one. Anyway, given that we are now in the [Post-Genomics Era](http://evolgen.blogspot.com/2005/12/genomics-in-post-genomics-era.html) it seems quite antiquated to exist at the convergence of evolution and genetics. After all, genetics is so [nineteenth century](http://en.wikipedia.org/wiki/Mendel). Genomics — that’s the science of the late twentieth century (we always were a decade or so behind the times).


And the part about Evolution Blog — well, that’s there due to the [confusion](http://mikethemadbiologist.blogspot.com/2006/05/interesting-perspective-on-science.html) [people](http://jgrr.blogspot.com/2006/05/emerging-answer-to-seeds-question.html) seem to have in telling the difference between a [Math Professor](http://scienceblogs.com/evolutionblog/) and [Grad Student](http://scienceblogs.com/evolgen/about.php). 


And, no, this blog does post on more than just changes to our banner. We’re working on some other stuff. Whether that other stuff gets published depends on our editors. PS, we are our own editors.


And, yes, we do like to start paragraphs with conjunctions, so long as our editors allow it.

#The Evolution of My Thinking about the Evolution of Gene Regulation
Date: May 31, 2006

…or how a learned to stop worrying and love evo-devo.


As my mind gets a chance to process some of the stuff I heard and talked about at [the meeting](http://www.smbe.org/geb/) I just returned from, I’ll post some thoughts that will help me organize my ideas (hopefully better organized than that last sentence). This is the first (of perhaps few, perhaps many) of those (possibly incoherent) ramblings — interrupted by as few paranthetical remarks as possible.


In this post, I’ll try to tie together:


- A talk by [Sean Carrol](http://www.molbio.wisc.edu/carroll/) on the [evolution](http://dx.doi.org/10.1038/nature03235) of [wing pigmentation](http://dx.doi.org/10.1038/nature04597).
- A talk by [Peter Andolfatto](http://www.biology.ucsd.edu/labs/andolfatto/index.html) on the [evolution of *Drosophila* non-coding DNA](http://dx.doi.org/10.1038/nature04107).
- A talk by [Carlos Bustamante](http://www.bscb.cornell.edu/~bustamante/) on [signatures of natural selection in human genes](http://dx.doi.org/10.1038/nature04240).


All of the work has been published (follow the links above), so I’m not going to spend much time reviewing the content. Instead, I’ll discuss how these different results relate to each other, and how they are helping me formulate a coherent concept of evolution at the DNA level. There is, of course, more below the fold.





At the meeting, I attended a session on *cis*-regulatory evolution. As one of the presenters noted, everyone began their talk with a nod to [King and Wilson’s 1975 paper](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&amp;db=pubmed&amp;list_uids=1090005&amp;dopt=Abstract) (full text available [here](http://www.jstor.org/view/00368075/ap004370/00a00060/0) if you have access to JSTOR). King and Wilson postulated that differences in protein sequence between humans and chimpanzees could not explain the phenotypic differences between the two species (I have [written about this before](http://evolgen.blogspot.com/2005/07/genotype-and-phenotype-and-how-to.html)). Instead, changes in the regulation of when and where the genes encoding those proteins are expressed are responsible for phenotypic evolution. This model spawned the field of [evo-devo](http://www.worldwidewords.org/turnsofphrase/tp-evo1.htm), which did not get a clever name until some twenty years after King and Wilson’s seminal paper.


Sean Carroll is one of the leading researchers bridging the gap between evolutionary and developmental biology. His students’ work on the evolution of *Drosophila* wing pigmentation (reviewed [here](http://scienceblogs.com/pharyngula/2006/04/evolving_spots.php) and [here](http://scienceblogs.com/pharyngula/2006/04/evolving_spots_again_and_again.php) by PZ Myers) revealed that changes in the *cis*-regulatory elements (CREs) flanking the *yellow* gene are partially responsible for the gain and loss of wing spots during evolution. The Carroll lab has not unraveled the entire story, but they have shown that changes in the expression of a gene (rather than its protein coding sequence) can lead to novel phenotypes.  It would be interesting to find out why this trait is sexual dimorphic (only males have wing spots, so some sex-specific upstream transcription factors are probably involved in determining the phenotype) and what other genes are involved the pigmentation patterning (the transgenic *D. melanogaster* have dark pigmentation in the anterior-distal portion of their wing, but this pattern is not as crisp as in the species with endogenous pigmentation).


While the Carroll lab is focusing on an individual gene, Peter Andolfatto is taking a whole genome approach to understand the evolution of non-coding DNA. I wrote a [summary](http://evolgen.blogspot.com/2005/10/evolution-of-drosophila-non-coding-dna_31.html) of Andolfatto’s paper for my old blog; go there if you’re unfamiliar with this research. Some non-coding sequences are conserved across long evolutionary distances — these sequences are probably constrained by purifying selection because they are essential for the proper expression of nearby (and maybe not so nearby) coding sequences. Andolfatto has shown that many other non-coding sequences are not conserved because they are under positive selection (and other sequences are turning over at a neutral rate). This inference was made based on the relationship between [polymorphism and divergence](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php) at these loci. King and Wilson’s hypothesis only distinguishes between the evolution of protein coding sequences and regulatory sequences — they do not explicitly promote a selectionist explanation over a neutral one. Andolfatto’s result, however, indicates that if changes to CREs are primarily responsible for phenotypic evolution, that evolution is probably driven by positive selection.


King and Wilson also failed to distinguish between gene regulation in *cis* and *trans*, although their thesis that changes in protein coding genes cannot explain morphological evolution seems to imply that they would favor *cis*-regulatory evolution (if they were familiar with the distinction). One of Sean Carroll’s former students, Tricia Wittkopp, [showed](http://dx.doi.org/10.1038/nature02698) that changes to CREs could explain more differences in gene expression between *Drosophila melanogaster* and *D. simulans* than *trans* changes. This result seems to trivialize the role of protein coding sequences in the evolution of the phenotype. Sure there are changes in protein coding sequences (Wittkopp did observe evidence of *trans*-regulatory changes), but the majority of gene expression evolution is due to CREs. Intuitively, this makes sense; CREs only regulate a single gene, whereas transcription factors regulate multiple genes. The expression of a single gene can be modified by a change to its CREs, but a change in a transcription factor could have deleterious effects on the other genes it regulates.


So, what role does that leave for protein coding sequences? Changes in the [protein coding sequence](http://dx.doi.org/10.1126/science.1113722) of [some genes](http://dx.doi.org/10.1126/science.1116815) may be partially responsible for the evolution of the human brain (although this result is [far from conclusive](http://dx.doi.org/10.1093/hmg/ddl126)). Working in the same theoretical framework as Andolfatto, Carlos Bustamante and colleagues compared polymorphism and divergence (from chimps) at both synonymous and non-synonymous sites for over 11,000 human genes. These data were collected by resequencing these genes, not by scoring SNPs, so they should be immune from ascertainment bias. Nine percent of the genes had an excess of a non-synonymous substitutions between humans and chimps, indicating that these sequences were under positive selection. Interestingly, many transcription factors have this signature of positive selection. It looks like *trans*-regulation ****does**** play an important role in evolution.


Does phenotypic evolution occur via *cis* or *trans* changes? As with most scientific controversies, the truth lies somewhere in between the caricature of the two camps. Carroll and colleagues were the only group to look directly at morphology, but they only studied a single gene. Wittkopp looked at more genes than Carroll’s group, but she was examining gene expression — a bit less complex than morphology. Wittkopp ****did**** show that *trans* changes contribute to the evolution of gene expression, and Bustamante and colleagues showed that transcription factors are under positive selection. But so are CREs, as Andolfatto demonstrated.


Sean Carroll likes to point out how important CREs are in the evolution of form. Bustamante joked (with Carroll in the room) that his data paint a very different picture, with transcription factors playing an important role. Why do they take such a polarizing stance? Well, controversy fuels excitement. If we all just acknowledged that *cis* and *trans* changes are important, we could move on toward understanding how these two regulatory mechanisms work together to contribute to evolutionary change. Carroll’s story is incomplete without understanding why the wing spot phenotype is sexually dimorphic — probably due to some *trans* regulation. Furthermore, Bustamante and Andolfatto have shown that both transcription factors and CREs contain signatures of positive selection. Wittkopp was the only one to simultaneously examine *cis* and *trans* factors, finding evidence that both are evolutionarily important. Let’s end the stupid bickering– it’s not doing anyone any good.

#A List of Doctors to Avoid
Date: June 19, 2006

The [Physicians and Surgeons for Scientific Integrity](http://www.pssiinternational.com/) [*sic*] (aka, Doctors Doubting Darwin) are planning to [Resolve the Conflict](http://www.pssiinternational.com/save_the_date.htm) between Darwin and Design. Something tells me the conflict will not be resolved using rational discourse, but rather apologetics and obfuscation. I have reproduced a list of physicians and surgeons to avoid below the fold.


(Via [Red State Rabble](http://redstaterabble.blogspot.com/2006/06/brainstorming.html).)





This list is updated at least monthly and was last updated on June 15, 2006. Members are listed<br/>
alphabetically.


Name Medical Practice City and State


Dr. Katherine A. Anderegg Pathology Sullivan’s Island, South Carolina<br/>
Dr. Robert W. Arnold Pediatric Ophthalmology Anchorage, Alaska<br/>
Dr. James A. Avery Internal Medicine/Pulmonary New York, New York<br/>
Dr. E. Harold Bammel Dentistry Kerrville, Texas<br/>
Dr. Jon W. Beasley Family Practice Clearwater, Florida<br/>
Dr. Michael R. Buglione Dentistry Vestal, New York<br/>
Dr. Jon P. Burdzy Family Practice Fort Myers, Florida<br/>
Dr. Barry L. Carey Emergency Medicine Lakeland, Florida<br/>
Dr. Robert E. Carpenter Dentistry Kerrville, Texas<br/>
Dr. L. Nichols Cook Ophthalmology Knoxville, Tennessee<br/>
Dr. Peter E. Dawson Dentistry St. Petersburg, Florida<br/>
Dr. Robert W. Dettmer Nephrology Fort Wayne, Indiana<br/>
Dr. Edmund T. Dombrowski Orthopaedic Surgery Redlands, California<br/>
Dr. Michael R. Egnor Pediatric Neurosurgery Stony Brook, New York<br/>
Dr. Donald R. Eubanks Cardiology Largo, Florida<br/>
Dr. Ivette P. Farmer General Practice Sapulpa, Oklahoma<br/>
Dr. Ilona J. Farr Family Practice Anchorage, Alaska<br/>
Dr. Mark W. Fitzmaurice General Practice Hornsby, Australia<br/>
Dr. Dennis K. Gabos Cardiology Allison Park, Pennsylvania<br/>
Dr. Stanley B. Gathinston III Immunology Chicago, Illinois<br/>
Dr. James P. Gills Ophthalmology Tarpon Springs, Florida<br/>
Dr. John T. Goodgame General Surgery Belleair, Florida<br/>
Dr. Terry J. Hamblin Hematology/Oncology Bournemouth, United Kingdom<br/>
Dr. Donald E. Hedrick General Dentistry Whittier, California<br/>
Dr. Marvin M. Hewlett, Jr. General Dentistry Greenville, Alabama<br/>
Dr. Jay L. Hollman Cardiology Baton Rouge, Louisiana<br/>
Dr. Liang Hong Dentistry Overland Park, Kansas<br/>
Dr. David K. Hopson Pathology Saint Charles, Missouri<br/>
Dr. Geoffrey J. Howe Orthopaedic Surgery Marietta, Georgia<br/>
Dr. Michael D. Jacobson Addiction Medicine/Family Practice West Chester, Ohio<br/>
Dr. Howard M. Johnson Family Physician Ellerbe, North Carolina<br/>
Dr. Thomas D. Jones Pediatrics Donalsonville, Georgia<br/>
Dr. Matthew P. Kaul Physical Medicine/Rehabilitation Portland, Oregon<br/>
Dr. Vernon S. Kent Obstetrics &amp; Gynecology Homer Glan, Illinois<br/>
Dr. Rick L. Kimbrel Dentistry Vidalia, Georgia<br/>
Dr. Carl J. Kissling Family Practice Lincoln, Nebraska<br/>
Dr. Charles H. Lasley Thoracic and Cardiovascular Surgery Belleair, Florida<br/>
Dr. Paula F. Leis Dermatology Houston, Texas<br/>
Dr. Jean K. Lightner Veterinary Medicine Painesville, Ohio<br/>
Dr. Douglas A. Lipperd Family Practice Rainbow City, Alabama<br/>
Dr. Gary E. Marais Cardiology Yucaipa, California<br/>
Dr. Paul K. Mensah Obstetrics &amp; Gynecology Dumfries, Scotland<br/>
Dr. Donald R. Moeller Maxillofacial Surgery Columbus, Georgia<br/>
Dr. Frank H. Moore Emergency Medicine Eagle River, Alaska<br/>
Dr. Daniel W. Nesselroade Obstetrics &amp; Gynecology Springfield, Ohio<br/>
Dr. John K. Olinger Internal Medicine Clearwater, Florida<br/>
Dr. Daniel M. Palmquist Family Practice Cloquet, Minnesota<br/>
Dr. William J. Parsons Cardiology Raleigh, North Carolina<br/>
Dr. Lewis F. Powell Ophthalmology Belleair, Florida<br/>
Dr. Richard A. Perallon Dentistry Palatka, Florida<br/>
Dr. Robert A. Perallon Clinical Pharmacology Tampa. Florida<br/>
Dr. Cesar E. Restrepo Orthopaedic Surgery Miami, Florida<br/>
Dr. F. Hall Reynolds II Dermatology Chattanooga, Tennessee<br/>
Dr. James A. Rogers Family Practice Cloquet, Minnesota<br/>
Dr. Andrew D. Rule Nephrology &amp; Hypertension Rochester, Minnesota<br/>
Dr. Matti S. Salo Anesthesiology Raisio, Finland<br/>
Dr. Robert M. Scovner Family Practice Frederick, Maryland<br/>
Dr. Geoffrey S. Simmons Internal Medicine Eugene, Oregon<br/>
Dr. James O. Smith Cardiology Tampa, Florida<br/>
Dr. Thomas B. Stogdill Family Practice Bluffton, Indiana<br/>
Dr. Gary L. Treece Endocrinology/Metabolism Yakima, Washington<br/>
Dr. Brian A. Trimble Neurology Eagle River, Alaska<br/>
Dr. Mikko J. Tuuliranta General Surgery Jyvaskyla, Finland<br/>
Dr. David J. Turell Internal Medicine Tomball, Texas<br/>
Dr. Lasse J. Uotila Clinical Chemistry &amp; Pathology Helsinki, Finland<br/>
Dr. John R. Vasko, Jr. Radiology Sammamish, Texas<br/>
Dr. Grahame J. Wells Family Practice St. Albans, United Kingdom<br/>
Dr. Robert F. White Anesthesiology Knoxville, Tennessee<br/>
Dr. Bruce D. Woodall Family Practice St. Maries, Indiana<br/>
Dr. Thomas H. Yau Ophthalmology Silver Spring, Maryland

#Better Wine Through Science
Date: June 19, 2006

Humans have been genetically engineering agricultural crops ever since plants were domesticated. Notice the difference between [maize](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-07b6d1395abb83aa3de25c4fafcd4506-maize.JPG) and [teosinte](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7d704b9036962aadd7e3276b35976661-teosinte.JPG). Or how about what we have done to create modern<br/>
[wheat](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-e0f0c2197df894cd28f715270487aee2-wheat_polyploid.png). By either imposing an artificial selection pressure on a few important loci (in the case of maize) or encouraging larger, polyploid individuals (wheat), we have been manipulating the genome of plants for a long, long time. More genetic manipulation of agriculture below the fold.





The difference between the genetic engineers of yore and those of today are the tools at their disposal. The molecular revolution of the past few decades has led to faster mechanisms for manipulating genomes. It all began with selective breeding of individuals with favorable characteristics — ie, larger fruits, increased milk production, etc.


Another simple trick involves crossing your strain of interest with another strain that has some property that you would like in your strain. For example, say your favorite tree species has been devastated due to an invasive pathogen, but a related species from the pathogen’s native habitat carries one or multiple resistance genes. If you can cross the two species, you can select hybrids that are resistant to the pathogen. In order to recover the traits characteristic of your species, you will then backcross the hybrids to your species, again selecting for resistant individuals. You will continue these backcrosses for multiple generations until you have created a resistant strain of your species. The goal in these hybrid backcross projects is to introgress the resistance genes (and only the resistance genes) into your species from the other species.


While the traditional approaches work, they can be extremely time consuming. That’s why scientists are looking more and more towards using molecular tools to genetically engineer agricultural plants and animals. Grape wine cultivars have been bred for many generations to produce certain flavors and aromas in the finished product. If a certain strain of grapes are susceptible to a pathogen, viticulturalist are tentative to perform hybrid backcrosses to transfer resistance genes from another strain for fear of losing the characteristics that give those grapes their desirable properties.


A group at Missouri State University is working to [characterize genes in American grape varieties that make them resistant to fungal infections](http://www.livescience.com/technology/ap_060615_better_wine.html). They plan to transfer those genes into European strains so that they can be grown in the humid Midwestern climate. Missouri’s not exactly a hotbed for wine production, but that could change if the vineyards are able to produce grapes that can tolerate the new environment.

#The Evolution of Women in Academia
Date: June 20, 2006

Two months ago, I [wrote](http://scienceblogs.com/evolgen/2006/04/women_in_the_sciences.php) about Teri Markow resigning as President of the [Society for the Study of Evolution](http://www.evolutionsociety.org/) in protest of the appointment of the new Editor in Chief. Long story short, Markow was unhappy that women were not considered for the editorship of the Society’s journal, [*Evolution*](http://lifesciences.asu.edu/evolution/). Don Waller, the current president of the Society, and Sally Otto, Vice-president and Associate Editor of *Evolution*, have both published letters to *Nature* (available [here](http://dx.doi.org/10.1038/441812c) and [here](http://dx.doi.org/10.1038/441812b)) in which they take issue with the journal’s coverage of Markow’s resignation. Waller points out that the Society actively recruited women for its Editor in Chief position:





> 
It is true that we have had only one female editor-in-chief. However, five women were approached about their interest in the position before Mark Rausher was appointed, and all declined. The selection process involved a committee of six instead of three as stipulated in our by-laws, but this widened input and involved more women — including Theresa Markow. She raised no procedural issues until after the new editor-in-chief had been selected, agreed to serve and started to appoint other editors. The officers and council responded promptly to these issues in early 2006, but voted to approve the new editors rather than repeat the search.



Markow’s resignation is beginning to look rather foolish if this is the case. Not only were women given the opportunity to serve as Editor in Chief, Markow had no objections to the hiring process while it was going on. Otto suggests some reasons as to why women were turning down the offers:


> 
In the face of the many obstacles to women in science, those of us who have established successful research careers have done so by learning to say “no”. The task of editor-in-chief is onerous, time-consuming and largely thankless. Potential candidates recognize that saying “yes” to this particular job comes at an enormous personal cost — to one’s research career, students and family.



Incidentally, the [Society for the Study of Evolution’s annual meeting](http://life.bio.sunysb.edu/ee/sse2006/) kicks off this weekend. [Mike](http://scienceblogs.com/mikethemadbiologist/) and I will be there, will you? Give a holler in the comments.

#Donors Choose Update
Date: June 20, 2006

In case you forgot, the evolgen [DonorsChoose Blogger Challenge](http://scienceblogs.com/evolgen/2006/06/a_great_opportunity_to_support_1.php) is [still going on](http://scienceblogs.com/evolgen/2006/06/donors_choose_update.php). My great readers have [donated $135](http://www.donorschoose.org/locale0/challenge.php?id=143) towards public science education (as of 8:00am on June 20). That’s 27% of the way towards our goal of $500. There are still [many proposals](http://www.donorschoose.org/locale0/challenge.php?id=143#proposals) left to be funded, and I don’t think anyone would complain if we surpass our goal. You have until July 1 to contribute to the ScienceBlogs Blogger Challenge if you would like to be entered in the contest to win [great prizes](http://scienceblogs.com/evolgen/2006/06/a_great_opportunity_to_support_1.php).


Click [here](http://www.donorschoose.org/locale0/challenge.php?id=143) to contribute. Any size contribution would be greatly appreciated by the students and teachers involved.


[<img border="0" src="http://www.donorschoose.org/images/link/link_donorschoose_medium.gif"/>](http://www.donorschoose.org/challenge.php?id=143)

#Mendel’s Garden Call for Submissions
Date: June 22, 2006

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://mendels-garden.blogspot.com/)


Hsien Lei will be hosting the second edition of [Mendel’s Garden](http://mendels-garden.blogspot.com/2006/06/mendels-garden-2-call-for-submissions.html) at [Genetics and Health](http://www.geneticsandhealth.com/). Submission information is available [here](http://blogcarnival.com/bc/eprof_4399.html).

#Live Blogging from Evolution 2006
Date: June 25, 2006

Blogging’s been light recently. I spent the last week frantically preparing my talk for the [Evolution Meeting](http://life.bio.sunysb.edu/ee/sse2006/). I was analyzing data until late Thursday evening. Then, I still needed to finish putting together my talk. We spent Friday driving to the meeting, and I gave my talk yesterday morning (Saturday). It went well — I got a packed (albeit, small) room and had enough time for two questions (one coming from a guy whose research I deeply admire).


You probably won’t see any new posts until later next week. I’ll be spending my time listening to other people talk about their research and catching up with folks I haven’t seen since the last time we were at a meeting together. I’ve been able to chat with a few of those people, but there are still more I need to track down.


[Mike](http://scienceblogs.com/mikethemadbiologist/)‘s also here, but we didn’t get much time to talk. Hopefully I’ll run into him again (I’m gonna try to make it to his session). I’m also gonna try to keep my eyes peeled for [Carl](http://scienceblogs.com/loom/).

#Archival ‘n Chips
Date: June 26, 2006

I [said](http://scienceblogs.com/evolgen/2006/06/live_blogging_from_evolution_2_1.php), “You probably won’t see any new posts until later next week.” Well, today is later this week than yesterday. Also, this post hardly counts as anything new. As you will see, it’s a link to all the evolgen content you should have already read. If you haven’t, read it. It’s good. We like it, that’s why we link to it.


When [some blogs](http://scienceblogs.com/mikethemadbiologist/) go on vacation, they repost old content from the archives. At evolgen, there are not archives, [only Zuul](http://imdb.com/title/tt0087332/quotes). Actually, there are archives, and you can find them [here](http://scienceblogs.com/evolgen/archives.php). But those are just archives of posts that have been posted since we moved to [Los Bloggos de Ciencia](http://scienceblogs.com/) . . . post. Here’s a list of the content you should read that’s sitting on the Blogger server waiting for the guvmint to classify as terrorist correspondence:





- [Cryptic Species](http://evolgen.blogspot.com/2005/01/cryptic-species.html)
- [Human Inversion Under Selection](http://evolgen.blogspot.com/2005/02/human-inversion-under-selection.html)
- [The Cambrian Explosion Never Happened](http://evolgen.blogspot.com/2005/02/cambrian-explosion-never-happened.html)
- [On the origin of life](http://evolgen.blogspot.com/2005/03/on-origin-of-life_09.html)
- [Chimera](http://evolgen.blogspot.com/2005/05/chimera.html)
- [Random Mutation and Natural Selection](http://evolgen.blogspot.com/2005/06/random-mutation-and-natural-selection.html)
- [The Genotype and the Phenotype and How to Measure Divergence](http://evolgen.blogspot.com/2005/07/genotype-and-phenotype-and-how-to.html)
- [Can You Marry Your Half-Sister’s Daughter?](http://evolgen.blogspot.com/2005/10/can-you-marry-your-half-sisters_16.html)
- [Species Sampling for Whole Genome Sequencing](http://evolgen.blogspot.com/2005/12/species-sampling-for-whole-genome.html)
- [A Few Words on Speciation](http://evolgen.blogspot.com/2005/12/few-words-on-speciation.html)
- [Mean Fitness, Genetic Load, and the Misapplication of Population Genetics Metrics](http://evolgen.blogspot.com/2005/12/mean-fitness-genetic-load-and.html)
- [Why Study Speciation Genes?](http://evolgen.blogspot.com/2005/12/why-study-speciation-genes.html)


**Are Deletions Deleterious?**


- [Part 1](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-1.html)
- [Part 2](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-2.html)
- [Part 3](http://scienceblogs.com/evolgen/2006/01/are_deletions_deleterious_part.php)


**Detecting Natural Selection**


- [Introduction](http://evolgen.blogspot.com/2005/11/detecting-natural-selection_20.html)
- [The Organization of the Genome](http://evolgen.blogspot.com/2005/11/detecting-natural-selection-part-1.html)
- [The Organization of Genes](http://evolgen.blogspot.com/2005/11/detecting-natural-selection-part-2.html)
- [Codon Based Models for Detecting Selection](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-3.html)
- [Phylogenetics and Relative Rates](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-4.html)
- [Allele and Genotype Frequencies in Populations](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-5.html)
- [Calculating Nucleotide Sequence Polymorphism](http://evolgen.blogspot.com/2006/01/detecting-natural-selection-part-6.html)
- [Nucleotide Polymorphism and Selection](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php)
- [Polymorphism and Divergence](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php)

#Home and Tired
Date: June 28, 2006

I got back from the [Evolution Meeting](http://life.bio.sunysb.edu/ee/sse2006/) late last night. The first thing on my docket was clearing flies so I could collect virgins this morning. The rest of this week will be spent getting myself back up to speed. I had some fruitful discussions at the meeting, attended some interesting talks, and met some cool people. I even got to talk to a couple other ScienceBloggers ([Mike](http://scienceblogs.com/mikethemadbiologist/) and [Carl](http://scienceblogs.com/loom/)). Also the Kitzmiller session was quite good — and an excellent respite from the heavy science stuff in all the other session. If your brain’s been working hard thinking about science, creationism is the best form of rest. I’ll have more to say about the meeting in subsequent posts.

#Flock of Dodos
Date: June 29, 2006

[They](http://life.bio.sunysb.edu/ee/sse2006/) showed us the movie [Flock of Dodos](http://www.flockofdodos.com/) at the Evolution Meeting. Randy Olson (the director) was also there to field questions. So was [Eugenie Scott](http://scienceblogs.com/evolgen/2006/02/eugenie_scott_in_cell.php). Someone asked about scientists boycotting the Kansas Kangaroo Court. Olson thought it was a bad idea because it made it look like the scientists couldn’t deal with the “criticisms of evolution”. Scott defended the boycott by arguing that because the school board’s decision was predecided, scientists would have only legitimized Connie Morris and her posse of ignorance. In case you care, I agree with Genie.


I think Flock of Dodos is a movie worth seeing (especially if you can swing a free screening). Olson’s thesis is that the creationist movement is bad for the American education system, but biologists aren’t doing a very good job dealing with it. He’s really into catch phrases and framing, but it would be much nicer if we could educate the public to the point where it can move beyond advertising tactics.  Until then, we need something equivalent to “Teach the Controversy” for our side. Any suggestions?

#Donors Choose – Just a Few Days Left
Date: June 29, 2006

The[ ScienceBlogs Donor’s Choose Blogger Challenge](http://scienceblogs.com/evolgen/2006/06/a_great_opportunity_to_support_1.php) is nearing completion. You have until July 1 to donate to the [evolgen challenge](http://www.donorschoose.org/challenge.php?id=143) and forward your confirmation email to [this address](mailto:sb.donorschoose.bonanza@gmail.com) if you want to win any of the [awesome prizes](http://scienceblogs.com/evolgen/2006/06/a_great_opportunity_to_support_1.php).


If we reach our goal of $500, Donors Choose will kick in an extra 10% for every dollar contributed — we’re 42% of the way to our goal. Let’s see if we can make a big push in these last few days. Janet has compiled a [list of blogs](http://scienceblogs.com/ethicsandscience/2006/06/last_leg_of_sbdonorschoose_dri.php) that are close to their goals, but she left evolgen off the list. We’re just as close (if not closer than those other guys), so you should [donate to the evolgen challenge](http://www.donorschoose.org/locale0/challenge.php?id=143) so that we can reach our goal.


[<img border="0" src="http://www.donorschoose.org/images/link/link_donorschoose_large.gif"/>](http://www.donorschoose.org/challenge.php?id=143) 

#A Scientist in Time
Date: July 20, 2006

****If you could have practiced science in any time and any place throughout history, which would it be, and why?****


That’s what [they](http://scienceblogs.com/seed/2006/07/ask_a_sb_july_19.php#commentsArea) are asking us this week. And, once again, I’m going to skirt the question. You see, it depends on whether the future counts as a “place throughout history.” Currently, the future is not history, but it will be history once the future becomes the past. You’ll probably need a few minutes to digest that, as I must have just blown your mind. Or not.





My chosen place in time (yeah, I abandoned the whole history thing): the day of the $1000 genome. This is the population geneticist’s dream. All that data for next to nothing in costs. So many hypotheses to test. So many discoveries to make. We ain’t post-genomics until genome sequencing is dirt cheap and accessible to everyone, regardless of their taxon of choice.


If I am restricted to only time periods that are currently considered history, then I’d have to go with [Thomas Hunt Morgan](http://nobelprize.org/nobel_prizes/medicine/laureates/1933/morgan-bio.html)‘s lab at Columbia in the 1910s. At that time, [Alfred Sturtevant](http://www.nap.edu/readingroom/books/biomems/asturtevant.html) was constructing the first ever genetic map, and the lab was giving birth to the field of genetics.

#El Jardín de Mendel Numero 3
Date: July 21, 2006

The third edition of Mendel’s Garden — **the** genetics blog carnival — has been posted at [Viva la Evolucion](http://vivalaevolucion.blogs.ie/2006/07/21/mendels-garden-3/). Nothing like a blog with a Spanish language flavor hosted by an Irish website. Check out los artículos de genética.

#I Write, Nicholas Wade Listens
Date: July 21, 2006

Or maybe his copy editor reads this blog. Either way, there are changes afoot at the NY Times.


Three days ago I [ragged](http://scienceblogs.com/evolgen/2006/07/decoding_the_1000_genome.php) on NY Times science reporter Nicholas Wade for using the word ‘decode’ when describing genome sequencing. In his [latest article](http://www.nytimes.com/2006/07/20/science/20cnd-neanderthal.html) he has improved. Last time he wrote about cheap whole genome sequencing; this time he has written about sequencing of a Neanderthal nuclear genome. Now, Wade hasn’t entirely kicked the decoding habit:





> 
The project is a collaboration between Dr. Svante Paabo of the Max Planck Institute for Evolutionary Anthropology in Leipzig, Germany, and 454 Life Sciences, a Connecticut company that has developed a new method of sequencing, or decoding, DNA.



[454](http://www.454.com/) is getting some sweet free advertising from one of the world’s largest newspapers. Wade’s also performing better. He’s started to refer to genome sequencing as what it is: sequencing, [not decoding](http://scienceblogs.com/evolgen/2006/01/chad_is_my_muse.php). But he still feels the need to qualify it by referring to it as decoding. He even slips back into his decoding ways later in the article:


> 
The first goal of the project will be to sequence three billion units of Neanderthal DNA, corresponding to the full length of the Neanderthal genome. This will require decoding 20 times as much DNA, because so much of the DNA in the Neanderthal bones belongs to bacteria.
Genomes must usually be decoded several times over to get a complete and accurate sequence, but the first three billion bases of Neanderthal should “hit all the essential differences,” Dr. Egholm said.



The rest of the article is pretty good, and Nicholas Wade is a good science writer. If we could only get him to stop butchering the terminology.

#Welcome to the Blogiversity
Date: July 22, 2006

I just realized that the (relatively) recent ScienceBlogs addition [Dynamics of Cats](http://scienceblogs.com/catdynamics/) is authored by a faculty member from my university. It only took me two months after he came on board to notice [Steinn Sigurðsson’s academic affiliation](http://scienceblogs.com/catdynamics/about.php). The two of us, along with [Monsieur Bérubé](http://www.michaelberube.com/), are representing the school quite well. Additionally, my alma mater has a [few](http://migration.wordpress.com/) [bio](http://helixer.blogspot.com/)–[bloggers](http://evolutionlist.blogspot.com/). Sadly, there is also a [growing movement of anti-science](http://positiveliberty.com/2005/11/cornells-new-idea-club.html) on the hill.


How is the blogging scene at your University? If you aren’t an academic, how’s the blogging scene in your town or community?

#Jesus’s Cheerleader
Date: July 23, 2006

Have you ever wondered what’s happened to the line between church and state? Are you curious as to whether the line has been blurred or erased altogether? Clay at DeadlyHippos investigates the issue by [visiting a megachurch in Tennessee](http://www.deadlyhippos.com/columns/Churchtastic.htm). The church is run by Maury Davis, a man convicted of first degree murder who subsequently found God-uh and was saved-ah. Pastor Davis is also the man behind the DVD **Islam: An Evil Religion (A four part series)**. Oh the sweet, sweet irony. In between pop songs, unraveling of enormous American flags, and fireworks displays, Davis cheers the virtues of traditional American “values” like ignorance and the cult of domesticity. This particular bullet from the post sums up the religious right pretty well:


> 
Davis finally quotes the bible at 7:42. However, he does so without referencing the Bible or even having a copy of the book himself. In fact, I realize I haven’t seen a single Bible or any other book since I entered the church.



The American Jesus don’t need no books. He don’t need no bible. He’s got his megachurch, his fifty foot flag, and his xenophobia.


(Via [Deadspin](http://deadspin.com/sports/tag/leftovers-189199.php).)

#Honey, I Blew Up the Bugs
Date: July 23, 2006

Check out [these pictures](http://pishmo.com/macro/) of tiny little critters up close. Wow! Thanks to [Neil](http://psychro.bioinformatics.unsw.edu.au/neil/index.php?option=com_jd-wp&amp;Itemid=70&amp;p=166) for pointing this out.

#More Critter Pictures
Date: July 27, 2006

I’m in the process of exhuming myself from under a mountain of work, that’s why the posting’s been ultra-light. My [last link](http://scienceblogs.com/evolgen/2006/07/honey_i_blew_up_the_bugs.php) to cool pictures of bugs went over well, so I’m giving you a few more pictures. These come from a [friend’s website](http://www.pbase.com/andywilson/insects) — be sure to check out the bee and its hitchhiker.

#Neutral Markers and Conservation Genetics
Date: July 31, 2006

I have [mentioned before](http://scienceblogs.com/evolgen/2006/07/who_or_what_deserves_protectio_1.php) that at one point in my life I wanted to study conservation genetics. This field can be thought of a subdiscipline of molecular ecology — wherein researchers use molecular markers to test hypotheses regarding demography in their population of interest. Jacob at Salamander Candy has a [post on the usefulness of neutral markers in conservation](http://www.salamander-candy.com/2006/07/how_useful_are_neutral_genetic.htm). Here’s a taste:





> 
The problem is, a growing body of evidence suggests that patterns of variation and divergence in adaptive traits are not well reflected by neutral markers (refs 1-8). In the hypothetical species mentioned above, a small amount of gene flow between east and west would be enough to swap small numbers of alleles. This would hardly affect the divergent neutral genotypes at all, but newly introduced advantageous alleles would increase in frequency even if they were originally rare. For example, maybe all the northern ones have adaptations for cold temperatures and the southern ones are adapted to warmth. This pattern would not be reflected in the neutral markers.



Neutral markers are only useful for inferring recent demographic history. I don’t think anyone has ever argued that they can be used to identify selection history. It has been pointed out, however, that [many molecular ecologists don’t understand the assumptions](http://scienceblogs.com/evolgen/2006/07/empty_dna.php) that go along with using certain molecular markers or [don’t perform the appropriate statistical tests](http://scienceblogs.com/evolgen/2006/02/the_coalescent.php) when analyzing their data. Even if we understand the limits of molecular markers for studying populations, how useful are they? Jacob makes a good point regarding what we are ultimately interested in conserving:


> 
It’s much easier to just take some tissue samples, extract DNA, and look at neutral molecular patterns. This convenience, combined with the “coolness factor” of new technology, and the large amount of information obtainable from molecular analysis, has made neutral markers popular in conservation biology. Remember, though, we’re not really interested in conserving neutral biodiversity per se, so its only useful to the extent that it’s a proxy for adaptive biodiversity.



There are definitely people who spend a fair bit of time characterizing adaptations in natural populations. These researchers also identify non-neutral characteristics that differentiate populations. The problem is that these projects are much more time consuming than ones which sample a random collection of neutral loci. Sometimes we only have the time and resources to infer demographic history rather than unique adaptations within populations.


These issues are related to [stuff](http://scienceblogs.com/gnxp/2006/07/but_they_did_interbreed_maybe.php) [Razib](http://scienceblogs.com/gnxp/2006/07/limelight_on_a_stupid_commente.php) [has been](http://scienceblogs.com/gnxp/2006/07/introgression_whats_in_a_word_1.php) [writing about](http://scienceblogs.com/gnxp/2006/07/introgression_related_papers.php) at Gene Expression. Razib’s posts deal with human population history and the introduction of advantageous alleles from archaic humans. In the case of the conservation examples, selection prevents the introgression of alleles between populations if there are different selection pressures in the two populations. Introgression between populations will differ depending on the selective regimes around a particular locus. That means that the majority of loci will reveal the general history of a population, whereas individual loci may tell a unique story if they were under selection.

#Jean Informatique Organizes a Cold Spring Harbor Meeting
Date: July 31, 2006

A few weeks ago I [introduced](http://scienceblogs.com/evolgen/2006/07/more_uses_for_the_ome_suffix_1.php) y’all to Genoinformatics, the hot new abbreviation for Genome Informatics (some sort of derivative of Bioinformatics). I pointed out that I have quite a few international collaborators in this research area, including people in Italy (Geno Informatico), Germany (Jan Informatik) and Mexico (Juan Informatica). I’m hoping that one of them makes it to [this CSH/Wellcome conference](http://meetings.cshl.edu/meetings/infouk06.shtml) on Genome Informatics in September. Maybe they could even give a presentation on the Genome-ome.


(Via [Post-Genomics](http://www.medbioworld.com/postgenomics_blog/?p=102).)

#No More Napa Valley Wines?
Date: July 31, 2006

[This week’s Ask A Science Blogger](http://scienceblogs.com/seed/2006/07/ask_a_sb_july_26.php) deals with global warming and wine production:


> 
****I heard that within 15 years, global warming will have made Napa County too hot to grow good wine grapes. Is that true? What other changes are we going to see during our lifetimes because of global warming?…****



I am not a climate scientist, and I don’t know enough about viticulture to offer an informed response regarding the effects of temperature changes on grape production. I do know, however, that Napa Valley wines tend to be very generic stereotypes of the European varieties they’re imitating. This wouldn’t be terrible, except that they are a bit too pricey for what you get. I’m usually a big fan of Chardonnay (or pretty much anything dry), but the Napa productions that I’ve tried taste of oak chips and artificial fruitiness. Don’t get me wrong, I like an oakey Chardonnay, but there’s a limit to how much tree I can tolerate in my wine. As long as South American (cheap), Australian (well priced), and Burgundy (best tasting) vineyards aren’t affected by global warming, I’ll be happy.


P.S. Yes, I realize that global warming will probably affect wine making all over the world. This post is mostly about my distaste for California wines (and American wines in general) rather than an intelligent commentary on global warming.

#Michael Egnor is a Crappy Neurosurgeon Who Will Cut out Your Brain and Eat It
Date: August 23, 2006

I gave Lawrence Krauss [some shit](http://scienceblogs.com/evolgen/2006/08/science_religion_and_ignorance.php) regarding his double standard towards the tolerance of willful ignorance. He’s cool with calling intelligent design proponents ignorant, but won’t go all the way and say all religion is anti-knowledge. Judging by the [letters in the NYTimes](http://www.nytimes.com/2006/08/22/science/22letters.html), there are some people who took umbrage at Krauss calling anyone ignorant:


> 
To the Editor:






> 
I disagree with Dr. Krauss’s essay “How to Make Sure Children are Scientifically Illiterate.” Darwinists are not advancing science by seeking federal court injunctions against criticism of Darwinism. Science is not a body of knowledge; it’s a method of inquiry. Freedom of inquiry is indispensable to science.
Students should be taught to question prevailing theories — even entrenched dogma like Darwinism — with rigor and with civility. Contra Dr. Krauss, the opposite of science is not ignorance; it’s censorship.
<p>Michael Egnor, M.D.<br/>
Stony Brook, N.Y.<br/>
The writer is a professor of neurosurgery at the State University of New York at Stony Brook.</p>



<img alt="i-b964467c8ef3f3e5a4396394dd3ba28f-dog_poo_science.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b964467c8ef3f3e5a4396394dd3ba28f-dog_poo_science.gif"/>


And if I took a big ol’ steaming shit on a piece of paper and wanted that taught as science, I should get my wish. Because if you say shit on paper isn’t science that’s censorship. Damn you and your censoring school board.


Dr. Egnor, what if I wanted you to teach your med-school students that messages are propagated along the axon of a neuron by little trucks found in myelin? Oh, wait, that’s a testable hypothesis. We could reject the hypothesis because it’s complete and utter bullshit. We can’t even test intelligent design because it doesn’t even propose hypotheses. We don’t waste time teaching complete bullshit in science classes just because some religious nutjob thinks it’s a good idea. That’s not censorship; it’s not wasting valuable class time on the myelin truck theory.


(Via [Evolutionary Times](http://evolutionarytimes.blogspot.com/2006/08/drosophila-melanogaster-my-cousin.html).)

#Polar Bear Shrinkage
Date: August 23, 2006

**To be filed under: “Every dude who’s gone swimming in a cold pool in the Hamptons could have told you that.”**


[Polar bear genitals are shrinking](http://www.livescience.com/animalworld/060823_polar_bears.html):


> 
The icecap may not be the only thing shrinking in the Arctic. The genitals of polar bears in east Greenland are apparently dwindling in size due to industrial pollutants.
Scientists report this shrinkage could, in the worst case scenario, endanger polar bears there and elsewhere by spoiling their love lives and causing their numbers to peter out.



<br/>
<img alt="i-3c76ee851cefacd837cef4cceba16f2a-george_costanza.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3c76ee851cefacd837cef4cceba16f2a-george_costanza.jpg"/>


Is it possible that the shrinkage could be due to the fact that they live in the arctic? You know, where it’s really freakin’ cold. And they go swimming in the water. That’s been known to cause significant shrinkage in a certain other mammalian species.


Also, mad props to Charles Q. Choi, the author of this article, for using the following words: shrinkage, peter, and “bearded seal”. In case you’re wondering, this is what we’re talking in terms of size for a polar bear package:


> 
The adult polar bear testicles the researchers examined were on average roughly three inches across and 1.8 ounces in weight, although they could dramatically enlarge during the height of sexual activity from January to July. Their bacula, or penis bones, were on average nearly seven inches long.


#Interesting Reviewers
Date: August 24, 2006

Jim Crow has published a [perspective in Genetics](http://www.genetics.org/cgi/content/full/173/4/1833) on his favorite reviewers from 1952-1956 when he was associate editor of the journal. He prefaces it by writing:


> 
As far as I can ascertain, the editorial correspondence from that period is lost, so I am writing this from memory. Naturally, my recollections of events half a century ago are fallible, but I think I remember the essence. The identity of reviewers was confidential, so some of what follows is a breach of that confidence. I do not know whether there is a statute of limitations, but it seems reasonable that after half a century some revelations are permissible. I have, however, not mentioned the authors in most cases.



I’ll leave it up to our [in house ethicist](http://scienceblogs.com/ethicsandscience/) to say whether this is an acceptable practice. Most of the characters are deceased, so I don’t think they’ll be complaining to the current editorial board. I’m just a sucker for this historical stuff. I’ve reproduced a few choice quotes below the fold. This is more anecdotal than anything else, but it does allow us to peer into the lives of our academic predecessors.





On [Sewall Wright](http://en.wikipedia.org/wiki/Sewall_Wright), who made sure to check your math:


> 
Sewall Wright was a particularly thorough reviewer. When he received a manuscript for review, he typically dropped other activities and went over the copy in great detail. Usually this involved his redoing all the calculations and reanalyzing the data. Alex [**RPM: managing editor Alexander Brink**] and I were convinced that he was spending too much time on other people’s data, at the price of not getting his own more important work done. For that reason, we employed him sparingly, only where his unique insights were essential.



But sometimes Wright’s services were necessary. He could even be impressed if the work was exceptional:


> 
My final example again involves Wright as reviewer. In 1953 the Genetics Society of America met in Wisconsin and a young Japanese mathematical geneticist was in attendance. He had with him a manuscript ([KIMURA 1954](http://www.genetics.org/cgi/reprint/39/3/280)) that he had written during the long sea voyage across the Pacific. He showed that, even in an infinite population, random fluctuation of selection intensities can mimic the effect of random allele-frequency drift in a finite population. Kimura’s diffusion equation was bewilderingly complex, but he found a transformation that converted it into the simple equation for heat conduction, whose solution is well known and familiar to every physics student.
We gave the manuscript to Wright to review. Wright was not given to verbal effusion, but his review was closer to unequivocal praise than any that he had done earlier. It was the beginning of Wright’s life-long admiration of Kimura’s work; and Kimura’s first GENETICS reviewer was his idol, Sewall Wright. Soon after, they both moved to Wisconsin, to the delight of all three of us.



On knowing about your equipment:


> 
The third manuscript involved neutron radiation. In this case, the astute reviewer, Aaron Novick, knew enough about the physics in the author’s laboratory to be aware of gamma contamination in the neutron source and to adjust the dosimetry accordingly. The data now fitted the expectations much more closely and the manuscript was much improved. The author was both chagrined not to notice this error and thankful for the reviewer’s insight.



On the rift between Sturtevant and Dobzhansky (you can read about it [here](http://www.enovitski.com/sturtevant-dobzhansky-novitski/frame.html)):


> 
My fifth example involves the well-known mutual dislike between Sturtevant and Dobzhansky (e.g., [NOVITSKI 2005](http://www.drosophilahistory.com/sturtevant-dobzhansky-novitski/body.html)). Nevertheless, in all my contacts with both men over the years, and these were extensive, I never heard either of them say anything derogatory about the other, although both of them were aware that I knew of the rift.
But on one occasion Sturtevant did make his feelings known, although with characteristic subtlety. GENETICS had received two manuscripts. One was by a young cytogeneticist and the other by Dobzhansky. Sturtevant reviewed both. His reply was essentially as follows: The first paper is careful work by a serious, deserving young scientist, but it does not quite measure up to GENETICS standards. I say, reject with regret. The Dobzhansky paper must surely be published. But it is too long for its content and generally overstated. I say, accept with regret.


#Positive About Prokaryotes
Date: August 24, 2006

In [last week’s edition](http://scienceblogs.com/evolgen/2006/08/phylogeny_friday_18_august_200.php) of Phylogeny Fridays, I mentioned [an essay](http://dx.doi.org/10.1038/441289a) that argued that biologists should refrain from using the term “prokaryote” because its definition is entirely negative. The author, Norman Pace, writes, “no one can define what is a prokaryote, only what it is not.” Furthermore, I pointed out that prokaryotes are a paraphyletic taxon (a statement that drew some criticism in the comments), and that alone should lead to the disuse of the term.


In a [letter to Nature](http://dx.doi.org/10.1038/442868c), Bill Martin and Eugene Koonin point out that there **is** a positive definition of prokaryotes:


> 
Prokaryotes are cells with co-transcriptional translation on their main chromosomes; they translate nascent messenger RNAs into protein. The presence of this character distinguishes them from cells that possess a nucleus and do not translate nascent transcripts on their main chromosomes.



Pace, however, had mentioned that archaeal transcription differs from bacterial transcription; archaea use TATA-binding proteins, much like eukaryotes, whereas bacteria do not. While the coupling of transcription and translation is a unifying property of archaea and bacteria, it may be a shared ancestral character (a symplesiomorphy). Shared ancestral characters are phylogenetically uninformative and a bane of cladists.

#Did Evolution Go Missing?
Date: August 24, 2006

The Chronicle of Higher Education [reported](http://chronicle.com/temp/reprint.php?id=v6pywllczrz22q3ybkb4b94qrx35ckr7) earlier this week that the U.S. Department of Education neglected to include “Evolutionary Biology” in it’s list of eligible majors for the new Smart Grant program (the [NYTimes](http://www.nytimes.com/2006/08/24/washington/24evo.html?pagewanted=print) and [New Scientist](http://www.newscientist.com/article/dn9833-us-department-of-education-overlooks-evolution.html) also report on this, as does the [NCSE](http://www.ncseweb.org/resources/news/2006/US/364_what_happened_to_evolutionary__8_22_2006.asp)). This drew extra attention because the omission left a blank space where the option had been removed. Blank spaces were also created when they eliminated the “Exercise Physiology” and “Heating Ventilation and Air Conditioning Technology” majors from the list. This leads one to believe the Department of Education encourages fat and sweaty college kids as much as it discourages evolutionary biology.


<img alt="i-5ab8ace5f813f38c46b208c5c5a68d67-MissingEvolution.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5ab8ace5f813f38c46b208c5c5a68d67-MissingEvolution.jpg"/>


The removal of the Evolutionary Biology option does not mean that evolution has been expunged from the list of majors entirely; The “Ecology, Evolution, Systematics and Population Biology” parent category is still there, along with all of the majors within, except for Evolutionary Biology.


Does this mean that the US government is on a crusade against evolution — not allowing undergrads who study evolution to get federal money? Keep in mind what Dobzhansky wrote:


> ****Nothing in biology makes sense except in the light of evolution.****


As long as there are biology options (ecology, population biology, systematics, molecular biology, cell biology, genetics, etc) evolution will be prominent. By removing the Evolutionary Biology major, it’s as if the Department of Education is saying that’s it’s ridiculous to offer an “evolution” option when all of biology is evolution. Of course, that may just be wishful thinking on my part.

#Birth Rates, Politics, and Education
Date: August 24, 2006

A [couple](http://scienceblogs.com/pharyngula/2006/08/i_guess_you_can_be_innumerate.php) of [people](http://stevereuland.blogspot.com/2006/08/idiots-are-multiplying.html) are talking about [this Wall Street Journal article](http://www.opinionjournal.com/editorial/feature.html?id=110008831) which claims that Republicans are popping out more babies that Democrats, and this disparity will lead to an relative increase in the frequency of Republicans. It’s been pointed out that the statistics and logic behind this argument are flawed, and the math in the article is atrocious.


Depending on your political persuasion, the outcome suggested in the article may be viewed as a good thing or a bad thing. Regardless of the political direction you’d like to see our country take, however, I think we can all agree that we’d like to see intelligent people spreading their seed. The problem is the well educated (allow me use this as a proxy for intelligence) are often burdened with the task of learning and spreading knowledge to the point where their reproductive success suffers. Floyd Reed and Chip Aquadro offer a solution to this paradox in a [recently published Opinion](http://dx.doi.org/10.1016/j.tig.2006.07.005) in Trends in Genetics:


> 
Genetic and cultural fitness can also directly conflict with each other. A clear example is the role of celibacy in many religions, which allows individuals to redirect their energy from child rearing to proselytism, but at the expense of darwinian fitness [[42](http://www.simonyi.ox.ac.uk/dawkins/WorldOfDawkins-archive/Dawkins/Work/Books/selfish.shtml)]. Higher education can also present a similar cultural-genetic conflict; that is, training others, and spreading the cultural trait, comes at the expense of genetic fitness [[43](http://dx.doi.org/10.1016/j.evolhumbehav.2005.07.004)]. (However, the positive correlation between income and number of offspring [[43](http://dx.doi.org/10.1016/j.evolhumbehav.2005.07.004)] suggests a way society could mitigate this conflict to some degree — by increasing the salaries of educators, especially during their reproductive years!)



The rest of the article, dealing with topics in human evolution, is a good read for [those people](http://scienceblogs.com/gnxp/) interested in such boring topics.

#Phylogeny Friday – 25 August 2006
Date: August 25, 2006

[Last week’s Phylogeny Friday](http://scienceblogs.com/evolgen/2006/08/phylogeny_friday_18_august_200.php) introduced the three domains of life: bacteria, archaea, and eukaryotes. The bacteria and archaea are commonly referred to as prokaryotes, although that creates a paraphyletic taxon. Today, we will focus on the eukaryotes (organisms with nuclei and organelles). This taxon contains plants, animals, fungi, and a bunch of other lineage with which you are probably not familiar. Recently, a group of Canadian researchers [reviewed](http://dx.doi.org/10.1016/j.tree.2005.09.005) our current understanding of the evolutionary relationships of the eukaryotes, creating the following phylogeny.





<img alt="i-5feb4cbd88250bf87fba504db8ff9f8c-Eukaryote_tree.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5feb4cbd88250bf87fba504db8ff9f8c-Eukaryote_tree.jpg"/>


The eukaryotes can be divided into five supergroups (they left off the [Damn Yankees](http://en.wikipedia.org/wiki/Damn_Yankees_%28band%29)), but the relationship of these groups is unresolved. The green supergroup in the upper left contains the photosynthetic lineages (both the green and red ones), including the land plants. In the bottom right are the unikonts, a taxon that contains the fungi ([ascomycetes](http://www.tolweb.org/tree?group=Ascomycota), [basidiomycetes](http://en.wikipedia.org/wiki/Basidiomycota), [zygomycetes](http://en.wikipedia.org/wiki/Zygomycota), [chytrids](http://en.wikipedia.org/wiki/Chytrid), and [microsporidia](http://en.wikipedia.org/wiki/Microsporidia) which were recently classified as fungi) and animals (in the red box). The other three supergroups — excavates, rhizaria, and chromalveolates — are made up entirely of organisms commonly known as protists.


What does this tree tell us about eukaryotes? Animals and land plants, the two taxa people are most familiar with, are represented by single lineages — twigs on the tips of the tree. The fungi make up a substantially larger part of the tree, but they are also extremely diverse relative to animals and land plants (in terms of ecology and physiology). The rest of the lineages of the tree (including the other unikonts and non-land plants) are lumped into the paraphyletic taxon [protists](http://en.wikipedia.org/wiki/Protist). Considering the diversity of this group, and the fact that these organisms make up the majority of the tree, we could say that protist is a synonym for eukaryote.


Next week we’ll take a closer look at one of the many lineages of the eukaryotes: the animals. Why? It could be because animals are the most interesting, it could be our bias (we are animals and we’re interested in learning about ourselves), or it could be because I know more about this lineage than any of the other ones.


---



Keeling, PJ, G Burger, DG Durnford, BF Lang, RW Lee, RE Pearlman, AJ Roger, MW Gray. 2005. The tree of eukaryotes. *Trends Ecol Evol* 20: 670-676. doi: [10.1016/j.tree.2005.09.005](http://dx.doi.org/10.1016/j.tree.2005.09.005)

#American Idol and Grant Review
Date: August 25, 2006

As a fledgling scientist, I am not privy to the process of grant view. It may as well occur behind a green curtain, and all I get to see is the hologram of the intimidating wizard in the form of an email announcing that I did or did not get the tiny morsel of cash I so politely requested (sorry for the bad *Wizard of Oz* metaphor). But real people review those grants, and those real people have personalities. Some are surly and dominant. Others are polite and passive. In fact, we can draw parallels between reviewers and *American Idol* panelists, as this [Correspondence to *Cell*](http://www.cell.com/content/article/fulltext?uid=PIIS0092867406010245) cleverly does:





> 
A typical day at one of the many NIH study sections goes something like this. Of approximately 50-70 investigator-initiated/R01 applications reviewed, about half are triaged and the rest are subjected to lengthy discussion, despite the fact that in most of the cases the initial scores are close. Like the amateur singers on the television talent show American Idol, each grant application is evaluated by three reviewers. And, when opinions are conflicting, the three reviewers may display a peculiar resemblance to the American Idol judges, Paula Abdul (sympathetic), Randy Jackson (neutral), and Simon Cowell (hostile). Due to the specialization of science, the discussion is often limited to the three reviewers, with the other study section panelists rarely participating. Indeed, sometimes, while the three reviewers wrangle over a particular application, others are busy on their laptop computers. It is difficult to determine whether these panelists are reading the application under discussion, preparing for the next discussion, or answering their emails. The necessarily inexpert or distracted panelist often sides more easily with the Cowellesque reviewer, who is trashing the application, especially when there is not enough money to go around. This leads to the perception that “the nasty reviewer always wins.” Remember, everyone on the study section votes to determine the final score — even those who are busy with their emails.



The author, Michele Pagano, goes on to point out the great costs that go into the grant review process, from the time spent preparing the grant proposal to assembling the study sessions. And this doesn’t include the actual funding requested by the applicant. Pagano suggests ways to streamline the grant review process including prescreening applications using letters of intent, turning the focus of the evaluation from nitpicky criticisms to big picture issues like the impact of the proposed research, eliminating detailed timelines from proposals, and using electronic forms of communications to save money on travel to a common meeting place. He argues that the peer review process for publication proceeds without face to face meeting, so the grant review process should be able to as well.


As a neophyte outsider to the process, I’m hardly in a position to offer much of an opinion. Do any of my more experienced readers have something to add?

#Mendel’s Garden — Second Call for Submissions
Date: August 28, 2006

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://blogcarnival.com/bc/cprof_349.html)


This is a reminder that I will be hosting the fifth edition of [Mendel’s Garden](http://mendels-garden.blogspot.com/). Mendel’s Garden is a blog carnival devoted to genetics. If you have written a genetics related entry on your blog and would like it to be included in the carnival, either email me (evolgen [at] yahoo [dot] com) or use the [Blog Carnival submission page](http://blogcarnival.com/bc/cprof_349.html). You may submit entries relating to quantitative genetics, molecular genetics, evolutionary genetics, developmental genetics, or anything else genetical in nature. The deadline for submission is this Friday (September 1).

#Incorrect Guide to Science
Date: August 29, 2006

The Panda’s Thumb is compiling a chapter by chapter rebuttal of [*The Politically Incorrect Guide to Darwinism and Intelligent Design*](http://www.pandasthumb.org/archives/2006/08/the_politically_1.html). This book is part of a series of shoddy academic endeavors that includes [*The Politically Incorrect Guide to American History*](http://evolgen.blogspot.com/2005/03/politically-incorrect-history.html); they could remove the word “politically” from the cover and capture a more accurate representation of the contents within. PZ Myers has posted his reply to the chapter on embryology [here](http://scienceblogs.com/pharyngula/2006/08/the_politically_incorrect_guid.php), and he [points out](http://scienceblogs.com/pharyngula/2006/08/more_pigdig_is_on_the_way.php):


> 
“It really is a ghastly, badly done book, and unfortunately, while it only takes one dishonest fool to spin a lie, it takes a whole team to undo it.”



Every sentence in one of these anti-science texts requires a full paragraph to explain why it’s total bullshit. It’s real easy to lie and spread misinformation; it’s a lot harder to actually spread knowledge honestly. Anyway, to help in the fight against willful ignorance, link back to the Panda’s Thumb with the title of the book like so: [*The Politically Incorrect Guide to Darwinism and Intelligent Design*](http://www.pandasthumb.org/archives/2006/08/the_politically_1.html).

#Human Rights, Respect for Science, and the Judicial Process
Date: September 22, 2006

Five nurses and one doctor have been imprisoned in a Libyan jail since 1999, accused of infecting more than 400 children with HIV. They were condemned to death in 2004, but the verdict was overturned. Their retrial is due to wrap up very soon, and they await their new sentencing. The problem: the scientific evidence indicates that the medics were not responsible for the infections, but the Libyan courts threw out that evidence. For more information, please read the [news item](http://dx.doi.org/10.1038/443254b) in *Nature* as well as [this editorial](http://dx.doi.org/10.1038/443245b).


The blogosphere has been quite active in the cause to free the Tripoli six (the name given to these medical workers). Declan Butler, a senior reporter for *Nature*, has written about this on [his blog](http://declanbutler.info/blog/?p=59). He has also compiled a [list of links to the coverage](http://www.connotea.org/user/Declan/tag/tripoli%20six?num=25). I particularly like [what Revere at Effect Measure has to say](http://scienceblogs.com/effectmeasure/2006/09/the_tripoli_six_campaign.php):


> 
This is not just a human rights story, although if it were “just” that it would have a major claim on our interests and our hearts. But it is also a science story: how scientific evidence, presented by one of the world’s leading HIV scientists, was rejected out of hand, the only exculpating evidence possible in favor of these five nurses and a doctor.



This is an issue involving human rights. This is an issue involving respect for science. And this is an issue involving the judicial process. We should all care about these things, especially human rights. If you would like to help out with this matter, Mike Dunford has [some suggestions](http://scienceblogs.com/authority/2006/09/the_tripoli_six_do_something_a.php).

#The Old Guy Strikes Again
Date: September 23, 2006

I saw [him](http://scienceblogs.com/evolgen/2006/09/meeting_local_retirees.php) again at the supermarket today. This is the second time I’ve seen him at the supermarket — and third time overall. But this is the first time I was stealth, without any identification of my alma mater.


In case you’re late to the game, this guy is an emeritus professor at my current university, and he also graduated from my *alma mater*. I’ve run into him twice around town, both times I had something that gave away my former school (either a hat or a license plate holder). Like the last time I saw him in the supermarket, we were both in the produce section. However, today I was sporting the hat from our local minor league baseball team. I was incognito.


I recognized him, but I don’t think he recognized me. I didn’t want to say anything so as to give up my presence. He asked one of the box boys where the hydroponic lettuce is. Seriously. That’s how this man rolls. Anyway, I’ll see if I can observe *Retiree alumnicus* in the wild again. Any notes will be reported here.

#A Dozen or So Songs About Me
Date: September 24, 2006

**I’m so vain I think I every single song is about me.**


[Dave](http://scienceblogs.com/worldsfair/2006/09/if_i_were_to_make_a_this_is_my.php) wants us to “make a music mix that is a reflection of your informative years.” I have no idea what he means by informative years, so I’m going to give you a list of a dozen or so songs (Dave’s suggested amount) that I like, each by a different band. The order makes sense to me, but it probably won’t make sense to you.





<li>
**Faith No More – Falling to Pieces**<br/>Faith No More’s *The Real Thing* was the first album I ever bought with my own money. I would listen to the album over and over, singing along to the lyrics in the liner notes. Once I memorized all the words to “Falling to Pieces”, I would sing them nonstop.
</li>
<li>
**Skid Row – 18 and Life**<br/>Skid Row? Yeah, Skid Row. I liked Skid Row. Still do a little bit, I guess.
</li>
<li>
**Guns N’ Roses – Welcome to the Jungle**<br/>The video for this song came out right about the time my friends and I discovered MTV. We stayed up late at night during sleepovers so that we could catch the video. *Appetite for Destruction* is still a great album nearly twenty years after its original release.
</li>
<li>
**Red Hot Chili Peppers – I Could Have Lied**<br/>*Blood Sugar Sex Magik* was our favorite album throughout middle school and at the beginning of high school. “Our” being me and my friends. I remember learning how to play the intro to this song on guitar, and I felt so cool.
</li>
<li>
**Pearl Jam – Daughter**<br/>I need to include one song from the mid 90s grunge explosion. I listened to Nirvana’s *Nevermind* and Pearl Jam’s *Ten* a lot when they first came out. But this song is on *Vs*. This is my list, so I get to make the rules. By the way, Scott Stapp has made Pearl Jam unlistenable. How? Well, every time I hear Eddie Vedder’s voice, I think of Stapp and get sick. Thank you Scott. I can’t listen to Nirvana anymore either, but I can get an indirect fix from Sonic Youth and the Pixies.
</li>
<li>
**Smashing Pumpkins – Frail and Bedazzled**<br/>I listened to *Pisces Iscariot* a lot. I liked it better than *Siamese Dream*, but they were both soundtracks to high school. I had a hard time to pick a song off of one of these albums as my favorite.
</li>
<li>
**Radiohead – Just**<br/>*The Bends* was my favorite album during that transition period from high school to college. This is one song off of that album.
</li>
<li>
**Beck – Pay No Mind (Snoozer)**<br/>Holy shit, Beck blew the lid off of the trashcan. *Mellow Gold*‘s got some weird stuff on it. This is song two on the album.
</li>
<li>
**The Clash – White Man In Hammersmith Palais**<br/>This song captures the fusion of reggae and 1977 punk that was The Clash. We’d all turn rebellion into money if we could.
</li>
<li>
**Rancid – Cash, Culture and Violence**<br/>*Life Won’t Wait* has been my favorite album for over five years. This is one song from that album, but I could have chosen a few others for this list.
</li>
<li>
**The Violent Femmes – Kiss Off**<br/>Good shit, yo. Good shit. And it’s not about Jesus, like some of their later stuff.
</li>
<li>
**Propagandhi/The Weakerthans – Anchorless**<br/>The Weakerthans have the best lyrics of any emo-country-pop-folk-punk band. They are also the only bad in that genre, but their lyrics are still amazing. This song represents the transition from the political punk of Propagandhi to the Canadian folk-pop of the Weakerthans.
</li>
<li>
**Fugazi – Waiting Room**<br/>If you don’t know Fugazi, well, I don’t know what to tell you. They definitely stand for their values, I can tell you that. And, by values, I don’t mean conservative Christian “values”.
</li>


These were the first 13 songs (a dozen or so) that came to mind. There are many others important songs that aren’t on the list. Maybe I’ll do a favorite albums list or favorite bands list in the future. Or maybe not.

#Yet Another “-ome”
Date: September 25, 2006

<img alt="i-f6662302f177480e8c321774697ad564-swami_meditation.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-f6662302f177480e8c321774697ad564-swami_meditation.jpg"/>


As if we didn’t have enough “-omes” in biology, I have come across yet another one: the methylome. It’s beginning to sound like a Hindu temple up in here. My fascination with the -ome is [well documented](http://scienceblogs.com/evolgen/2006/07/more_uses_for_the_ome_suffix_1.php). I was made aware of the newest -ome in my collection by [this preview](http://dx.doi.org/10.1016/j.cell.2006.09.002) of [this article](http://dx.doi.org/10.1016/j.cell.2006.08.003) in *Cell*. The preview is entitled “The First High-Resolution DNA ‘Methylome'” and describes a paper announcing a high resolution map of DNA methylation in the *Arabidopsis* genome.


We can thank Andrew Feinberg for [introducing the term](http://dx.doi.org/10.1038/83825) methylome, which he defines as “a neologism that describes the complete set of DNA methylation modifications of a cell.” Check out [this definition](http://medical-dictionary.thefreedictionary.com/Neoligism) of neologism: “A meaningless word used by a psychotic.” Exactly. Enough with the -ome already.

#Circus of the Spineless Gets Inked
Date: September 25, 2006

Ever wanted a tattoo of your favorite invertebrate? I hear they’re inking insects, molluscs, cnidarians, and all kinds of other spineless critters in Monterey. To get your tat, visit the newest edition of [Circus of the Spineless](http://deepseanews.blogspot.com/2006/09/circus-of-spineless_25.html) at Deep-Sea News.

#An Informal Poll: Coffee Drinkers
Date: September 26, 2006

If it weren’t for coffee, much of the research that gets done today wouldn’t get done. Or so I am led to believe based on the reverence people seem to have for the sacred bean. Frankly, I’d trade a cup of coffee for a good night’s sleep. Hell, I’d trade a cup of coffee for pretty much anything. That’s because I don’t drink coffee. That sound you hear is from all the coffee drinkers (yourself included, probably) letting out a gasp as they wonder how someone can function without a chugging liquid caffeine.


I don’t really have a secret. I just never got addicted — to coffee or caffeine. I’ve limited my caffeine intake to the occasional piece of chocolate, and now I feel like a social outcast within academic circles. Even my non-coffee drinking advisor keeps himself well caffeinated with an endless supply of Coca-Cola products. Thankfully, I still imbibe vast quantities of alcohol, the lubricant for both scientific and flirtatious discourse. But pounding a sixer of Natty Light doesn’t go over so well in the lab, unlike the ubiquitous coffee mugs and doubled-up cups with cardboard sheaths to protect your delicate hands from hot-hot heat generated by your favorite bean grinder’s blend.


So what’s a non-coffee drinker to do when everyone around him drinks the brew like it’s going out of style? And don’t say tea; it’s just caffeine in different packaging. I tried drinking tea at a seminar once and had a head-ache for the rest of the afternoon. I’m not bothered by the coffee, but by the ceremony surrounding the whole process. I feel left out — like I don’t belong — when it’s time to grind the beans or brew a pot. And “getting a cup of coffee” has become synonymous with “an informal conversation”. Can I just talk to you for a little while without your circadian crutch?


Anyway, this is my long winded way of asking you whether you drink coffee or not. If you do, are non-coffee drinkers like an exotic creature to you? If not, do you feel ostracized by the caffeine-o-philes?

#Tangled Bank #63
Date: September 27, 2006

[<img alt="The Tangled Bank" class="inset" height="31" src="http://pharyngula.org/images/tbbadge.gif" width="88"/>](http://tangledbank.net/)


The [sixty-third edition of Tangled Bank](http://www.indiancowboy.net/blog/?p=329) has been posted by the Indian Cowboy. Go read yourself some science.

#Near Blog Encounters
Date: September 28, 2006

There is a general disconnect between the people I know from blogging and the people I know in real life. I’ve only met a couple of my fellow SciBlings (ScienceBlogs siblings): [John Lynch](http://scienceblogs.com/strangerfruit/), [Mad Mike](http://scienceblogs.com/mikethemadbiologist/), and [Carl](http://scienceblogs.com/loom/). I have only once had a real world friend or colleague overtly mention to me in real life that they read or know of my blog. That’s probably to be expected given that I blog pseudo-pseudonymously (quite an appropriate title given my study species).


But there are people in the bloggy world who have figured out who I am in the real world. I don’t mind, I just ask that they not broadcast it all over the internets because that would clog up the tubes. The tubes don’t want to be clogged. There are also a couple of real world people who have found the blog. [This post](http://evolgen.blogspot.com/2005/03/ripping-someone-new-asshole-using-only.html) got some attention at one meeting because the paper it deals with was a hot topic of conversation. One graduate student I know from another university has even gone so far as to leave comments. Who knows how many others know about my internet presence.


At a meeting earlier this year, a post-doc came thisclose to saying, “You’re the dude who writes that crappy blog, evolgen.” We were talking about speciation or human evolution when he said to me, “Like you said, humans and chimps didn’t speciate.” I hadn’t said that during our conversation. I had written it [here](http://scienceblogs.com/evolgen/2006/05/human_and_chimp_speciation_1.php), on my blog. So, what I’m wondering is if very few people know about the blog, and they’ve commented or alluded to it conversation? Or if a bunch of people know about it, but they haven’t said jack?

#Is the Y Chromosome as Flawed as mtDNA for Studying Demographic History?
Date: September 30, 2006

Via [Yann Klimentidis](http://yannklimentidis.blogspot.com/2006/09/male-vs-female-genetic-histories-among.html) come [this paper](http://dx.doi.org/10.1093/molbev/msl088) comparing patterns of polymorphism in mitochondrial DNA (mtDNA) and Y chromosomes from Native American populations. The premise is that mtDNA reveals the maternal demographic history (because mitochondria are only passed from mother to child) of a population and the Y chromosome reveals the paternal history (Y chromosomes are only passed from father to son). But we know that [mtDNA is not a good marker for studying demographic history](http://www.scienceblogs.com/evolgen/2006/07/empty_dna.php) because there is no recombination in the mitochondrial genome. That means that evidence of demographic events will be swept away by natural selection anywhere on the molecule.


How about the Y chromosome? Most of the Y chromosome is non-recombining, although there is some genetic exchange with the X and some intra-Y chromosome recombination. The lack of recombination in the mitochondrial genome coupled with evidence for natural selection on that molecule make any mtDNA locus a poor neutral marker. The Y chromosome is thought to have ‘decayed’ do to recurrent fixations of deleterious mutations — these alleles cannot be removed from a population because they are genetically linked to advantageous alleles elsewhere on the Y chromosome.


So, I ask you, is the Y chromosome a poor molecular marker for studying demographic history? If so, what do comparisons of polymorphism between the Y chromosome and mtDNA tell us? Note that the genealogical relationships (ie, the tree topology) should not be affected by the lack of recombination. But the shape of the tree will be dramatically altered every time natural selection alters the frequency of an allele. This shouldn’t affect our ability to determine maternal and paternal relationships amongst close relatives; it **will** affect our ability to say whether there was a recent change in population size (ie, within the last <strike>2N</strike> 1N generations<sup>1</sup>) and determine gene flow between populations.


---



1- It’s 2N not 4N because it’s a haploid genome, right? [**Edit: As John McDonald points out in the comments, it’s 1N because the Y-chromosome is only found in males. Duh! Looks like I’ve gotta retake popgen.**]

#Insert Clever Title Playing on ‘Bitch’
Date: September 30, 2006

Thomas Huxley was known as “[Darwin’s Bulldog](http://en.wikipedia.org/wiki/Thomas_Henry_Huxley#Darwin.27s_bulldog)” for his many public defenses of Darwin’s theory of evolution by natural selection. Richard Dawkins has been labeled “[Darwin’s Rottweiler](http://www.discover.com/issues/sep-05/features/darwins-rottweiler/)“, drawing a parallel between Dawkins’s current role as a popularizer of science (and defender of reason) and Huxley’s as a public face for Darwin. All that is well and good, and you probably knew it already. But when can we start calling [PZ](http://scienceblogs.com/pharyngula/2006/09/you_must_be_kidding_mr_unwin.php) [Myers](http://scienceblogs.com/pharyngula/2006/09/a_devils_catechism.php) “Dawkins’s Pit Bull”?

#…and Geico Commercials
Date: October 24, 2006

<img alt="i-f52b1bb64c9d4886baf0b9be650ba678-caveman.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-f52b1bb64c9d4886baf0b9be650ba678-caveman.jpg"/>


Some biologist in Poland [said this](http://news.yahoo.com/s/afp/20061023/sc_afp/polandpoliticsreligionscience):


> 
“A scientist showed me a picture of an American boxer. He had all the traits of Neanderthal man. These people are among us. They are part of the human race, probably more prevalent once upon a time, but who still exist.”



Apparently those dudes from the Geico commercials are practicing the sweet science. But there ****are**** Neanderthals among us. Razib knows about them . . .


Big ups to whoever comes up with the best Polish invention for this item.


(Via [The Panda’s Thumb](http://www.pandasthumb.org/archives/2006/10/more_zaniness_f.html).)

#They Live . . . in the North
Date: October 24, 2006

No ocean, river, or fjord is safe. [The manatees are attacking](http://www.cnn.com/2006/US/10/24/memphis.manatee.ap/index.html). We’ve [warned you](http://scienceblogs.com/evolgen/2006/08/manatees_invade_the_northeast_1.php) before. Now is the time to take notice. Three times is a trend.

#Chuck Norris Hates Your Evolution (He Sound Like Uzbeki Talk About Science)
Date: October 25, 2006

Chuck Norris doesn’t “believe” in evolution. And he’s writing about it in the Wing Nut Daily. I won’t link to the actual article, but you can read about it [here](http://pandagon.net/2006/10/23/fact-chuck-norris-does-not-understand-irony/). Thanks [Bora](http://scienceblogs.com/clock/2006/10/chuck_norris_creationist_1.php) for ruining my appreciation for the Texas Ranger.


To make up for bringing you that bad piece of news, here are the first four minutes of Borat’s movie:


<br/>
<object height="350" width="425"><param name="movie" value="http://www.youtube.com/v/I1jt4dvxoWw"/><param name="wmode" value="transparent"/><embed height="350" src="http://www.youtube.com/v/I1jt4dvxoWw" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>

#I Push Flies Like Weight
Date: October 25, 2006

<img alt="i-a4cc5dc305fa2019974e9d1573009ad4-Dpse.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-a4cc5dc305fa2019974e9d1573009ad4-Dpse.gif"/>


At the same time this post is scheduled to go up on ScienceBlogs I should be a few thousand feet above Middle America. I’ll be spending the rest of this week in Tucson, Arizona (which I like to pronounce “Tuck-Sun”, like how we pronounce Phoenix “Pa-Ho-Nix”) learning about *Drosophila*. Sure, I already know a bit about these [pomace flies](http://www.genetics.org/cgi/content/full/162/1/1), but after this weekend I’ll have **mad** *Drosophila* skills. They should provide an excellent complement to my [chicken-head-eating skills](http://scienceblogs.com/evolgen/2006/09/i_am_not_a_nerd.php).


This is my way of saying posting will be light — kinda like the weight I push — over the next few days.

#Vegas, Baby!
Date: October 25, 2006

I’m currently waiting to board my third flight of the day (for [this trip](http://scienceblogs.com/evolgen/2006/10/pushing_flies.php)), sitting in the Las Vegas airport. The Las Vegas airport has free wireless! That’s a good thing. It’s also packed with slot machines going “bleep-bleep”, “ding-ding”, and “do-do-do-do”. That’s kind of annoying.


The southwest is sunny and warm. That’s also a good thing. Considering that this is what I left this morning:





<img alt="i-c8d966cb56a9c49c734adfe7d6c3f1e8-PA_weather.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c8d966cb56a9c49c734adfe7d6c3f1e8-PA_weather.gif"/>


And this is where I’m going to be tonight:


<img alt="i-9281444ca1de6bc41f5db70b077e3343-AZ_weather.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9281444ca1de6bc41f5db70b077e3343-AZ_weather.gif"/>

#Neofunctionalization is Overrated
Date: October 26, 2006

At least in polyploids.


So says [this review](http://www.scq.ubc.ca/?p=596) of [this paper](http://www.pnas.org/cgi/content/abstract/103/8/2730). Take home message: polyploids use duplicates to buffer against mutations to important genes.


The review is brought to you by the [Science Creative Quarterly](http://scq.ubc.ca/). SCQ is brought to you by David Ng, who has just introduced [The Filter](http://scq.ubc.ca/filter/). If you don’t know about SCQ, you should check it out. And if you know about SCQ but haven’t discovered The Filter, you should check that out.

#Support Science Education in Ohio
Date: October 26, 2006

On November 7, the citizens of the State of Ohio will be voting a bunch of things, just like all Americans. One of the things they’ll be voting for are seats on the State Board of Education (see Ed’s discussion [here](http://scienceblogs.com/dispatches/2006/08/help_defeat_a_creationist_in_o.php)). In one of the races, an anti-science incumbent, Deborah Owens-Fink, is running against a pro-science challenger, Tom Sawyer. Owens-Fink has been far more successful at raising money than Sawyer. To donate money to Sawyer’s campaign visit [here](http://www.votetomsawyer.com/contact.htm#don).


More about the push for donations can be found below the fold.





Taken from [EvolDir](http://evol.mcmaster.ca/~brian/netevoldir/Other/Ohio.election):


> 
Dear Fellow Scientists,
In 2003, our community pulled together in response to an attack on the teaching of evolution in Ohio. We electronically circulated a petition that was signed by over 6,800 scientists across our country. Many of you also wrote passionate comments about how this 10th grade “critique of evolution” lesson plan was just flat out bad science. We were able to take those signatures and comments to the Ohio Board of Education and show the moderates on the board the strength of support for the teaching of evolution within the scientific community. Unfortunately, at the time there were not enough moderates on the Board to overturn the lesson plan &amp; it narrowly passed.
The architect of that anti-science lesson plan, Deborah Owens-Fink, is up for re-election on Nov. 7th. About one month ago, we again electronically circulated a similar request for help to our scientific community, this time requesting a small donation to help us to replace Owens-Fink with a pro-science candidate (former congressman Tom Sawyer) for the Ohio Board of Education. We assumed that if ~50% of those 6,800 people would donate $5-10, we would be in a strong position to get our pro-science message out &amp; counter what would likely be a well funded Owens-Fink campaign.
Unfortunately, our community has yet to pull through for us this time. We have only received 191 donations, &amp; over half of these have been from non-scientists concerned with the state of science education in our country. Owens-Fink has raised three times as much money as we have, with her average donor giving over $1000. She has been sending out more mailings &amp; buying more advertising than we have been able to afford.
We would like to ask you to reconsider helping us out. If you would simply go to www.votetomsawyer.com and donate $10 and then pass this message to a handfull of your colleagues (much the same as you did with our 2003 petition), we will be able to afford the mailings we need to get our word out in these crucial final 2 weeks of the campaign. We have the chance to depose the single most effective promoter of Intelligent Design in Ohio, and for a mere $10, you can help us to eliminate the need to have to circulate the NEXT petition to once again implore the moderates on the Ohio BOE to reject the next anti-science lesson plan that Owens-Fink will assuredly propose if she is re-elected!
Thank you for your help,
<p>Steve Weeks<br/>
Randy  Mitchell<br/>
Peter Niewiarowski<br/>
Francisco Moore<br/>
Todd  Blackledge<br/>
Rich Londraville</p>
PS please act soon – the Election will be November 7, and we need to be active well ahead of then.


#Lots of Papers Coming out on the Honeybee Genome
Date: October 26, 2006

There will be a ton of papers coming out over the next couple of weeks on the recently sequenced honeybee (*Apis mellifera*) genome. Carl Zimmer has a [review](http://scienceblogs.com/loom/2006/10/25/to_bee_1.php) of some of the topics that will be addressed in the upcoming publications. The NHGRI also has [some information](http://www.genome.gov/19518753).

#The Good, The Bad, and The Pretty
Date: October 30, 2006

Big ups to the grad students in the Ecology and Evolution Department at the University of Arizona for putting on a killer Halloween party and letting a couple of uncostumed drosophilists share in the festivities. That’s the good.


The bad: US Airways. I just found out my last flight (of three) on my way home was canceled. Instead of arriving home at 9:30am, I won’t get there until the early afternoon. There goes my plan to get work done on Monday.


Said drosophilists who were treated to vast amounts of ethanol by the Arizonans also got to see some really cool looking flies. I took some pictures, and once I get back home I’ll upload them to my computer. Keep your eyes peeled for some pretty pictures of flies with striped abdomens and colorful testes on this very website.

#Drosophila Pictures (Part 1)
Date: October 31, 2006

As [promised](http://scienceblogs.com/evolgen/2006/10/the_good_the_bad_and_the_prett.php), I’ll be posting some pictures of flies we caught in and around Tucson, Arizona. But before we can talk about the flies, we need to talk about the local flora. The host plants for these flies serve as a restaurant, [discotheque](http://comp.uark.edu/~karbuck/song.html), bedroom, and nursery. In Arizona, those plants tend to be cacti. Below the fold I have posted a picture of a cactus that met a sad fate: it was struck by a bold of lightning. But from the ashes of the dead saguaro came a diverse collection of *Drosophilids*.





*Drosophila* are ****NOT**** [true fruit flies](http://en.wikipedia.org/wiki/Tephritidae) (see [here](http://www.genetics.org/cgi/content/full/162/1/1)), and they prefer rotting plants to fresh fruit. For this reason, they are not an agricultural pest — the plants they hang out on have already been done in by various microbes, including yeast which ferment the sugar.


<img alt="i-42750a9b8ba6111295a0400cf85ec268-cactus_night.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-42750a9b8ba6111295a0400cf85ec268-cactus_night.gif"/>


Many of the flies in the Southwest United States and Mexico feed, mate, and oviposit in rotting cacti. A cactus may have a gash in which yeast have infected, fermenting the sugar found in the fleshy insides. Or a cactus could be struck down by a bolt of lightning:


<img alt="i-d05f89faf5f0cb9340ee2b7f6bce9400-cactus_lightning.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d05f89faf5f0cb9340ee2b7f6bce9400-cactus_lightning.gif"/>


When we cut open one of the fallen arms, this is what we found:


<img alt="i-0e103f7c8163985b955efbbaf16e6039-cactus_rot.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-0e103f7c8163985b955efbbaf16e6039-cactus_rot.gif"/>


As a point of comparison, the [flesh of a saguaro](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-f41a806e34e185b7cb3bc7f2ef160761-saguaro.jpg) is usually bright green or white. This plant has rotted away from the inside, making it a haven for insects of all varieties. In the next post, I’ll show you some of the flies we caught feasting on this banquet of fermenting cactus.

#Are There Any Evolutionary Biologists Doing Hard Time?
Date: November 22, 2006

Colin Purrington, he of the great [textbook disclaimer stickers](http://www.swarthmore.edu/NatSci/cpurrin1/textbookdisclaimers/) (and whose [website](http://www.swarthmore.edu/NatSci/cpurrin1/index.html) is a treasure trove of information), is [looking for some prison statistics](http://evol.mcmaster.ca/~brian/netevoldir/Other/Morality.and.Evolution):


> 
A common theme in anti-evolution writings is that teaching kids about Darwin will cause them to lose all touch with morality, and will end up committing crimes.  It might follow that evolutionary biologists themselves would be especially overrepresented in the criminal justice system.  Therefore, I’d like to make a bar graph that shows the number of individuals with PhDs in evolutionary biology who are currently in prison.  A possible control group to eliminate educational level might be “PhDs in theology” in prison (e.g., Kent Hovind).
Anyone have any ideas on how to get the data?



I’m not sure there will be enough data to make some sort of statistically meaningful comparison. One possible solution is to compare juvenile crime rates in districts where evolution is taught with those where evolution is not taught (controlling for income level and other variables, of course).


You can [email Colin](mailto:cpurrin1@swarthmore.edu) your suggestions of where he can find the data he is looking for.

#The Secretome
Date: November 23, 2006

****Happy Turkey Day!!**** As a special gift, I give you yet another -ome to spread on your Thanksgiving bird. You see, I’m [fascinated by -omics](http://scienceblogs.com/evolgen/2006/10/now_available_the_omeome.php), the idea that you can affix “-ome” to your favorite biological system to prime the hype machine. And what I have for you this time makes me think of the juices of a turkey that’s been cooked just right.


With that in mind, I give you the [secretome](http://www.biomedcentral.com/1741-7007/4/40). From what I can gather, these are the proteins secreted by an organism. The authors of the article linked above are interested in the evolution of the mollusk shell, but [mice](http://www.genome.org/cgi/content/abstract/13/6b/1350), [humans, pufferfish, and pigs](http://dx.doi.org/10.1093/nar/gkh286) all have secretomes. The mouse paper defines the secretome as “the set of soluble proteins secreted into the extracellular environment of a cell.”

#Nature Goes Bucknutty for Copy Number Polymorphism
Date: November 23, 2006

There’s an old paradigm in human population genetics that we each differ from each other by less than one percent at the DNA sequence level. While that may be true for our DNA sequences, [recent work](http://scienceblogs.com/evolgen/2006/02/copy_number_polymorphism.php) indicates that there’s also quite a bit of variation amongst individuals in the actual content of their genomes. Such variation is known as copy number variation (CNV) or copy number polymorphism (CNP). What it means is that some people may have one copy of a genomic region, other may have two, and even others may have none.


*Nature* thinks this research on CNV is quite important, as there are five recent articles in *Nature* journals on this topic:





<li>
An [article](http://dx.doi.org/10.1038/nature05329) in *Nature* by Redon et al (a collaboration between Stephen Scherer’s group and Matthew Hurles’s group) reporting a map of CNV in human populations based on 270 individuals. The found 1,447 regions that varied in copy number (CNVRs) which make up 360 megabases or 12% of the human genome. Many of these regions contain functional genes, which has implications for evolution and disease.
</li>
<li>
*Nature* has also published a [news item](http://dx.doi.org/10.1038/news061120-9) on this research.
</li>
<li>
And *Nature* has a [News and Views article](http://dx.doi.org/10.1038/444428a) on this research.
</li>
<li>
And *Nature* has a [Key Contributors piece](http://dx.doi.org/10.1038/7118xic) on how some of the research was carried out.
</li>
<li>
Not to be left out, *Nature Genetics* will be publishing [an article](http://dx.doi.org/10.1038/ng1921) that compares the two human genome project sequences (that of [Celera](http://dx.doi.org/10.1126/science.1058040) and the one from the [public project](http://www.pnas.org/cgi/content/abstract/101/7/1916)). The comparisons are used to identify structural differences between the two sequences, such as CNVs, inversions, insertions, and deletions.
</li>


While DNA sequence differences or single nucleotide polymorphisms (SNPs) may explain some of the variation within populations and differences between them, structural variations are responsible for a great deal of the phenotypic polymorphism in human populations. And while [some reports](http://www.newscientist.com/article.ns?id=dn10646) are selling this research as revolutionary and groundbreaking (the one I linked to isn’t that bad), keep in mind what [Susumo Ohno](http://www.nap.edu/readingroom/books/biomems/sohno.html) wrote in 1970:


> 
Had evolution been entirely dependent on natural selection, from a bacterium only numerous forms of bacteria would have emerged. The creation of metazoans, vertebrates and finally mammals from unicellular organisms would have been quite impossible, for such big leaps in evolution required the creation of new gene loci with previous nonexistent functions.



Any good evolutionary biologist realizes that all differences between taxa were, at some point, polymorphism within populations. The reason these recent papers are important is not because they shift paradigms, but because they represent [technological achievements](http://scienceblogs.com/evolgen/2006/11/the_most_important_development.php). It’s because of our new-found sequencing and computing powers, along with dedicated lab work, that we can study this important feature of genome evolution at a population genetic level.


But these catalogues of structural polymorphisms are only the first step in understanding the how CNV affects evolution. Many more projects need to be completed to determine the phenotypic affects important CNPs (such as [effects of a polymorphic inversion](http://evolgen.blogspot.com/2005/02/human-inversion-under-selection.html) on fecundity). Some of these may represent disease loci. Others will play a role in differences between individuals. And some may even explain differences between human populations.

#The European Creationism Problem
Date: November 24, 2006

Remember [Maciej Giertych](http://scienceblogs.com/evolgen/2006/11/the_dumb_polack_jokes_write_th.php)? He’s the Polish representative to the European Parliament who wants creationism taught in schools alongside evolution. And he’s got a degree in tree physiology, but he claims to be a population geneticist. Anyway, his [letter to *Nature*](http://www.nature.com/nature/journal/v444/n7117/full/444265d.html) made it seem like Poland has this big problem with creationism. Boy was I mislead.


It turns out the problem isn’t just localized to Poland; it’s spread throughout Europe. *Nature* [reports](http://dx.doi.org/10.1038/444406a) that creationists in Great Britain, using the name Truth in Science, are trying to undermine the teaching of evolution. And Letizia Moratti, the former education minister in Italy, removed the teaching of evolution from the nation’s middle schools. And the ARCTUR Research Geological Lab in Moscow, Russia is trying to find geological evidence for creationism. That should keep them busy for <strike>a while</strike> eternity.


The *Nature* article argues that the European creationism movement is more diverse than the American version. They claim that American creationism is mostly limited to evangelical Protestants, whereas European creationists may be Protestant (England), Catholic (Poland, Italy), or Muslim (England, France, and the capital of European creationism, [Turkey](http://www.seedmagazine.com/news/2006/11/not_in_kansas_anymore.php)).

#Yo, Dudes, Surf’s Up on this Meme
Date: November 28, 2006

Everyone’s jumping on [this meme](http://acephalous.typepad.com/acephalous/2006/11/measuring_the_s.html), but I’m not gonna do it. Ooops . . . Looks like I just did it. Anyway, blogging’s been light recently. Hopefully I can get my act together and have some posts up later this week. And by “get my act together” I mean procrastinate from doing real work by writing blog posts.


[They’re](http://scienceblogs.com/retrospectacle/2006/11/im_from_florida_i_have_no_acce.php) [also](http://scienceblogs.com/purepedantry/2006/11/do_you_have_an_accent.php) [all](http://scienceblogs.com/cortex/2006/11/my_accent.php) [doing](http://scienceblogs.com/terrasig/2006/11/i_have_an_accent.php) [this meme](http://www.gotoquiz.com/what_american_accent_do_you_have). Turns out, I’m a “SoCal surfer”. Makes sense — I grew up a few blocks from the ocean. Of course, it could be worse. I could have grown up in Huntington Beach. Then I’d really sound like [Spicoli](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-99ac7cf7c532273bacf59b77a8133400-fast_times.jpg).




<td colspan="2" style="background: white; color: black; padding: 5px;"><b style="font: bold 20px 'Times New Roman', serif; display: block; margin-bottom: 8px;">What American accent do you have?</b> </td>

#Geneticists as Auto Mechanics
Date: November 29, 2006

A couple of years ago, one of the rear tail lights went out on my car. I’m kinda handy with tools, so I figured it would be an easy fix. I poked around in the trunk area with my screw driver, removing every screw within spitting distance of the lamp. I tugged on the lamp in every possible direction, but the sucker wouldn’t budge. Just because I could sequence DNA or parse a genome into the interesting bits and discard the boring stuff, didn’t mean I could perform simple maintenance on my car. I admitted defeat and had the lamp fixed the next time I brought my car in for an inspection.


Why did I tell you this? Partially because of [this story](http://www.lablit.com/article/176/) on LabLit. In that story, Ian Brooks (who has, on more than one occasion, forced me to drink one more Guinness even though it was a Monday night and I needed to get up early the next morning) manages to fix his rear tail light even though he’s a trained molecular geneticist. Ian also manages to get in a few jabs at the automobile inspection standards in Pennsylvania, West Virginia, and Tennessee.


But the real reason I’m writing about auto repair and geneticists is because they’re the topic of a great description of what geneticists do. Bill Sullivan’s “[The Salvation of Doug](http://bio.research.ucsc.edu/people/sullivan/savedoug.html)” is the story of a biochemist and geneticist who try to figure out how cars work. The geneticist performs elegant experiments (creating “mutations”) while the biochemist toils away at grinding things up and mixing things together.


For the sake of fairness, I will point out that Douglas Kellogg (the Doug from Sullivan’s story) has written the biochemist’s version of the tale: “[The Demise of Bill](http://bio.research.ucsc.edu/people/kellogg/contents/Demise%20of%20Bill.html)“. The Bill in question is obviously Dr. Sullivan.

#Free Virtual Conference
Date: November 29, 2006

[<img alt="i-eebecd91f07422e8bc28b8903320e0a2-EMBL_online_PhD.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-eebecd91f07422e8bc28b8903320e0a2-EMBL_online_PhD.jpg"/>](http://onlinesymposium.predocs.org/)


A [bunch of grad students](http://onlinesymposium.predocs.org/about/organizers/) at the [European Molecular Biology Laboratory](http://www.embl.de/) (EMBL) are organizing an [online symposium](http://onlinesymposium.predocs.org/) in the Life Sciences. They’ve got a [list of speakers](http://onlinesymposium.predocs.org/media/overview-of-the-speakers/) and you can upload your own presentation. The conference runs from December 4-8, but I’m not sure how exactly this virtual conference works. If you’re interested, check out their [website](http://onlinesymposium.predocs.org) and read [what other people have to say](http://postgenomic.com/story.php?cluster_id=2793).


(Via [Public Rambling](http://pbeltrao.blogspot.com/2006/11/embl-online-phd-symposium-via-notes.html).)

#Are You a Grad Student Close to Finishing Your PhD?
Date: November 29, 2006

If you’re getting near finishing your PhD (within a couple of years or less), check out [this article](http://dx.doi.org/10.1371%2Fjournal.pcbi.0020121) from PLoS Computational Biology. It offers ten suggestions to consider when searching for a post doc. The suggestions are not comp-bio specific, but I’d be interested in hearing if people have other ideas. I’ve definitely been thinking about post docs, but I have no concrete plans as of yet.

#Update on Carl Zimmer’s Ford Pinto
Date: November 30, 2006

[Last week](http://scienceblogs.com/evolgen/2006/11/the_discovery_institute_blows.php) I linked to Carl Zimmer’s take down of [Casey Luskin](http://scienceblogs.com/loom/2006/11/13/getting_the_mooney_treatment.php). Notice how the text containing the link to Carl’s post says “Casey Luskin”. It’s called a Google bomb — we’re trying to make Carl’s post the number one search result when one enters [Casey Luskin](http://scienceblogs.com/loom/2006/11/13/getting_the_mooney_treatment.php) as a query in Google. It only works if a bunch of people link to Carl’s post like this: [Casey Luskin](http://scienceblogs.com/loom/2006/11/13/getting_the_mooney_treatment.php).


As of November 29, 2006, Carl’s post is the 22nd result when you [Google Casey Luskin](http://www.google.com/search?q=casey+luskin). That puts him on the third page. In fact, Carl’s post ranks behind a post from [Afarensis](http://scienceblogs.com/afarensis/2006/11/20/casey_luskin_crash_test_dummy/) and one from [Migrations](http://migration.wordpress.com/2006/11/20/zimmer_on_luskin/) that both link to Carl’s post (and both of those posts employ the Google bomb strategy!).


This is, obviously, not satisfactory. We must increase Carl’s post to the number one search result when one Google’s the Disco’s Lawyer/Anti-scientist’s name. So, if you’ve got a blog or webpage, give Carl a link and help take down [Casey Luskin](http://scienceblogs.com/loom/2006/11/13/getting_the_mooney_treatment.php).

#Doc Bushwell Joins the Anti-omics Brigade
Date: November 30, 2006

Dr. Joan Bushwell (not her real name) has [hurled some poo](http://scienceblogs.com/bushwells/2006/11/the_memeome_1.php) at the -omicists. You know, [those people](http://scienceblogs.com/evolgen/2006/07/more_uses_for_the_ome_suffix_1.php) who [attach -omics](http://scienceblogs.com/evolgen/2006/10/now_available_the_omeome.php) to [everything](http://scienceblogs.com/evolgen/2006/11/the_secretome.php) and act like they’ve come up with a brand new research discipline. I imagine them making the “guitar player changing chords” face (go to 1:09 in [this video](http://www.youtube.com/watch?v=0ponOKWMazk)) when they coin a new “ome”.


As one last hurrah for omeomics, Doc Bushwell coined the [meme-ome](http://scienceblogs.com/bushwells/2006/11/the_memeome_1.php). That would be a collection of all the memes floating around the blogosphere. Or is it all memes everywhere? Who knows? But it is the perfect congruence of two terms that have worn out there welcome.


And systems biologists, don’t think you’re immune from the criticism. We all know you’re just computer scientists who think they’ve invented physiology.


P.S. I posted this in the [Philosophy of Science](http://scienceblogs.com/channel/philosophy-of-science/) category because, well, I can.

#Studying Cancer from an Evolutionary Framework
Date: December 25, 2006

Carl Zimmer has an [article](http://www.sciamdigital.com/index.cfm?fa=Products.ViewIssuePreview&amp;ARTICLEID_CHAR=634610CD-2B35-221B-6741D56AB263FFE2) in the upcoming edition of *Scientific American* that looks at cancer from the perspective of evolutionary biology. The most obvious parallel is that of cancer cells within an individual modeled as an evolving population:


> 
Rare mutations, for instance, may cause a cell to lose restraint and begin to multiply uncontrollably. Other mutations can add to the problem: They may allow deranged cells to invade surrounding tissues and spread through the body. Or they may allow tumor cells to evade the immune system or attract blood vessels that can supply fresh oxygen.
Cancer, in other words, re-creates within our own bodies the evolutionary process that enables animals to adapt to their environment. At the level of organisms, natural selection operates when genetic mutations cause some organisms to have more reproductive success than others; the mutations get “selected” in the sense that they persist and become more common in future generations. In cancer, cells play the role of organisms. Cancer-causing changes to DNA cause some cells to reproduce more effectively than ordinary ones. And even within a single tumor, more adapted cells may outcompete less successful ones.



But multicellular organisms have also evolved mechanisms to suppress such runaway populations of cells. A group of researchers, however, noticed that there is a tradeoff between regulating cell division and aging. Their study of the tumor suppressor gene p16 in mice revealed that:


> 
Natural selection favors anticancer proteins such as p16, but only in moderation. If these proteins become too aggressive, they can create their own threats to health by making bodies age too quickly.



The effects of natural selection on tumor suppression genes is greatest prior to and during prime reproductive age — once you can no longer reproduce, your ability to transmit your alleles is greatly limited (grandmother hypotheses aside). Therefore, our body is optimized to fight early onset cancers, but is not so great at dealing with those that strike at old age.

#A Clash of Cultures on the Gridiron
Date: December 25, 2006

<img alt="i-77f8065bd8b32788e2167439db966685-tamu_haight.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-77f8065bd8b32788e2167439db966685-tamu_haight.gif"/>


[The University of California, Berkeley](http://www.berkeley.edu/) and [Texas A&amp;M University](http://www.tamu.edu/) will square off in this year’s [Holiday Bowl](http://www.holidaybowl.com/game.htm) football game on Thursday, December 28. While not quite as complex as [last year’s national championship game pairing](http://evolgen.blogspot.com/2006/01/red-state-blue-state-rose-bowl.html), there is a nice dichotomy when one considers the stereotypes of Cal and Texas A&amp;M. In general, Cal is home to a bunch of liberal, tree hugging, pinko hippies, whereas the Texas A&amp;M student body is made up of gun wielding, warmongering good ol’ boys (and [Nick](http://scienceblogs.com/scientificactivist/)). The tale of the tape is laid out in [this article](http://www.signonsandiego.com/sports/college_football/20061224-9999-1s24holiday.html) from the San Diego Union-Tribune — San Diego is home to the Holiday Bowl. A few highlights can be found below the fold.





> 
**School bosses:**
Texas A&amp;M: The school president, Robert Gates, was recently chosen by President Bush to supervise the war in Iraq as the new Secretary of Defense.
Cal: Chancellor Robert Birgeneau, a native of Toronto, lists on his curriculum vitae an editorial he once wrote for the Toronto Star entitled “Celebrating Sexual Diversity.”



> 
**Campus faces:**
Texas A&amp;M: Football games are attended by former President George H.W. Bush, whose presidential library is located on campus.
Cal: A student became known as “The Naked Guy” in the 1990s by attending classes nude.



> 
**Girls:**
Texas A&amp;M: The school was all-male for several decades. It didn’t open its doors to women students until 1963, on a limited basis. Full admission for women began in 1971, with the Corps of Cadets first allowing women to join in 1974.
Cal: The first female student enrolled in 1870. One its most famous former students is Betty Friedan, co-founder of the National Organization of Women, and author of the 1963 book “The Feminist Mystique.”



(Via [The Wizard of Odds](http://thewizardofodds.blogspot.com/2006/12/culture-clash.html).)

#James Brown — Overpowered by Funk
Date: December 25, 2006

<img alt="i-874dffafdb5ad6498089e7cf373d443a-james_brown.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-874dffafdb5ad6498089e7cf373d443a-james_brown.jpg"/>


James Brown, the Godfather of Soul, has [passed away](http://www.nytimes.com/2006/12/25/arts/music/25cnd-brown.html) at the age of 73 in Atlanta, Georgia. The world is a slightly less funky place today. I’ve posted some videos — in honor of the hardest working man in show business — below the fold.





The early years:


<object height="350" width="425"><param name="movie" value="http://www.youtube.com/v/bs1HUbMCZKc"/><param name="wmode" value="transparent"/><embed height="350" src="http://www.youtube.com/v/bs1HUbMCZKc" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>


Eddie Murphy as James Brown in “Celebrity dHot Tub Party”:


<object height="350" width="425"><param name="movie" value="http://www.youtube.com/v/Ly_CGin6QS8"/><param name="wmode" value="transparent"/><embed height="350" src="http://www.youtube.com/v/Ly_CGin6QS8" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>


James Brown — along with Clive Owen, Gary Oldman, and Marilyn Manson — in BMW Films’ “[Beat the Devil](http://www.imdb.com/title/tt0338768/)“. Brown plays a Robert Johnson character looking to renege on his deal with the devil:


<object height="350" width="425"><param name="movie" value="http://www.youtube.com/v/jNYSEUOnbPs"/><param name="wmode" value="transparent"/><embed height="350" src="http://www.youtube.com/v/jNYSEUOnbPs" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>

#Chance, Stochasticity, Probability and Evolution
Date: December 26, 2006

[John Wilkins](http://scienceblogs.com/evolvingthoughts/2006/12/epicurus_moran_and_chance_1.php) has replied to [Larry Moran](http://sandwalk.blogspot.com/2006/12/10-truths-about-atheism.html) on the role of “chance” in evolution (incidentally, [Moran replies](http://sandwalk.blogspot.com/2006/12/can-you-be-theist-and-believe-in.html) to Wilkins on the same topic, but a [different post by Wilkins](http://scienceblogs.com/evolvingthoughts/2006/12/god_evolution_and_variation.php)). Here’s what Larry wrote:


> 
Nobody denies the power of natural selection and nobody claims that natural selection is random or accidental. However, the idea that everything is due to natural selection is the peculiar belief of a relatively small number of people, of whom Richard Dawkins is the most outspoken.
A great deal of evolution is the result of chance or accident, as is a great deal of the rest of the universe. It’s perfectly okay to say, as a first approximation, that lots of evolution is random or accidental. This is a far closer approximation to the truth than saying it’s the all the result of design by natural selection.



This is a favorite topic of Moran’s (see [his webpage](http://bioinfo.med.utoronto.ca/Evolution_by_Accident/Evolution_by_Accident.html) for more). Wilkins takes a closer look at Moran’s point, focusing especially on some philosophical issues, but the argument is lacking due to a sloppy treatment of the statistics and a failure to adequately define “chance” from a statistical perspective.


First, a quibble on probability distributions. Wilkins writes:





> 
There are a number of meanings to “chance” in this case. One of them is that without a perturbing cause, ensembles of events will tend to form a Poisson distribution – the “bell curve” of beginner’s statistics.



I like where he was going with a definition of “chance” — random draws from a probability distribution — but the “‘bell curve’ of beginner’s statistics” is the [normal distribution](http://en.wikipedia.org/wiki/Normal_distribution). Given enough trials, a [Poisson distribution](http://en.wikipedia.org/wiki/Poisson_distribution) will converge on a normal distribution (the [central limit theorem](http://en.wikipedia.org/wiki/Central_limit_theorem)). But a Poisson random variable specifically refers to how many events will occur in a certain window of time. For example, one could model the number of cars driving past your house every hour using a Poisson distribution. The inverse of this is the [exponential distribution](http://en.wikipedia.org/wiki/Exponential_distribution), which deals with the issue of waiting time — the expected length of time between the events (how long do we expect to wait between cars driving past your house). But I digress.


Throughout his entire argument, Wilkins never gives a true definition of “chance” (neither does Moran, for that matter, but his blog entry was much shorter). And, while Wilkins does hint that he means random draws from a probability distribution, he also suggests something else:


> 
Yes, molecular changes can be “random” in the sense that they are external to the theory of evolution. They are not external to the theory of chemistry. In the domain of subatomic physics, the randomness of radioactive decay or gamma radiation has to do more with statistical properties than actual chance.



“Statistical properties” mean, to me, random draws from a probability distribution. So, what is chance if chance is not a stochastic process? Oftentimes, people conflate randomness with a [uniform distribution](http://en.wikipedia.org/wiki/Uniform_distribution_%28mathematics%29) — equal probabilities of all possible outcomes. But when we model a random process, we assume some distribution that approximates the randomness of the natural event we’d like to simulate. In evolutionary biology, this is often done with the [binomial distribution](http://en.wikipedia.org/wiki/Binomial_distribution) — either allele A<sub>1</sub> or allele A<sub>2</sub> get passed on to a child, either a locus obtains a mutation or it does not, either two alleles coalesce at generation t-1 or they do not.


The examples mentioned above are the chance aspects of evolution. Evolution, in a nutshell, results from differential inheritance of alleles. The expected frequency of a neutral allele in the next generation is simply its frequency in the previous generation. But, in finite populations, there exists some variance around that mean. Processes that can be modeled as random draws from probability distributions — mutation and drift, for example — can lead to differential inheritance of alleles. Moran is arguing that these types of processes trump natural selection in terms of importance throughout evolution history. Many of the arguments Wilkins makes are irrelevant to that point, instead focusing on the theistic implications of “chance” (which I won’t touch with a ten foot cross).


But the big question remains: is Moran correct? Is evolution due mostly to the stochastic processes described above, or does natural selection (non-random draws from a probability distribution) drive most changes? And that’s what a lot of people are studying. Of course, it depends on which aspects of evolution one is most interested in. If one is studying evolution at the DNA sequence level, then, yes, evolution is mostly due to the mutations that accumulate over a given amount of time (which can be modeled as a Poisson distribution). If, however, one is interested in the evolution of the amino acid sequences encoded by protein coding genes, then we begin to see many more examples of both selective constraint (stasis due to natural selection) and adaptive evolution (natural selection driving change). Finally, the morphology of organisms probably contains the greatest evidence for natural selection. A [cetacean](http://en.wikipedia.org/wiki/Cetacean)‘s streamlined morphology, a bird’s wing, and every type of eye out there were all shaped by natural selection.


Moran’s peeve — and one that I agree with — is that people tend to interpret everything in an adaptationist light. This is especially problematic when it comes to molecular evolution. Much of what people explain using natural selection may merely be a [spandrel](http://en.wikipedia.org/wiki/Spandrel_%28biology%29).

#Scientific American Also Mixes Terminology
Date: December 28, 2006

The editors at *Scientific American* are [afraid of PLoS ONE](http://scienceblogs.com/evolgen/2006/12/science_reporters_lament_the_a.php), but they’re more than happy to [publish articles](http://www.sciam.com/article.cfm?chanID=sa004&amp;articleID=C61B0F6E-E7F2-99DF-3BB99A2F0F7F8802) about [*Nature* papers](http://scienceblogs.com/evolgen/2006/12/on_the_genetics_of_virgin_birt.php). Their [coverage](http://www.sciam.com/article.cfm?chanID=sa004&amp;articleID=C61B0F6E-E7F2-99DF-3BB99A2F0F7F8802) of the Komodo dragon virgin births contains the following lead in:





> 
The “immaculate conception” of Komodo dragons at two English zoos might provide one explanation why Jesus was not a clone of Mary



And now a freakin’ Jew will explain why this is wrong. You see, the [Immaculate Conception](http://en.wikipedia.org/wiki/Immaculate_Conception) refers to the conception of Mary, free from original sin. This allowed her to be inseminated by God and give birth to the baby Jesus — God wouldn’t boink any ol’ Bethlehem floozy. The article’s metaphor is mixed and bereft of meaning.


Allow me also to take umbrage with this:


> 
Embryos of some reptiles–notably crocodiles and turtles–don’t have any sex chromosomes; rather, the incubation temperature dictates their gender.



They mean that the incubation temperate of the eggs dictates the animals’ sex, but they were afraid to type the word sex. Maybe they couldn’t find the “X” key on their keyboard (it’s located between “Z” and “C”, beneath “S” and “D”), or maybe they giggle when the word is used. Either way, gender is not quite appropriate here. Sex is the right word. Gender is merely a social construct that dictates how certain sexes should behave. Sex refers to penises and vaginas and their equivalents in other taxa. You wouldn’t refer to male and female plants and genders, would you?

#Barbaro NOT Cured by Jesus
Date: December 28, 2006

<img alt="i-640b5d46f7876fbe5c60cc42a68ac860-pray_for_barbaro.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-640b5d46f7876fbe5c60cc42a68ac860-pray_for_barbaro.jpg"/>


[Barbaro](http://en.wikipedia.org/wiki/Barbaro), the 2006 Kentucky Derby winner who broke his leg during the Preakness Stakes, is [almost healthy enough](http://www.msnbc.msn.com/id/16331036) to be released from his ICU stall at University of Pennsylvania’s New Bolton Center. He has been the focus of hopes and prayers of many horse racing fans — and little kids who are fans of horsies in general. The chief surgeon at the center has this to say about Barbaro’s recovery:


> 
“It’s not a miracle. It’s anything but that,” he said, sitting next to a Christmas tree topped with a stuffed Barbaro.
“Some of the Barbaro fans aren’t going to like that, perhaps. I’m a scientist, I’m a doctor. I’m not a faith healer or a religious person. I believe in the application in science and I think nothing that’s happened to him is particularly miraculous.”



(Via [Deadspin](http://www.deadspin.com/sports/barbaro/barbaro-fans-finally-go-too-far-piss-off-dr-richardson-224719.php).)

#It Happens Again
Date: December 28, 2006

Why, it seems like [just last week](http://scienceblogs.com/evolgen/2006/12/seven_simple_letters.php) that we were kvetching about the **ad nominum** attacks upon our character — or, at least, the spelling of our blog’s name. The people who add an extra “V” (turning evolgen into evolvgen) are lightweights compared to [those who confuse us with Evolution Blog](http://www.scienceblogs.com/evolgen/2006/05/once_twice_three_times_a_name.php). As of [today](http://scienceblogs.com/seed/2006/12/post_2.php), we can add Page <strike>6.0221415 × 10<sup>23</sup></strike> 3.14 to the list:


> 
<p>**From Evolgen, [A Boom in Bible Publishing](http://scienceblogs.com/evolutionblog/2006/12/boom_in_bible_publishing.php)**.<br/>
Jason Rosenhouse comments on an article in The New Yorker that casts Bible printing as a booming business.</p>



I am not, nor have I ever claimed to be, named [Jason Rosenhouse](http://scienceblogs.com/evolutionblog). I am an evolutionary geneticist; Jason is a mathematician. Jason writes about the cultural impact of evolution and religion as well as chess. I write about population genetics, molecular evolution, and genomics. I suck at chess and lack any semblance of cultural decency.


In honor of the latest confusion over our name, we have a new banner with a new slogan. The banner can be seen by visiting [the blog](http://scienceblogs.com/evolgen).

#The Rift in Evolutionary Biology
Date: December 29, 2006

The rift in the biological sciences may lie [between computational biologists and wet labs](http://scienceblogs.com/evolgen/2006/08/the_rift_in_the_biological_sciences.php), but when we look at individual fields, we see other divisions. In an [essay in PLoS Computational Biology](http://dx.doi.org/10.1371/journal.pcbi.0020156) Carl Zimmer describes the divide amongst evolutionary biologists. On one side are researchers who like to get their hands dirty — ecologists, paleontologists, and others that fall under the label ‘naturalist’. And on the other side we have the people that prefer to work with molecular tools; Zimmer calls these guys computational biologists, but they also generate their own data, so that label isn’t quite correct. The rift between evolutionary biologists essentially boils down to people studying morphological/phenotypic evolution and those studying molecular evolution.


Zimmer points out that this division is quite old — dating back to the early days of genetics. Only recently have we developed the tools allowing us to bridge that divide. The future of evolutionary research lies not in the study of the evolution of phenotypes or genomes alone, but in research programs that bridge the gap. For example, what are the phenotypic effects of [all that](http://scienceblogs.com/evolgen/2006/11/nature_goes_bucknutty_for_copy_1.php) [copy number polymorphism](http://scienceblogs.com/evolgen/2006/12/even_more_copy_number_polymorp.php)? The top research labs should view Zimmer’s rift as a great opportunity for interdisciplinary research.


See also: [Carl Zimmer’s blog entry](http://scienceblogs.com/loom/2006/12/29/up_and_down_lifes_staircase.php).

#Save the Flies!!
Date: December 30, 2006

Endangered Ugly Things has [a post](http://endangered-ugly.blogspot.com/2006/12/hawaii-fly-o.html) on the eleven *Drosophila* species listed as endangered and the one listed as threatened in 2006. These flies are known as picture winged for the patterns found on their wings. (The obligatory pictures can be found below the fold.) I’m especially happy that these flies are listed as “pomace flies” on the [NatureServe](http://www.natureserve.org/explorer/servlet/NatureServe) website (search for “Drosophila”), and I bet [Mel Green](http://www.genetics.org/cgi/content/full/162/1/1) is as well. But what are these beautiful creatures doing on a blog called [Endangered Ugly Things](http://endangered-ugly.blogspot.com/)?


I found [this website](http://www.msu.edu/user/taylo110/drosophila.htm) with more information on Hawaiian *Drosophila*.





<img alt="i-7fcb0507538c36cac3a405b58a41e4c0-DHeteronuera.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7fcb0507538c36cac3a405b58a41e4c0-DHeteronuera.JPG"/><br/>
*Drosophila heteronuera*, one of species listed as endangered in 2006.


<img alt="i-42ac6f091108d1eec88ab3f0c043d102-Drosophila_heteroneura2.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-42ac6f091108d1eec88ab3f0c043d102-Drosophila_heteroneura2.jpg"/><br/> Two *D. heteronuera* showing off their wings to each other.


<img alt="i-058b309d96880985da25f3e5d9a9247c-Drosophila_mulli.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-058b309d96880985da25f3e5d9a9247c-Drosophila_mulli.jpg"/><br/>[*Drosophila mulli*](http://hbs.bishopmuseum.org/good-bad/mulli.html) which was listed as threatened in 2006.

#Lab Coats & Gloves
Date: December 31, 2006

Biologists often wear rubber gloves when doing their research to protect them from the nasty chemicals they’re working with and to protect their samples from contamination. I’ve been known to [bitch](http://scienceblogs.com/evolgen/2006/11/keep_your_gloved_hand_away_fro.php) about people not removing those gloves prior to entering common areas — touching things they shouldn’t be touching with gloves. This could lead to mutagens, carcinogens, and simply abrasive reagents on doorknobs and elevator buttons that other people touch with their bare hands. Or it could lead to environmental particles on the gloves which could contaminate and ruin some experiment. The chances of such events are fairly low, but failure to remove gloves indicates that you are inconsiderate to the safety of other people and could care less about the fidelity of your results.


It appears the equivalents in the medical setting are lab coats and scrubs. From [this Science Creative Quarterly article](http://www.scq.ubc.ca/?p=659) by Timon Buys:


> 
But this is just so incredibly thoughtless that it just screams to be railed against. God only knows what kind of work these people do. Can you imagine if someone from a level 3 lab was pulling this stunt? I would hope that people working with that level would have more sense – and it’s likely that they do – but we can’t know for sure when we see that white coat out in public, can we? I mentioned that grease stains would make me avoid a befouled seat like the plague. In the case of the public lab coaters who take up residence in my coffee shop, I might be avoiding those seats so I won’t catch the Plague.



Just as with the gloves, the chances someone will fall ill because they sat in the same seat as a lab-coat-wearer are low (although the possibility is there). It just makes you look like an inconsiderate (or ignorant, or stupid, or fill-in-the-blank) moron.

#Where do Babies Come from and Other Stupid Questions
Date: January 17, 2007

*Esquire* runs a regular article called “Answer Fella” in which stupid questions get stupid answers. In [this month’s edition](http://www.esquire.com/features/articles/2006/070106_mfe_February_07_answerfella.html) we find out whether cloned humans have souls, why South Dakota’s badlands are called badlands, and how many potatoes make up a bag of chips. My favorite answer, however, is to the question of what happens to the bodies of large animals when they die at the zoo:


> 
According to Brandie Smith, director of conservation and science for the Association of Zoos and Aquariums, a necropsy–an animal autopsy–is performed first, to determine the cause of death and for research. “We can check all kinds of things, like cholesterol levels or measure the size of the heart,” she tells AF. “We use it as an opportunity to learn more about them as a species.” After the necropsy, Smith says, the goal is to use the remains to advance science and education. When an elephant dies, for example, the ivory may go to a classroom, the skull to a museum, and the penis to Tijuana<sup>1</sup>.
**1- The penis-to-Tijuana thing is a joke. They actually ship it to Keith Olbermann for his collection.**



(Via [blog.bioethics.net](http://blog.bioethics.net/2007/01/will-human-clones-have-souls.html))

#On the Use of Elevators in Research Buildings
Date: January 18, 2007

Allow me to set the stage. I just emerged from the autoclave room with a cart full of hot, steamy, dirty vials and bottles of *Drosophila* media in tow (see image below the fold). The glassware had been the home for thousands of flies for a period of over a month. What started out as a mixture of agar, cornmeal, yeast, molasses was churned up and excreted into by tons of larvae. All this nastiness was then heated at high pressure, releasing all kinds of aromas that I have the pleasure of dragging around our building. I’m a real popular guy.


I had to push the cart from the autoclave room to the elevator and from the elevator to our lab. Anyone unfortunate enough to encounter me in the hallways or elevator (or use the autoclave room within a day of me) gets to enjoy my unique scent. They should feel honored. As I walked down the hallway, I noticed a fellow grad student waiting for the elevator to arrive. I politely allowed her to ride alone so as not be trapped in a confined space with my cart o’ stank.


We were on the sixth floor. I needed to ride the elevator because my lab is on the floor below and my cart wouldn’t fare too well on the stairs. The other grad student had nothing with her — no cart, no bag, no nothing — and was presumably planning to ride to the ground floor. She was going to ride the elevator **down**. Not up, which would be somewhat understandable. ****Down!**** And she’s a marathon runner. Seriously. Like competitive and shit. She didn’t **need** to ride the elevator; she **chose** too.


Which brings me to the question: in what situations should you ride the elevator in a research building?





<img alt="i-36a6ce0a6bd9d9aa44cc0adfd4c23ffe-fly_cart.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-36a6ce0a6bd9d9aa44cc0adfd4c23ffe-fly_cart.gif"/><br/>**You do not want to ride in an elevator with this bad boy.**


My personal opinion is that an elevator should only be used if it is needed. If you can physically walk up the stairs without undue discomfort, don’t ride the elevator. If you’ve got a cart that looks like the one shown above, you get to ride the elevator. If you’re on crutches, in a wheelchair, or are otherwise physically handicapped, you get to ride the elevator. If you’re carrying bins full of field samples to your lab on the fourth floor, you get to ride the elevator.


Our building doesn’t have one of those new fangled speedy elevators you see in the glistening buildings built within the past decade. This sucker’s slow. Not only does it not go between floors with any sense of urgency, it also takes a while to arrive when it’s called. It’s pretty much always faster to walk the stairs than to ride the elevator. That means people who ride the elevator when they could take the stairs are just lazy.


And screwing it up for all of that need to use the elevator to lug carts up and down between floors. Because the lazy people ride the elevator so much, it takes longer than it should to arrive when called. The extra traffic wastes everyone’s time. So I’ve been tempted to post a sign in each of our two elevators — right next to the “IN CASE OF FIRE USE STAIRS” sign — that reads:


**IN CASE OF SELF-MOTIVATION USE STAIRS**


I’m not entirely happy with the phrasing, but it’s a start. Incidentally, I once saw an IN CASE OF FIRE placard with the following edit:


> 
IN CASE OF FIRE<br/>DO NOT USE ELEVATOR<br/>USE <strike>STAIRS</strike> WATER



But my fellow grad students (marathon runners or not) are not the worst elevator hogs. Sure, there are a bunch of people who ride the elevator from the ground floor to the sixth floor, which is somewhat understandable — although that hike is not too physically challenging. The worst elevator hogs are the undergrads who ride the elevator from the ground floor to the first floor.

#Mendel’s Garden & Other Announcements
Date: January 20, 2007

Three things:


<li>
A new edition of [Mendel’s Garden](http://scienceblogs.com/neurotopia/2007/01/mendels_garden.php) has been posted at Neurotopia. Go read the latest genetics blogging.
</li>
<li>
The anecdote at the beginning of my [rant about elevator usage](http://scienceblogs.com/evolgen/2007/01/on_the_use_of_elevators_in_res.php) needs a slight correction: I think the grad student who took the elevator down has a bum knee (it’s a new injury). I’m not too disappointed that she took the elevator, as walking down stairs sucks when your knee’s screwed up. That’s what I get for passing judgment without knowing all the facts. But that doesn’t make up for all the perfectly healthy people who ride the elevator despite the fact that they don’t need to.
</li>
<li>
The motherboard on my laptop is fried, so I’m using a backup computer until it gets fixed. Blogging may be near non-existent during that time, as I no longer have a working computer at home. As if anyone really cares.
</li>

#The Motherboard Has Landed
Date: January 22, 2007

I got my [motherboard](http://scienceblogs.com/evolgen/2007/01/mendels_garden_other_announcem.php) replaced a couple of hours ago, so I now have my old forms of procrastination at my disposal — blogging and blog reading.


I’ve got one link for you that is relevant to [Gregg Easterbrook’s anti-multi-author screed](http://scienceblogs.com/evolgen/2007/01/post.php). This one comes from [BioCurious](http://biocurious.com/fifth-author-twice-removed); it’s an [article](http://dx.doi.org/10.1371/journal.pbio.0050018) on attributing credit in multi-author papers. Enjoy.

#Species Concepts are Dynamic . . . and Crap
Date: January 23, 2007

**Because I haven’t riled up [Wilkins](http://scienceblogs.com/evolvingthoughts/) in a while.**


I was chatting with a friend who has published a fairly high profile article on speciation about species concepts. We came to the conclusion that species concepts are useless unless they guide future research. Okay, we were just echoing [Coyne ‘n Orr](http://www.sinauer.com/detail.php?id=0914).


As crappy as the biological species concept is, the BSC is really good for guiding research on the evolution of pre-zygotic, intrinsic reproductive isolation. I realize this is entirely circular (the BSC states that speciation has occurred when reproductive isolation is achieved). All of the research I follow on speciation falls into this category; I don’t read about speciation in non-sexually reproducing organisms.


How do people study speciation in asexually reproducing organisms? Better yet, is it even worthwhile to study speciation in asexually reproducing organisms? Just because speciation in sexually reproducing organisms (and, by speciation, I mean intrinsic prezygotic isolation) is really cool — and it is, not just because I have friends doing it — doesn’t mean that people need to study “speciation” in other taxa.


So, species concepts are useless unless you can define speciation in those taxa. If it’s just divergence, it don’t count. Wilkins, the ball’s in your court. Tell me why I’m wrong — include lots of links to articles I should read before I engage in uninformed punditry.

#Wilkins Replies
Date: January 24, 2007

Wilkins has [replied](http://scienceblogs.com/evolvingthoughts/2007/01/species.php) to [my post](http://scienceblogs.com/evolgen/2007/01/species_concepts_are_dynamic.php) on species concepts.


The gist: there are a bunch of species concepts, many of which are pretty darn good.


My reply: that’s awesome as long as they guide future research. The BSC provides a framework for studying reproductive isolation. Ecological species concepts are useful when studying adaptation to niches. But I still see nothing interesting that can be garnered from purely phylogenetic- or divergence-based species concepts.

#Another Creationist Group Comes to my University
Date: January 25, 2007

We’ve got [Phil Skell](http://evolgen.blogspot.com/2005/04/appeal-to-authority.html), and we can’t get rid of him. Both Michael Behe and [William Harris](http://evolgen.blogspot.com/2005/09/guess-whos-coming-to-dinner.html) have rolled through my parts in the past few years. Tonight we get disciples of [Adnan Oktar](http://en.wikipedia.org/wiki/Adnan_Oktar) (Harun Yahya), the muslim creationist described in [this article](http://www.seedmagazine.com/news/2006/11/not_in_kansas_anymore.php) on creationism in Turkey. He’s also an alleged [holocaust denier](http://www.talkorigins.org/faqs/organizations/#bav).


When I got my “Invitation to attend Evolution lecture tonight” (reproduced below the fold) from the president of the Muslim Student Association, I didn’t know what to think. I’m not sure if I’ll be attending the talk (I’ve got real work to do), and Oktar won’t even be there; he’s sending out two of his “students” to give the presentation. (I put students in quotes because students require teachers, and teachers are supposed to spread knowledge encourage independent thinking, not banish understanding from their realm of impact.)


And even if I decide to attend, I probably won’t blog on it. This shit just saps all my desire to live.





Here is the email sent to a bunch of graduate students inviting us to the talk:


> 
Dear Fellow Student,
I am writing to cordially invite you to our lecture tomorrow on “The Collapse of Darwinism and the Fact of Creation.” Our speakers are scientists and students of Turkish writer Harun Yahya, who has sold millions of books around the world. The scientific approach they have taken to deconstructing Darwinism has earned them great renown throughout the Muslim world. 
I understand that much of the mainstream scientific community is reluctant to consider such theories anything better than pseudo-science. If this is true, then this makes the attendance of graduate students in an acclaimed biology program such as [**university name redacted**]’s such as yourself even more important. While we respect the viewpoints of our guests, we are in search of the truth and discussion is always beneficial. 
Thus, I hope you accept our sincere invitation to attend the lecture tomorrow night. I am certain the entire university community could benefit from your informed perspective in the post-lecture Q/A session.
Best regards,
<p>[**name redacted**]<br/>
President, Muslim Student Association<br/>
[**name redacted**] University</p>



Here is the advertisement for the talk:


> 
<p>______________________________<br/>
The Collapse of Darwinism and the Fact of Creation:<br/>
A Harun Yahya Conference</p>
<p>A thought-provoking presentation by renowned scientists<br/>
Dr. Oktar Babuna, M.D. and Dr. Ali Sadun, Ph.D.<br/>
Members, Science Research Foundation, Istanbul, Turkey</p>
Followed by an extended interactive discussion and Q/A session
<p>7:00 pm, Thursday, January 25th<br/>
[**address redacted**]</p>
<p>Sponsored by the Muslim Student Association and Dialog Forum<br/>
(of the [**name redacted**] University)</p>
Harun Yahya is one of the most prolific writers of the Islamic world; his books have been translated into 57 languages and millions of copies have been sold. He is most renowned for his scientific arguments for the creation of Man and the Universe. We invite the [**university name redacted**] community to attend this event and to gain valuable insight from this unique perspective. Students and faculty specializing in the life sciences are especially encouraged to contribute to the after-event discussion.


#Week of Science Challenge — One Week Away
Date: January 29, 2007

The [Week of Science Challenge](http://scienceblogs.com/evolgen/2007/01/the_week_of_science_challenge.php) (official website [here](http://www.justscience.net/)) begins next Monday (5 February 2007). During the time of the challenge, all participating bloggers will post at least once per day on science and only science — no anti science or non-science material.


If you would like to participate, you can sign up [here](http://www.justscience.net/signup.php) by providing your blog name, email address, blog URL, and blog RSS feed. There will be an official Week of Science feed made up of all the participating blogs.

#Of Genes and Species
Date: January 30, 2007

I straddle the line between being a population biologist and a molecular geneticist. That’s a self-congratulatory way of saying that I am an expert in neither field. But existing in the state I do allows me to observe commonalities shared by both. For example, both fields have terminology (or what the uninitiated would call jargon) that lack sufficient definitions.


Amongst my minimal postings from last week were included a couple of riffs on species and speciation ([the first](http://scienceblogs.com/evolgen/2007/01/species_concepts_are_dynamic.php), [the second](http://scienceblogs.com/evolgen/2007/01/wilkins_replies.php)) getting at the lack of a coherent definition of species. My conclusion is that there is not, nor will there ever be, a definition of species that performs satisfactorily in every taxon. John Wilkins has studied species concepts much more than I have, and he offered his opinion [here](http://scienceblogs.com/evolvingthoughts/2007/01/species.php). But that’s all old news.


What’s also old news is the debate over what constitutes a gene. And the debate rages on in the science blog-o-sphere — [PZ Myers](http://scienceblogs.com/pharyngula/2007/01/basics_what_is_a_gene.php), [Sandy Porter](http://scienceblogs.com/digitalbio/2007/01/what_is_a_gene_my_definition_i.php), [Larry](http://sandwalk.blogspot.com/2007/01/richard-dawkins-definition-of-gene-is.html) [Moran](http://sandwalk.blogspot.com/2007/01/what-is-gene.html), and [Greg](http://gregladen.com/wordpress/?p=162) [Laden](http://gregladen.com/wordpress/?p=217) have all offered their two cents (Moran’s subject to an exchange rate, of course). While there is considerable overlap between the definitions, each person offer’s his or her unique approach toward defining a gene. Just like species concepts, there is not a single definition of gene that everyone finds satisfactory.





Whether we’re arguing species concepts, debating genes, or complaining about any other ephemeral terminology, there is one thing in common: the appropriate definition is not what someone else says it should be, but what fits your research best. A developmental biologist will be interested in how genes function (by exploring the phenotypes of individuals with mutated versions or examining when and where genes are expressed), whereas an evolutionary geneticist will be interested in how sequences (which may or may not be called genes) change over time. Sometimes people will define genes as protein coding sequences, sometimes they’ll include introns, other times they’ll include *cis* regulatory regions. You’ll also see nontranslated RNAs lumped into the class “gene” if that’s appropriate for whatever someone’s studying.


The take home message is that this terminology (all terminology?) is not fixed; it depends on context and the person delivering the information. Just as “gene” may refer to a sequence that gets transcribed, a sequence that gets translated, or a sequence with function, “species” may refer to unique gene pools, reproductively isolated populations, or ecologically differentiated populations. You can squabble about what a gene is, what a species is, or even what constitutes “life”, but these terms are dynamic, and their definitions are often implied by the context in which they are placed.


Rather than using these poorly defined terms, why not use a more detailed term? If you’re talking about protein coding sequences, don’t call them genes when you could refer to them as protein coding sequences. The same goes for transcribed sequences or any other sequence that falls into the catchall “gene”. That was the point I was trying to get at with my [posts](http://scienceblogs.com/evolgen/2007/01/species_concepts_are_dynamic.php) on [species](http://scienceblogs.com/evolgen/2007/01/wilkins_replies.php) — if you’re studying reproductive isolation, call it that rather than speciation.

#Circus of the Spineless #17
Date: January 31, 2007

Jeremy has posted the newest edition of the [Circus of the Spineless](http://thevoltagegate.blogspot.com/2007/01/circus-of-spineless-17-symbology-of.html) — the blog carnival dedicated to invertebrates — at the Voltage Gate. We’ll excuse the carnival organizers for creating a carnival organized around a paraphyletic taxon this one time because Jeremy included one of our posts on wasps. He’s also got stuff on cephalopods, shrimp, snails, spiders, and a bunch of other spineless critters.

#Speciation, Natural Selection, and Karyotypes
Date: February 5, 2007

I’ve been chatting up [Wilkins](http://scienceblogs.com/evolvingthoughts/) about the role of natural selection in speciation (and when I say “speciation” I mean “reproductive isolation”). Wilkins listed a few cases where speciation would occur independently of natural selection. Amongst the mechanisms in Wilkins’s list was speciation via karyotypic changes (polyploidy, inversions, fusions or fissions). I cried shenanigans, and this is why.


The karyotype refers to the organization of an organism’s genome — chromosome number, fusions/fissions of chromosomes, and gene order within chromosomes. One way to change the karyotype is to duplicate the entire genome (tetraploidization), which often results in reproductive incompatibilities between the tetraploid individuals and the diploid (normal) individuals. If the tetraploid individuals do find a way to reproduce (either by selfing, finding another tetraploid individual, or mating successfully with a diploid individual), there’s a good chance that the new karyotype (the tetraploids) will be reproductively isolated from the diploids. In this case, the karyotypic mutation itself has led to reproductive isolation without natural selection acting on any other variation.


But what about other karyotypic changes? Do they result in “instantaneous” reproductive isolation?





As I have [mentioned before](http://scienceblogs.com/evolgen/2007/01/chromosome_number_and_chromoso.php), chromosomal fusions and fissions do not, in and of themselves, prevent matings between individuals with different karyotypes. Individuals with fused chromosomes should be able to successfully mate with individuals with unfused chromosomes producing heterokaryotypic progeny. Unlike the case with polyploidy, fusions and fissions do not induce instantaneous reproductive isolation.


But what I’m more interested in are chromosomal inversions. The meiotic effects of these karyotypic rearrangements are better studied than fusions and fissions. We also know about a lot of examples of inversions segregating within natural populations. It was once hypothesized that inversions could induce instantaneous speciation much like polyploidy can. But the fitness costs of carrying two different arrangements differentiated by an inversion are too low; this is inferred based on the amount of inversions segregating in natural populations.


Here’s the twist: inversions can and do play a role in speciation, and it’s in conjunction with natural selection (ie, Wilkins was wrong). Rather than acting as an intrinsic cause of reproductive isolation, inversions are the substrate upon which the causes of reproductive isolation arise and evolve. That’s a mouthful and rather unintelligible without the details, some of which I’ll provide for you below. Keep in mind that it’s not the inversions themselves that allow for reproductive isolation, but without them speciation would proceed much more slowly.


First, let’s lay out our assumptions. We will assume that two populations have diverged to some extent such that interpopulation hybrids (individuals produced by matings between individuals from different populations) are less fit than progeny produced by intrapopulation matings (we call this “post-zygotic isolation”). It follows that natural selection will favor pre-zygotic isolating barriers (ie, prior to the fusion of egg and sperm) between the populations to prevent the production of interpopulation hybrids. If the populations are not entirely isolated (they come in contact and can mate with each other) these barriers must be behavioral, and, if they are to be acted on by natural selection, must be heritable (ie, genetic).


Given that there is some gene flow between the populations, the genetic factors for prezygotic reproductive isolation within one population could be transmitted to the other population during the interpopulation matings that occur. Individuals carrying the alleles that prevent them from mating with the other population are more fit (because their progeny are more fit), but if those alleles move between populations due to interpopulation hybridization, the alleles will not be as effective at preventing interpopulation hybridization. How can the movement (or migration) of the alleles from within one population into the other be prevented? This is where inversions fit in.


Inversions suppress recombination in heterokaryotypic individuals. An individual with two different versions of a particular chromosome (they are differentiated by at least one inversion) tends to produce gametes with very few crossing over events along that chromosome (all the alleles within the inversion are tied together). Now, imagine our two populations not only have some postzygotic isolating barriers between them, but are also karyotypically differentiated by at least one inversion on at least one chromosome. The inversion does not prevent interpopulation hybrids, but what if the genes for reproductive isolation are located with the inversion? Rather than being transmitted as single, independent alleles between populations, they are locked within the inversion and must be transmitted as a unified group.


A similar scenario has been modeled by [Navarro and Barton](http://dx.doi.org/10.1554/0014-3820(2003)057[0447:APIGIP]2.0.CO;2), and they found that inversions significantly accelerate the accumulation of postzygotic isolating barriers between populations. It’s not the inversions themselves that lead to reproductive isolation; it’s the genes contained with the inversions. Gene flow between the populations is drastically suppressed within the inverted regions because the inversions contain genes that decrease fitness in interpopulation hybrids, and they are transmitted as a block because of the inversion. And natural selection can act on the genes within the inverted regions because they either confer some intrinsic fitness benefit or they favor reproductive isolation between the populations.


Not only does the model work out well on paper, the model was [tested](http://dx.doi.org/10.1554/04-174) by Mohamed Noor’s lab using the model organisms from the *Drosophila pseudoobscura* subgroup. Noor’s group mapped hybrid male sterility factors (genes that decrease fertility in interspecific hybrid males) between two species pairs. The first pair, *D. pseudoobscura* and *D. persimilis*, have overlapping ranges in the northwestern United States, while the second pair, *D.p. bogotana* and *D. persimilis*, do not (*D.p. bogotana* is located in northwestern South America, in and around Bogota, Columbia).


[<img alt="i-b53588284a49c414659ddeb17bf556ae-Dpse_speciation(sm).GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b53588284a49c414659ddeb17bf556ae-Dpse_speciation(sm).GIF"/>](http://dx.doi.org/10.1554/04-174)


Navarro and Barton’s model predicts that gene flow between populations will lead to genes for reproductive isolation mapping to within inverted regions. It makes no such predictions when there is no gene flow between populations. Consistent with the model, the hybrid male sterility factors between *D. pseudoobscura* and *D. persimilis* (the species pair with overlapping ranges) map to regions of the genome which contain inversions that differentiate the two species. In the non-overlapping species pair (*D.p. bogotana* and *D. persimilis*), the genes show no preference for inverted regions.


Additionally, there is some evidence that inversions may play a role in the reproductive isolation of a classic model organism in speciation research: *Rhagoletis pomonella*. These fruit flies appear to be speciating due to the recent introduction of cultivated apples into North America. Prior to the introduction of apples, the flies fed, mated, and laid their eggs primarily on hawthorns (another fruit), but they can now be found on both plants. The two plants fruit at different points in the season, and this has led to two separate host races that each prefer one fruit over the other. Jeff Feder and colleagues mapped loci for host plant preference and [found evidence](http://www.genetics.org/cgi/content/abstract/163/3/939) that they mapped to within inverted regions (identifying inversions in *Rhagotelis* is much more difficult than in *Drosophila*). They have also [hypothesized](http://www.pnas.org/cgi/content/abstract/100/18/10314) that the inversions (and possibly the genes that predisposed the species for a speciation event) were introduced into the population via individuals migrating in from Mexico.


In these examples, the inversions themselves are not under natural selection and do not induce speciation. But the genes contained within are responsible for traits involved in pre- and post-zygotic reproductive isolation between populations. It’s these genes that are under selection and responsible for the evolution of reproductive isolation (speciation). Therefore, to say that speciation via karyotypic mutations proceeds without selection is misleading. Yes, polyploid speciation occurs via the intrinsic reproductive isolation of tetraploids from diploids, but inversions play a much more intricate role in the speciation process, in concert with genes responsible for reproductive isolation and selection acting upon those genes.


---



Brown KM, Burk LM, Henagan LM, Noor MAF. 2004. A test of the chromosomal rearrangement model of speciation in *Drosophila pseudoobscura*. *Evolution* **50:**1856-1860. doi:[10.1554/04-174](http://dx.doi.org/10.1554/04-174)


Feder JL, Berlocher SH, Roethele JB, Dambroski H, Smith JJ, Perry WL, Gavrilovic V, Filchak KE, Rull J, Aluja M. 2003. Allopatric genetic origins for sympatric host-plant shifts and race formation in *Rhagoletis*. *Proc. Natl. Acad. Sci. USA* **100:**10314-10319. [LINK](http://www.pnas.org/cgi/content/abstract/100/18/10314)


Feder JL, Roethele JB, Filchak K, Niedbalski J, Romero-Severson J. 2003. Evidence for Inversion Polymorphism Related to Sympatric Host Race Formation in the Apple Maggot Fly, *Rhagoletis pomonella*. *Genetics* **163:**939-953. [LINK](http://www.genetics.org/cgi/content/abstract/163/3/939)


Navarro A and Barton NH. 2003. Accumulating postzygotic isolation genes in parapatry: a new twist on chromosomal speciation. *Evolution* **57:**447-459. [doi:0014-3820(2003)057[0447:APIGIP]2.0.CO;2](http://dx.doi.org/10.1554/0014-3820(2003)057[0447:APIGIP]2.0.CO;2)

#Mutation
Date: February 6, 2007

**Where the variation comes from.**


Evolution proceeds by the action of many different evolutionary forces on heritable variation. Natural selection leads to the increase in frequency of variation that allows individuals to produce more offspring who, themselves, produce offspring. Genetic drift changes the frequency of variation through random sampling of individuals from one generation to the next. Population subdivision divides the variation into isolated groups where other forces (selection, drift, etc) act upon it. But where does all this variation come from?


Given the title of the post, the subtitle of the post, and your general understanding of biology, it should be pretty obvious that the variation comes from mutation. The purpose of this essay is to explain the different types of mutations that can contribute to heritable variation in populations. We will also explore how evolutionary forces act upon the different mutations.





Genomes are divided into chromosomes. Each chromosome contains a unique set of DNA sequences. Some organisms contain two copies of each chromosome (one from their mother and one from their father), and others only have one copy of each chromosome (inherited from a single parent). Each chromosome is made up of sequences that perform specific functions (some encode proteins, some determine when the protein coding sequences are expressed, and others encode other function elements) and non-functional sequences (junk DNA). We will not concern ourselves much with the specific categories the DNA sequences can fall into.


For the purpose of this treatment of mutations, we will divide the types of mutations into four classes:


<li>
**Substitutions:** changing the information in the genome.
</li>
<li>
**Rearrangements:** rearranging the information in the genome.
</li>
<li>
**Insertions:** increasing the amount of information in the genome.
</li>
<li>
**Deletions:** decreasing the amount of information in the genome.
</li>


**Substitutions**<br/>Substitutions, also known as point mutations, result in the change of a nucleotide into another nucleotide. DNA sequences are made up of a series of four different nucleotides: adenine, thymine, guanine, and cytosine (symbolized by the letters A, T, G, and C, respectively). A genome sequence consists of an arrangement of these nucleotides in a specified order (think about it as a book written in a language consisting of four letters). If one of the nucleotides is changed, a substitution or point mutation has occurred.


The effects of a substitution depend on within which type of sequence the mutation occurs. If it occurs in a DNA sequence that lacks any function (so-called junk DNA), it will be neutral. Point mutations can also occur in sequences that encode proteins or sequences that regulate the expression of protein coding sequences. A substitution within a protein coding sequence may alter the protein that the sequence encodes. If so, the protein may be rendered nonfunctional, in which case the individual harboring that mutation will be less fit than other individuals. There is also the possibility that the mutation renders the individual more fit because the new protein sequence is better than the old one. Similar scenarios could be imagined for point mutations in other functional DNA sequences.


**Rearrangements**<br/> Genomic rearrangements include events such as fusion and fissions of chromosomes, inversions, and translocations (see figure below). The scale of these events can range from a region as small as a gene (a small part of a chromosome) to large portions of chromosomes to entire chromosomes. Fusion events, such as the one that occurred in the human genome after the divergence with chimpanzees, join together two **complete** chromosomes, whereas translocations occur when **part** of a chromosome is moved to another part of the same chromosome or to a different chromosome.


<img alt="i-dba84ef01ec00adf9b7fd1d1a434e052-chr_muts.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-dba84ef01ec00adf9b7fd1d1a434e052-chr_muts.JPG"/>


Inversions rearrange the genetic content within a single chromosome. The can also play an important role in [speciation](http://scienceblogs.com/evolgen/2007/02/speciation_natural_selection_a.php) or contain [alleles](http://evolgen.blogspot.com/2005/02/human-inversion-under-selection.html) that confer fitness benefits. One reason is that inversions suppress recombination between different arrangements (chromosomes carrying different inversions do not easily exchange alleles). But it’s important to understand that rearrangements occur at a much lower frequency than point mutations. Comparing fitness effects of rearrangements and substitutions is a bit trickier, and we won’t deal with that here.


**Insertions**<br/>The next two classes of mutation result in the addition or loss of genetic material in the genome. Insertions increase the net content of a genome. There are multiple sources of the genetic material that enters the genome. It may arise *de novo* (synthesis of novel sequence), it may come from outside the genome, or it may be a duplicated copy of something from within the genome of interest. Insertions of novel sequence tend to be small (on the order of a few nucleotides), whereas insertions of material from outside or within the genome can be as large as a single gene or even multiple genes. We will focus on these types of insertions.


The ability of material from outside of a genome to insert into the genome in question (and our understanding of such events) depends on what types of sequence we are studying. In eukaryotes (plants, animals, fungi, and various microbes), viral sequences often move from individual to individual, inserting themselves into genomes. This horizontal transmission of genetic information is even more common in bacteria due to biological properties of these organisms. The uptake of genetic material from extragenomic sources have the potential to cause disease (ie, viruses) or may lead to novel genes which allow an organism to perform a new function (ie, bacteria picking up genes for antibiotic resistance from other individuals or from the environment).


Genomic information can also be duplicated within a genome. This can occur via various mechanisms, sometimes even aided by viral like sequences moving within a single genome. Entire blocks of genetic material can be duplicated via mechanisms that we’re still working to understand. Another common mechanism occurs when DNA sequences are transcribed to RNA, then reverse transcribed back into DNA and inserted back into the genome. Duplications allow genes or other DNA sequences to explore mutational space that would be inaccessible if they only existed in a single copy. That’s because many point mutations are deleterious, but if there is a copy of a sequence that maintains the original function, a duplicate copy can accumulate mutations that interfere with the original function. Many of these mutations will lead to a non-functional duplicate copy, but some may lead to a sequence with a new function that would not be possible with a single copy because the single copy must maintain the original function.


**Deletions**<br/>In addition to accumulate new content, a genome can also lose content. These events can occur on various scales, ranging from a few nucleotides to large chunks of chromosomes. Larger duplications tend to be more deleterious than smaller ones, but the quality of the content of the deletions affect of the fitness costs of the deletions as well. For example, the deletion of a small region containing an essential gene will be far more deleterious than a large deletion of non-functional sequence. Not all deletions will be deleterious, and it’s possible that they may even confer a fitness benefit if they delete a sequence that is deleterious. We can gain a fair bit of understanding of the evolutionary dynamics by exploring the frequencies of deleted sequences in natural populations. Three recent studies (reviewed [here](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-1.html), [here](http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-2.html), and [here](http://scienceblogs.com/evolgen/2006/01/are_deletions_deleterious_part.php)) have performed such an analysis and found many common deletions within human populations.


---



The different types of mutations vary in the frequency at which they occur (point mutations are more common than rearrangements, insertion, and deletions), but there is also variation within the classes. For example, different sizes and types of rearrangements, insertions, and deletions occur at different frequencies. Additionally, certain substitutions are more frequent than others (see [here](http://evolgen.blogspot.com/2005/06/random-mutation-and-natural-selection.html) for more details). And the fitness costs of these mutations depend on multiple factors, including the size of the events, in which types of sequence they occur or which sequences they contain, and what other mutations are associated with the mutations.

#Shotgun Sequencing a Eukaryotic Genome
Date: February 7, 2007

Shotgun sequencing refers to the process whereby a genome is sequenced and assembled with no prior information regarding the genomic location of any of the DNA we sequence. There are quite a few steps that you have to go through before you have an assembled genome sequence. We’re going to cover isolating DNA, putting the DNA in bacteria, sequencing the DNA, and assembling those sequencing into a complete genome.


Sandy has been running a series on sequencing genomes (parts [1](http://scienceblogs.com/digitalbio/2007/01/basics_how_do_you_sequence_a_g_1.php), [2](http://scienceblogs.com/digitalbio/2007/01/basics_how_do_you_sequence_a_g.php), [3](http://scienceblogs.com/digitalbio/2007/01/basics_how_do_you_sequence_a_g_2.php), [4](http://scienceblogs.com/digitalbio/2007/01/basics_how_do_you_sequence_a_g_3.php), [5](http://scienceblogs.com/digitalbio/2007/01/sequencing_a_genome_part_v_che.php), [6](http://scienceblogs.com/digitalbio/2007/02/sequencing_a_genome_part_vi_ch.php), [7](http://scienceblogs.com/digitalbio/2007/02/sequencing_a_genome_part_vi_ah.php)). You should go check it out even if you read this post; while I’m going to deal with some of the basics of shotgun sequencing, she goes over some things that I will not. This post will cover how genome sequencing projects go from organisms to assembled genomes, but there are certain details that I will be leaving out that Sandy has explained quite well.





**Isolating and Cloning DNA**<br/>Before we can begin sequencing DNA, we must isolate it from dead organisms (or parts of those organisms). There are multiple ways to isolate DNA, but they all involve breaking down the tissues and cells to isolate the nuclei (the membrane bound intracellular structure containing the genomic DNA), then breaking down the nuclei to remove the DNA inside. Once we have the DNA, we can directly sequence it **only if we have prior information** regarding the some of the sequence of the region we would like to sequence. If we want to sequence an entire genome, we will not have enough sequence information to directly sequence the genomic DNA.


Once we have isolated the DNA, we break it into fragments of different sizes (for reasons discussed below). Those fragments are then mixed with bacteria, and some of the bacteria take up the DNA which gets incorporated into extragenomic DNA sequences called **plasmids**. Because we know the sequences of those plasmids, we can easily sequence the fragments that are inserted into the plasmids (the fragment shown as a red block in the figure below). Each of those plasmids is known as a **clone**.


<img alt="i-235250bc58e21f7af2db1e983004c636-cloning.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-235250bc58e21f7af2db1e983004c636-cloning.gif"/>


**Sequencing**<br/>In order to sequence the DNA the old fashioned way (there are some new fangled techniques we won’t deal with here), we use primers to initiate the sequencing reaction. Those primers are designed to match the known sequence of the plasmids flanking the region containing our DNA of interest that was inserted into the plasmid (shown as green arrows in the figure above). DNA sequencing can only handle a few hundred **nucleotides** (DNA letters), and the genomic fragments are on the order of thousands of nucleotides. That means we don’t get the entire sequence of the fragment, but we do generate sequences of the ends of the fragments (squiggly lines in the figure). Furthermore, we can keep a record of from which clones the end sequences come, so we know that each pair of end reads should be located in the same genomic region.


**Assembling Shotgun Reads**<br/>This aspect of shotgun sequencing will receive the brunt of my focus. Hopefully I’ve set this up properly by describing end sequencing of reads because that is secret to shotgun sequencing. That’s a hint to go back and read the previous paragraph and look at the previous figure if you skimmed it over. The sequencing strategy is important. Real important.


Once the DNA sequencing is completed, the sequences are assembled like a puzzle. Ideally, the fragments overlap each other so that the sequences that partially overlap each other will be joined together to form larger sequences. We also would like to have small, medium, and large fragments covering each region (see below). This process continues until all the overlapping sequences are assembled into a bunch of really long sequences known as **contigs**.


<img alt="i-d2cdf8bba659e42b1aa656f364868574-Genomic_region.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d2cdf8bba659e42b1aa656f364868574-Genomic_region.gif"/>


But the contigs only cover portions of each chromosome, and the goal is to have a single sequence that covers an entire chromosome. For various reasons (including repetitive DNA and lack of sequence for all genomic regions) there are genomic sequences that fail to assemble into the contigs. The next best thing we can do is try to fit the contigs together into a single sequence known as a **scaffold**.


<img alt="i-6f91a625bac0c878ebd94ab1e3f61ed0-contig_scaffold.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-6f91a625bac0c878ebd94ab1e3f61ed0-contig_scaffold.gif"/>


In the figure shown above, three contigs are combined into a single scaffold. The arrows indicate paired end reads of clones — the red arrows are from one clone and the blue arrows are from a different clone. If multiple paired end reads are located at the ends of two contigs, we can join the contigs into a single scaffold. The red region of the scaffold is the sequence that came from the contigs, and the black region is the sequence inferred to be located between the contigs. Only we don’t know what those black sequences are, so we fill them with unknown nucleotides and refer to that region as a **gap**. While it may seem problematic to introduce gaps into a genome assembly, we make up for the cost of the gaps because they allow us to assemble contigs into scaffolds.


The scaffolds are then assigned to chromosomes using a few different strategies. If we know something about the molecular genetics of the organism we’re studying, we can identify genes that have been previously mapped to chromosomes within our scaffolds. If a scaffold contains a gene (or, even better, multiple genes) that is known to be located on a particular chromosome, the scaffold most likely came from that chromosome. And if we know the order of the genes on the chromosome, we can designate and orientation to the scaffold and order multiple scaffolds on a chromosome.


If we lack known genetic markers or the marker set is poor, we can take the some of the clones that we sequenced and map them to chromosomes using *in situ* hybridization of clones to the actual chromosomes. This involves attaching a fluorescent tag to the cloned sequence and mixing the tagged clone with cells from the organism. The nuclei of the cells can be observed under a microscope and the chromosomes visualized. The clone will anneal to the chromosome from which in came, allowing you to map the clone to a particular chromosome. The contig and scaffold containing the sequences from that clone can then be assigned to that chromosome.


---



Once all of the sequences have been assembled into contigs, the contigs assembled into scaffolds, and the scaffolds assigned to chromosomes, we have a “draft” assembly of a genome — not until we minimize the gaps to an acceptable standard can we call the assembly “finished”. Most sequenced genomes do not make it beyond the draft stage, as the finishing process is expensive; a draft sequence is usually good enough for most genomes. The genome sequence then gets **annotated**, a process that involves finding genes and other sequences in the scaffolds. This is done using gene prediction algorithms, comparisons with other annotated genomes and known genes, and other computational techniques.

#Science & Technology
Date: February 8, 2007

I have previously mentioned, in passing, a pet peeve of mine: when people conflate ecology with environmentalism (see [here](http://www.scienceblogs.com/evolgen/2006/12/nonoverlapping_magisteria_and.php) and [here](http://www.scienceblogs.com/evolgen/2006/12/are_there_any_ecologists_in_th.php) for examples). It’s an odd pet peeve for an [admitted non-ecologist](http://scienceblogs.com/evolgen/2006/01/welcome_to_evolgen_at_scienceb.php), but it falls under the umbrella of distinguishing science from technology which is at the heart of the real pet peeve. It just happens that the ecology/environmentalism issue pops up more often than other science/technology issues in my daily life (I don’t deal with stem researchers or people cloning whole organism).


Before I get too far ahead of myself, allow me to define what I mean by science and what I mean by technology. These words (like all words) are just placeholders for two processes: the pursuit of knowledge and understanding of the natural world and the application of that knowledge for human uses. Science is the pursuit of that knowledge and technology is its application. Using ecology and environmentalism as an example, [ecology](http://thevoltagegate.blogspot.com/2007/02/basic-concepts-what-is-ecology.html) is the study of organisms in their environment, whereas environmentalism is the movement to conserve those organisms and their environments. Ideally, environmentalists use findings from ecological research to justify their conservation efforts (ie, they apply the knowledge gained from the science of ecology).





Given what I just wrote, it should come as no surprise that I bristled when someone in my department justified his efforts to encourage people in the department to recycle by referring to himself as a “hardcore ecologist”. “You mean you’re a hardcore **environmentalist**,” I replied. His comeback was that environmentalism is a subset of ecology and mine was, obviously, that ecology is science and environmentalism is application. I think the whole exchange came off as a big dork-fest to those watching, and I appeared to be quite an asshole for trying to show him up (or [science prick](http://scq.ubc.ca/sciencescouts/index.htm)).


So, why should we care about the distinction between science and technology? Some interesting points came up in a back and forth between [Janet Stemwedel](http://scienceblogs.com/ethicsandscience/) and [Larry Moran](http://sandwalk.blogspot.com/index.html). It all [started when Larry asked](http://sandwalk.blogspot.com/2007/01/ethical-issues-in-science.html) what role technology and ethics should play in teaching genetics and biochemistry to undergraduates. Janet [offered her take](http://scienceblogs.com/ethicsandscience/2007/01/teaching_about_ethics_and_othe.php) as a philosopher of science specializing in ethics, and [Larry replied](http://sandwalk.blogspot.com/2007/01/teaching-ethics-in-science-science-v.html) that the ethical issues all arise when we discuss the application of science, not the science itself. The final two posts (one from [Janet](http://scienceblogs.com/ethicsandscience/2007/01/is_what_is_this_good_for_a_que.php) and one from [Larry](http://sandwalk.blogspot.com/2007/01/teaching-ethics-in-science-science-v_25.html)) focus on the role that the application of knowledge should play in teaching science.


In regards to the pedagogical aspects, Janet argues that the applications of the knowledge unify the individual pieces of information:


> 
A laundry list of isolated facts is not the kind of thing the students want to learn, nor the kind of thing the science teachers want to teach. What keeps the facts from being isolated — what imposes a coherent structure where they’re connected to each other — almost always involves a storyline about what various bits of the knowledge are good for.



What Janet means by “good for” is the application of that knowledge (from what I can gather). While this may be true in organic chemistry (the example Janet provides), this is hardly the case in biology. A unified presentation of molecular genetics (from DNA to proteins) can be done without ever discussing the applications of the knowledge. I empathize with Larry in his goal to get students excited in the science, rather than have them think “what good is this for?”


In teaching scientific content, it is often useful to point out some applications of the concepts. But the course should not be unified around the applications or the pieces of information. Instead, it would be interesting to see a course organized around the scientific method itself (ie, how the science got done and the knowledge accumulated). This allows the educator to both introduce why researchers were interested in understanding something (eg, the structure of DNA or the genetic code) as well as how they went about discovering the bits of knowledge. Along with the issue of why the scientists wanted to understand a particular phenomenon comes how that discovery fits into a larger system (eg, how solving the genetic code relates to transcription and translation).


This is a shift in emphasis from important pieces of information to important concepts, where science as a process is emphasized over science as a collection of facts. The process then gets tied in with a growing body of knowledge that gets developed in order to get a complete picture of a natural process that cannot be understood by individual facts. This provides the motivation for the individual experiments and provides context for the pieces of knowledge.


And, before you rail on me for creating a false dichotomy between science and technology, I understand that the division can be quite ambiguous in places. But I like to think of the motivations: are they for increased understanding or the application of knowledge that does not increase understanding on its own (note: if the technology can be applied for increasing understanding, it still doesn’t count as science)? And then there’s the issue of technology as science, which I’ll leave for people to hash out in the comments.

#Mendel’s Garden #11
Date: February 9, 2007

Hsien at Genetics &amp; Health has [posted the newest edition](http://www.geneticsandhealth.com/2007/02/09/we-love-genetics-we-love-genetics-not/) of my favorite (just) science blog carnival: Mendel’s Garden. There are oodles of reasons why you should love genetics (and only a couple why you should not).


If you would like to host a future edition of Mendel’s Garden, visit [the carnival’s blog](http://mendels-garden.blogspot.com/) and leave a comment or email [Paul](http://theforcethat.blogspot.com/).

#NESCent
Date: February 12, 2007

Does anyone know anything about [NESCent](http://www.nescent.org/) (the National Evolutionary Synthesis Center in North Carolina)? They’re affiliated with Duke, UNC, and NC State (the Research Triangle universities), and they offer [postdoctoral fellowships](http://www.nescent.org/science/postdoctoral.php). One of my committee members suggested it as a possible location for a postdoc, but I’m concerned that any potential project would be way heavy on computational work. I’m just trying to get a feeling for the center from anyone familiar with the research going on there.

#A Question about Retroposition
Date: February 13, 2007

Duplicated genes can arise via various mechanisms — [polyploidization](http://en.wikipedia.org/wiki/Polyploidy), [chromosomal duplication](http://en.wikipedia.org/wiki/Aneuploidy), [segmental duplication](http://mbe.oxfordjournals.org/cgi/content/abstract/22/1/135), and retroposition — and we can usually distinguish the various mechanisms as each has distinct signatures. For example, retroposed duplicates arise when an RNA transcript is reverse transcribed back into DNA and re-inserted into the genome. This is how many transposable elements (TEs) and viruses propagate throughout genomes, but the reverse transcriptase encoded by TE and viral genomes can be used on endogenous transcripts as well. Because they arise via the [reverse transcription](http://www.web-books.com/MoBio/Free/Ch4J1.htm) of [mRNA](http://en.wikipedia.org/wiki/Mrna), retroposed duplicates lack [introns](http://en.wikipedia.org/wiki/Intron) and often have a [poly A tail](http://en.wikipedia.org/wiki/Poly_A_tail).


When [searching for retroposed genes](http://www.genome.org/cgi/content/abstract/12/12/1854) in sequenced genomes, researchers will look for pairs of duplicates in which one copy is missing introns. They may also try to identify a poly A tail, but that signature will decay more rapidly than the missing introns (i.e., a run of A’s will only be useful for identifying recent retroposition events, whereas missing introns are useful for identifying both new and old events). Because they give us more power to characterize duplications, structural signatures (presence or absence of types of sequences, which part of the gene is duplicated, etc) are more reliable than sequence signatures.


Oftentimes, however, genes that have no introns are duplicated via retroposition. Or only an exonic portion of a gene is duplicated and we would like to determine whether it arose via retroposition or some other mechanism. With that in mind, I was wondering if anyone has any information about the reliability of a second structural signature (missing introns being the first): which part of a gene (5′, 3′, or middle) gets duplicated.


Reverse transcription proceeds from the 3′ to the 5′ end of the transcript (creating a [cDNA](http://en.wikipedia.org/wiki/CDNA) product in the 5′ to 3′ direction). If the reverse transcriptase terminates prior to reaching the 5′ end of the transcript (where the start codon is located), only the 3′ end of the gene (where the stop codon is located) will be duplicated. We see evidence for this when looking at human retroposed duplications (the 3′ ends are overrepresented in the genome relative to the 5′ ends). Can we use this logic to decide whether a duplication missing other signatures (no intron signature and no poly A tail) is retroposed or not? If a duplicated gene only contains a portion of the terminal exon or the 3′ end of a single exon gene, is this sufficient evidence that the gene arose via a  retroposition event (assuming that a poly A signal is not present)?

#Snow Dayz!!
Date: February 14, 2007

In honor of all the snow being dropped on the Northeast US, I give you “Things that rock &amp; things that suck.” Cue the theme music…


<li>
**Things that rock:** Snowfall measured in feet.
**Things that suck:** Freezing rain.
</li>
<li>
**Things that rock:** Showing up to class/work on cross-country skis.
**Things that suck:** Cancelled classes.
</li>
<li>
**Things that rock:** Watching ill-prepared undergrads try to navigate an unplowed path or snow drift.
**Things that suck:** Navigating an unplowed path or snow drift.
</li>
<li>
**Things that rock:** Riding your bike in the snow.
**Things that suck:** Snow removal services that plow snow onto bike racks.
</li>


Now, I don’t want to get off on a rant here, but they aren’t really “removing” the snow, so much as “moving” it to another location. There are certain places that moved snow tends to accumulate — and it usually piles up right where you want to be. Am I right here? There’s the pile of snow blocking your driveway, the pile that blocks your side street, and the pile that keeps you from getting into your parked car, let alone trying to move it from the curb to the road. There’s gotta be a way to plow two orthogonal streets without creating the freakin’ Himalayas blocking one of them. I mean, I have a better chance of climbing K2 sans sherpa than trying to get from my little side avenue onto Main Street. Of course, that’s just my opinion. I could be wrong.

#Junk DNA, Revisited
Date: February 15, 2007

Some bio-bloggers are atwitter over an [article](http://www.sciam.com/askexpert_question.cfm?articleID=B6C523E9-E7F2-99DF-3590E74C7E7C14B8&amp;ref=rss) by Wojciech Makalowski on Scientific American’s website about Junk DNA. I’m a little late to the game because, well, I’ve been really busy looking at sequences to determine if they are junk DNA. Is it irony? Is it coincidence? Who cares? It’s an opportunity to discuss semantics, and I love semantics.





Those of you who have hung around here for a while know this topic often comes up at evolgen (remember [this](http://scienceblogs.com/evolgen/2006/03/junk_dna_is_junk.php), [this](http://www.scienceblogs.com/evolgen/2006/03/junk_dna_redux.php), and [this](http://scienceblogs.com/evolgen/2006/04/junkity_junk_mcjunkerson_1.php) . . . hell, [here’s what a search for Junk DNA](http://www.google.com/search?q=junk+DNA&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search) turns up). Long story short, I can’t stand the term junk DNA, but I do agree with [Dan Graur](http://scienceblogs.com/evolgen/2006/03/junk_dna_is_junk.php#comment-43636) that Junk DNA is a valid null hypothesis. And that’s it. The majority of any eukaryotic genome is made up of non-functional sequences that are slightly deleterious, but they persists because selection against these sequences is too weak to purge them (i.e., [nothing in evolution makes sense except in the light of population genetics](http://scienceblogs.com/evolgen/2006/09/_in_the_light_of_well.php)).


The blogorific bruh-ha-ha started with [Larry Moran’s response](http://sandwalk.blogspot.com/2007/02/junk-dna-scientific-american-gets-it.html), and then [Alex Palazzo jumped in](http://scienceblogs.com/transcript/2007/02/junk_dna_origin_of_the_term.php). The biggest flaw with Makalowski article, and one that Larry points out, is that Wojciech attempts to answer the question “What is junk DNA, and what is it worth?”, but instead spends most of the time describing repetitive DNA. He never actually gets around to answering the question. Makalowski goes on to point out a bunch of examples of repetitive DNA being coopted (exapted?) by genomes to become transcriptional enhancers. At least he avoids saying that transposable elements don’t become parts of protein coding genes (see [here](http://dx.doi.org/10.1016/j.tig.2006.03.006) for why that’s not surprising).


Even though **some** repetitive sequences become functional elements, **most** of those repeats are just filler. Junk. Not useful, but not bad enough to be worth eliminating. I’m sure there’s an appropriate analogy, but I’m not clever enough to coin it. But even though the junk can perform fancy molecular tricks, like [induce rearrangements](http://www.genetics.org/cgi/content/abstract/107/4/657), it is still junk. You can think of those pieces of junk as mutational hotspots. The junk is isolated to regions of the genome where it can do as little harm as possible (the hypothetical analogy I would have introduced were I more clever would have been extended here). That is the evidence for selection on these sequences — to purge them from near functional regions.


I’m usually annoyed by uses of the term “junk DNA” in the popular literature because they treat the discovery of functional non-protein coding sequences as some big surprise. This time, I’m bothered by a treatment that suggests a function for all junk DNA. I guess I can’t win.

#So Many Badges, So Little Time
Date: February 15, 2007

[<img alt="i-a83255badf800c778491d4063397dbbe-00OOTSSOERAAAP.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-a83255badf800c778491d4063397dbbe-00OOTSSOERAAAP.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html)


Those kooky Canucks at the Science Creative Quarterly have started a new club: the [Order of the Science Scouts of Exemplary Repute and Above Average Physique](http://scq.ubc.ca/sciencescouts/). Anyone is free to join, provided they’re not a teetotaling, lying, world-dominating, badge-hater, ’cause they’re really into badges. To profess their love of badges, the Science Scouts have produced [quite a collection](http://scq.ubc.ca/sciencescouts/#badges), some of which I feel qualified to wear on my Science Scout Sash. You see, these badges aren’t handed out by some higher authority; you simply claim all the badges that apply to you and post them on your website, like so:


<br/>
[<img alt="i-239834a1c75fd5320c616a18f3536786-01talk.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-239834a1c75fd5320c616a18f3536786-01talk.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#1)


**The “**talking science**” badge.**<br/>
Required for all members. Assumes the recipient conducts himself/herself in such a manner as to talk science whenever he/she gets the chance. Not easily fazed by looks of disinterest from friends or the act of “zoning out” by well intentioned loved ones.


**Well, if it’s “required for all members”, then I guess I have no choice.**


[<img alt="i-8b22e0ab5c3bdd21f652446a4824fd8c-06blog.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-8b22e0ab5c3bdd21f652446a4824fd8c-06blog.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#6)


**The “**I blog about science**” badge.**<br/>
In which the recipient maintains a blog where at least a quarter of the material is about science. Suffice to say, this does not include scientology.


**This is a blog. There’s a lot of science here (check the [archives](http://scienceblogs.com/evolgen/archives.php)). And there’s no scientology.**


[<img alt="i-1ab399efc43aa20d270954ac38c2edef-08openflame.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-1ab399efc43aa20d270954ac38c2edef-08openflame.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#8)


**The “**I’m pretty confident around an open flame**” badge.**<br/>
Recipients have demonstrated proficiency around open flames in laboratory settings.


**See [here](http://scienceblogs.com/evolgen/2006/09/dispatches_from_the_lab.php).**


[<img alt="i-9f988e2325a7eafa86663faa5b270d77-14sexing.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9f988e2325a7eafa86663faa5b270d77-14sexing.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#14)


**The “**sexing up science**” badge.**<br/>
In which the recipient has had experience with things such as selective breeding, crossing, mate selection, prokaryotic conjugation, fertility studies, STD related microbiology, and/or any other acceptable interpretation of the badge.


**I’ve done mutant screens, done crosses to study recombination rate and meiotic segregation, inbred animals for many consecutive generations, and bred animals for the sole purpose of grinding them up and extracting their DNA. And I came up with design for the badge. Yeah, I think I qualify for this one.**


[<img alt="i-23f3c3c5a5557fb295a9ad3593d8f5e4-16prick.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-23f3c3c5a5557fb295a9ad3593d8f5e4-16prick.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#16)


**The “**I can be a prick when it comes to science**” badge.**<br/>
In which the recipient can be so passionate about things of a scientific nature, that he/she may appear surly, rude, and/or unpleasant.


**See [here](http://scienceblogs.com/evolgen/2007/02/science_technology.php) and scan the [archives](http://scienceblogs.com/evolgen/archives.php). I’m not one to keep my opinions on matters of science to myself.**


[<img alt="i-56f630f4b782e689c6d214a76ea9dc42-31useless.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-56f630f4b782e689c6d214a76ea9dc42-31useless.jpg"/>](http://scq.ubc.ca/sciencescouts/index.html#32)


**The “**I’ve done science with no concievable practical application**” badge.**<br/>
There are probably more who are deserving of this badge than you would expect.


**I study the genome of an inconsequential fly that feeds on rotting fruit and vegetation (i.e., it’s not an agricultural pest) and the NSF helps pay for it, which, by definition, means my research has absolutely no practical application.**


<img alt="i-476dfe44836e89851ec5a96794b96bcd-36cloner.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-476dfe44836e89851ec5a96794b96bcd-36cloner.jpg"/>


**The “**cloner**” badge.**<br/>
In which the recipient has cloned something or other. Rules to a drinking game concerning this badge will be forthcoming.


**Isolating DNA, amplifying regions of interest, cloning chunks of DNA, sequencing DNA — I’ve done it all. And any badge with an associated drinking game is okay by me.**

#Scientist Refutes the Darwinian Model
Date: March 20, 2007

Darwinian evolution means different things to different people. To me, and [many other](http://dx.doi.org/10.1038/nature04240) [population geneticists](http://dx.doi.org/10.1126/science.1123539), it refers to [positive selection](http://depts.washington.edu/jtlab/positiveSelection.html). To Jeffrey Schwartz, an anthropologist at the University of Pittsburgh, [Darwinian evolution means gradual change](http://www.sciencedaily.com/releases/2007/02/070210170623.htm). By the way, [Schwartz also thinks](http://dx.doi.org/10.1038/308501a0) humans are more closely related to orangutans than they are to chimps and gorillas.


I’m having a hard time making heads or tails of [Schwartz](http://www.pitt.edu/~jhs/). He seems hypercritical of all molecular evolution — to the point where one wonders whether he even understands the field. He appears to not understand that evolution at different levels (DNA sequence, protein sequence, cellular, anatomical, etc) can occur at different modes and tempos. Many DNA mutations are neutral — they confer no phenotypic effect and no fitness benefit or cost — and these changes are remarkably useful for inferring evolutionary relationships. When looking over short time scales (millions of years), DNA sequences can be used reconstruct phylogenies because they evolve quite rapidly. But protein sequences are more useful over longer time scales (tens to hundreds of millions of years) because they evolve slower.


Furthermore, when one looks at different time scales, the gradual nature of evolutionary change manifests itself at different evolutionary levels. One would struggle to find evidence that amino acid sequences evolve gradually within a species or between closely related species. There simply aren’t enough changes over this short time scale, and when we do see a lot of changes they are examples of evolutionary bursts. But DNA sequences do evolve gradually at this time scale. Schwartz seems to not understand these important points.

#SCIENCE SPRING SHOWDOWN: Octopus Region — 2nd Round Results (Part 1)
Date: March 21, 2007

[SECOND ROUND PREVIEW](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu1.php) | [PRESS CENTER](http://scienceblogs.com/worldsfair/2007/03/science_spring_showdown_2007_t.php) | [PRINTABLE BRACKETS](http://scienceblogs.com/worldsfair/2007/03/the_2007_science_spring_showdo_1.php)


<img alt="i-c9c9bf116e1fd187c7fcd8ae416de8e5-biolbracket2.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c9c9bf116e1fd187c7fcd8ae416de8e5-biolbracket2.jpg"/>


**Welcome, ladies and gentlemen, to our comprehensive coverage of the second round of the Science Spring Showdown. We had two great games yesterday, and we have two more match ups today. The game between Phylogenetics and Unipotent is just underway, and the final game of the round, between HIV and Psychology, is coming up this afternoon.**





**Jim Pipetman:** Yesterday’s first game, between the top seeded Invertebrates and the ninth seeded Surgeons, turned out like most of the experts predicted. The Surgeons came into the game bragging about their backbone, but they couldn’t identify the Intvertebrates’ hearts. Literally. The Surgeons began by attacking the [arthropods](http://en.wikipedia.org/wiki/Arthropod) from the ventral side, only to discover the body plan was reversed from the vertebrate plan they were so used to.


<img alt="i-2c72bc415a97c962d8cd16623d0346c8-insect_anatomy.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-2c72bc415a97c962d8cd16623d0346c8-insect_anatomy.JPG"/>


**Sally Agar:** That’s right, Jim. As you can see here, the insects have a DORSAL heart. The Surgeons employed their best cardiac specialist on that initial maneuver, but they simply couldn’t reach the heart from the ventral side. If they had only sent their neurosurgeons in on the ventral attack, they may have been able to slice the main nerve chord then and there — rendering the insects utterly helpless.


**Jim Pipetman:** That pretty much summed up the Surgeons’ day. The arthropods were able to build a big lead before they passed off the scoring duties to the mollusks. And if you thought grasshoppers and spiders would give the Surgeons fits, you should have seen the looks on their faces when they encountered the clams.


**Sally Agar:** In fact, it was the [bivalves](http://en.wikipedia.org/wiki/Bivalves) in general that wreaked havoc on the Surgeons. When it came time to deploy the mollusks, you’d think the Invertebrates would have gone with the [colossal squid](http://scienceblogs.com/pharyngula/2007/02/colossal_squid.php). But they caught the Surgeons totally off guard with a [lamellibranchia](http://www.answers.com/topic/lamellibranchia-2) attack.


**Jim Pipetman:** Upon hearing about their heavy reliance on the foot, the Surgeons sent their podiatric specialists after the bivalves. But they couldn’t figure out what to do with it. After the game, Dr. Tarsus Phalanges remarked, “There were no bunions, no calluses, no nothing to operate on. We were lost.”


**Sally Agar:** In the end, the Invertebrates destroyed the Surgeons, 158-37. The grasshopper led the Invertebrates in scoring, with 62 points. The combination of his quickness and agility was too much for the Surgeons to handle. But the game wasn’t without controversy. Here’s our own [Jean Informatique](http://scienceblogs.com/evolgen/2006/07/jean_informatique_organizes_a.php) with a breakdown from Louis Pasteur Arena.


**Jean Informatique:** Thank you, Sally. Yes, there was some controversy when the Invertebrates inserted a [lancelet](http://en.wikipedia.org/wiki/Cephalochordata) into the lineup. The Surgeons’ head coach claimed that the lancelet is a fish, and fish are vertebrates. But the official judge on systematics and taxonomy pointed out that the small cephalochordate, while a member of the phylum [chordata](http://en.wikipedia.org/wiki/Chordate), is not a vertebrate because it lacks a spine. This all happened after the game was pretty much in the bag, so it probably shouldn’t affect the Invertebrates as they get ready for their game in the next round.


**Jim Pipetman:** Thanks for the report, Jean. In the second match up yesterday, Genomics and Photosynthesis squared off in what can only be described as the marquee game of the second round. This was a back and forth battle between two evenly matched teams fighting for the right to play the Invertebrates in the Sweet 16. Here’s Sally with the highlights.


**Sally Agar:** Yes, Jim, it was quite a battle. Photosynthesis loaded their starting line up with eukaryotes, hoping to protect their chemical reactions in [intracellular organelles](http://en.wikipedia.org/wiki/Chloroplast). Genomics responded by [sequencing](http://www.pnas.org/cgi/content/abstract/94/11/5967) [the](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Retrieve&amp;dopt=Citation&amp;list_uids=16247559) [entire](http://dnaresearch.oxfordjournals.org/cgi/content/abstract/10/2/59) [chloroplast](http://www.biomedcentral.com/1471-2148/6/37) [genome](http://repositories.cdlib.org/lbnl/LBNL-56161/) [of](http://www.biomedcentral.com/1471-2229/6/21) [every](http://nar.oxfordjournals.org/cgi/content/abstract/31/2/716) [plant](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2770692&amp;dopt=Abstract) the Photosynthesis squad put on the court. Everyone in Charles Darwin Arena was impressed by their ambition, but no one could figure out how exactly this approach would help them crack Photosynthesis. To the end of the first half, where the score was 20-20. With under one minute to play in the half, Photosyntesis head coach [Melvin Calvin](http://en.wikipedia.org/wiki/Calvin_cycle) called a time out and substituted all the plants out of the game, replacing them with [cyanobacteria](http://en.wikipedia.org/wiki/Cyanobacteria). This strategy helped Photosynthesis close out the half on an 8-0 run and take a 28-20 lead into the locker room.


**Jim Pipetman:** It’s not like [Craig Venter](http://collections.plos.org/plosbiology/gos-2007.php) and his Genomics squad didn’t see this one coming. They came out in the second half with a wide array of environmental sequencing projects — unleashing a combination of [Sanger and 454](http://scienceblogs.com/evolgen/2007/03/454_is_so_last_week.php) for quite a double punch. Their goal: characterize all the bacterial [photoautotrophs](http://en.wikipedia.org/wiki/Photoautotroph) at the genomic level. And the Photosynthesis squad kept sending more and more previously uncharacterized organisms at the Genomics team. The microbial diversity was simply too much for the genomics squad to handle, and it seemed like there was no way they could make up the eight point halftime deficit.


**Sally Agar:** But with under two minutes to play in the game, Genomics only found themselves trailing by five — an amount they could make up with some clever multiplexing. You could hear the cheers from the genomics section: “[Phred](http://scienceblogs.com/digitalbio/2006/07/dna_sequence_quality_part_i_qu.php)! [Phrap](http://scienceblogs.com/digitalbio/2006/08/sequencing_a_genome_the_video.php)! Autotrophs ain’t all that!” Genomics pounded away, driving at the Photosynthesis squad with all their sequencers, assembly algorithms, and powerful computers. But, alas, it was too little, too late, and Photosynthesis held on to win 59-58.


**Jim Pipetman:** What does this loss mean for the future of Genomics? For an expert opinion we go to the [Applied Biosystems](http://www.appliedbiosystems.com/) headquarters.


**ABI Spokesman:** Genomics employed a good strategy, but it ultimately backfired. The high volume attack works well for generating lots of data, but, in the end, the Genomics team was overwhelmed by too much data. In the span of a 40 minute game, you simply do not have enough time to assemble and annotate all those sequences. This is a team designed to complete over an entire season not win a one-and-done tournament. You put them in a best of seven series, and they take the whole thing.


**Jim Pipetman:** Can we match the down to the wire excitement of that game? Phylogenetics has jumped out to an early 10-4 lead over Unipotent, and HIV and Psychology is coming up soon. We’ll bring you results of both of those games as soon as they go final.

#SCIENCE SPRING SHOWDOWN: Octopus Region — 2nd Round Results (Part 2)
Date: March 21, 2007

[SECOND ROUND PREVIEW](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu1.php) | [PRESS CENTER](http://scienceblogs.com/worldsfair/2007/03/science_spring_showdown_2007_t.php) | [PRINTABLE BRACKETS](http://scienceblogs.com/worldsfair/2007/03/the_2007_science_spring_showdo_1.php)


<img alt="i-c9c9bf116e1fd187c7fcd8ae416de8e5-biolbracket2.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c9c9bf116e1fd187c7fcd8ae416de8e5-biolbracket2.jpg"/>


**Welcome back to our coverage of the second round of the Science Spring Showdown. We had [two great games](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu_2.php) yesterday, and another one finished earlier today. The final game of the round, between HIV and Psychology, is just wrapping up now; we’ll bring you the result as soon as that one goes final.**





**Tom Ribosome:** Earlier today, Phylogenetics took on Unipotent in one of the more non-traditional match ups. Coming off of games against classic rivals, Taxonomy and Totipotent, no one was sure what to expect when the tree builders and fated cells squared off. Phylogenetics struggled early, unable to construct any evolutionary relationships. Unipotent weren’t used to playing a team that lacked any sort of cell types. Despite each of their early difficulties, both teams managed to put a couple of points on the board, and the score was 24-21 in favor of Phylogenetics as we went to halftime.


**Betty Myelin:** Phylogenetics, which had become so accustomed to building trees with DNA data, had to try a different strategy. Co-coach Masatoshei Nei noticed how much the [*C. elegans* fate map](http://www.wormatlas.org/Sulstonemblin_1983/results.html) resembled a phylogeny. During halftime, the Phylogenetics team came up with a tree building algorithm to show the developmental relationships of different cellular lineages. They unveiled this strategy on their first offensive possession of the second half, and all the Unipotent team members quickly realized that, if they traced their ancestry far enough back, they were all, at some point, totipotent. With no answer to the Phylogenetic trickery, Unipotent fell into an insurmountable hole. Final score: Phylogenetics 67, Unipotent 38.


**Tom Ribosome:** Phylogenetics co-head coach, Joe Felsenstein, had this to say about a potential match with the Invertebrates in the regional finals:


> 
Yes, we realize that they’re [paraphyletic](http://scienceblogs.com/evolgen/2007/03/new_terms_in_phylogenetics.php). We’ll be bringing up the issue with Competition Committee, but we’re not looking for an easy way out of this. We’ll play them as they are currently organized. We’ll play them split into monophyletic taxa. We’ll even play the entire animal kingdom if that’s what it comes down to. The issue of monophyly is one that we’re not going to linger on, but we will take advantage of it if it presents itself.



**Tom Ribosome:** And this just in: the game between HIV and Psychology has gone final. This was a game where HIV started out very aggressive, taking advantage of its high [virulence](http://en.wikipedia.org/wiki/Virulence). The Psychology team realized, however, that as long as they did not engage in any [dangerous behaviors](http://www.cdc.gov/hiv/resources/factsheets/transmission.htm), they would be fine. In a true show of sports psychology, head coach Phil Jackson made sure his Psychology team avoided all sexual contact, blood transfusions, breast milk, and needle sharing for the entire contest. But, while this was an excellent defensive strategy, it was hardly practical at the offensive end. The score was even at 0 as the teams headed into the intermission.


**Betty Myelin:** That’s right, Tom; these teams had scored just as many points as you and I. It was quite a pitiful showing. This wasn’t even a defensive battle; it was pure futility. But this is the kind of match Psychology is used to; after being held scoreless in the first half of their game against Neuroscience, they mounted a miraculous comeback for the win. Could they do the same against the top seeded HIV team?


**Tom Ribosome:** In a word, Betty: no. We could see HIV mutating and evolving within a single host throughout the first half even though they were unable to score. In the locker room at halftime, HIV coach Reverse Transcriptase must have hit upon the perfect combination of virulence and transmission because they came out with a vengeance in the second half. I mean, they went pandemic. There were compromised immune systems around the entire arena, and I could have sworn that I saw some cases of full blown AIDS. There was absolutely nothing that the Psychologists could do. Listening to your problems can only go so far. HIV took this one going away: 48-3


**Betty Myelin:** To recap, the Invertebrates handle Surgery, 158-37. Photosyntheis squeaked by Genomics, 59-58. Phylogenetics knocked off Unipotent, 67-38. And HIV infected the Psychologists, 48-3. That sets up two interesting games in the Sweet 16: the top seeded Invertebrates take on Photosynthesis, in a battle between heterotrophs and autotrophs, and Phylogenetics and HIV meet up <strike>for the first time ever</strike> in a game with [some tradition](http://scienceblogs.com/evolgen/2006/06/phylogeny_friday_9_june_2006_1.php). That’s all for now. On behalf of the evolgen sports network, Basic Concepts, and Tom Ribosome, this is Betty Myelin reminding you to maintain homeostasis.

#Manatee Genomics
Date: March 27, 2007

We’ve [told you](http://scienceblogs.com/evolgen/2006/08/manatees_invade_the_northeast_1.php) about [the manatees](http://scienceblogs.com/evolgen/2006/10/they_live_in_the_north.php) making their move from the southeastern United States to the northeast. We warned you that the [sea cows ain’t as dumb](http://scienceblogs.com/evolgen/2006/10/they_live_in_the_north.php) as you thought they were. We took you [inside their training camps](http://scienceblogs.com/evolgen/2007/01/youre_only_going_to_make_them.php). We showed you [the future of the manatees](http://scienceblogs.com/evolgen/2007/01/i_warned_you_about_the_manatee.php).


But now, thanks to a patriotic group of researchers, we have [information on the organization of the manatee genome](http://dx.doi.org/10.1186/1471-2148-7-6) — an important first step in preventing the manatee invasion. The researchers, from the University of Florida and the National Cancer Institute, conclude that the closest living relatives of the manatees, dugongs, and Stellar’s sea cow are the elephants. Until they prove otherwise, we can assume the elephants are in cahoots with the manatees, harboring, aiding, and abetting their marine cousins.





The great Americans who published this study of manatee genomics can be further touted in all sorts of flag waving revelry because they have published their findings in an open access journal. This research — of utmost national security — is freely available to any person interested in learning about the greatest threat to our domestic security. But their patriotism is only equaled by their poor understanding of evolutionary biology:


> 
Sirenia (manatees, dugongs and Stellar’s sea cow) have no evolutionary relationship with other marine mammals, despite similarities in adaptations and body shape. 



Ouch! All mammals (and all animals, and all life) share **some** evolutionary relationship. Nothing shares “no evolutionary relationship”. It’s not an issue of being related (all life is related), but of the closeness of those relationships. Let’s hope NCI big wig and evolutionary geneticist [Stephen O’Brien](http://scienceblogs.com/evolgen/2007/03/current_biology_interviews_ste.php) had nothing to do with this butchering of evolutionary biology.

#The Sweetest Sixteen
Date: March 28, 2007

[PRESS CENTER](http://scienceblogs.com/worldsfair/2007/03/science_spring_showdown_2007_t.php) | [UPDATED BRACKET](http://scienceblogs.com/worldsfair/upload/2007/03/bracketss.pdf)


The folks that brought you the Second Round of the Octopus Region of the Science Spring Showdown ([part 1](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu_2.php), [part 2](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu_1.php)) will be bringing you one of the marquee match ups of the third round. Those folks are us, and the place is here at evolgen. We’re down to sixteen teams (some would even call this collection of teams “sweet”), which means there are [eight games](http://scienceblogs.com/worldsfair/2007/03/down_to_the_sweet_sixteen_the.php) that will be played on the internets over the course of the next week.


We’ll be playing host to a game from the Chair Region between Darwin and Corporate. That’s right, we’re delving into the Philosophy of Science. Can the seventh seeded Darwin squad continue their survival, or have they finally met their match in the eleven seed, Corporate? The entire Chair Region has been loaded with upsets; the other two teams in the Sweet Sixteen are 16 seed Kuhn (has a 16 seed ever made it this far?) and five seed Theory. (I thought the five seed is **the one that supposed to be upset** in the first round.)


There will be a preview of the big match up between the Victorian scientist and the industrial machine later this week. The result of the game will be reported next week. Keep your browser pointed at evolgen for all the news on this important showdown.

#Neutral Expectations
Date: March 29, 2007

Last week, I [linked](http://scienceblogs.com/evolgen/2007/03/on_human_genes_and_silent_muta.php) to an [article](http://www.seedmagazine.com/news/2007/03/the_sound_of_silence.php) in Seed about synonymous mutations with deleterious effects in humans. It’s heavy with errors, but I didn’t linger too much on them. [Larry Moran](http://sandwalk.blogspot.com/2007/03/silent-mutations-and-neutral-theory.html), on the other hand, got a bit more riled up than I did, and [John Logsdon](http://johnlogsdon.blogspot.com/2007/03/fundamentally-involvement-of-silent.html) (whose blog has the potential to be something cool) agrees with Larry. And I agree with both of them.


The issue here is with the neutral theory, which Larry describes quite well (see also the coverage in [this primer](http://www.routledge-ny.com/ref/philosophyscience/genetics.html) to population genetics). Many people misinterpret the neutral theory and think that it claims that all mutations in some class (non-coding, synonymous, etc) have no fitness cost or benefit. It does no such thing. The neutral theory merely says that there are many mutations with absolutely no fitness cost or benefit. The neutral theory — and every other reasonable alternative — allows for both adaptive and purifying selection in addition to neutral evolution.


There are a few graphs below the fold to illustrate this point.





When discussing different models of molecular evolution, we must understand what they say about the frequency of different types of mutations and the frequency of different types of fixations (also known as substitutions). For simplicity’s sake, we will discuss three categories of alleles: deleterious, neutral, and advantageous. Deleterious mutations will almost never fix in the population, neutral mutations will fix with a probability proportional to the inverse of the population size, and advantageous alleles will fix with a probability proportional to the fitness benefit they confer. Any and all models of molecular evolution use this paradigm, but they differ in the frequency of mutations in each category.


The fitness benefits of an advantageous mutation depends on the advantage individuals carrying the mutation have relative to individuals without the mutant allele (which we will call ‘**s**‘) and the population size (which we will call ‘**N**‘). A given advantageous mutation is more likely to fix in a large population than a small one because the product of the selection coefficient and population size (**Ns**) is greater in larger populations. The effect of population size is diminished for mutations with large fitness costs or benefits, **Ns**&lt;-1 or **NS**&gt;1.


A neutral mutation (one that confers no fitness cost or benefit) is more likely to fix in a small population because random fluctuations in allele frequency are greater in small populations. Please note, however, that the **rate of neutral fixations** is equal in all populations because large populations accumulate more mutations (because they have more individuals) than small populations — that makes up for the lower probability of fixation of any single neutral mutation in large populations.


The neutral theory (in our simplified model with only three categories of mutations) predicts that the majority of new mutations will be either deleterious or neutral. It allows for a very small class of advantageous mutations, of which many reach fixation. The deleterious mutations are purged from the population and nearly none reach fixation. Many of the neutral mutations fix.


<img alt="i-277843ec236d07929fdbd688c46ea5eb-mut_fix_neutral_model.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-277843ec236d07929fdbd688c46ea5eb-mut_fix_neutral_model.gif"/>


As you can see in the graph above, the majority of molecular evolution, according to the neutral model, occurs via non-adaptive processes. The narrow bar centered at zero indicates the frequency of neutral mutations (left) and neutral fixations (right). There are some fixations that result from Darwinian selection (adaptive evolution), shown by the small bump in the graph on the right. But the model predicts so few mutations in this class (they don’t even show up on the graph on the left) that most of the divergence between species occurs at sites that confer no selective cost or benefit.


As a contrast to the neutral model, I have created a caricatured adaptive model (see below). The main difference between this adaptive model and the neutral model is in the frequency of mutations that confer a fitness benefit. The adaptive model allows for a couple orders of magnitude more beneficial mutations, which means there are substantially more fixations driven by natural selection. In both models, the majority of new mutations are either deleterious or neutral. And, in both models, the deleterious ones are purged and the neutral ones fix with a probability dependent mutation rate. But, to hammer home the point, the adaptive model leads to many more adaptive fixations than the neutral model based solely on the frequency of beneficial mutations.


<img alt="i-30b708a8a0cc15b503be530f3ac6d285-mut_fix_adv_model.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-30b708a8a0cc15b503be530f3ac6d285-mut_fix_adv_model.gif"/>


Finally, I would be remiss to not mention the [nearly neutral theory](http://scienceblogs.com/evolgen/2006/02/more_on_misconceptions.php). In the neutral model, there is a class of mutations that confers no fitness cost or benefit (they are truly neutral). The nearly neutral model uses the idea that the fitness cost/benefit of a mutation is a product of the selection coefficient (**s**) and population size (**N**). If **Ns** &lt; 1, natural selection will remove the mutation from the population because it is deleterious. If **Ns** &gt; 1, a mutation has a high chance of fixing. But if **Ns** is in the ballpark of 1, a mutation will behave much like a neutral allele.


That means that the neutrality of a mutation depends not only the selective coefficient of a mutation, but also on the size of the population in which it arose. Small populations will have many more mutations with *Ns* near one than large populations because small populations have a smaller *N*. Because there are more mutations that are essentially neutral in a small population, small populations will experience more evolution due to drift. Large populations will have fewer mutations behaving as neutral variants — they’ll either be slightly deleterious or slightly advantageous — so you’ll see more fixations of alleles as the result of positive selection and more removal of alleles as the result of purifying selection.


<img alt="i-04e80ef4ae5c824c233c483862a6842c-mut_fix_nearly_neutral.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-04e80ef4ae5c824c233c483862a6842c-mut_fix_nearly_neutral.gif"/>


The expected distribution of mutations and fixations in different fitness categories for the nearly neutral theory (shown above) resembles that of the neutral theory with one major exception: the neutral mutations in the neutral theory have been replaced by a distribution of mutations that are nearly neutral (there aren’t values on the axes, but assume the distribution is in the ballpark of -1&lt;*Ns*&lt;1). The many deleterious mutations that arise in a population will still be removed by natural selection, and the few highly advantageous mutations will be fixed by selection. But the probability of fixation of the nearly neutral mutations (those in the distribution near zero) will depend on population size. In small populations, the fate of many will depend on random sampling (ie, genetic drift). In large populations, many will be removed or fixed by selection, but there are still other that will be removed or fixed by neutral processes.


In reality, we don’t see three discrete fitness categories; there is actually a continuum of fitness costs and benefits for new mutations which determine their probability of loss or fixation. The simple model presented above can be extended to a more realistic one if you so desire. But the purpose of this post was to illustrate a point, so I shied away from adding unnecessary complexities.


Which is the true model for molecular evolution? None of the above, that’s for sure. And, really, it depends on the locus and it depends on the time frame. Over short evolutionary time (within species or between closely related species), the many non-coding sites will evolve according to neutrality (fixed by chance, not by selection). But over longer time scales, even protein coding sequences appear to evolve neutrally — most likely because they evolve slower so there simply aren’t enough substitutions over short amounts of time to be able to observe neutral rates.


As for the utility of the neutral theory: it makes an excellent null hypothesis when searching for genomic regions under the influence of natural selection. [This recent review](http://dx.doi.org/10.1038/sj.hdy.6800937) by John Stinchcombe and Hopi Hoekstra does a good job explaining how we use the neutral theory to determine expected patterns of polymorphism and test for outlier loci that are not evolving neutrally (ie, aren’t evolving according to the manner expected of mutations with no fitness cost or benefit). It’s quite difficult (and impractical) to come up with a sweeping generalization regarding neutral and adaptive evolution across entire genomes. But the neutral theory has a very nice utilitarian practicality that cannot be neglected.


Getting back to [the article in question](http://www.seedmagazine.com/news/2007/03/the_sound_of_silence.php) (you know, the one that started this whole post), we see that the neutral theory has not been overturned, but it is constantly being overturned. Every time someone performs a test for natural selection (see [here](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php) and links therein), they are testing the neutral theory. And every time they identify a non-neutral pattern of evolution, they are overturning the neutral theory . . . for that particular locus.

#Philosopher Generalizes Beyond Experience
Date: March 29, 2007

[Janet](http://scienceblogs.com/ethicsandscience/2007/03/how_important_are_labs_for_lea.php) pointed me to a [post at the Philosopher’s Playground](http://philosophersplayground.blogspot.com/2007/03/does-learning-science-mean-doing.html) about doing away with laboratory courses in the science curriculum. Steve Gimbel, the philosopher doing the playing, teaches at Gettysburg College. He argues that the lab portions of science classes cause non-science majors to avoid those courses and not enroll in any science class not required for graduation. If science courses consisted of more theory and less labs (by theory he means lecture, and, by choosing that word, he indicates he doesn’t have much experience in non-physics science courses where the lectures consist of more than ‘theory’) non-science-majors would be more likely to take science courses, according to Gimbel.


Additionally, Gimbel points out that many experiments conducted in teaching labs lead to results that are inconsistent with the course material. That’s not because the course material is wrong, but because the experimental techniques employed in teaching labs are faulty. The combination of the time required and the crappy results lead Gimbel to conclude that we can do away with teaching labs. Both Gimbel and Janet are philosophers of science, specializing in physics and chemistry, respectively. As a biologist with way too much experience teaching undergraduate lab classes, I felt it necessary to make a few points defending their existence.





First off, we must make an important distinction between courses required for non-science majors and those required of science majors. Even though Gimbel tries to focus his argument on courses for non-science majors, he brings up some points that apply to both the groups. Gimbel claims that “students have a three to four hour chunk of their weekly schedule carved out in order to take any science class.” I seem to remember my chemistry lab classes taking three to four hours, but many biology labs are only booked for a far more manageable two hour block. Some of them, however, do require students to come in outside of their scheduled lab times to care for live organisms (ie, collect virgin *Drosophila* or maintain some plants), but a two hour block is far less obtrusive in one’s schedule than a three or four hour commitment.


But Gimbel claims the real costs are on the faculty side, where faculty must “teach the lecture course and a couple sections of the associated lab,” meaning they have fulfilled their teaching requirement with a single course. Instructors for non-lab courses, on the other hand, are required to teach multiple courses, increasing what Gimbel calls their “curricular footprints”. While this may be true for small colleges like Gettysburg, it doesn’t hold at large research institutions.


At larger schools, courses with lab components are quite balkanized. They are divided into the lecture component, which is handled by a faculty member or team taught by a couple instructors, and the laboratory section. The laboratory sections are usually taught by graduate students or advanced undergrads. The grad students get valuable teaching experience (and financial support) that would otherwise be unavailable without laboratory courses. Most faculty members are not overwhelmed by the lab portion of the course because they do not deal with the labs — that is left to teaching assistants and laboratory coordinators. This issue is more of a difference between small colleges and large universities than lab and non-lab courses.


What about the benefits of lab courses from a student’s perspective? I do acknowledge that the experiments performed in lab courses are clumsy and of poor quality. But, as I emphasize to my students, the purpose of a lab exercise is not to do amazing experiments, but to gain an understanding of how science is done. As I have [stated previously](http://scienceblogs.com/evolgen/2006/04/what_i_think_you_should_know_1.php), one does not understand science without understanding the hypothetico-deductive method. Yes, there is more to science than this simple approach, but it provides basic framework for understanding what scientists do.


And therein lies the value of laboratory courses. The students should perform experiments in which they develop hypotheses, test those hypotheses, and analyze their results relative to their hypotheses. I’ve [previously argued](http://scienceblogs.com/evolgen/2007/02/science_technology.php) that science courses should be organized around the scientific method rather than the memorization of facts or the practical application of the knowledge. A laboratory component that emphasizes the scientific method would complement such a lecture perfectly. This approach applies to both courses for majors and non-majors.


There are two more benefits of laboratory courses at large universities that I would like to point out. First, lab courses which require students to write up their results both emphasize the communication component of the scientific process and provide the students with an opportunity to write. Underclassmen don’t spend much time writing in their science lectures. They take exams, which consist of multiple choice and short answer questions, but the lecture courses are usually devoid of writing requirements. It’s my opinion that being able to write clearly is one of the most important skills for a student to develop. Forcing students to write and have their writing evaluated will, hopefully, improve the quality of their communication skills.


In addition to writing lab reports, lab and recitation components of courses at large universities allow students to interact with instructors or TAs in smaller classes. These lab sections usually contain about 20 students, whereas lectures may have hundreds of students. A good lab course will reinforce material covered in lecture and allow students to ask questions about the material in a more welcoming environment. This is often the only chance science majors get to take a small class at large universities in their first couple of years.


While there are drawbacks to lab courses, they are essential to a good education in the life sciences. As Gimbel points out, grading lab reports is hard (much more difficult and time consuming than grading exams), but, if a lab course is designed correctly, it will teach students important concepts that they would otherwise miss in a lecture only curriculum. This is especially true at large universities which have gigantic lecture sessions. In the end, the difference in conclusions (between Gimbel and me) may simply be the result of differences between small colleges and large universities and between the physical sciences and the life sciences.

#The Motivations of the Attack are Made Clear
Date: March 29, 2007

**URGENT:** We have a new update from the War Against [Manatees](http://scienceblogs.com/evolgen/biology/manatees/).


[<img alt="i-7f73c13c51b9b09b08ccb4354f6b45bd-bad_report_manatee_sm.GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7f73c13c51b9b09b08ccb4354f6b45bd-bad_report_manatee_sm.GIF"/>](http://www.sfgate.com/cgi-bin/article.cgi?file=/chronicle/archive/2007/03/28/DDASMUSSENBR.DTL)


The [Bad Reporter](http://www.sfgate.com/cgi-bin/article.cgi?file=/chronicle/archive/2007/03/28/DDASMUSSENBR.DTL) has a dispatch from the great [manatee](http://scienceblogs.com/evolgen/biology/manatees/) migration. While the enviro-terrorists are claiming global warming is to blame, we patriots know what’s up. It’s not warm northern waters that are drawing the sea cows north. They’re out to undermine our way of life.


That’s right — they’re on their way to Massachusetts (aka, the [Gay State](http://uncyclopedia.org/wiki/Massachusetts)) for some dirty man-on-man action. And maybe some woman-on-woman action. But definitely man-on-man. Hot, steamy, man-manatee-on-man-manatee action. DIRTY!!!


Because you can’t get that kind of nasty down in the gulf — unless you include Miami, which is really just an annex of New York City and doesn’t count as the south — they’re invading the northern waters. In the real south, where the manatees belong, that kind of behavior is not tolerated. So the manatees are moving to Gayassachusettes. For the gayness. And the hot, steamy man-on-man action. We are only left to wonder where their next destination may be. We’ve got our money on San Francisco.

#Phylogeny Friday – 30 March 2007
Date: March 30, 2007

**Mammals did not rapidly radiate after the K/T boundary.**


That’s the punch line of a [paper](http://dx.doi.org/10.1038/nature05634) published in this week’s issue of *Nature*. This has been all over the news, including the New York *Times* twice ([#1](http://www.nytimes.com/2007/03/29/science/29mammal.html) and [#2](http://www.nytimes.com/2007/03/28/science/28cnd-mammal.html)). You see, there’s this idea that when the dinosaurs (technically, the non-avian dinosaurs) disappeared, mammals quickly filled in the vacated niches. That means there should have been a rapid radiation of mammalian lineages following the dinosaur mass extinction — marked by the boundary between the Cretaceous and Tertiary periods (known as the K/T boundary).


The new study reveals that the extant mammalian lineages (those that are present today) did not originate immediately after the dinosaurs went extinct. [PZ Myers](http://scienceblogs.com/pharyngula/2007/03/dont_blame_the_dinosaurs.php) points out that our concept of a mammalian radiation is biased by our attention to charismatic megafauna. The little mammals — which make up the majority of mammalian diversity — wouldn’t have been as affected by the K/T extinction event as large mammals, so we won’t see much of a phylogenetic signature of the radiation of the big guys. Also, the study focused on extant mammals, and would have missed any radiation of extinct lineages.


I want to focus on another point: that of morphological differentiation and lineage diversification. Oh, and there’s a phylogeny below the fold.





The authors of the study constructed a phylogeny using DNA sequences from over 4500 mammalian species — nearly every extant species — and showed that the major extant mammalian clades (shown in five different colors on the tree below) originated prior to the K/T boundary (shown as a dashed circle). They also show that there is no evidence for increased phylogenetic diversification after the dinosaur extinction by plotting the amount of lineages versus time.


[<img alt="i-396262041845ed83796a7551cac673fe-mammal_tree_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-396262041845ed83796a7551cac673fe-mammal_tree_sm.JPG"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-6d30d90baea1c7c81a9d63fbf802e4d4-mammal_tree.jpg)<br/>*Click to enlarge.*


But does this really address the issue of mammals filling the niches vacated by non-avian dinosaurs? Evolutionary lineages are different than morphological diversification. The authors showed that there wasn’t any rapid diversification of the sampled lineages. But these are genetic lineages, which do not correlate perfectly with morphological differentiation. It’s possible that many of the mammalian lineages were present prior to the K/T boundary, but they didn’t morphologically differentiate until after the dinosaur mass extinction.


What we have is evidence against rapid speciation after the dinosaur extinction. We do not have evidence that there wasn’t rapid evolution of **phenotypes** after the extinction (sorry for the double negative). These are two separate hypotheses which must be addressed using different sets of data.


---



Bininda-Emonds ORP, Cardillo M, Jones KE, MacPhee RDE, Beck RMD, Grenyer R, Price SA, Vos RA, Gittleman JL, Purvis A. 2007. The delayed rise of present-day mammals. **Nature** **446:** 507-511. doi: [10.1038/nature05634](http://dx.doi.org/10.1038/nature05634)

#SCIENCE SPRING SHOWDOWN: Chair Region — Sweet 16 Preview (Corporate vs. Darwin)
Date: March 30, 2007

[PRESS CENTER](http://scienceblogs.com/worldsfair/2007/03/science_spring_showdown_2007_t.php) | [UPDATED BRACKET](http://scienceblogs.com/worldsfair/upload/2007/03/bracketss.pdf)


<img alt="i-3e2a252f6079206d0248609769ae4f13-darwin_sell-out.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3e2a252f6079206d0248609769ae4f13-darwin_sell-out.gif"/>


Early next week, the amorphous, indefinable entity that is Corporate will take on a man named Charles Robert Darwin in the [third round](http://scienceblogs.com/worldsfair/2007/03/down_to_the_sweet_sixteen_the.php) of the Science Spring Showdown. That’s right, we’re down to sixteen teams, including the eleven seed Corporate and the seven seed Darwin. This match up of Chair Region powerhouses will be presented on evolgen. A preview of this potentially epic battle can be found below.





Some may say that Charles Darwin was a tentative man. They base this claim on the fact that Chuck waited twenty years from his first ideas on evolution by natural selection to the first publication of *On the Origin of Species*. It was long thought that this tentativeness would be Darwin’s greatest obstacle in the Science Spring Showdown. But the [tide may be changing](http://www.journals.royalsoc.ac.uk/openurl.asp?genre=article&amp;id=doi:10.1098/rsnr.2006.0171) — Darwin’s delay may simply have been the result of a man with a lot of other stuff to write about.


The Corporate team, lead by Johnson &amp; Johnson and Dow Chemicals, have developed a game plan centered on Darwin’s reluctance to publish his seminal piece. These companies aggressively pursue patents, and now, with the [elimination of Genomics](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu_2.php), we can only imagine the Corporate team will have even more fan support (which may lead to even more patents). But the entire Corporate strategy may backfire given this new telling of Charles Darwin’s story.


Rather than dwelling on the minutia of one of the most important scientific theories of all time, Darwin focused his attention on the important things in life: invertebrates. And you can bet the inverts, coming off of a [blowout of Surgery](http://scienceblogs.com/evolgen/2007/03/science_spring_showdown_octopu_2.php), will have some vested interest in the game between Corporate and Darwin. The barnacles in particular will be playing close attention to Darwin, but we’re not sure which way they’ll be leaning…


Corporate is left to devise a new strategy in the few days remaining before the big game. How will they go after Darwin? How will they defend against the Victorian naturalist’s prolific prose? How will Chuck approach big business? Will he describe them in excruciating detail? We know his rigorous writing style would fit perfectly in the bureaucratic environment of industry, so he must develop a new strategy. Leave your predictions in the comments.

#A Rube Goldberg Machine….
Date: April 23, 2007

…that appears to do absolutely nothing ([link](http://www.chilloutzone.de/files/07022502.html)). Usually they do something mundane in a really complex manner. But all [this one](http://www.chilloutzone.de/files/07022502.html) does is open the curtains so that the credits can run. The device pictured below is different from the one linked above, but I figured this post needed something to fill up more space.


<img alt="i-fdd83989cb1505cc6fbac58feea21c1c-rube_back_scratch.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-fdd83989cb1505cc6fbac58feea21c1c-rube_back_scratch.gif"/>


(Via [The Disgruntled Chemist](http://thedisgruntled.blogspot.com/2007/04/awesome-rube-goldberg-machine.html).)

#Links Dumped
Date: April 23, 2007

Because my browser consists of a growing forest of tabs containing stuff waiting to be blogged, and there is no way I can write a complete entry on each one, and I want to at least link to the relevant sites so that I can close those tabs, I give you a link dump with maximal run-on sentences:


<li>
Remember my post on [the evolution of zombie populations](http://scienceblogs.com/evolgen/2007/01/on_the_evolution_of_zombie_pop.php)? If so, you’ve got a good memory. If not, who cares? Anyway, there’s an [article](http://www.arxiv.org/abs/physics/0608059) in arXiv about ghosts, vampires and zombies. (Via [Ortholog](http://www.ortholog.com/archive/deserving_memes/garlic_cross_stake_lotkavolterra_equatio.php)).
</li>
<li>
The [evolution of snake venom](http://www.biomedcentral.com/1471-2148/7/2) genes. They make up part of the **venome**.
</li>
<li>
Jonathan Eisen on the [Human Microbiome Project](http://phylogenomics.blogspot.com/2007/04/human-microbiome-program.html).
</li>
<li>
Share your presentations using [Slideshare](http://www.slideshare.net/). (Via [Neil Saunders](http://nsaunders.wordpress.com/2007/04/22/slideshare-and-wordpress/).)
</li>

#Junk DNA in Scientific American
Date: April 24, 2007

Would I write about junk DNA? [No](http://scienceblogs.com/evolgen/2006/03/junk_dna_is_junk.php). [Never](http://scienceblogs.com/evolgen/2006/03/junk_dna_redux.php). [Not me](http://scienceblogs.com/evolgen/2006/04/junkity_junk_mcjunkerson_1.php). Not even when [Scientific American](http://scienceblogs.com/evolgen/2007/02/junk_dna_revisited.php) publishes sub par articles on junk DNA. Well, they’re [at it again](http://www.sciam.com/article.cfm?chanID=sa003&amp;articleID=2090C7F6-E7F2-99DF-3E988265E9F00613).


The most recent [junk DNA article](http://www.sciam.com/article.cfm?chanID=sa003&amp;articleID=2090C7F6-E7F2-99DF-3E988265E9F00613) describes a study by Gill Bejerano of Stanford University which I can’t find published anywhere (neither can [ERV](http://endogenousretrovirus.blogspot.com/2007/04/more-junk-for-scientific-american.html)). Is Scientific American describing unpublished research on their website? Either way, they’re doing it poorly. The study described in the article identified non-coding sequences that were conserved across all sequenced mammalian genomes but missing from the chicken genome. Five percent of the sequences resemble known transposable elements, and the rest are of unknown origin. The Scientific American article said so much in nine paragraphs, and included the (unnecessary) word “junk” nine times (including twice in the headline and twice in a figure caption which is just a picture of a landfill).


Once again, I’ll say that I don’t mind the term “junk DNA” so long as it’s used appropriately. Throwing it around willy-nilly like every popular science writer seems to do is not the appropriate usage. Also, claiming that we have absolutely no freakin’ idea what non-coding DNA does is highly misleading. We know a fair bit about the function of non-protein coding DNA, including how it regulates transcription, how it encodes non-translated RNAs, and many studies have identified conserved non-coding elements. That said, a lot of it probably is junk. There are a ton of DNA sequences in any eukaryotic genome that serve absolutely no function, but they’re not deleterious enough to be purged from the population. So they’re just there, and that’s all there is to it.

#Junk in the Media
Date: April 26, 2007

<img alt="i-f356d3c680563d9f75006916fbde8514-junk.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-f356d3c680563d9f75006916fbde8514-junk.JPG"/>


The recent [Scientific American article](http://www.sciam.com/article.cfm?chanID=sa003&amp;articleID=2090C7F6-E7F2-99DF-3E988265E9F00613) on junk DNA (discussed [here](http://scienceblogs.com/evolgen/2007/04/junk_dna_in_scientific_america.php)) has instigated a quite a furor in the bioblogosphere. Here is a collection of links:


- [ERV linked](http://endogenousretrovirus.blogspot.com/2007/04/more-junk-for-scientific-american.html) with a tone of disgust.
- I [restated my frustration](http://scienceblogs.com/evolgen/2007/04/junk_dna_in_scientific_america.php) with the term [junk DNA](http://www.google.com/search?q=junk+dna&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search).
- JR Minkel, the author of the Scientific American article, [responded](http://blog.sciam.com/index.php?title=the_dna_formerly_known_as_junk_aka_calli) to my criticisms.
- Ryan Gregory [replied](http://genomicron.blogspot.com/2007/04/junctional-dna.html) to Minkel’s SciAm blog post, introducing the term “junctional DNA” to <strike>replace junk DNA</strike> describe sequences with unknown function. Gregory also <strike>tells us [how a genome is like an onion](http://genomicron.blogspot.com/2007/04/onion-test.html) (let me count the ways), or something of the sort</strike> wonders why [onions have really big genomes](http://genomicron.blogspot.com/2007/04/onion-test.html).
- If you have any further questions about junk DNA, Ryan Gregory is also [collecting questions](http://genomicron.blogspot.com/2007/04/questions-about-junk-dna.html) about junk DNA. Word on the street is he may even answer some of them.


What do I think about junk DNA? Should we replace it with another term (ie, junctional DNA, funk DNA)? I’d rather just see the whole thing scrapped. Abandoned. Junked.


**Update:** At Gregory’s request, I changed his bullet point. I’ve been accused of “sloppy reporting”, but I prefer to think of it as playing fast and loose. I shoot from the hip. I’m a hip shooter. And I thought the <strike>sarcasm</strike> <strike>irony</strike> intentional-douchebaggery was clear. Also, John Timmer [writes about the actual paper](http://arstechnica.com/journals/science.ars/2007/04/26/abit-of-useful-junk-mobile-dna-and-gene-regulation).

#Around the Bioblogosphere
Date: April 26, 2007

Ever wonder what biobloggers are blogging about on their blogs? Here’s what:


- Razib [posts](http://www.gnxp.com/blog/2007/04/against-evo-devo.php) part of a paper by Jerry Coyne and others (which I can’t seem to track down) which questions the role *cis* regulatory elements play in adaptive phenotypic evolution. This all part of [Coyne’s war on evo-devo](http://evolgen.blogspot.com/2005/06/coyne-versus-carroll.html).
- Another [post at GNXP](http://www.gnxp.com/blog/2007/04/population-gentics-of-deletion.php) (this one by p-ter) describes a polymorphic deletion that is associated with resistance to retroviral infection. P-ter tries to throw me a bone by mentioning the relevance of *Drosophila* research, but ends up shooting himself in the foot (how’s that for mixed metaphors?).
- Orac’s got a [long post](http://scienceblogs.com/insolence/2007/04/peter_duesberg_chromosomal_chaos_and_can.php) (Orac’s not good with concise, me not good with grammar) on a [Scientific American article](http://www.sciam.com/article.cfm?chanID=sa006&amp;colID=1&amp;articleID=E68F1707-E7F2-99DF-3C3FB15F1C7AAF61) by Peter Duesberg about chromosomal aberrations and cancers. Interestingly, the article is written by a guy who doubts the connection between HIV and AIDS. Also interesting is how Duesberg toots his own horn on the connection between screwed up chromosomes and cancer even though it’s not his idea (despite the way he presents it), it’s not a minority viewpoint, and he neglects a fair bit of literature on other causes of cancers. Like I said to Orac, this stuff can’t be all that revolutionary if I — a dude who doesn’t know jack about cancer — know about the connection between aberrations and cancer.
- P-ter has also blogged on [constraint of gene expression](http://www.gnxp.com/blog/2007/04/breaking-people-are-different.php) profiles. He also tells us that human populations differ genetically.
- Jacob is clipping toes and [writing about it](http://www.salamander-candy.com/2007/04/catch_frogs_release_toed_part.htm) at Salamander Candy.
- The [worst science news article ever](http://www.abc.net.au/news/newsitems/200704/s1906743.htm), and it doesn’t even mention junk DNA (via [Neil](http://nsaunders.wordpress.com/2007/04/26/awful-science-reporting/)).
- And, finally, ScienceBlogger “Shelly Bats” (*sic*) was [threatened](http://scienceblogs.com/retrospectacle/2007/04/when_fair_use_isnt_fair_1.php) by an Editorial Assistant from the Journal of the Science of Food and Agriculture. The junior editor emailed Shelley that lawyers from John Wiley &amp; Sons would pursue legal action against her if she did not remove a table and part of a figure from an article published in the Journal that she included in [a post](http://scienceblogs.com/retrospectacle/2007/04/antioxidants_in_berries_increa_1.php) about that article.

#Phylogeny Friday – 27 April 2007
Date: April 27, 2007

This week’s phylogeny comes from [this paper](http://dx.doi.org/10.1016/j.tree.2006.11.013) on molecular dating of speciation events. I won’t be addressing molecular dating *per se*, but I will be dealing with what molecular clocks tell us. Like, do they actually reveal the speciation time of a pair of species?





The divergence date of a pair of species can refer to two things: when the two populations became two species (no longer exchanging alleles) or when the two genetic lineages split. The splitting of genetic lineages happens ****prior**** to the speciation event. That’s because within a population there is variation throughout the genome. It’s that variation, when it gets fixed between species, which contributes to the initial divergence between the species. Most of the molecular variation gets fixed randomly, but the variation originated prior to the split of the species. When we use divergence times to date speciation events, we’re actually looking at the date when that variation arose.


[<img alt="i-2341dc55fac6efe27378f8de4f8068c5-species_gene_tree.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-2341dc55fac6efe27378f8de4f8068c5-species_gene_tree.jpg"/>](http://dx.doi.org/10.1016/j.tree.2006.11.013)


This figure displays the aforementioned concepts. The red line shows the genetic relationships of members of Species 1 and Species 2, and the black outer lines show the relationships of the organisms. The molecular divergence of species 1 &amp; 2 is marked by “Cc”, while the actual point at which the lineages become true species is indicated by the blue line. Between Cc and the blue line, alleles are segregating in an ancestral population. Some of those alleles end up fixing along the lineage leading to Species 1, and some fix on the lineage leading to Species 2.


For recently diverged species, this can be problematic. Take, for example, the hypothetical species pair in the figure. The amount of time from the present back to the speciation event is approximately equal to the amount of time from the speciation event to the molecular common ancestor (Cc). That means that estimates the divergence time of the two species using molecular data will not be a good approximation of the speciation time of the two species; the molecular divergence time of the species will be a gross overestimate of the speciation time.


A good example of the disagreement between speciation times and divergence times is the human-chimpanzee species pair. Estimates of the human-chimp divergence using molecular data center around 6 million years ago (see [here](http://dx.doi.org/10.1073/pnas.0509585102  ) for an example). Recently, a paper was published that dated the speciation time of humans and chimps at approximately 4mya (available [here](http://dx.doi.org/10.1371/journal.pgen.0030007)). These are not conflicting estimates, as the divergence time is expected to predate the speciation time.


If you followed the discussion this far, it should be clear to you that this problem goes away when we’re dealing with divergence times further back in the past. The time between Cc and the speciation event stays constant, but the distance from the present to the speciation event increases over time. Eventually, the time from the present to the speciation event dwarfs the time from Cc to speciation, making the estimate of divergence time equivalent to the estimates of speciation time.


---



Hobolth A, Christensen OF, Mailund T, Schierup MH. 2007 Genomic relationships and speciation times of human, chimpanzee, and gorilla inferred from a coalescent hidden markov model. PLoS Genet 3: e7 doi: [10.1371/journal.pgen.0030007](http://dx.doi.org/10.1371/journal.pgen.0030007)


Kumar S, Filipski A, Swarna V, Walker A, and Hedges SB. 2005. Placing confidence limits on the molecular age of the human-chimpanzee divergence. PNAS 102: 18842-18847. doi: [10.1073/pnas.0509585102](http://dx.doi.org/10.1073/pnas.0509585102)


Pulquerioa MJF and Nicholsa RA. 2007. Dates from the molecular clock: how wrong can we be? Trends Ecol Evol 22: 180-184. doi: [10.1016/j.tree.2006.11.013](http://dx.doi.org/10.1016/j.tree.2006.11.013)

#Genetics of Speciation
Date: April 27, 2007

The [most recent issue of the **Journal of Heredity**](http://jhered.oxfordjournals.org/content/vol98/issue2/index.dtl) contains a bunch of articles from a symposium on the “Genetics of Speciation” organized by [Loren Rieseberg](http://dx.doi.org/10.1093/jhered/esl066). Included in the collection is an [article by Allen Orr](http://dx.doi.org/10.1093/jhered/esl060 ) and two of his students on speciation in *Drosophila*, which discusses mapping speciation genes, the role of meiotic drive in speciation, and Dobzhansky Muller incompatibilities via gene translocation (the latter two are the topics of recent papers from Orr’s group — [here](http://dx.doi.org/10.1534/genetics.104.033274) and [here](http://dx.doi.org/10.1126/science.1128721)). Also in the special issue is an [article from Mohamed Noor’s lab](http://dx.doi.org/10.1093/jhered/esl065 ) on mapping inversion breakpoints between two sibling species of *Drosophila*, an [article on expression of duplicate genes](http://dx.doi.org/10.1093/jhered/esl061) in polyploid and hybrid plants, and a few others.

#Scientific American Editor Says, “I’d Hit That”
Date: April 28, 2007

In a round-up of some of the coverage of [Shelley’s run-in with Wiley](http://scienceblogs.com/retrospectacle/2007/04/what_i_want_out_of_this_is.php), Scientific American’s Nikhil Swaminathan [wrote the following](http://blog.sciam.com/index.php?p=527&amp;more=1&amp;c=1&amp;tb=1&amp;pb=1&amp;showComments=1):


> 
Anyway, on Tuesday, over at the ScienceBlog Retrospectacle, neuroscience PhD student Shelley Batts (who based on her pictures alone seems to be both attractive and avian-friendly) posted an analysis of a study appearing in the Journal of the Science of Food and Agriculture, which suggested that the antioxidants properties in fruits were boosted by alcohol.



So, Nik wants to hit that shit. This is, like, two steps away from passing Shelley a note in homeroom that reads “Do you like me?” and has two empty boxes marked “Yes” and “No”. The rule is Shelley is supposed to check one of them and pass the note back to Nik — in case you didn’t know. Now, this isn’t all that professional of a representative of Scientific American, but some guys ain’t as smooth as the other side of the pillow. What? I fucked that line up? Well, screw me with a screen door on a submarine.


Here’s the interesting twist: Shelley [linked](http://scienceblogs.com/retrospectacle/2007/04/scientific_american_blog_weigh.php) to the SciAm blog post without commenting on Nik’s complement/sexist-remark. This was brought up in the comments, and Shelley is yet to respond. Whether she does or not is up to her. But I’m wondering which box she’s checked.


As expected, [Zuska is pissed](http://scienceblogs.com/thusspakezuska/2007/04/sciam_reports_scienceblogger_i.php), but that was expected, as expected.

#A Post on Sex Chromosomes
Date: April 29, 2007

A few articles have come out recently dealing with sex chromosomes in a variety of taxa. Here are some links to those articles, in list form:


- Given [all we know](http://scienceblogs.com/evolgen/2006/09/phylogeny_friday_8_september_2.php) about [vertebrate sex chromosomes](http://scienceblogs.com/evolgen/2006/12/origins_of_amniote_sex_chromos.php), it’s surprising that we don’t know how sex determination works in many fish, including the pufferfish *Takifugu rubripes*. That’s especially surprising given that the fugu genome was [sequenced](http://www.jgi.doe.gov/sequencing/why/pufferfish.html). That is, we didn’t know much until researchers mapped a sex determining locus and reported it in [this paper](http://dx.doi.org/10.1534/genetics.106.069278). You see, fish use a wide range of mechanisms for sex determination, including environmental cues and genetic signals, and some even change from one sex to another in midlife. Just figuring out how a species of fish determines sex is an important step in understanding the biology of the organism.
- In contrast to fishes, the sex determination system in placental mammals is well understood. The Y chromosome contains a gene (*SRY*) that initiates the development of male characters. [This paper](http://dx.doi.org/10.1534/genetics.106.066779) reports on the divergence of *SRY* amongst cats. The authors observe different rates of protein evolution in different lineages, suggesting that natural selection on *SRY* is not constant even along short evolutionary time scales.
- If we’re going to discuss the evolution of the mammalian Y chromosome, we should give equal time to the X chromosome. [Here](http://dx.doi.org/10.1534/genetics.106.069419) is a study of nucleotide diversity on the X chromosome and two autosomes in a few subspecies house mice. The authors found reduced polymorphism on the X relative to autosomes, and the diversity could not be explained by demography. They conclude that either adaptive evolution is greater on the X (reducing polymorphism) or X chromosome loci introgress less easily between subspecies.

#Giant Squids, Other Squids, and Manatees
Date: April 30, 2007

Dr. Doris [Haggis-on-Whey](http://www.haggis-on-whey.com/index.php) and her husband Benny, as part of their H-O-W series of books, have produced an absolutely dispensable piece of misinformation, the third in a series of we can only hope not too many, ineloquently titled [*Animals of the Ocean: In Particular the Giant Squid*](http://www.amazon.com/Animals-Ocean-Particular-Giant-Squid/dp/1932416390/). They claim that their World of Unbelievable Brilliance series, of which *Animals of the Ocean* is the third in a series of we can only hope . . . um, they claim that their series . . . well, I don’t really know what they claim, because they never get around to it.


Dr. and Mr. Haggis-on-Whey are neither not biologists nor are they not experts on the ocean. They are not experts on knots either, but that’s not neither for not now nor not . . . later. Their lack of expertise shines through quite clearly in their latest work — a flaw obvious even to the most naive reader, your not-so-humble reviewer. Sure, the Haggis-on-Wheys are full of themselves, but that’s expected given Doris’s impressive **curriculum vitae** (that’s pointy headed ivory tower talk for **resume**, which is French for the shit you’ve done and stuff). Sadly, for all their qualifications, they know absolutely nothing about the ocean, the sea, el mar, or any other large body of salty water. I will show you evidence to support this claim in the following paragraph.





This is that paragraph mentioned in the previous sentence (located in the previous paragraph, the one right above this sentence &amp; paragraph). First of all (first in the list of evidence that the Haggis-on-Wheys know nothing about salt water, interrupted by the requisite parentheticals), the book, as the title proudly proclaims, deals primarily with the giant squid. This would not be problem if the giant squid made up the majority of the material in the ocean. This is not the case. <strike>This</strike> The ocean is mostly water. Second in the list of things in the ocean (not to be mistaken for the list of things Dr. and Mr. Haggis-on-Whey get wrong) is salt. That’s why they (not Dr. and Mr. Haggis-on-Whey, but people who actual know about the ocean, ie, ocean professionals, ie, oceanographers, oceanoligicians, and oceanists) call it “salt water”. Third is pie, which means the Haggis-on-Wheys actually get something right when they tell us that squids eat mostly pie.


But the giant squid makes up a small fraction of the entire ocean. Even amongst very large squids, the giant squid isn’t even the giantest. That honor belongs to the colossal squid. And squids aren’t even the most numerous, let alone the most interesting, organims in the organisms — those honors belong to some microscopic thing I can’t be bothered to look up and the sailfish, respectively. Yet the Haggis-on-Wheys have devoted a large fraction of their book to squids and other tentacled creatures (17 of 62 pages to be precise, not including other pages where squids are mentioned but not the focus of discussion).


Not only do cephalopods receive a treatment disproportionate to their importance — they’re really bit players in any decent ocean adventure (yeah, Jules, I said it: **20,000 Leagues** sucked!) — but we’re also treated to this image, without context:


[<img alt="i-5814b94579b5813cc8540cfde70415c3-cuttlefish_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5814b94579b5813cc8540cfde70415c3-cuttlefish_sm.gif"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9dead449a8ed0639097dae9aa6c4449e-cuttlefish_lg.gif)


I want to know what each of those species is named. I want to know where they can be found. And I want to know which ones taste best pickled, which ones taste best fried, and which ones I should avoid eating at all costs. A better book would have done that, but that book would not have been written by the Haggis-on-Wheys because that book would be good.


Of interest to interested evolgen readers is the not-so-interesting treatment of manatees. In a world in which the Haggis-on-Wheys wrote informative books, they would have pointed out the [threat manatees pose](http://scienceblogs.com/evolgen/biology/manatees/) to the safety of all Americans (and all lovers of freedom throughout the world). This is not that world. Instead, we get a paragraph about some manatee named Fred who developed “a look”. Nothing about Fred stockpiling weapons. Nothing about Fred attempting to purchase enriched uranium. Nothing about Fred’s mobile biological weapons labs. Just Fred and his look, which isn’t actually described, but we can imagine it involves a trucker hat and overalls.


In summary, conclusion, and wrapping-up, the Haggis-on-Wheys’ **Animals of the Ocean** falls short when it comes to information about animals and information about the ocean. If you are looking for a book with this information, do not purchase this book. If you’re looking for a book about squids and other ocean stuff written in a manner resembling the ill-informed ramblings of some senile, ignorant, washed up scientists, the Haggis-on-Wheys have produced the perfect book for you.


Oh, and I almost forget, if you’re going to compare West Coast Squids with Dirty South Squids, the least you can do is mention the throw down that went down when the Humboldts rolled into the Gulf of Mexico and started static with the squid there. And don’t forget that Duurrrtyy South Squids wear ice in their grill, whereas West Coast Squids fix their rides up so they bounce. But, yeah, West Coasters are all about the fold-over sandwich baggies, and they like zip-locks in the Dyeiuuurrrtttaaiiiyyy South.


---



**Note:** Your not-so-humble reviewer has a long running feud with Dr. Haggis-on-Whey regarding the role of phytoplankton in ecosystems devoid of overbearing mollusks. Obviously, Haggis-on-Whey thinks mollusks rock and can’t even be bothered to discuss small things she can’t see with the unaided eye. This reviewer won’t stop poking the woman in the arm.

#Redefining the Binomial
Date: May 16, 2007

There’s an interesting post over at [Statistical Modeling, Causal Inference, and Social Science](http://www.stat.columbia.edu/~cook/movabletype/archives/2007/05/y1n2_instead_of.html) on calculating probabilities. Traditionally, if you observe a certain number of events (*y*) in some number of trials (*n*), you would estimate the probability (*p*) of the event as *y*/*n*. To calculate the variance around this estimate, you would use this equation: *p*(1-*p*)/*n*.


This leads to two problems. First, if you never observe the event, your estimate of the probability of the event is zero; if you observe the event in every trial, your estimate is one. This leads to a [deterministic](http://en.wikipedia.org/wiki/Deterministic_system_(mathematics)) model even if the unobserved event is possible. Second, if *p* is estimated as zero or one, then the estimated variance is zero (once again, suggesting a deterministic model).


To get around these problems, the formula (**y**+1)/(**n**+2) is proposed for calculated *p*. Using this formula, you can never get a probability of zero or one, and the variance will always be greater than zero. There is further discussion of the implications of this calculation  at [SMCISS](http://www.stat.columbia.edu/~cook/movabletype/archives/2007/05/y1n2_instead_of.html).

#Math for Biologists
Date: May 17, 2007

Keith Robison, at Omics! Omics!, [asks and answers](http://omicsomics.blogspot.com/2007/05/what-math-for-biology.html) the question, “What math courses should a biologist take in college?” His answer: a good statistics course is a must (one where you learn about experimental design and Bayesian statistics), and a survey course that covers topics like graph theory and matrix math would provide a nice introduction to important topics (that course probably doesn’t exist at most colleges). He also advocates taking a programming class and turning math education into something more stimulating rather than rote drilling (easier said than done).


This being a blog, I, obviously, have an opinion. While I can’t say what math courses are best for all biologists, I can suggest which ones are very helpful for a quantitative biologist/geneticist, if not essential. I’ll do so by first describing my math education, and then saying what I would have changed. When I arrived at college, I found that I had completed the math requirement for my biology major by taking two semesters of calculus in high school – my college required either take semester of calculus or one semester of calculus plus a semester of statistics for the major. As an incoming freshman being told that I didn’t have to take any more math, I was ecstatic. My academic advisor at the time was a faculty member in the religious studies department, and he didn’t know any better to suggest I take any more math classes (one of the detriments of a liberal arts education, I guess).





What I should have done was take linear algebra during my freshman year, but I didn’t realize that until a couple of years later. I wanted to be a geneticist (at that point I didn’t even know what population genetics was, let alone that I would be interested in it), and I thought math wouldn’t be all that important. I did end up taking two math courses later on in my undergrad career — one in introductory statistics and one in dynamic modeling — but, by the time I realized that I should have taken linear algebra, it was too late. It had been a few years since my calculus class, and I knew it would be damn near impossible for me to jump right back into a real math class.


I was exposed to eigenvectors and eigenvalues for the first time in the dynamic modeling course. The professor assumed we understood those concepts, so I had to visit with the TA to have him explain them to me. I wasn’t required to understand how eigenvectors and eigenvalues are calculated, only how to interpret them in terms of transition matrices and how to get Matlab to calculate them for me. It was at that point that I realized how valuable a course in linear algebra would have been. Since then, I have encountered various aspects of matrix math and continually lament not enrolling in at least one more semester of math upon arriving at college.


I agree with Keith that a statistics course is essential for any undergrad biology major. My undergrad statistics course was taught through a biostats department, and I had a better experience than Keith did in his stats course with business majors. The course covered all the basics of hypothesis testing, including the major distributions and tests (normal, t-test, chi-square, ANOVA, regression, etc). This course was, in my opinion, the ideal introductory statistics course. For someone interested in quantitative genetics, I would also recommend a course in probability theory. As a grad student, I took a course in which we started with first principles and derived all the major distributions. While this course wasn’t as practical as my undergrad statistics course, it was extremely useful for developing an understanding of how the statistical tests work and where they come from.


With all that said, here is the mathematical coursework I would recommend for anyone interested in quantitative genetics. I generally agree with Keith, although my list draws upon courses that I have seen offered at universities.


- Three to four semesters of the university’s calculus sequence — calculus 1 &amp; 2, plus linear algebra and maybe differential equations.
- An introductory statistics course and possibly a course on probability theory. Ideally, the intro statistics course would also teach students how to use [R](http://www.r-project.org/), rather than use MiniTab.
- At least one programming course. Yes, I realize that this isn’t math *per se*, but Keith included it in his post, so I’m including it in mine. Ideally, the student would take two courses. The first course would teach programming in C, C++, Java, Python, or an equivalent language. The second course would teach scripting such as PERL for various applications (this type of course is already offered under the guise of a bioinformatics course at some universities).

#A Question for the Colorblind
Date: May 17, 2007

I’m working on a few graphs for a presentation. In a previous incarnation, I distinguished two partitions of my data using the colors red and green. This made sense intuitively (the red ones had something broken, and the green ones were a-ok), but I realized that people with [red-green colorblindness](http://en.wikipedia.org/wiki/Color_blindness#Red-green_color_blindness) would not be able to distinguish the different graphs. I switched the color scheme to white and gray, which should enable everyone to distinguish the two groups.


<img alt="i-d84a62bfb1ffc7f1a3f9f28bc5e2d69d-blue_yellow_graph.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d84a62bfb1ffc7f1a3f9f28bc5e2d69d-blue_yellow_graph.gif"/>


I also have a second way of partitioning the data, and I don’t want to use the same color scheme for both. I initially used blue and yellow for these two groups, but that [may be problematic](http://en.wikipedia.org/wiki/Color_blindness#Blue-yellow_color_blindness). I can’t use gray and white for this partition because then the two different partitions would look the same. Does anyone have any suggestions for another color scheme for this data set? I’d like to find two color schemes that are both acceptable for the colorblind and unique enough that the graphs showing different partitions won’t be confused.

#Attack of the Hybrids
Date: May 25, 2007

For some reason, I have been collecting links to articles involving hybridization. That, on its own, would call for a massive link dump, but a recent news item makes for a nice contrast. First, the hybrids:


- Where better to start than [this review](http://dx.doi.org/10.1038/nature05706) of hybrid speciation — a topic I’ve [discussed](http://scienceblogs.com/evolgen/2006/12/hybrid_speciation_strikes_agai.php) [previously](http://scienceblogs.com/evolgen/2007/04/a_blow_against_hybrid_speciati_1.php). The take home message: we first thought that hybrid speciation only happened in combination with polyploidy in plants, then we found out that hybrid speciation can also occur without polyploidization in plants, and then we discovered that the same thing happens in animals.
- Here’s an [article on the ecology of hybrid zones](http://dx.doi.org/10.1038/sj.hdy.6800969). The authors present on a model that incorporates habitat disturbance in contact areas between plant species.
- If modeling hybrid zones isn’t your cup of tea and you want data from real populations, [here](http://dx.doi.org/10.1371/journal.pone.0000357) is a paper on a hybrid zone between a threatened species and an invasive species.
- And speaking of environmental effects on hybridization, [here](http://dx.doi.org/10.1111/j.1558-5646.2007.00104.x) is a paper in which the authors report the results of a study to map QTLs for reproductive isolation between two populations of *Drosophila mojavensis*. Bill Edges and colleagues recorded the mating success of F2 hybrid males with females from one of the populations and analyzed the courtship song of the males (*Drosophila* males rub their wings together, producing a song that gets the females turned on — kind of like Barry White for insects). What makes this study unique is that the males were reared on two different host plants — one native to each of the two populations — and the effect of each QTL, each host, and the genotype by environment interaction were determined. They found evidence for all three effects on reproductive isolation.
- *Helianthus* sunflowers are one of the classic models of hybrid zones and hybrid speciation. Laboratory crosses between *H. annuus* and *H. petiolaris* show strong reproductive isolation between the two species. Loren Rieseberg and colleagues, however, [demonstrate that there is little genetic differentiation](http://dx.doi.org/10.1534/genetics.106.064469) between the two species despite the fact that they rarely interbreed. Rieseberg is also author on [a pape](http://www.journals.uchicago.edu/AN/journal/issues/v169n6/41905/brief/41905.abstract.html)r in which he and his co-authors study the effects of natural selection on hybrid speciation in *Helianthus*





And, finally, in the antithesis of hybridization, someone told a hammerhead shark to go fuck herself, and [she took it literally](http://dx.doi.org/10.1098/rsbl.2007.0189). Rather than soil her personal gene pool by outcrossing, she produced a pup, not by cloning herself, but by screwing around with meiosis.

#Selection on Mitochondrial Genomes
Date: May 27, 2007

Given the amount of attention I devoted to the effect of selection on the relationship between mitochondrial DNA polymorphism and population size (see [here](http://scienceblogs.com/evolgen/2006/07/empty_dna.php), [here](http://scienceblogs.com/evolgen/2006/06/do_they_even_read_the_literatu.php), [here](http://scienceblogs.com/evolgen/2006/09/is_the_y_chromosome_as_flawed_1.php), [here](http://scienceblogs.com/evolgen/2006/10/why_the_empty_population_size.php), [here](http://scienceblogs.com/evolgen/2006/10/another_nail_in_the_coffin_for.php), and [here](http://scienceblogs.com/evolgen/2006/12/filling_the_empty_dna.php)), it’s only appropriate that I link to [this article](http://dx.doi.org/10.1016/j.tig.2007.03.008) by Meiklejohn, Montooth, and Rand on selection on mtDNA. Here’s the abstract:


> 
Several recent studies have confirmed that mitochondrial DNA variation and evolution are not consistent with the neutral theory of molecular evolution and might be inappropriate for estimating effective population sizes. Evidence for the action of both positive and negative selection on mitochondrial genes has been put forward, and the complex genetics of mitochondrial DNA adds to the challenge of resolving this debate. The solution could lie in distinguishing genetic drift from ‘genetic draft’ and in dissecting the physiology of mitochondrial fitness.



This paper leans heavily on the [Bazin et al paper](http://dx.doi.org/10.1126/science.1122033) from last year (that’s one of the “recent studies” that showed no relationship between mtDNA polymorphism and population size) and a [follow-up paper](http://dx.doi.org/10.1126/science.1132585) which showed a relationship between mtDNA polymorphism and population size in mammals. But the authors also present some neat analysis in which they break the Bazin meta-data-set into sub-sets. As a recap, Bazin et al found more evidence for adaptive amino acid changes in invertebrates than vertebrates by comparing [polymorphism and divergence](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php) in amino acid coding sequences from mtDNA. They used this pattern to argue that adaptive evolution of mtDNA sequences erases any relationship between mtDNA polymorphism and population size.





Many of the published mtDNA sequences come from cytochrome oxidase subunit (COX) genes and the cytochrome B (cytB) gene. Meiklejohn et al show that COX genes show more evidence for adaptive evolution in invertebrates than vertebrates, but cytB does not. The heterogeneity in adaptive evolution between mtDNA genes is not remarkable on its own, but it’s important because the vertebrate and invertebrate data sets have nearly inverse frequencies of sequences from these two genes (the vertebrate data set consists of mostly cytB, while the invertebrate data set is mostly COX). Therefore, some of the evidence for adaptive evolution could merely be the result of biased data.


But the most interesting aspect of this paper is how the authors turn Bazin et al’s model of adaptive evolution on its head by hypothesizing that the excess amino acid changes observed in invertebrates are the result of a higher rate of *deleterious* amino acid evolution in the invertebrates. Ironically, this hypothesis is based on the genetic draft model — the same model Bazin et al use when hypothesizing that adaptive fixations in invertebrates decouples mtDNA polymorphism and population size. Meiklejohn et al point out that [Gillespie’s model of genetic draft](http://dx.doi.org/10.1111/j.0014-3820.2001.tb00732.x) predicts that there will be more deleterious fixations in larger populations because there are more linked loci fixed by adaptive evolution in large populations (the deleterious alleles are just along for the ride and get fixed as would any linked neutral variant). Because the test for adaptive evolution merely looks for an excess of amino acid changes, elevated rates of deleterious amino acid evolution and adaptive amino acid evolution will both leave the same signature.


This leads to a paradox: if deleterious amino acid changes are fixing due to draft, what are the loci under adaptive evolution that are dragging those deleterious alleles along for the ride? One possibility is that there are a few mtDNA mutations that are responsible for the sweeps, but Meiklejohn et al hypothesize that cytoplasmic parasites such as [*Wolbachia*](http://en.wikipedia.org/wiki/Wolbachia) may be the cause. By manipulating the reproductive success of their hosts, these symbionts could affect the frequency of different mtDNA haplotypes in a population. Two taxa with high rates of symbiont infections, insects and round worms, also show elevated rates of mtDNA amino acid evolution.


---



Bazin E, Glemin S, Galtier N. 2006. Population size does not influence mitochondrial genetic diversity in animals. Science 312: 570-572. doi:[10.1126/science.1122033](http://dx.doi.org/10.1126/science.1122033)


Gillespie JH. 2001. Is the population size of a species relevant to its evolution? Evolution 55: 2161-2169. doi:[10.1111/j.0014-3820.2001.tb00732.x](http://dx.doi.org/10.1111/j.0014-3820.2001.tb00732.x)


Meiklejohn CD, Montooth KL, Rand DM. 2007. Positive and negative selection on the mitochondrial genome. Trends Genet. 23: 259-263. doi:[10.1016/j.tig.2007.03.008](http://dx.doi.org/10.1016/j.tig.2007.03.008)


Mulligan CJ, Kitchen A, Miyamoto MM. 2006. Comment on “Population size does not influence mitochondrial genetic diversity in animals”. Science 314: 1390. doi:[10.1126/science.1132585](http://dx.doi.org/10.1126/science.1132585)

#The Future of Eukaryote Genome Sequencing
Date: May 28, 2007

A couple of weeks ago I [suggested](http://scienceblogs.com/evolgen/2007/05/the_future_of_genome_sequencin.php) that the National Human Genome Research Institute ([NHGRI](http://www.genome.gov/)) would no longer be funding *de novo* genome sequencing projects via white papers. They appear to be shifting their focus to resequencing projects to study variation (ie, [this](http://genome.gov/10001551)) and take a closer look at well studied organisms (ie, [ENCODE](http://www.genome.gov/10005107), which now has [**Drosophila** and **Caenorhabditis** versions](http://www.genomeweb.com/issues/news/140000-1.html?CMP=OTC-RSS)). But the distribution of genomic resources is extremely biased towards a few species. What should those researchers who work on organisms without genome sequences do if they can’t solicit funds from the NHGRI?





As pointed out in the [comments](http://scienceblogs.com/evolgen/2007/05/the_future_of_genome_sequencin.php#comment-429527) of my previous post, *de novo* sequencing of microbial genomes is still going strong. This research is heavily funded by the Joint Genome Institute ([JGI](http://www.jgi.doe.gov/)), so a change in focus at the NHGRI won’t affect microbiologists. In fact, this is pretty much true for anyone interested in sequencing a small genome (bacterial, archael, or eukaryotic). A moderate research grant from JGI or the National Science Foundation ([NSF](http://www.nsf.gov/)) would fund a *de novo* whole genome sequencing project for many (most?) species (sorry for the bias toward US funding agencies, but those are the ones with which I am familiar). One recent example (from outside the US) is reported in [this paper](http://dx.doi.org/10.1073/pnas.0608218104) from Ken Wolfe’s group in which they analyze gene loss following whole genome duplication in a yeast species that diverged from *S. cerevisiae* soon after a whole genome duplication event. The authors were able to sequence and analyze a new yeast genome (*Kluyveromyces polysporus*) in such a matter of fact way because of the small size of these yeast genomes and the availability of multiple other yeast genomes for comparison. In fact, it’s not obvious from the title of the paper that they are reporting the whole genome sequence of a new species — that’s how ho-hum *de novo* sequencing projects have become in some taxa.


Resequencing projects and moderately funded *de novo* whole genome sequencing projects are only possible for species from well studied taxa or those with small genomes. How should one go about soliciting funds to sequence the genome of a poorly studied species or one with a large genome (presumably due to a large amount of repetitive DNA)? Some may argue that next generation sequencing technologies (eg, [454](http://evolgen.blogspot.com/2005/04/bead-based-resequencing.html) or [Solexa](http://scienceblogs.com/evolgen/2007/03/454_is_so_last_week.php)) will allow for *de novo* sequencing of whole eukaryotic genomes in the near future. I disagree. These new technologies can be used for resequencing of large genomes (for instance, [James Watson](http://www.bio-itworld.com/newsitems/2007/march/03-23-07-watson)) or sequencing genomes from well studied taxa for which there are closely related species with available whole genome sequences to aid in the assembly (the [Neanderthal projects](http://scienceblogs.com/evolgen/2006/11/more_sexy_caveman_news.php) come to mind). But the read lengths generated from 454 and Solexa simply aren’t long enough to go after a large genome from a species for which there is not available sequence from a close relative to aid in assembly.


Given that the funds are not available for *de novo* shotgun sequencing of large genomes via the Sanger technique, and we’re not quite to a point where the new technologies can replace the Sanger method, what can we do to study the genomes of these “non-sequencable” species? My solution: use 454 to sequence [cDNA libraries](http://en.wikipedia.org/wiki/CDNA_library). First of all, this isn’t really my solution, but one that I have been hearing from a few different labs that work on “non-sequencable” species. Second of all, yes, I realize that this is not a great substitute for a whole genome project. A sequenced cDNA library will only provide information about the transcribed sequences in a genome (and the annotation will be biased toward protein coding sequences), but this is far superior to no sequence. Furthermore, funding for a sequenced cDNA library for a single species can be incorporated in a modest grant proposal, and said proposal could even include funds to sequence multiple individuals from a species or multiple closely related species.


With a large fraction of protein coding sequences, a fair bit of analysis can be performed. The scope of a study can be greatly improved if polymorphism data or sequences from closely related species are generated. From what I have heard, the cost of sequencing a cDNA library using 454 is not much more than $10,000, so comparative transcriptomics is quite feasible without much of an investment. Additionally, these sequences would provide a valuable resource for researchers whose study organisms lack good sequences from which to develop molecular markers. A moderate front end investment to sequence an entire cDNA library for a poorly studied organism could yield substantial benefits for future molecular work in that species.


Keep in mind, I do not propose that 454 sequencing of cDNA libraries is the finish line for “non-sequencable” species. Rather, we are currently caught in a holding pattern during which the funding and technology are not available for *de novo* whole genome sequencing of these species. Once the cost and quality of next generation sequencing technologies reach the appropriate levels to allow for sequencing of whole genomes from any and all species, then we can make the switch to those technologies. Until then, sequencing cDNA libraries offers an affordable alternative that lies within the budgetary constraints of many research labs.

#On Being Rated
Date: May 29, 2007

Amongst the other TAs and the lab coordinators in my department, I have a reputation of being a tough grader. At the end of the semester, when the course admins calculate grades, my students invariably get a few points added to their lab scores — this is done to bring lab scores more in line with lecture exam scores. Does that mean I’m a bad teacher who doesn’t explain the material well enough, but grades as if it were explained clearly? Or do I explain the material perfectly fine, but expect too much from my students?





Because I’m such a hard-ass, I often get complaints from my students — either through course evaluations, hearing them gripe about me in class, or from second-hand sources (other students, other TAs, etc). I try hard to teach the students both the material they are responsible for and the manner in which they are expected to present their knowledge to me. And then I evaluate their work in a critical manner with the goal of improving both their understanding of the material and their ability to communicate their knowledge. For this, I get critiques (via [Rate My Professors](http://www.ratemyprofessors.com/index.jsp)) such as:


> 
Honestly sucks. Thinks he knows a lot but really doesn’t. Grades horribly; you will not get away with forgetting to put a comma. I suggest switching if you have him.



What can I glean from this? Well, apparently I don’t know jack shit. This evaluation is from a student in an introductory Ecology and Evolution course. Damn right I don’t know jack shit about ecology. I’m not an ecologists, but I do know more about ecology that the student. And I grade horribly because I correct grammatical errors (I guess I’m supposed to let bad writing go unnoticed in a writing intensive course). I appreciate criticism, but I rarely get anything all that helpful. Maybe I can try to present less of an air of knowledge (or, as one student put it, “Plays favorites. Snobby.”), but this evaluation sounds more like that of a bitter student than someone with a real understanding of what I don’t do well.


My teaching philosophy is one in which I attempt to challenge the students. I don’t like to give away “the right answer”; I prefer the students arrive at the solution via a little work. I like to reward students who put in extra effort — I try to be as helpful as I can if a student comes to office hours — but I do not expect them to go beyond what is covered in class to get an excellent grade. That said, many students don’t understand the material covered in class (often covered multiple times on different days), and they don’t bother to seek help (which is freely available to them) that would greatly improve both their understanding and their grade.


Then there’s the whole issue of whether students are the appropriate evaluators of teaching quality. This is discussed often at [Rate Your Students](http://rateyourstudents.blogspot.com/), and I won’t get into it much here. Thankfully, my teaching performance (evaluated however you chose) does not affect my status as a graduate student — although my poor evaluations may prevent me from earning one of my department’s coveted teaching awards (coveted because of the monetary portion of the award). Tenure track faculty, on the other hand, have their job security determined, in part, by those student evaluations. That means an excellent teacher who grades hard (earning poor evaluations) may be a disadvantage compared to a poor teacher who hands out easy grades (and gets good evaluations). When I got a poor grade in college, I blamed myself for poor study habits. When some of these students get poor grades, they blame their instructor, and the instructor’s career is put in jeopardy.


There are students that appreciate the work their instructors put into teaching. They even seem to accept that they won’t get a good grade unless they put some effort into the assignments they submit. I really like it when I get evaluations like these:


> 
He does a good job. Not too tough, but not too easy either. At least he makes lab semi-fun.
He’s tough, but you can learn a lot from him. He really knows his****.



But I would benefit more from students who could offer thoughtful criticism of my teaching abilities. Even the best students, however, struggle with this. They simply do not understand the material well enough to offer a sophisticated critique. For that, one must turn to peers, but we rarely get those types of evaluations.

#Always Choose “Same”
Date: May 30, 2007

The beginning of many [Ultimate](http://www.upa.org/) (nee, Frisbee) games is marked by flipping discs to decide which team must pull (kick off) and which goal each team will defend at the start of the game. This is sort of like the coin flip before an American Football game. Two players — one from each team — flip a disc in the air. A third player — a representative from one of the teams — calls “same” or “different”, referring to whether both discs land with the same side (top/bottom or heads/tails) facing up or different sides facing up. If he guesses right, his team gets to choose whether they want to pull or receive to start the game or if they would like to choose which end of the field to defend.


From my casual observations, most players tend to pick “same” or “different” quite randomly. That would be all well and good if the probability of each event were equal, but they are probably not. If the probability the disc lands top side up (we’ll call that heads) is equal to *p* and the probability it lands bottom side up (tails) is equal to *q*, then we can calculate the probability of the same and different outcomes of two flips of the disc. The probability both discs land heads up is *p*<sup>2</sup> (assuming the two flips are independent), and the probability that both discs land tails up is *q*<sup>2</sup>. That means the probability of “same” is *p*<sup>2</sup>+*q*<sup>2</sup> because the two outcomes are mutually exclusive. The two discs can land with different sides up if the first disc lands heads up and the second lands tails up (this occurs with probability *pq*) or if the first disc lands tails up and the second lands heads up (with probability *qp*). Because these two events are also mutually exclusive, the probability of “different” is 2*pq*.





Given those two equations, we can calculate the probability of “same” and different” for different probabilities of heads and tails. I have graphed the probability of same and different versus the probability of heads, as shown below.


<img alt="i-24b551dbc11d714a3e0a453514b2eb31-same_different.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-24b551dbc11d714a3e0a453514b2eb31-same_different.gif"/>


As you can see, the probability of “different” is maximized when the probability of heads (and tails) is equal to 0.5. That maximal probability, however, is only 0.5. Therefore, if the probability of heads and tails are equal, you have a 50% chance of winning the flip if you chose “different”. But if the probability of heads and tails are not equal, the probability of “different” drops below 50%. That means there is never a scenario in which choosing “different” is a smart bet (greater than 50% chance of winning), but choosing “same” is a smart bet unless heads and tails are equally likely (and, even then, it’s even money).


So, I say to all ultimate players out there, ****ALWAYS CHOOSE SAME****.


---



**UPDATE:** See the [follow-up post](http://scienceblogs.com/evolgen/2007/05/usually_choose_same.php).

#Usually Choose Same
Date: May 31, 2007

In [yesterday’s post](http://scienceblogs.com/evolgen/2007/05/always_choose_same.php), I argued that, when flipping two unfair discs (or coins), there is a greater chance that both discs will land with the same side up than different sides up. As pointed out in [the comments](http://scienceblogs.com/evolgen/2007/05/always_choose_same.php#comment-449600), I was assuming that the probability of heads is equal for both discs:


> 
Aren’t you assuming that p (and q=1-p) are the same for both discs? But isn’t it more reasonable to assume that, while no disc has a perfect p=0.5 probability of landing ‘heads’, the p’s of no two discs are likely to be the same? (Assume, perhaps, that each disc’s p is drawn independently from some kind of larger distribution, maybe dependent on something like manufacturing or the way the person throwing it holds it, before throwing).



What happens if we allow for different probabilities of heads on the two discs?





<img alt="i-31a3c4c76a858874d27504cb3a13adca-same_different2.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-31a3c4c76a858874d27504cb3a13adca-same_different2.gif"/>


This is a graph of the probability of same for various probabilities of heads for each disc. The probabilities of heads on disc 1 is shown on the X-axis, and the probability of heads on disc 2 are 0.1, 0.3, 0.5, 0.7, and 0.9 (the five different lines). As you can see, the probability of same is greater than 0.5 when the two discs are biased in the same way (i.e., both have a probability of heads greater than 0.5, or both have a probability of heads less than 0.5). On the other hand, the probability of different is greater than 0.5 when the discs are biased in opposite directions (one with a probability of heads greater than 0.5, and one with a probability less than 0.5).


So, I say to all ultimate players out there, CHOOSE SAME if you think the discs have the same biases, and CHOOSE DIFFERENT if you think they are both biased, but in different directions.

#Troubleshooting with the Hive Mind
Date: May 31, 2007

Late last week, my PCRs stopped working. One day I was able to amplify DNA from multiple different templates using different primers, and the next day I couldn’t. This is a major setback for me — a huge chunk of the remaining work I need to complete for my PhD involves doing PCR. If I can’t get my PCRs to work, I’m royally screwed.


<img alt="i-a8b0c1abe4213cc9f81382b38a348d21-gel_blank.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-a8b0c1abe4213cc9f81382b38a348d21-gel_blank.jpg"/>


As soon as I couldn’t get any PCR products, I went into troubleshooting mode. I had just made a new batch of dNTPs, so those were the first thing to get replaced. No dice. I thought there might be something wrong with my water, so I grabbed a new bottle and made another batch of dNTPs. Nothing. I replaced my buffer solution. No luck. I made a new dilution of each primer. Nope. I bought new *Taq* DNA polymerase and new stock solutions of each dNTP, and I remade my dNTP solution again. Nada. I prepped a new dilution of my template DNA (and I had been testing my PCRs using multiple templates that had worked in the past), but that didn’t help. I tried multiple other primers, two different thermal cyclers (I’m currently trying a third machine tonight), different containers of tubes and tips, but none of it helps. When I run my gels, I still get no bands in any of my lanes.


I can pretty much rule out the possibility that I’m getting amplification, but my gels are screwy for two reasons. First of all, my ladder runs perfectly fine in all the tests. Secondly, I reran some PCR products from a couple of weeks ago (back when I was able to perform a successful PCR), and they looked a-ok.


So I’m turning to you guys. Now that I have replaced every reagent, every solution, and my template, what can I do? Is there anything I haven’t thought of (tinkering with Mg<sup>2+</sup> concentration, perhaps?) that will fix this problem? Keep in mind, I went from really nice PCR products to none at all, and my primers are designed to the genome sequence of the species and strain I’m amplifying. What’s a helpless grad student to do?

#Lots of Pretty Wings in PLoS ONE
Date: June 1, 2007

In a [new article in PLoS ONE](http://dx.doi.org/10.1371/journal.pone.0000487), a group of researchers led by Kevin Edwards present a collection of images of Hawaiian *Drosophila* wings. Here’s one figure from the paper showing the evolutionary relationships of a bunch of different clades and some representative wing patterns:


[<img alt="i-3cda87ef6d121aecfc08e66d65424863-drosophila_wings.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3cda87ef6d121aecfc08e66d65424863-drosophila_wings.gif"/>](http://dx.doi.org/10.1371/journal.pone.0000487)


The authors point out that, with the availability of the [**Drosophila grimshawi** genome sequence](http://rana.lbl.gov/drosophila/grimshawi.html), biologists can now use molecular techniques to understand the genetic changes that give rise to the various pigmentation patterns.


---



Edwards KA, Doescher LT, Kaneshiro KY, Yamamoto D. 2007. A database of wing diversity in the Hawaiian **Drosophila**. PLoS ONE 2: e487 doi:[10.1371/journal.pone.0000487](http://dx.doi.org/10.1371/journal.pone.0000487)

#New York Times Cracks
Date: June 3, 2007

Let’s go through the basics again. [Cracking the genetic code](http://genomicron.blogspot.com/2007/05/cracking-code.html) refers to figuring out how DNA encodes the information to make proteins — that was done decades ago. Sequencing a genome does not mean that you have [decoded](http://scienceblogs.com/evolgen/2006/07/decoding_the_1000_genome.php) [the genome](http://scienceblogs.com/evolgen/2006/10/10_million_to_decode_nicholas.php); presumably, decoding a genome would mean you’ve figured out the function of every part of a sequenced genome, but there really isn’t a proper definition. In genetics, [mapping](http://scienceblogs.com/evolgen/2006/10/genome_cartographers.php) refers to determining the location of genetic elements, which is different than sequencing. And [deciphering](http://scienceblogs.com/evolgen/2007/06/will_nicholas_wade_ever_learn_1.php) has no real meaning.


With that said, check out the [newest New York *Times* article](http://www.nytimes.com/2007/06/03/weekinreview/03harm.html?ex=1338523200&amp;en=aa0ccf5c14b312cf&amp;ei=5088&amp;partner=rssnyt&amp;emc=rss) on the sequencing of James Watson’s genome. Here’s the first sentence:


> 
JAMES D. WATSON, who helped crack the DNA code half a century ago, last week became the first person handed the full text of his own DNA on a small computer disk.



Jim Watson did not “crack the DNA code”. Watson helped figure out the structure of DNA and double strand pairing of DNA strands. But it only gets <strike>better</strike> worse:


> 
Soon enough, scientists say, we will all be able to decipher our own genomes — the six billion letters of genetic code containing the complete inventory of the traits we inherited from our parents — for as little as $1,000.



Decipher our own genomes? What the fuck does that mean? Six billion letters of genetic code? The genetic code consists of triplets of nucleotides called codons, which you can view in this handy dandy [codon table](http://en.wikipedia.org/wiki/Genetic_code#RNA_codon_table). Considering that most of the  human genome does not encode protein coding sequences, far less than six billion nucleotides are even involved in the genetic code. And that bit about our genome “containing the complete inventory of the traits we inherited from our parents” — I ain’t touching that with a poorly mixed metaphor.





In the first two paragraphs, we’ve already hit a trifecta: attributing some accomplishment to James Watson that he did not do, mixing up sequencing and decoding, and deciphering the spirit of Nicholas Wade. But that was only the first race of the day. In the showcase, we have Decode, a favorite at 3:2 odds to irk any respectable geneticists. The decode horse comes out of the gates fast and takes a huge lead coming around the first bend:


> 
As thousands of people decode their DNA over the next few years, they are likely to find themselves facing a genetic mirror whose reflection changes on an almost daily basis.



Oh, no, she didn’t? Oh, yes, she did! Everyone is going to be ****<u>decoding</u>**** their genomes in the next few years. I know I want to ****<u>decode</u>**** my genome. Our craptacular author, Amy Harmon, is just throwing around words willy nilly, with no regard for what they really mean. She’s probably been reading the articles by Wade and other journalists with a poor command of terminology. Now, with her chance to shine, she doesn’t want to write a boring article in which she uses the word “sequence” (and its variants) over and over again. So she samples from the works of others — the mistakes they’ve made become her mistakes. The dastardly diction disease is spreading through the science journalism community like my inability to craft clever metaphors.


And if you haven’t had enough (and if you’re not sick of me scratching my pet peeve), here’s Harmon inventing a new false synonym for DNA sequencing:


> 
There are other reasons to unravel your genome. Embracers of nature over nurture may sift through their 20,000 genes to find an explanation for personality traits thought to have a partial genetic basis — like early rising, risk-taking, shyness and addiction.



Yes, my friends, soon you too will be able “unravel your genome” for the low cost of $1,000. I’m guessing they’ll be using [helicase](http://en.wikipedia.org/wiki/Helicase) to do the unraveling. So, let’s add another word to the list of those that make a science writer look ignorant. Stephen, add “unravel” to the big board.


<img alt="i-651474221babca09e3353a59fea4ddab-OnNotice_decode.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-651474221babca09e3353a59fea4ddab-OnNotice_decode.jpg"/>


Thanks Stephen.

#Two Genetics Blog Carnivals for Today
Date: June 3, 2007

There are two genetics blog carnivals available for your perusing today. First is [Gene Genie](http://www.eyeondna.com/2007/06/03/gene-genie-in-mendels-garden/) hosted by Hsien at Eye on DNA. The second is [Mendel’s Garden](http://scienceblogs.com/transcript/2007/06/mendels_garden_15_summer_readi.php), over at The Daily Transcript. [Hsien](http://www.eyeondna.com) will also be hosting the next edition of [Mendel’s Garden](http://mendels-garden.blogspot.com/). Visit the [Mendel’s Garden webpage](http://mendels-garden.blogspot.com/) if you’d like to contribute to the next edition.

#SMBE 2007
Date: June 5, 2007

[<img alt="i-dc7d546879e62b180ce14cd48647fd7a-SMBE07.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-dc7d546879e62b180ce14cd48647fd7a-SMBE07.gif"/>](https://smbe2007.dal.ca/)


This year’s meeting of the [Society for Molecular Biology and Evolution](http://smbe.org/) will take place at Dalhousie University in Halifax, Nova Scotia ([conference webpage](https://smbe2007.dal.ca/)). I’m going, and so are [Reed Cartwright](http://dererumnatura.us/archives/2007/05/smbe_talk_accep.html) and [Jason Stajich](http://fungalgenomes.org/blog/). Additionally, [Rosie Redfield](http://rrresearch.blogspot.com/) and [John Logsdon](http://johnlogsdon.blogspot.com/) have each been invited to give a talk at the conference. As you can see, there will be a blogger presence at the meeting.


Are you going to the conference? If you are (whether you’re a blogger or not), you should join the [SMBE 2007](http://network.nature.com/group/smbe2007) group on the Nature Network. And if you haven’t already signed up for the [Nature Network](http://network.nature.com) (it’s like Facebook for science dorks), set up an account and add yourself to the SMBE 2007 group.

#Tangled Bank #81
Date: June 6, 2007

Matt’s posted the [newest edition of the Tangled Bank](http://matt-at-berkeley.blogspot.com/2007/06/tangled-bank-81.html) (the original science blog carnival) at his Behavioral Ecology Blog. There’s a link to [a post on Mike Lynch’s anti-adaptationist paper in PNAS](http://thinkevolution.net/archives/80) and another to a [post by Pedro’s on protein evolution](http://pbeltrao.blogspot.com/2007/05/protein-evolution-what-constrains-and.html).

#On the Causes of Variation in the Rate of Molecular Evolution
Date: June 16, 2007


Last year, Katie Pollard and colleagues published a couple of papers in which they [identified regions of the human genome](http://dx.doi.org/10.1371/journal.pgen.0020168) that had recently undergone an acceleration in their rate of evolution and [characterized the expression pattern](http://dx.doi.org/10.1038/nature05113) of an RNA gene located in one of those regions. The RNA gene is expressed in the developing brain, which lead people to speculate that it played some important role in making humans smarter than chimps (my round-up and stab at speculation can be seen [here](http://scienceblogs.com/evolgen/2006/08/you_got_a_fast_gene.php)). Their approach toward identifying those regions is quite simple, but the cause of accelerated evolution in those regions is unclear. Are they evolving rapidly because of natural selection fixing beneficial mutations, or is evolution sped up because of higher mutation rates?


Pollard and colleagues started by searching for sequences that are highly conserved (ie, have very few changes) amongst all mammals, excluding humans. They then searched within the highly conserved sequences for those with the fastest rates of evolution along the lineage leading to humans, after the divergence with chimpanzees (see [here](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-4.html) for a review of relative rate tests). They dubbed these rapidly evolving human sequences “human accelerated regions” (HARs). The fastest evolving HAR contained a sequence encoding an RNA that is expressed in the developing neocortex, but not translated into a protein. The authors also attempted to detect signatures of positive selection using polymorphism data (see [here](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.phpl) and [here](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php)), but they did not find much evidence for adaptive evolution from either the publicly available SNP data or their own resequencing data.


In their interpretation of the results, Pollard et al suggest that it’s unclear whether the HARs are rapidly evolving because of natural selection or higher rates of mutation in humans. But in an [opinion piece published in Trends in Genetics](http://dx.doi.org/10.1016/j.tig.2007.03.011), Nicolas Galtier and Laurent Duret argue that variation in mutation rate is a more likely explanation for many of the HARs. Their argument is based upon the types of changes that have occurred within HARs and their location in the genome.





One nice feature of Pollard et al’s study is the fact that they exclude relaxed selective constraint as an explanation for accelerated evolution along the human lineage. They do so by comparing the rates of evolution of HARs against that of four-fold synonymous sites (third positions in codons which do not change the amino acid encoded by the codon), which are assumed to evolve neutrally. If the HARs evolved at a rate equivalent to that of the synonymous sites, then the accelerated evolution would most likely be the result of relaxed selective constraint along the human lineage. But the HARs evolve much faster than the neutral rate, suggesting that the changes along the human lineage were driven by natural selection.


The changes along the human lineage are also biased towards going from adenine (A) and thymine (T) to guanine (G) and cytosine (C). These types of changes are referred to as weak to strong because A and T are held together by two hydrogen bonds, while G and C are held together by three hydrogen bonds. Throughout the majority of a genome, weak to strong and strong to weak changes should occur at approximately equal frequencies. At recombination hotspots, however, weak to strong changes will dominate. That’s because the recombination events are initiated by double strand breaks (DSBs) in DNA, which are then repaired by one of a few DSB repair pathways. If an individual is heterozygous for an A/T and a G/C nucleotide at a site near a DSB, one of the DSB repair pathways is known to preferentially replace the A/T with G/C during the repair event (known as biased gene conversion, or BGC).


That’s the mechanism by which mutational bias would allow for accelerated evolution in one region of a genome but not another. This would not explain accelerated evolution along any particular lineage unless the recombination hotspots differed between species. And they do! Gil McVean and colleagues used [polymorphism data to study recombination](http://dx.doi.org/10.1371/journal.pgen.0020148) in the human genome (reviewed [here](http://scienceblogs.com/evolgen/2006/11/recombination_rate_dna_polymor.php)). They found that recombination hotspots are ephemeral, changing often since the divergence of humans and chimps. If recombination hotspots have changed, and those hotspots induce weak to strong mutations, and HARs are biased for weak to strong mutations, it follows that HARs can be explained by mutational bias.


[Galtier and Duret](http://dx.doi.org/10.1016/j.tig.2007.03.011) use that logic to argue that many of the HARs identified by Pollard et al evolved rapidly because of mutation bias, not natural selection. Pollard et al do acknowledge the possibility that BGC could be responsible for the HARs, but nowhere near as aggressively as Galtier and Duret. Not only do the HARs have an excess of weak to strong substitutions, they are also located near recombination hotspots, further supporting the mutation explanation. Additionally, if natural selection were responsible for the HARs, you would expect the signature of selection spread widely around the region under selection. BGC, on the other hand, will produce a much more localized signature, which is what Pollard et al observed.


If Galtier and Duret’s hypothesis is correct, then these HARs represent an “Achilles’ heel” of the genome — essentially the opposite of the natural selection hypothesis. Rather than being shaped by natural selection, the HARs may be highly mutable regions which must be tolerated despite the fact that they occur in functional regions (remember, they are highly conserved amongst all mammals studied except for humans). If a recombination hotspot were to appear in a functional sequence (or a functional sequence were to migrate into a highly recombining region, like [**Fxy** in mice](http://dx.doi.org/10.1016/S0960-9822(99)80430-8)), you would observe accelerated evolution in that region. Natural selection would need to be even stronger to maintain the function of that sequence, but many of those sequences would rapidly evolve despite being under strong selective constraint.


---



Galtier N and Duret L. 2007. Adaptation or biased gene conversion? Extending the null hypothesis of molecular evolution. Trends Genet 23: 273-277 doi:[10.1016/j.tig.2007.03.011](http://dx.doi.org/10.1016/j.tig.2007.03.011)


Pollard KS, Salama SR, King B, Kern AD, Dreszer T, et al. 2006. Forces shaping the fastest evolving regions in the human genome. PLoS Genet 2: e168 doi:[10.1371/journal.pgen.0020168](http://dx.doi.org/10.1371/journal.pgen.0020168)


Pollard KS, Salama SR, Lambert N, Lambot M-A, Coppens S, et al. 2006. An RNA gene expressed during cortical development evolved rapidly in humans. Nature 443: 167-172 doi:[10.1038/nature05113](http://dx.doi.org/10.1038/nature05113)


Spencer CCA, Deloukas P, Hunt S, Mullikin J, Myers S, et al. 2006. The influence of recombination on human genetic diversity. PloS Genet 2: e148 doi:[10.1371/journal.pgen.0020148](http://dx.doi.org/10.1371/journal.pgen.0020148)

#Drosophila Smell and Taste
Date: June 22, 2007


Sensing and reacting to one’s environment is necessary for survival. Different species have different expertise in regards to how they sense their environment. Humans, for example, have reduced olfactory abilities relative to other mammals, but excellent color vision. Cats have good night vision, but poor vision during day light. These proficiencies and deficiencies in sensory abilities hold for non-mammalian taxa as well.


[Olfaction](http://scienceblogs.com/evolgen/2007/01/odorant_receptor_evolution.php) and taste have been well studied in a variety of taxa. Amongst the invertebrates, the genes responsible for olfaction and taste in *Drosophila* are one of the best studied systems. Much of the recent work mapping genes to functions in the olfactory system has been done in [John Carlson’s lab](http://www.biology.yale.edu/facultystaff/carlson.html), where they have figured out which genes are expressed in which neurons and which odors those neurons detect. Carlson’s work was done in *D. melanogaster* (the common work-horse in molecular biology and genetics), but with the availability of [completely sequenced genomes](http://rana.lbl.gov/drosophila/) from 11 other *Drosophila* species, researchers have begun to study the evolution of olfactory and gustatory genes within the entire genus.





As the genomes of multiple *Drosophila* species became available, multiple labs mined the sequences for olfactory receptor (**Or**) and gustatory receptor (**Gr**, or taste) genes. Two papers from two different research groups were published earlier this year reporting on the evolution of *Drosophila* **Or** genes ([here](http://dx.doi.org/10.1093/molbev/msm038) and [here](http://www.pnas.org/cgi/content/abstract/104/17/7122)). Additionally, at least two other groups have performed their own computational analysis of *Or* genes in *Drosophila* genomes (based on titles I’ve seen for talks and posters that will be presented at the [SMBE meeting](https://smbe2007.dal.ca/)). The *Drosophila* *Or* repertoires evolved independently from the *Or* and *Gr* genes found in mammals. These studies found that the sizes of the gene family has remained relatively constant, despite frequent duplication and loss of Or genes. The evolution of *Or* genes appears to fit a [birth-and-death](http://www.pnas.org/cgi/content/abstract/94/15/7799) model for gene family evolution.


Mining genomes for your favorite gene family is expected as genomes become available. While these studies are important in that they contribute necessary annotations to these genomes, the science is fairly mundane. Also, there’s something a bit questionable about publishing a paper in which all of the research reported is based on data that are publicly available (the genome assemblies), but the people who generated the data have not yet published their paper. In a way, they violated an unwritten moratorium in the genomics community. The labs who published papers on the *Or* genes in the 12 *Drosophila* genomes should have waited for the genome papers to be published before publishing their own analyses of those sequences. Or, better yet, they should have joined the consortium of labs analyzing the 12 genomes and published their papers along with the genome papers from the consortium.


There is also some really interesting research being done on *Drosophila* *Or* and *Gr* genes. These studies explicitly address questions that get at the biology of the organisms being studied. In one study, [Lindy McBride compared the *Or* and *Gr* repertoires](http://dx.doi.org/10.1073/pnas.0608424104) of the sister species *D. simulans* and *D. sechellia*. *D. simulans* is a generalist when it comes to host plant preference, whereas *D. sechellia* only feeds on the fruits of *[Morinda citrifolia](http://en.wikipedia.org/wiki/Morinda_citrifolia)* — a plant that produces toxins that all other drosophilids find objectionable. McBride found that *D. sechellia* is losing *Or* and *Gr* genes much faster than *D. simulans*, and the rate of protein evolution in these genes is higher in *D. sechellia* than *D. simulans*.


The changes in rates of evolution along the *D. sechellia* lineage could be the result of positive selection or relaxed selective constraint. Ideally, one would use population genetic data to differentiate between the two, but *D. sechellia* has very few polymorphisms segregating in natural populations, limiting the power of the applicable analyses. Instead of using polymorphism data, McBride analyze where the substitutions were occurring within the genes. She found that *D. sechellia* *Or* genes have changes in the same parts of their proteins as *D. simulans* and *D. melanogaster* (the outgroup species) *Or* genes, but *D. sechellia* *Gr* genes do not. She uses this observation to argue that adaptive evolution has driven the accelerated evolution of *D. sechellia* *Or* genes (they are still functionally constrained because the changes can’t happen anywhere in the protein), whereas relaxed selective constraint is responsible for accelerated evolution of the *Gr* genes.


And if you’re more interested in the function of the genes responsible for the host shift in *D. sechellia*, [this paper](http://dx.doi.org/10.1371/journal.pbio.0050118) might tickle your fancy. Takashi Matsuo and colleagues used *D. melanogaster* lines with knocked-out odorant binding protein genes (which are different from *Or* genes) to identify genes involved in the tolerance to and preference for the Morinda plant. They showed that one gene is differentially expressed in *D. sechellia* compared to its close relatives, and knock-outs of that gene (along with another gene) allow *D. melanogaster* to tolerate some of the toxic volatiles produced by the Morinda fruit.


Interestingly, *D. sechellia* diverged from *D. simulans* approximately 1/2 million years ago, but the Morinda plant that *D. sechellia* feeds on was introduced to the Seychelles islands (home to *D. sechellia*) by humans (within the last few thousand years). That means the speciation event between *D. simulans* and *D. sechellia* was not driven by host adaptation of *D. sechellia* to the Morinda plant on the Seychelles. I can think of three scenarios for the host shift observed in *D. sechellia*:


1. *D. sechellia* was adapted to Morinda (or some other plant that produces similar toxins) on mainland Africa (home to the native range of *D. simulans* and presumably the native range of the ancestor of the two species) and then colonized the Seychelles upon the introduction to the plant.
1. *D. sechellia* and 
#On the Road to Halifax
Date: June 23, 2007

When this post appears, I’ll be on the road to Halifax for the [SMBE meeting](http://scienceblogs.com/evolgen/2007/06/smbe_2007.php). That’s right — on the road. As in an 18 hour drive. I’m not doing it alone, mind you. It’s me and ten other grad students and post-docs packed into two vans. And we’re not doing the whole thing in one stretch — not everyone is hardcore enough for that — so we’re spending Saturday night in Bangor, Maine. I guess that’s probably for the best because it would totally suck if we’re already burnt out by the first day of the meeting.


Anyway, for those of you going to the meeting, I’ll see you in Halifax. We’re planning a bloggers’ dinner on Tuesday night (see [here](http://network.nature.com/forums/smbe2007/181)), but non-bloggers are welcome to come along (if you read the blog, you’re blogger enough). I’ll be giving a talk on Tuesday afternoon (in the session on Drosophila Genome Biology) entitled, “Evolutionary fates of duplicated genes: measuring the amount of selective constraint on diverging paralogs in the **Drosophila pseudoobscura** genome.” I was hoping to win the contest for Longest Title Involving a Colon, but someone in my session managed to out-do me.

#Muller’s Wheel
Date: June 27, 2007

H.J. Muller is famous for (among other things) his argument for the evolution of recombination involving the purging of deleterious alleles (dubbed [Muller’s Ratchet](http://en.wikipedia.org/wiki/Muller's_ratchet)). In a nutshell, Muller observed that, in the absence of recombination, deleterious mutations will fix in populations because every chromosome will, eventually, obtain a mutation which decreases the fitness of the organism. Recombination allows for the movement of deleterious mutations off of chromosomes, which decreases the genetic load in a population (because fewer deleterious mutations will fix).


Much of the recent work in population genetics centers on the effect of recombination rate on the efficiency of natural selection. Regions of low recombination have lower effective population sizes; because natural selection is most effective in regions of higher recombination, more deleterious mutations and fewer advantageous mutations are expected to fix in regions of low recombination. (Please note that there is [evidence suggesting](http://dx.doi.org/10.1186/gb-2007-8-2-r18) that all that is required for selection to work is some recombination — and selection works equally well in regions of low and high recombination — but what follows can be best viewed as rampant speculation, so I will conveniently ignore the aforementioned reference.) We also know that selective sweeps greatly reduce neutral polymorphism around the site under selection (decreasing effective population size, as well).


So, given that natural selection is most effective in regions of high recombination (ie, large effective population size), and natural selection reduces effective population size, I present a new “tool”: Muller’s Wheel. Here’s my logic:


- Natural selection is most effective in regions of the genome with large effective population size (high recombination rate).
- Selective sweeps (which are fairly common if you believe [some](http://dx.doi.org/10.1073/pnas.0701572104) [people](http://dx.doi.org/10.1038/nature04107)) will reduce the effective population size surrounding a region under selection.
- With a reduced effective population size, natural selection will be less efficient, thereby allowing recombination to increase the effective population size.
- The large effective population size allows for more efficiency of selection, which leads to a selective sweep, which decreases effective population size, which decreases the efficiency of selection, etc., etc.


I need something to drink.

#Back from Halifax
Date: June 29, 2007

After driving through the night, we got home [from Halifax](http://scienceblogs.com/evolgen/2007/06/on_the_road_to_halifax.php) this morning. The [SMBE meeting](http://scienceblogs.com/evolgen/2007/06/smbe_2007.php) was excellent as usual, and I agree with Dan Hartl that it’s the best meeting for evolutionary genetics. With excellent talks and posters on topics ranging from population genetics to comparative genomics (and many people discussing both and everything in between in a single 15 minute talk), it’s hard to find a better meeting.


I also had the opportunity to meet [Rosie Redfield](http://rrresearch.blogspot.com/), [John Logsdon](http://johnlogsdon.blogspot.com/), [Jason Stajich](http://fungalgenomes.org/blog/), [Reed Cartwright](http://dererumnatura.us/), and [Jacob Tennessen](http://www.salamander-candy.com/). We got together [for dinne](http://www.pandasthumb.org/archives/2007/06/smbe_2007_scien.html)r, along with Professor Steve Steve, to discuss blogging and open science. It turns out that I also have an academic connection with a bunch of these bloggers — Jason and I graduated from the same college in the same year, Reed’s PhD advisor was my advisor’s advisor, and Rosie did her post-doc in the same lab as my advisor. And I was surprised that I hadn’t met Jason before, given that we share overlapping interests (he was a co-author on the [human-chimp copy-number divergence paper](http://scienceblogs.com/evolgen/2006/12/the_6_rule.php) from Matt Hahn’s lab).


Finally, it was announced that John’s school, the [University of Iowa](http://www.uiowa.edu/), will be hosting the meeting in 2009. Next year’s meeting will be in Barcelona — the combination of cost and the timing of the meeting will probably keep me from going — but I’ll most likely be in Iowa City the following year.

#Eight Things to Hate about Me
Date: July 8, 2007

I’m fucking sick of blog memes. Not only do I find online surveys totally lame, I also never get tagged. Boohoo, nobody likes me. Now, [John Logsdon orders me](http://johnlogsdon.blogspot.com/) to tell you eight things about me. I’m only doing this ’cause we had [dinner together](http://www.pandasthumb.org/archives/2007/06/smbe_2007_scien.html) last week. Here are the rules:


1. We have to post these rules before we give you the facts.
1. Players start with eight random facts/habits about themselves.
1. People who are tagged need to write their own blog about their eight things and post these rules.
1. At the end of your blog, you need to choose eight people to get tagged and list their names.
1. Don’t forget to leave them a comment telling them they’re tagged, and to read your blog.


The eight things can be found below the fold, but I’ll put my taggees right here:


- [Kevin](http://other95.blogspot.com/)
- Both of the [Biocuriosities](http://biocurious.com/) (that counts as two)
- [Jacob](http://www.salamander-candy.com/)
- [I’m a chordata](http://convoluta.ucdavis.edu/research/)





**Eight things to hate about me:**


**Where I’m from:** I was born and raised in the Los Angeles area. I actually drove by the set of Baywatch on my way to high school every morning. I fucking hate that city and will never live there again.


**I was an aspiring child star:** I got my big break when I worked my stuff on the cat walk for a toddler fashion show; I looked dashing in my little sailor suit. I then auditioned for an Oscar Meyer commercial and got a call back. But, during my second audition, I spotted my mom in the audience and burst into tears. Needless to say, I didn’t get the part, and my career was in ruins before I entered kindergarten.


**I’m a Lakers fan:** Being born and raised in Los Angeles, I grew up a fan of the Dodgers, Kings, Raiders (they hadn’t moved back to Oakland yet when I formed my allegiances), and Lakers. I remember jumping on my parents’ bed when the Dodgers won the World Series in 1988, and I went to the Stanley Cup finals the one year the Kings made it (I also got to see Gretsky break the all time scoring record). But the best year was the 1987-88 NBA season when the Lakers won their second championship in a row. I went to every home game that season (regular season and playoffs) and ran on the court when they won game seven (back when they still let you do that). Oh, I also made a sign that I took into all the games during the championship series against Detroit that read “Pistons Piss Tons,” which my seven year old sensibilities found downright hilarious.


**Soon, they’ll convert my dead relatives:** I was a voice actor for a series of Mormon audio tapes. Some Mormon dude called the director of my high school’s theater program asking for a student to play the role of a kid in some tapes he was making for the Church of the Magic Decoder Glasses.


**I’m tone deaf:** Despite the fact that I’ve taken lessons on three different instruments (piano, saxomophone, and guitar if you’re scoring at home), I can’t carry a tune — and even if I could carry a tune, I wouldn’t be able to find it.  In spite of my shortcomings, I managed to score a roll in a musical while in high school — a singing role, no less (no solos, thank god). I mean, I’m helpless when I comes to telling if an instrument is tuned properly, and I couldn’t identify a key change if [something clever] [something else clever] locksmith.


**I’m bad with metaphors:** I couldn’t construct a clever turn of phrase if . . . well, there you have it.


**My academic pedigree:** Continuing the trend set by John and [Jonathan](http://ttaxus.blogspot.com/), I’d like share with you some of the famous academics in my lineage. Neither my undergrad advisor nor my grad advisor are members of the National Academy, but my grad advisor did his PhD with an academy member and his post-doc with another academy member. Both of those gentlemen were students of Dobzhanksy.


**Cleaning up loose ends:** Before he passed away last year, [Ed Novitski](http://dx.doi.org/10.1534/genetics.104.65953) sent some emails to various *Drosophila* geneticists, including Dick Lewontin, regarding some experiments by Dobzhansky and colleagues. Lewontin forwarded the emails to my advisor, who passed them along to me. I was getting ready to do some experiments somewhat related to the ones required to examine the hypothesis Novitski proposed to explain Dobzhanksy’s data. I did the experiments and found no support for Novitski’s hypothesis. Sadly, we did not publish the results until after Novitski passed away.

#Evo-Devo Revisited
Date: July 9, 2007

I thought I had come to grips with [evo-devo](http://scienceblogs.com/evolgen/2006/05/the_evolution_of_my_thinking_a_1.php). Then along come [Hopi Hoekstra and Jerry Coyne to call shenanigans](http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x) on Sean Carroll’s model of evo-devo. This is [nothing new](http://evolgen.blogspot.com/2005/06/coyne-versus-carroll.html) for Coyne, but I can’t recall Hoekstra ever getting involved in the debate before now.


Before we get to Hoekstra and Coyne, let’s allow Carroll to describe evo-devo in his own words. His most recent summary of the evolution of transcriptional regulatory regions can be found in [this paper](http://dx.doi.org/10.1073/pnas.0700488104) from a recent National Academies Sackler Colloquium. Here’s how he and his colleagues explain the importance of *cis* regulatory regions (CREs) in morphological evolution:


> 
[R]ecent studies have provided direct evidence of the role of CRE evolution in morphological evolution. More importantly, these detailed functional analyses have revealed some surprising and previously unanticipated features of how gene regulation evolves at the molecular level that, we suggest, reflect general principles. The goal of this article is to articulate these emerging principles, namely how regulatory evolution: (i) proceeds using available preexisting genetic components, (ii) introduces discrete changes in gene expression thus minimizing deleterious effects and fitness penalties, and (iii) allows the association between any transcription factor and any downstream gene and thereby provides immense potential for evolutionary novelty. These principles explain both how and why regulatory sequence evolution is a pervasive, although not the exclusive, mechanism underlying morphological diversification.



So, Carroll thinks that CREs are important for morphological adaptation/diversification/evolution, but he does allow for some protein sequence evolution. Hoekstra and Coyne, on the other hand, argue that Carroll’s claims lack evidence. In their article, they say that we currently lack the evidence to decide whether CREs or proteins are responsible for morphological evolution, but the current state of affairs suggest that protein coding sequences are more important.





In making their argument, Hoekstra and Coyne redefine regulatory and structural mutations from how Carroll and colleagues use the terms. Hoekstra and Coyne argue that trying to differentiate between regulatory and structural protein coding genes can get tricky, so they treat all mutations in protein coding regions as structural and all non-protein coding mutations as regulatory. From here, their goal is to determine whether published results containing more evidence supporting regulatory or structural mutations as the main drivers of morphological evolution. Keep in mind that they are partitioning the data in a slightly different manner than Carroll and colleagues.


Before Hoekstra and Coyne evaluate the evidence for and against regulatory changes driving morphological evolution, they question why morphological evolution should be any different than the evolution of other phenotypes. Carroll’s theory of evo-devo concerns only the evolution of form, not “behavior, biochemistry, metabolism, and physiology”. Why not? Here’s what Hoekstra and Coyne think:


> 
The reason, then, why the evolution of anatomy is a “more open” question than that of physiology is not because there is some fundamental biological difference between the two classes of traits. It is only because we have less evidence about the nature of change affecting form, and therefore are less constrained by facts in speculating about its genetic basis.



Ouch! They also point out that the fossil record allows us to evaluate form, while other aspects of the phenotype are more difficult to examine in extinct organisms. But there is no real reason to say that regulatory changes are responsible for one aspect of phenotypic evolution, while showing no preference for regulatory changes for other aspects of phenotypic evolution.


Another aspect of Carroll et al’s argument in favor of *cis* regulatory changes is that they are the only way to escape the pleiotropic effects of most mutations. Because structural mutations will affect all substrates upon which a protein acts, and *cis* regulatory changes only affect the expression of the gene  in question, *cis* regulatory mutations should be more tolerable. Carroll and colleagues claim that gene duplication does not make up for the pleiotropic effects of structural mutations because gene duplications are too rare. They base this claim, however, on a misinterpretation of [Lynch and Conery’s result](http://dx.doi.org/10.1126/science.290.5494.1151) that there is approximately one duplication per gene per 100 million years. While this suggests that duplication is rare on a per gene basis, the estimate translates to over 100 duplications per genome per million years.


But all that is just idle hypothesis. What do the data reveal about the importance of *cis* regulatory changes in the evolution of form? First of all, there aren’t any thorough studies of the genetic causes of the evolution of form. So we must turn to studies that address parts of the big question; one part is the role of *cis* and *trans* changes in the evolution of gene expression. And what studies do Hoekstra and Coyne turn to? Oh, just the same three papers I discussed last year in [my post](http://scienceblogs.com/evolgen/2006/05/the_evolution_of_my_thinking_a_1.php) on the evolution of gene expression ([1](http://dx.doi.org/10.1038/nature04107), [2](http://dx.doi.org/10.1038/nature04240), [3](http://dx.doi.org/10.1038/nature02698)). I won’t get into the details of Hoekstra and Coyne’s argument because you can just read my post (in fact, this section of their article reads like they did just that).


What other data support the Hoekstra and Coyne’s hypothesis that Carroll is being premature in concluding *cis* regulatory changes drive morphological evolution? Hoekstra and Coyne argue that studies of the morphological effects of mutations in single loci indicate no preference for regulatory versus structural mutations. Also, many studies of phenotypic evolution reveal the importance of protein coding sequence changes. And, finally, no Coyne paper would be complete without a treatment of speciation. He and Hoekstra point out that all known “speciation genes” are mutations in protein coding sequences, although it’s hard to see how this is related to the evolution of form.


In the end, Hoekstra and Coyne are not saying that *cis* regulatory changes aren’t important for the evolution of form, only that Carroll is being far too premature in arguing that they are the predominant mechanism.  Additionally, a survey of the literature reveals that structural changes may be more important. This is the same argument that Coyne made in his [critique of Carroll’s book](http://evolgen.blogspot.com/2005/06/coyne-versus-carroll.html), expanded into a complete article. And, guess what, they have a valid a point. The research by <strike>Coyne</strike> Carroll and colleagues is shedding light on the morphological implications of regulatory evolution, but it’s not appropriate to play up that research as if that’s the only way organisms evolve in form.


**Related:** [John Timmer](http://arstechnica.com/journals/science.ars/2007/05/29/what-we-know-about-the-evolution-of-gene-regulation) comments. Also, see [this article](http://dx.doi.org/10.1038/sj.hdy.6801000) on natural selection and the evolution of gene expression and [this article](http://dx.doi.org/10.1126/science.1140247) on the evolvability of gene expression.


---



Hoekstra HE and Coyne JA. 2007 The locus of evolution: evo devo and the genetics of adaptation. Evolution 61:995-1016 doi:[10.1111/j.1558-5646.2007.00105.x](http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x)


Prud’homme B, Gompel N, Carroll SB. 2007. Emerging principles of regulatory evolution. Proc Natl Acad Sci USA 104:8605-8612 doi:[10.1073/pnas.0700488104](http://dx.doi.org/10.1073/pnas.0700488104)

#I’ve got Math and CS Envy
Date: July 10, 2007

Dave wants to know [whether we biologists have physics envy](http://scienceblogs.com/worldsfair/2007/07/do_biologists_have_physics_env.php), as the physicists often claim. I’m quite happy being a biologist and I wouldn’t want to study physics, but there are certain skill sets I wish I had. My envy is not for the questions other fields address, but for the tools other people in my field have at their disposal.


Enough with the foreplay; here are my answers to Dave’s three questions:


<li>
What’s your current scientific specialty?
Evolutionary genetics, in general. Specifically, the molecular biology and evolutionary dynamics of genomes.
</li>
<li>
Were you originally pursuing a different academic course? If so, what was it?
Ever since I arrived at college, I have been a biologist. I’ve been dead set on evolutionary genetics since my second year of undergrad.
</li>
<li>
Do you happen to wish you were involved in another scientific field? If so, what one?
I’m peachy keen with the field I’m in, but I’m envious of all the kids who can write good code and solve partial differential equations.
</li>

#Looking for Darwin with a Bad Pair of Eyes
Date: July 11, 2007

**Bad tests for natural selection are bad at detecting selection.**



Austin Hughes has published a [fairly critical review](http://dx.doi.org/10.1038/sj.hdy.6801031) of some methods used to detect natural selection in protein coding sequences. His attack on current methods for detecting natural selection is threefold. First, he claims that comparing non-synonymous to synonymous substitutions (see [here](http://evolgen.blogspot.com/2005/12/detecting-natural-selection-part-3.html)) does not allow one to differentiate between adaptive evolution and relaxed selective constraint. Second, he argues that comparing polymorphism and divergence of synonymous and non-synonymous sites (see [here](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php)) does not allow one to differentiate between adaptive and demographic explanations for departures from neutrality. And, third, he declares that both of these approaches are flawed because they assume that selection will act on protein coding sequences by fixing multiple amino acid substitutions since the divergence of the two sequences being compared.





I don’t think you’ll find disagreement from anyone that comparing non-synonymous and synonymous substitutions per site (dN and dS, respectively) is a pretty poor way to detect selection. I’d argue that it’s poor not because it cannot be used to tell the difference between positive selection and relaxed constraint, but because it’s got such low power. Technically, dN&gt;dS is evidence for natural selection, but there are other implementations of the test which compare dN/dS across multiple branches of a phylogeny; the latter approaches would yield false positives if relaxed selective constraint leads to an elevated dN along a particular lineage. And Hughes has a valid point that this test (along with some others) also assume that natural selection will fix multiple amino acid changes — a violation of this assumption makes the test even less powerful.


Not only does Hughes criticize the conservative dN/dS as a threat to yield too many false positives, some of his criticisms of the McDonald-Kreitman (MK) test (which compares dN and dS with synonymous and non-synonymous polymorphisms) don’t lead to the conclusions he’d like for you to believe. I’ll discuss some of those criticisms in a subsequent post, but I’d like to include one of them here. The MK test measures within species polymorphism by counting the number of nucleotide sites that vary within the sample. This does not present a complete picture of nucleotide polymorphism; it’s common to also measure the average differences between all pairs of sequences. Hughes correctly points out that deleterious mutations may be segregating as rare polymorphisms, which would elevate the amount of non-synonymous polymorphism in the data. Rather than leading to incorrect inferences of natural selection, this would actually make the test more conservative because it would take an even greater excess of non-synonymous differences between species to reject the null hypothesis and infer natural selection.


The Achilles heel of Hughes’s article, however, is that he attacks only a subset of the approaches used to detect natural selection. The article does not mention any tests that use polymorphism data, other than the McDonald-Kreitman test. Analyses that look at the site frequency spectrum of DNA sequence polymorphism (see [here](http://scienceblogs.com/evolgen/2006/01/detecting_natural_selection_pa.php)) or [haplotype blocks](http://dx.doi.org/10.1371/journal.pbio.0040072) are able to detect recent selection events even if only a single nucleotide is under selection. By excluding a large swath of tests from his article, Hughes allows himself to attack a straw man of current approaches toward detecting natural selection. Ironically, he devotes a sizable chunk of his article toward defending Kimura’s neutral theory against historical attacks that were based on a misunderstanding of the model.


Hughes concludes that codon based approaches toward detecting selection on DNA sequences are flawed and that we must use new techniques to detect natural selection in non-coding regions. He also includes a fair bit of text defending the importance of transcriptional regulatory regions in adaptive evolution (this part actually offers some solid criticisms of the Hoekstra and Coyne article reviewed [here](http://scienceblogs.com/evolgen/2007/07/evodevo_revisited_1.php)). But if he had performed an adequate survey of techniques that use polymorphism data to detect natural selection, he may have realized that they have the power to identify selection in non-coding regions. Even though it would be cool to be able to use gene expression data to detect natural selection, we’re still missing the appropriate algorithms for such an analysis.


---



Hughes AL. 2007. Looking for Darwin in all the wrong places: the misguided quest for positive selection at the nucleotide sequence level. Heredity *In press* doi:[10.1038/sj.hdy.6801031](http://dx.doi.org/10.1038/sj.hdy.6801031)

#Evolutionary Biology and Human Health
Date: July 12, 2007

The American Institute of Biological Sciences has [posted talks](http://live.blueskybroadcast.com/bsb/client/CL_DEFAULT.asp?Client=26&amp;PCAT=335&amp;CAT=335) from their meeting on Evolutionary Biology and Human Health. Not only have they provided audio and video of the talks, but there are also transcripts and slides that go along with the talks. Very cool.

#Junk on Junk
Date: July 15, 2007

I’ve recently come across two articles on junk DNA. The first one, from **New Scientist**, includes a pretty thorough coverage of recent studies that have identified functions for non-protein-coding regions of the human genome ([“Why ‘junk DNA’ may be useful after all”](http://www.newscientist.com/channel/life/mg19526121.500-why-junk-dna-may-be-useful-after-all.html)). The article is set up as if it will present the demise of junk DNA, but it paints the accurate picture that a large portion of the human genome is non-functional. And [TR Gregory like the article](http://genomicron.blogspot.com/2007/07/new-scientist-gets-it-right.html) too.


The second article tries to do that, but fails. It’s from *The Scientist* and entitled [Junk Worth Keeping: Is it time to retire provocative descriptors such as “junk DNA”?](http://www.the-scientist.com/article/home/53300/). This editorial starts out by presenting the evidence for the demise of junk DNA, including a nod to Francis Collins’s belief that there is no junk in the human genome (providing further evidence that [Frank knows nothing about evolution](http://scienceblogs.com/evolgen/2006/06/francis_collins_opens_mouth_in.php)). From here, the author attempts to defend the term “junk DNA”, but does so in a very poor manner. While I agree with his conclusion (at least I think I do — it’s hard to parse from stuff on framing and intelligent design), the entire article is just a mess suggesting the author really doesn’t understand what he’s writing about.

#The Frailty of Nearly Neutral Hypotheses
Date: July 16, 2007


Mike Lynch has been getting a fair bit of hype recently for his nearly neutral model of genome evolution (see [here](http://dx.doi.org/10.1093/molbev/msj050) and [here](http://dx.doi.org/10.1073/pnas.0702207104)). The nearly neutral theory riffs off the idea that the ability of natural selection to purge deleterious mutations and fix advantageous mutations depends on the effective population size of the population in which the mutations arise. From here, the nearly neutral theory predicts that more slightly deleterious mutations and fewer slightly advantageous mutations will fix in small populations compared to large populations (see [here](http://scienceblogs.com/evolgen/2006/02/more_on_misconceptions.php) and [here](http://scienceblogs.com/evolgen/2007/03/neutral_expectations.php) for previous posts on this topic).


Lynch argues that the evolution of genomes can be best understood in a nearly neutral framework rather than a purely neutral or adaptive one (see [here](http://dx.doi.org/10.1038/ng0707-811) for a review by Dan Hartl). He claims that features of eukaryotic genomes (ie, introns, transcriptional regulatory regions, and large genomes in general) are slightly deleterious because they increase the size of mutational targets which can lead to a decrease in fitness if they get hit with a mutation. These features evolved (despite their fitness costs) because of small populations in eukaryotes, relative to prokaryotes, which allowed for the fixation of these slightly deleterious features. Furthermore, microbial eukaryotes tend to have larger population sizes than multicellular eukaryotes, and microbial eukaryotes have more streamlined genomes, on average, than multicellular ones.


While Lynch’s model deals with trends observed over multiple comparisons, some people are using nearly neutral models to explain the evolution of individual genomic features. I’ll take a look at how nearly neutral explanations are invoked in the evolution of repetitive sequences in the human genome ([Gherman et al 2007](http://dx.doi.org/10.1371/journal.pgen.0030119.eor)) and the evolution of protein coding sequences (the [Hughes paper](http://dx.doi.org/10.1038/sj.hdy.6801031) I discussed [previously](http://scienceblogs.com/evolgen/2007/07/looking_for_darwin_with_a_bad.php)). I question the appropriateness of the demographic explanations proposed by these authors and present alternative explanations that are equivalent models.





**Demographic versus Molecular Explanations of Transposable Element Distributions**


Human genomes are loaded with junk — DNA sequences with no function. A large fraction of that junk is made up of transposable elements (many of which spread throughout genomes by getting transcribed into RNA, then reverse transcribed back into DNA and inserted in another spot in the genome) and processed pseudogenes (transcribed genes which have been reverse transcribed and inserted into the genome). A smaller fraction of the junk consists of fragments of the mitochondrial genome that have inserted themselves into the nuclear genome (these are known as numts).


A [paper in the pipeline at PLoS Genetics](http://dx.doi.org/10.1371/journal.pgen.0030119.eor) reports on the temporal distribution of numt insertion events (see [here](http://www.eurekalert.org/pub_releases/2007-07/jhmi-neh070907.php) for a press release and [here](http://genomicron.blogspot.com/2007/07/should-scientists-nit-pick.html) for TR Gregory’s smack down of the crappy coverage). The authors find that a disproportionate amount of numts in the human genome were inserted at about 54 million years ago. This date corresponds to both the approximate divergence time of new world monkeys from old world monkeys and apes (see [here](http://www.tolweb.org/Primates/15963) for a phylogeny) and to the [Paleocene-Eocene boundary](http://en.wikipedia.org/wiki/Paleocene-Eocene_Thermal_Maximum). Previous research showed that there was also a burst of transposable element insertions (specifically [**Alu** elements](http://en.wikipedia.org/wiki/Alu_sequence)) around this same time.


Both numts and transposable elements, for the most part, range in fitness costs to the host organism from neutral to deleterious. Recall that the nearly neutral theory predicts that slightly deleterious mutations have a higher probability of fixation in small populations. The authors of the paper, led by Nicholas Katsanis, conclude that, because independent non-functional sequences experienced a simultaneous increase in the rate of insertion, the ancestral populations along the lineage to humans went through a population bottleneck during this time. That decrease in population size allowed for a relaxation of selection against these insertions.


This nearly neutral explanation for the current genomic distribution of *Alu*s and numts makes a nice just-so story, but it suffers from the same limitations as adaptationist stories. First of all, what makes the bottleneck along the catarrhine lineage (apes and old-world monkeys) different from other bottlenecks — allowing for the accumulation in this bottleneck but not others? Does it have something to do with the Paleocene-Eocene boundary? If so, did other lineages experience the same burst in the accumulation of transposable elements and other junk?


It seems like a purely molecular explanation is just as plausible as the nearly neutral one. It is known that organisms evolve mechanisms to suppress the activity of transposable elements. This differs from purging transposon insertions via natural selection; natural selection involves a decrease in fitness of individuals carrying insertions, which means they don’t pass them on to subsequent generations. The molecular mechanism is independent of natural selection — transposable elements are simply kept from replicating themselves throughout genomes. It’s hard to deny that there was a burst of numt and *Alu* insertions in the human lineage around 54 million years ago, but the demographic explanation is as much of a spandrel as adaptive hypotheses for the evolution of of this genomic feature.


**Nearly Neutral Explanations of the McDonald-Kreitman Test**


The second paper was discussed [here](http://scienceblogs.com/evolgen/2007/07/looking_for_darwin_with_a_bad.php) a couple of days ago. Briefly, in [this paper](http://dx.doi.org/10.1038/sj.hdy.6801031) Austin Hughes argues that current approaches toward detecting natural selection in DNA sequences are flawed. My previous critique pointed out that Hughes neglects a large subset of methods used to detect evidence for natural selection, focusing on only two approaches. Here, I will point out that his criticism of one of those approaches contains faulty reasoning regarding nearly neutral expectations.


Hughes criticizes the McDonald-Kreitman (MK) test for its inability to distinguish between adaptive and demographic explanations for departures from neutrality. This test compares polymorphism and divergence at synonymous and non-synonymous sites using a 2×2 [contingency table](http://en.wikipedia.org/wiki/Contingency_table) (reviewed [here](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php)). An excess of non-synonymous differences between species, relative to non-synonymous polymorphisms within one species, can be explained by invoking either adaptive or nearly neutral mechanisms. The common interpretation is that excess non-synonymous differences were fixed by selection (this is the adaptive explanation). Alternatively, if there was a population bottleneck at some point in a species’ evolutionary history, slightly deleterious non-synonymous differences may have been fixed, which would also result in an excess of non-synonymous differences between the species.


Hughes points out that MK tests performed on data from *D. melanogaster* identifies more amino acid changes fixed by selection than tests performed on human data. He offers the following nearly neutral explanation:


> 
The high rate of ‘positive selection’ detected by the MK test in Drosophila can be explained by fixation of slightly deleterious mutations during a bottleneck in the process of speciation (Ohta, 1993). The level of nucleotide diversity in **D. melanogaster** is at least five times as great as that in the human species, indicating a much larger long-term effective population size in the former than in the latter (Li and Sadler, 1991). With an origin in Sub-Saharan Africa, this species was largely unaffected by Pleistocene glaciation, a major cause of bottlenecks in species of the North Temperate zones (Hughes and Hughes, 2007). Given a large effective population size for a long time, the nearly neutral theory predicts that slightly deleterious mutations will have a good chance of being purged by purifying selection. Thus, the highly effective purifying selection within **D. melanogaster**, by lowering Pn [the frequency of non-synonymous polymorphisms], causes Dn [the frequency of non-synonymous fixed differences] to appear large by comparison.



As cool as this nearly neutral explanation is, it’s not the only one we can pull out of our ass. How do you like this one: because *Drosophila* have historically larger effective population sizes than hominids they have been able to fix more slightly advantageous amino acid mutations. That leads to an excess of amino acid substitutions along the lineage leading to *D. melanogaster*. What makes my nearly neutral hypothesis any better or worse than Hughes’s hypothesis? You see, two can play at this just-so-story game.


**Conclusions**


If you’re going to attack adaptationist explanations of molecular evolution for lacking rigor, you should present a reasoned explanation with empirical evidence. That’s not to say that the abundance of TEs in the human genome that were inserted about 54 million years ago aren’t the result of a population bottleneck. But where is the evidence for that bottleneck? And if you’re going to present one nearly neutral hypothesis as an alternative to an adaptationist story, you should also be aware that there are slightly adaptationist explanations for the same data. As I’ve [said before](http://scienceblogs.com/evolgen/2007/07/whither_adaptation.php), a demographic explanation requires the same quality of evidence as an adaptationist one, and, without such evidence, is open to the same ridicule as a spandrel ridden selectionist story.


---



Gherman A, Chen PE, Teslovich T, Stankiewicz P, Withers M, et al. 2007. Population bottlenecks as a potential major shaping force of human genome architecture. PLoS Genet. In press. doi:[10.1371/journal.pgen.0030119.eor](http://dx.doi.org/10.1371/journal.pgen.0030119.eor)


Hughes AL. 2007. Looking for Darwin in all the wrong places: the misguided quest for positive selection at the nucleotide sequence level. Heredity *In press* doi:[10.1038/sj.hdy.6801031](http://dx.doi.org/10.1038/sj.hdy.6801031)


Lynch M. 2006. The origins of eukaryotic gene structure. Mol Biol Evol 23:450-468. doi:[10.1093/molbev/msj050](http://dx.doi.org/10.1093/molbev/msj050)


Lynch M. 2007. The frailty of adaptive hypotheses for the origins of organismal complexity. Proc Natl Acad Sci USA 104:8597-8604 doi:[10.1073/pnas.0702207104](http://dx.doi.org/10.1073/pnas.0702207104)

#Score One for the Lumpers
Date: July 18, 2007

<img alt="i-b9fa330c6fdfd937bb97b977bc1bff1e-clouded_leopard.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b9fa330c6fdfd937bb97b977bc1bff1e-clouded_leopard.jpg"/>


Remember that [new species of leopard](http://www.sciencedaily.com/releases/2007/03/070315075842.htm) that was “discovered” earlier this year? Well, it wasn’t really discovered so much as [recategorized](http://dx.doi.org/10.1016/j.cub.2006.08.066) as a [unique species](http://dx.doi.org/10.1016/j.cub.2006.10.066) (it was originally discovered in the early nineteenth century). That’s a picture of it on the right if you don’t remember.


Anyway, there’s an [Editorial in PLoS Biology](http://dx.doi.org/10.1371/journal.pbio.0050194) arguing that we’re creating too many damn species. We’re not really “creating” them, mind you, but categorizing what were previously subspecies as distinct species. The authors of the editorial think that this is getting out of hand; they’re taking the lumping position in the lumpers vs. splitters debate. On the other side are the splitters, who can be humorously represented by [C. Hart Merriam](http://en.wikipedia.org/wiki/Clinton_Hart_Merriam) — the man who split North American brown bears into 82 species. Merriam’s overzealous taxonomic practices are supposed to be a warning of what happens when splitters take control of the species naming business.


There is actually a practical reason for this scholarly debate, and it’s all about conservation. Some argue that it’s easier to pass legislation to conserve a group of organisms if those organisms are members of a species rather than a subspecies or population. These authors disagree with that hypothesis ([pdf](http://www.ias.ac.in/currsci/apr102004/897.pdf)), pointing out that for every new “species” that is deemed conservation worthy, another species will lose the protection it requires because of limited funds.


I’m waiting for [Wilkins](http://scienceblogs.com/evolvingthoughts/) to say something intelligent about the philosophical implications of all this. Anyone else can chime in on the policy.

#What is Open Access and How Should We Pay for It?
Date: July 19, 2007

Bill Hooker has [taken Nature editor Maxine Clark to task](http://www.sennoma.net/main/archives/2007/07/open_access_is_not_a_marketing.php) for her claims about the open access status of the online features offered by the Nature Publishing Group. Maxine points to the various free online services offered by Nature — including [Nature Precedings](http://precedings.nature.com/), [Nature Reports](http://www.nature.com/reports/index.html), [Nature Network](http://network.nature.com/), [Scintilla](http://scintilla.nature.com/), and the journal [Molecular Systems Biology](http://www.nature.com/msb/index.html) — in claiming that Nature has “many open access projects and products”. Bill disagrees. You should read his [entire post](http://www.sennoma.net/main/archives/2007/07/open_access_is_not_a_marketing.php), but the punchline is that Clark is redefining Open Access to fit Nature’s model and to be used as a marketing device.


A big issue with Open Access publishing is how to pay for the various services offered. The traditional model of scientific publishing earns money for the publisher via subscriptions (both private and institutional) and advertising. Additionally, some publishers require the authors to pay a publication fee. Open Access publishers must survive without subscription fees, but they do have publication fees for authors. One nice feature of the Public Library of Science ([PLoS](http://www.plos.org/)), is that they will waive the publication fee if you cannot afford to pay it — they are the only Open Access publisher with whom I have dealt, so I don’t know if this policy applies to other publishers. This comes in handy when you would like to publish an article in an Open Access journal, but lack the funds to pay the often hefty fee requested by the publisher (generally over $1000).


That’s where my story comes in. I am currently preparing a manuscript, and I’ve narrowed down the list of journals — to whom I’ll give the honor of rejecting my work as unsatisfactory — to three. All three journals offer Open Access options, but only one is entirely Open Access (it’s a PLoS journal). I’d like to submit my manuscript to the PLoS journal, but I know I won’t be able to afford the publication fee. Both of the other journals also have publication fees, which are they are less than the PLoS fees and will be waived if I cannot afford them (they offer Open Access options at comparable prices to the PLoS journal).


Regardless of where I send the manuscript for review, I will be asking for a fee waiver. One journal isn’t very friendly when it comes to waiving publication fees. The PLoS journal won’t give me any flack, but I feel bad asking for a second waiver from PLoS in the span of a single year — like I’m duping them out of their money. I like PLoS and what they stand for, so I don’t want to be hurting their cause by not paying my publication fees. I don’t feel so bad asking for the fee waiver from the other two journals (one is the journal for a society to which I belong and the other is published by an academic press), but I probably won’t be able to go Open Access with them because they are unlikely to waive both the publication fee and the Open Access fee.


Any ideas? I’m especially keen on hearing from people who serve as editors for Open Access journals, work for publishers who offer Open Access options, or submit their work to Open Access journals.

#Language Log is Stealing My Business
Date: July 22, 2007

Mark Liberman at Language Log has been posting on genetics recently. A couple of days ago [he tried to track down the origins of the components of the gene name BTBD9](http://itre.cis.upenn.edu/~myl/languagelog/archives/004732.html). The letters and numbers in the name stand for complex-tramtrack-bric-a-brac-domain 9, which are hijacked from *Drosophila* nomenclature. Liberman then tries to figure out the origins of the names *tramtrack* and *bric-a-brac* using [FlyNome](http://www.flynome.com/) (a cool webpage that I hadn’t seen before) and [FlyBase](http://flybase.bio.indiana.edu/). In the end, he couldn’t track down the (clever) story behind either one.


I was amused by that post, and I was further impressed by Liberman’s range in [this post](http://itre.cis.upenn.edu/~myl/languagelog/archives/004736.html) on the news coverage of genome wide association studies. These studies look for single nucleotide polymorphisms (SNPs) that show a statistically significant association with a phenotype of interest. Liberman mentions two studies that mapped a genetic basis of restless legs syndrome, questioning the coverage of these papers in the popular press. Without presenting the frequency of the associated SNP, newspaper articles leave readers with the impression that individuals carrying one allele have a particular disease/syndrome/whatever, while individuals with the other allele do not. But, when one looks at the actual data, that is not the case — affected individuals may have an allele frequency of 80%, while the general population has an allele frequency of 70%. Liberman want these allele frequencies to be presented in the articles discussing the results.


And if you aren’t a regular reader of [Language Log](http://itre.cis.upenn.edu/~myl/languagelog/), bookmark it, add it to your RSS reader, or tattoo the URL to your forehead. It’s one of the few must read blogs in these internets.

#Selective Constraint on Ultraconserved Elements
Date: August 20, 2007


There’s been a lot of recent interest in sequences that are highly conserved between humans and other mammals (and even other non-mammalian vertebrates). These sequences are thought to be under purifying selection, which prevents the accumulation of substitutions after two evolutionary lineages diverge. We cannot rule out, however, that the sequences are conserved by either pure chance (ie, in a large enough dataset, there will be a substantial amount of outliers that are still part of the same distribution) or because they have a lower mutation rate.


[David Haussler](http://www.cbse.ucsc.edu/staff/haussler.shtml) is one of the leaders in the study of [ultraconserved sequences](http://scholar.google.com/scholar?q=haussler%20d%20conserved) in the human genome. In an attempt to understand the evolutionary dynamics of the ultraconserved sequences, [Haussler and colleagues studied](http://dx.doi.org/10.1126/science.1142430) the patterns of polymorphism in these regions. Population genetics theory predicts that, if natural selection were responsible for the conservation of these sequences, they should also have low levels of polymorphism because natural selection would be purging newly arising mutations before they reach appreciable frequencies.


Haussler and colleagues found that there is a marked excess of rare alleles segregating in human populations at these ultraconserved elements. New mutations in ultraconserved elements stick around long enough to be found in a couple of individuals, but they are removed from the populations. In fact, this excess of rare alleles is even greater than that in protein coding regions, indicating that the ultraconserved elements are under more purifying selection than regular genes.


---



Katzman S, Kern AD, Bejerano G, Fewell G, Fulton L, Wilson RK, Salama SR, and Haussler D. 2007. Human genome ultraconserved elements are ultraselected. Science. 317: 915. [doi:10.1126/science.1142430](http://dx.doi.org/10.1126/science.1142430)

#Confusion Regarding Long Author Lists
Date: August 21, 2007

Nautilus, **Nature**‘s blog for authors, has a [guest post](http://blogs.nature.com/nautilus/2007/08/whats_an_author.html) by [Robin Rose](http://www.forestry.oregonstate.edu/cof/fs/people/faculty/rose.php) on long author lists, entitled “What’s an author?”. The post is representative of a certain brand of curmudgeonliness mixed with a dash of either ignorance or naivete. Rose has seen author list with more than 20 authors, and he’s confused. Did each author contribute equally? How could the manuscript possibly have gotten written? How do you evaluate each author’s contribution? Should we cite these long author publications differently? These are all questions running through Dr. Rose’s mind, and he has bothered to share them with the world.


<img alt="i-6296a4f46682a10b582729e851f04654-human_genome_title.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-6296a4f46682a10b582729e851f04654-human_genome_title.JPG"/>


What Rose fails to realize, however, is that different fields of research collect and analyze data in different ways, and this gets represented in how researchers publish their results. As a member of a field that produces papers with long author lists (I’ve had the honor of being author #10 out of 50), I feel it’s my duty and responsibility to respond to Dr. Rose. You see, when a paper has a lot of authors — like [this one](http://dx.doi.org/10.1126/science.1058040), [this one](http://dx.doi.org/10.1126/science.287.5461.2185), or [this one](http://dx.doi.org/10.1038/nature01262) — it’s because a lot of people contributed to collecting and analyzing the data. Genome sequencing projects, for example, require a group of researchers to propose the sequencing project, at least one sequencing facility to generate the sequence data, another group of scientists to assemble the sequences into chromosomes (or parts of chromosomes), and yet more scientists to analyze the assembled sequences for various features (identifying genes, comparing those sequences other sequenced genomes, etc.). That’s a lot of people, each contributing in a substantial way to the data presented in the manuscript. This type of project cannot be carried out by a small group of researchers because of the diversity of laboratory tools and areas of expertise required to generate and analyze the data.


Just because there are no projects in your field that require manuscripts with large author lists doesn’t mean that you should look at papers with more than 20 authors with disdain. Simply put, some projects in some disciplines require many people to complete. And when those results are presented, there’s usually some rhyme and reason to how the author lists are assembled. In biology, for example, you can usually figure out the significance of someone’s role by their proximity to the beginning or end of the author list — people near the beginning usually did a lot of the work, and people near the end are usually responsible for either overseeing the project, designing the project, or instigating the research. The closer you are to one of the ends, the more significant your role. In other fields, the lists are compiled alphabetically (which I do not find as useful as how it’s done in biology).


[The post](http://blogs.nature.com/nautilus/2007/08/whats_an_author.html) has a minimal comment from Maxine Clark, Executive Editor at *Nature* pointing out that the journal does not have rules regarding the maximum number of authors or the order in which they are listed. She also makes the important point that it’s very nice when each author’s contribution is listed along with the article.

#Junk on Cancer
Date: August 23, 2007

The University of Michigan has put out a [press release](http://www.eurekalert.org/pub_releases/2007-08/uomh-bo082307.php) entitled:


**Bits of ‘junk’ RNA aid master tumor-suppressor gene**


With a title like that, how could I not blog the hell out of this bastard? I mean, they even put the [scare quotes](http://evolgen.blogspot.com/2005/10/evolution-of-drosophila-non-coding-dna_31.html) around “junk”. Like that — like I just did. Amazing!


The story is about three micro RNA genes (miRNAs) that interact with p53 — ****the**** cancer gene — and are not expressed properly in some lung cancer cells. Not only have these researchers cured cancer, Guido Bommer, the lead author, seems to think they’ve found the cure amidst piles of junk:


> 
“In the ‘junk’ lies treasure, in terms of critical knowledge about how normal cells stifle cancer or succumb to it.”



Let me tell you a little story about junk DNA, and I think Guido should also pull up a chair. You see, when a sequence of non-protein-coding DNA has a function (for example, it encodes a miRNA), it is not junk. In fact, when we say that the majority of the genome is junk, we aren’t assigning the title “JUNK” to certain sequences. We’re saying that there’s a fraction of the genome that has absolutely no function. It’s a statement of probability, not one of determination. It’s impossible to say which segments are functional and which are not without doing detailed molecular studies (like the one done by Dr. Bommer and colleagues). But we can infer the fraction of the genome that is functional by comparing genome sequences from within and between species.


Thankfully, the word “junk” does not appear at all in the actual scientific paper (available [here](http://dx.doi.org/10.1016/j.cub.2007.06.068)). Most misuse of the term junk DNA comes from the scientific press, but it’s a shame to see an author of the paper fall prey to such sloppy use of jargon.

#Zombie Labs Eat Brain Genes
Date: August 25, 2007


Remember the story about how [we inherited the gene that gives us human brains from Neanderthals](http://scienceblogs.com/evolgen/2006/11/your_great_1800_grandmother_wa_1.php)? The genetic data that were used to reach that conclusion (or a slightly less over-the-top conclusion) were part of [a couple](http://dx.doi.org/10.1126/science.1113722) of [other studies](http://dx.doi.org/10.1126/science.1116815) that identified signatures of adaptive evolution in genes involved in brain development. Those results were [poorly criticized](http://scienceblogs.com/evolgen/2006/07/rejecting_the_neutral_model.php) over a year ago, and they’ve [recently come under fire](http://scienceblogs.com/evolgen/2007/04/scientific_controversies.php) yet again — and [again](http://dx.doi.org/10.1126/science.1141705) (I’ve compiled a list of the relevant papers below the fold).


The two most recent attacks come in two different flavors. The first attack (previously discussed [here](http://scienceblogs.com/evolgen/2007/04/scientific_controversies.php)) points out that, if one uses empirical data to generate a null distribution, there is no evidence for positive selection on one of the brain genes (*ASPM*). Now, this is not the brain gene that introgressed from an archaic lineage (that was *microcephalin*). The Lahn lab (the group behind the original brain gene papers and the introgression paper) [argue](http://dx.doi.org/10.1126/science.1143658) that the data used to reach the conclusion in the dissenting paper suffer from ascertainment bias. There are a lot of technical details in both the attack and response to the attack that I have neither the energy nor the interest to get into here.


If the first attack is based on technical details, the [second attack](http://dx.doi.org/10.1126/science.1141705) is one of philosophies. Lahn and colleagues identified strong signatures of adaptive evolution in *ASPM* and *microcephalin*, and, because these genes are expressed in the brain, they argued that these may be important genes for the evolution of the human brain. Where the first critique questions whether there is evidence for positive selection on *ASPM*, the second presents evidence that alleles of these genes do not influence human brain phenotypes. [Mekel-Bobrov and Lahn](http://dx.doi.org/10.1126/science.1143658) point out that this is a far from novel result — [previous](http://dx.doi.org/10.1093/hmg/ddl126) [studies](http://dx.doi.org/10.1098/rsbl.2006.0586) (including [one from the Lahn lab](http://dx.doi.org/10.1093/hmg/ddl487 )) have failed to identify any relationships between these genes and human cognition or brain morphology — and does not challenge the conclusion that positive selection shaped the evolution of these genes. What does remain an open question, however, is what phenotype natural selection was acting upon to influence the patterns of both sequence divergence and polymorphism at these loci. The negative results surrounding association of brain phenotypes with the two genes suggest that natural selection may have been acting on a different feature.





---



Evans et al. 2005. *Microcephalin*, a gene regulating brain size, continues to evolve adaptively in humans. Science 309: 1717-1720 [doi:10.1126/science.1113722](http://dx.doi.org/10.1126/science.1113722)


Mekel-Bobrov et al. 2005. Ongoing adaptive evolution of *ASPM*, a brain size determinant in *Homo sapiens*. Science 309: 1720-1722 [doi:10.1126/science.1116815](http://dx.doi.org/10.1126/science.1116815)


Currat et al. 2006. Comment on “Ongoing Adaptive Evolution of **ASPM**, a Brain Size Determinant in **Homo sapiens**” and “**Microcephalin**, a Gene Regulating Brain Size, Continues to Evolve Adaptively in Humans”. Science 313: 172 [doi:10.1126/science.1122712](http://dx.doi.org/10.1126/science.1122712)


Mekel-Bobrov et al. 2006. Response to Comment on “Ongoing Adaptive Evolution of **ASPM**, a Brain Size Determinant in **Homo sapiens**” and “**Microcephalin**, a Gene Regulating Brain Size, Continues to Evolve Adaptively in Humans”. Science 313: 172 [doi:10.1126/science.1122822](http://dx.doi.org/10.1126/science.1122822)


Evans et al. 2006. Evidence that the adaptive allele of the brain size gene **microcephalin** introgressed into **Homo sapiens** from an archaic **Homo** lineage. PNAS 103: 18178-18183 [doi:10.1073/pnas.0606966103](http://dx.doi.org/10.1073/pnas.0606966103)


Yu et al. 2007. Comment on “Ongoing Adaptive Evolution of **ASPM**, a Brain Size Determinant in **Homo sapiens**“. Science 316: 370 [doi:10.1126/science.1137568](http://dx.doi.org/10.1126/science.1137568)


Timpson et al. 2007. Comment on Papers by Evans et al. and Mekel-Bobrov et al. on Evidence for Positive Selection of **MCPH1** and **ASPM**. Science 317: 1036 [doi:10.1126/science.1141705](http://dx.doi.org/10.1126/science.1141705)


Mekel-Bobrov and Lahn. 2007. Response to Comments by Timpson et al. and Yu et al. Science 317: 1036 [doi:10.1126/science.1143658](http://dx.doi.org/10.1126/science.1143658)

#Two Posts on Publishing
Date: August 26, 2007

Why should advisers encourage their students to publish? For the answer, read [this post](http://genomicron.blogspot.com/2007/08/why-would-advisors-encourage-students.html) by TR Gregory.


Why is the publishing industry afraid of open access? I can’t answer that question, but I can point you to the evidence for their fear: it’s [right here](http://www.prismcoalition.org/index.htm). Jonathan Eisen [points out](http://phylogenomics.blogspot.com/2007/08/prism-partnership-for-research.html) why PRISM, the anti-open access lobbying group, is total bullshit. The [Open Reading Frame](http://www.sennoma.net/main/archives/open_accessopen_science/) doesn’t like it either.

#Stealing from Other Schools
Date: August 29, 2007

[<img alt="i-768cecd32e219765b862494af0c2d96e-PennStateposter.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-768cecd32e219765b862494af0c2d96e-PennStateposter.jpg"/>](http://graphics.fansonly.com/schools/psu/graphics/07-fb-poster-psu-800.jpg)


That’s the poster for the upcoming football season at Penn State. It’s purpose is both utilitarian (it’s got the schedule of games printed on it) and motivational (it’s supposed to get you all geeked up for the upcoming season). This year’s poster is using the slogan “FIGHT ON!”, and I bet someone in the athletic department thought they were real clever when they came up with that. It also sounds mighty familiar, almost as if I’d heard it before. Oh, yeah, it’s the name of the USC fight song (see [here](http://www.usc.edu/about/usc_basics/)). It’s not as bad as Auburn printing “Roll Tide” on their poster or Ohio State printing “Go Blue” on their’s, but it’s pretty freakin’ stupid.


(Via [The Wizard of Odds](http://thewizardofodds.blogspot.com/2007/08/penn-state-fight-on.html).)

#In Vino (Genome) Veritas
Date: August 30, 2007

<img alt="i-85a23a55e23b4a60fb6f49d55ad8211d-pinot_noir_grapes.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-85a23a55e23b4a60fb6f49d55ad8211d-pinot_noir_grapes.jpg"/>


A group of researchers from France and Italy have [sequenced the genome](http://dx.doi.org/10.1038/nature06148) of the finest grape varietal, [Pinot Noir](http://en.wikipedia.org/wiki/Pinot_noir). The genome has hallmarks of ancient triploidization, shared by other [dicotyledons](http://en.wikipedia.org/wiki/Dicot), but there is no evidence for recent polyploidization. That meant sequencing and assembling this genome is easier than doing so for other agricultural plants that experienced genome doubling (and tripling) as a result of domestication.


Comparisons of gene content with *Arabidopsis thaliana* reveals an enrichment of various gene families responsible for protein products that produce favorable features in the wines produce from these grapes. For example, there are twice as many terpene synthases (which are responsible for the synthesis of important aromatics) in grapevine as in *Arabidopsis*. 



We are currently at the rise of alcohol related genomics — see [this comparative analysis](http://evolgen.blogspot.com/2005/07/hooray-beer.html) of various brewing yeasts as another example. What alcohol related project would you like to see done next? My nomination: metagenomics of [noble rot](http://en.wikipedia.org/wiki/Noble_rot).


More information on the grapevine project can be found at [Nature News](http://www.nature.com/news/2007/070820/full/070820-13.html) and [Nobel Intent](http://arstechnica.com/journals/science.ars/2007/08/27/heard-through-the-grapevine-the-origin-of-flowering-plants)


---



 French-Italian Public Consortium for Grapevine Genome Characterization. 2007. The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla. Nature **In press** [doi:10.1038/nature06148](http://dx.doi.org/10.1038/nature06148)

#Michael Ashburner on Open Access
Date: August 30, 2007

BioMed Central has posted videos of interviews of some notable scientists. Included is a [video](http://www.biomedcentral.com/profiles/movies/popup.asp?moviename=/movies/michaelashburner.flv) of friend of evolgen (and [enemy of Prof. Steve Steve](http://scienceblogs.com/evolgen/2007/03/steve_steve_has_the_smarch_of_3.php)) [Michael Ashburner](http://scienceblogs.com/evolgen/2006/05/michael_ashburner_1.php) espousing the virtues of open access. Ashburner was a signatory on the [letter to Science](http://dx.doi.org/10.1126/science.1060273) encouraging publishers to make their journals publicly available. He also walks the walk, proclaiming that he will not publish in non-open access journal, nor review articles for non-open access journals. Not only is he in favor of open access publishing, he also pushes for open access software.


One of Ashburner’s recent project has involved mining the scientific literature and providing smart databases. He points out that open access literature allows people who mine scientific papers to make their results available. Even thought he and his colleagues are free to search any article they have access to, they can’t share those results with people who can’t access the protected literature. In the video, he lists three reasons to publish in open access journals: it’s the moral high ground, it increases the amount of people who read your articles, and you maintain the copyright on your work. Additionally, he points out that non-open access journals are scam because universities end up paying for the research twice — once to generate the results and again to access them.


(Via [BioMed Central blog](http://blogs.openaccesscentral.com/blogs/bmcblog/entry/new_additions_to_the_video).)

#Sorting the Pancakes that Make up a Genome
Date: August 31, 2007

Genome rearrangements are fast becoming one of the most interesting aspects of comparative genomics (I may be slightly biased in my perspective). We have known for quite some time that genomes of different species (and even within species) differ by inversions of their chromosomes (this was [first studied in *Drosophila*](http://evolgen.blogspot.com/2005/10/human-genome-is-mess.html)). In fact, some of the [early work](http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16577723) on the evolutionary relationships of species was done using chromosomal rearrangements. Additionally, there’s a whole lot of important biological implications of rearrangements, including [speciation](http://scienceblogs.com/evolgen/2007/02/speciation_natural_selection_a.php), [human disease](http://dx.doi.org/10.1002/ajmg.1320450511), and the [function of genes contained within inversions](http://dx.doi.org/10.1073/pnas.0403090101).


Now that we have whole genome sequences from multiple closely related species, we can identify differences in gene order. Some species are so closely related that there are not many differences (eg, [humans and chimps](http://dx.doi.org/10.1038/nature04072)). Other species differ by many rearrangements (eg, [distantly related drosophilids](http://dx.doi.org/10.1101/gr.3059305)). For the closely related species, it’s pretty easy to reconstruct the rearrangement events. For the distantly related species, however, we need complex algorithms to solve these problems — problems that exist at the intersection of biology and computer science.





The problem of reconstructing the inversion events that differentiate two genomes is analogous to sorting a stack of pancakes (each with a different diameter) from largest to smallest using only a spatula to flip over the pancakes at the top of the stack — an analogy presented in [this article](http://www.americanscientist.org/template/AssetDetail/assetid/55848/) from American Scientist (via [3QD](http://3quarksdaily.blogs.com/3quarksdaily/2007/08/sorting-out-the.html)). The article itself is an interesting read, tracing the history of studying genome rearrangements from the early work on *Drosophila* to the early computational work on pancake stacking and flipping (some of which was done by Harvard dropout William Gates) to the merger of the computer science algorithms with the biology.


From a naive reading of the article, it would appear that many of the big questions have been solved. This is far from the truth, as the algorithms described don’t work very well for species that differ by a boatload of inversions. Sure, they work alright for [human and mouse](http://dx.doi.org/10.1073/pnas.1330369100) — these guys love to test out their work in mammalian genomes — but they fall flat when it comes to dealing with *Drosophila*, ironic considering that’s where this problem first presented itself. The problem with these algorithms lies in how they approach the problem at hand; they try to solve it by looking at the entire sequence of genes at once, shuffling them around until the end up with the same gene order in both species. This approach does not work when the amount of inversion events gets too large (and breakpoints get reused often).


If the [American Scientist article](http://www.americanscientist.org/template/AssetDetail/assetid/55848/) had been published a few months from now, it would have described analysis done on the [12 *Drosophila* genomes](http://rana.lbl.gov/drosophila/) that approaches the problem of genome rearrangement from a different perspective. I can’t say much else about this unpublished research, but it will give us a new way to study genome rearrangements. Additionally, this new approach captures the biological realities of the rearrangements in a much more accurate light than the work described in the article I’ve linked.

#I’M IN UR GENOME, ADAPTING UR HOST-SYMBIONT RELATIONSHIP
Date: August 31, 2007


Eukaryotic genomes are [chimeras](http://evolgen.blogspot.com/2005/05/chimera.html) of sequences from many different sources. There are the genes responsible for the normal functioning of the host, but there are also [transposable elements](http://en.wikipedia.org/wiki/Transposable_element) (TEs), sequences from mitochondria ([numts](http://scienceblogs.com/evolgen/2007/07/the_frailty_of_nearly_neutral.php)), and [endogenous retroviruses](http://en.wikipedia.org/wiki/Endogenous_retrovirus) (ERVs). In addition to those examples, other [symbionts](http://en.wikipedia.org/wiki/Symbiont) also infect eukaryotes and leave traces of their presence in the genomes of the hosts. One such parasite of many invertebrates is [*Wolbachia*](http://scienceblogs.com/loom/2006/08/21/down_with_the_malekillers_a_ta.php), a bacteria that invades the germ cells of its host and inflicts odd behaviors such as male killing, feminization of males, and reproductive isolation of infected males and uninfected females. Each of these strategies is intended to produce the same result: maximal transmission of the symbiont from one generation to the next.


Because symbionts live in such close quarters with (or within) their hosts, they often exchange genetic material with each other. We can search for evidence of these lateral gene transfers (LGT) by examining eukaryotic genomes for sequences that resemble those of a particular symbiont. These searches are often not possible with the off the shelf assemblies of genomes because bacterial sequences are usually excluded from those assemblies. A [paper in the pipeline in *Science*](http://dx.doi.org/10.1126/science.1142490) presents the results of a search for *Wolbachia* sequences in various invertebrate genomes. The authors searched these genomes for sequences with good matches to the *Wolbachia* genome, but they had to start from the raw data rather than any available assemblies.





They started with *Drosophila* genomes, and found a lot of evidence for LGT between *Wolbachia* and the flies. In fact, they found one example of the entire *Wolbachia* genome inserted into the genome of *D. ananassae*. On top of that, 2% of the genes from the *Wolbachia* genome are transcribed from the host genome. The authors are careful to point out that transcription of these genes does not necessarily mean they’re functional, but future experiments to determine if they are would be pretty cool.


In some of the press surrounding this article, adaptationist undertones are emerging. Take, for example, [this news item](http://www.nature.com/news/2007/070827/full/070827-6.html) from *Nature*:


> 
The discovery also hints that the bacterial genome must have provided some sort of evolutionary advantage to its host. “You’re talking about a significant portion of its DNA that is now from Wolbachia,” says Julie Dunning Hotopp, a geneticist at the J. Craig Venter Institute in Rockville, Maryland, who led the study. “There has to be some sort of selection to carry around that much extra DNA.”



Are these insertions under selection? Thankfully, I don’t have to debunk this claim, as [Jonathan Eisen has done a good job responding to the adaptationists](http://phylogenomics.blogspot.com/2007/08/adaptationomics-award-1-wolbachia-dna.html). Essentially, assuming that there’s some functional role for these sequences (especially the entire genome insertion) just because they’re there is absurd. Conversely, I’d also be careful before invoking any [nearly neutral explanations](http://scienceblogs.com/evolgen/2007/07/the_frailty_of_nearly_neutral.php) for these LGT events.


The authors also searched the genomes of some wasps and roundworms for evidence of LGT between *Wolbachia* and the hosts. Many of the *Wolbachia* insertions are species specific with high identity to the relevant reference *Wolbachia* genome. That suggests that these insertions are fairly recent. Additionally, they don’t present any evidence for ancient insertion events that have not degenerated. That sends up a red flag that these sequences are deleterious and cannot persist very long in the face of purifying selection.


Despite the adaptationist press surrounding this article, the science is very good. None of the crazy claims of selection favoring the LGT events are presented in the actual article, thankfully. But, sadly, the media need to spin this in such a manner to obfuscate the actual science.


[Jonathan Badger](http://ttaxus.blogspot.com/2007/08/horizontal-gene-transfer-from-bacteria.html) and [John Timmer](http://arstechnica.com/journals/science.ars/2007/08/30/the-remains-of-bacteria-may-lurk-in-animal-genomes) have also written about this article.


---



Hotopp et al. 2007. Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes. Science *In press* [doi:10.1126/science.1142490](http://dx.doi.org/10.1126/science.1142490)

#Tangled Bank #88
Date: September 12, 2007

Matt at the Behavioral Ecology Blog has posted the [newest edition of the Tangled Bank](http://blog.behavioralecology.net/2007/09/tangled-bank-88.html), the original science blogging carnival.

#Wheel Bugs
Date: September 12, 2007

As I was making my way back from a seminar on skin color genetics yesterday, I noticed a couple of bugs perched on the outer wall of my building. This wouldn’t be a blog worthy moment, except that the bugs were huge . . . and in mid coitus. I hurried inside and ran upstairs to grab my camera. I returned — with both my camera and another grad student — to find the bugs in the same position as when I left them.


[<img alt="i-d7befdefd89b47c0ab281d7c3ad53190-wheel_bugs_side_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d7befdefd89b47c0ab281d7c3ad53190-wheel_bugs_side_sm.JPG"/>](http://scienceblogs.com/evolgen/2007/09/wheel_bugs_side_lg.php)


This wasn’t exactly a high energy sexual encounter. In fact, the male appeared to be just sitting on the back of the female — as if he were in the process of pinning her in some kind of slow moving wrestling match. There was no humping, no bumping, no grinding. It was boring, like I imagine married people sex to be.


<br/>
[<img alt="i-1b9a6e686feb1b0ca46a19f2bdc84c39-wheel_bugs_side_vert_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-1b9a6e686feb1b0ca46a19f2bdc84c39-wheel_bugs_side_vert_sm.JPG"/>](http://scienceblogs.com/evolgen/2007/09/wheel_bugs_side_vert_lg.php)


As we leered at the bugs, they barely moved an antenna in response. I snapped pictures like an entomological peeping Tom, and they didn’t mind. Both my co-conspirator and I were amazed by the exotic appearance of these love<s>birds</s> bugs. The nifty stegosaurus-esque thoraxes screamed out that these creatures were not native to central Pennsylvania. It turns out that we were wrong, but more on that later.


It also seemed odd that, despite the fact that the smaller male had mounted the larger female (a pretty striking sexual dimorphism, if you ask me), it didn’t appear that he had brought his genitals in contact with her’s. Or, to use more colloquial terminology, he hadn’t put his doodad in her nanner. At least it didn’t look like it. Was he really boinking her?


I spotted a lone male standing atop the ashtray/trashcan that’s outside the door to our building (this is not the appropriate time for me to opine on biologists who smoke, so I won’t — except for this aside). Maybe he’d attempted to court the female, but failed. Or maybe he was just hanging out. Or maybe he was engaging the same peeping tom-foolery that my friend and I were perpetrating. Whatever the case may be, there he was.


[<img alt="i-674bc2955eb3a480ec97fbe9d6c9dd87-wheel_bugs_single_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-674bc2955eb3a480ec97fbe9d6c9dd87-wheel_bugs_single_sm.JPG"/>](http://scienceblogs.com/evolgen/2007/09/wheel_bugs_single_lg.php)


Curious as to what the hell these funny looking critters were (and still under the impression that they were something exotic — something from far far away), we discussed who we could cajole into helping us identify the buggers. It was past five o’clock, so our options were limited. Also, the Entomology Department was too far away. My partner in crime suggested one of the faculty members in our building who knows a thing or two about creepy crawly things.


She ran upstairs and grabbed a plastic bottle from her lab. When she returned, she delicately directed the lone male into her trap (a feat I’m not sure I would have been able to pull off). Luckily, he didn’t struggle much because, as we would soon learn, these guys are capable of delivering quite a bite. We brought our captor up to the bug guy’s office only to discover that he had gone home for the day. As we poked around his lab, we came across a new grad student who happened to have been trained as an entomologist. Hooray!!


[<img alt="i-3eb3e0066f2e38e0224475dfebb952bd-wheel_bugs_topview_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3eb3e0066f2e38e0224475dfebb952bd-wheel_bugs_topview_sm.JPG"/>](http://scienceblogs.com/evolgen/2007/09/wheel_bugs_topview_lg1.php)


Our new friend took one look at our prize and said, “That’s a [wheel bug](http://en.wikipedia.org/wiki/Wheel_bug).” We replied, “What’s a wheel bug and where is it from?” He told us that they’re [native to much of the United States](http://creatures.ifas.ufl.edu/trees/wheel_bug.htm#dis). What a bummer — this whole time we’d thought we had found some exotic creature. Looks sure can be deceiving.


Our newfound bug expert told us that these guys are [true bugs](http://tolweb.org/Heteroptera/10805). We were even able to identify it down to the genus and species name: *Arilus cristatus* (described by Linnaeus!). As the bug crawled out of the plastic bottle, he warned us to be careful because these guys bite. Then he calmly pushed it back into the bottle with his fingers. Later we would return him (the bug, not the grad student) to hang out with his wheel bug buddies outside.


We told bug expert that there was a pair of wheel bugs outside doing the nasty. He claimed to have seen them out there the previous day as well — what stamina! I kept my eyes peeled for them earlier today as I came back from lunch. Both the male and the female were still perched on the wall, only the male was no longer mounting the female. Instead, he was a few inches below her facing downwards toward the ground. I guess there’s only so much nookie one wheel bug can handle in a week.

#Family Values
Date: September 14, 2007

[<img alt="i-ec45e04f411abf5b7661e0e2a1e32437-spider_web.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ec45e04f411abf5b7661e0e2a1e32437-spider_web.jpg"/>](http://www.cnn.com/2007/US/08/30/spider.web.ap/)


I’m sure by now you’ve heard of the ginormous [spider web](http://www.cnn.com/2007/US/08/30/spider.web.ap/) that was spun in Texas. The thing was huge — 200 yards long — and it was spun by multiple different species. That interspecific collaboration got Bill Poser thinking, so he [blogged about it at Language Log](http://itre.cis.upenn.edu/~myl/languagelog/archives/004906.html):


> 
The web covers hundreds of square meters. Not only was it built by hundreds of spiders, who normally build isolated webs and eat each other if they get too close, but entomologist Allen Dean [reports](http://hosted.ap.org/dynamic/stories/G/GIANT_SPIDER_WEB?SITE=NCJAC&amp;SECTION=HOME&amp;TEMPLATE=DEFAULT) that they belong to twelve different **families**! We’re talking massive inter-**species** communication here folks, and not particularly closely related species either. It is comparable to communication and collaboration between human beings (family **Hominidae**) and [Lar Gibbons](http://en.wikipedia.org/wiki/Lar_Gibbon) (family **Hylobatidae**).



Twelve different families! To show you how diverse these spiders are, Poser reveals what species we’d have to collaborate with to accomplish such inter-species team work. Only he underestimates the scale of the difference. You see, Poser falls victim to taxonomic bias, assuming that a family in one taxon ([spiders](http://tolweb.org/Araneae/2546)) is equivalent to a family in another taxon ([primates](http://tolweb.org/Primates/15963)).





Before we get any further, allow me to remind you of the major taxonomic categories used for classifying animals:


     Kingdom<br/>
          Phylum<br/>
               Class<br/>
                    Order<br/>
                         Family<br/>
                              Genus<br/>
                                   Species



To use humans as an example, we’re in the kingdom [Animalia](http://en.wikipedia.org/wiki/Animalia), aka the animals. Our phylum is [Chordata](http://en.wikipedia.org/wiki/Chordate), which is made up predominantly of the vertebrates (ie, things with backbones). Our class is [Mammalia](http://en.wikipedia.org/wiki/Mammal), our order is [Primates](http://en.wikipedia.org/wiki/Primate), our family is Hominidae, and our species name (made up of the genus and species titles) is *Homo sapiens*. Many mnemonics have been developed to aid students in remembering the taxonomic groupings, playing off the first letter in each word. My favorite: King Plays Cards On Fat Girls’ Stomachs.


Anyway, mammalian families aren’t all that diverse. The family [Hominidae](http://en.wikipedia.org/wiki/Hominidae) contains only four extant species (humans, chimps, gorillas, and orangutans). If they were invertebrates, we’d probably classify them as a genus (or sub genus). But, because they’re closely related to us, they get their own family (that’s taxonomic bias). Other mammalian families are more diverse; [Muridae](http://en.wikipedia.org/wiki/Muridae) (mice, rats, and gerbils) contains hundreds of species, and it’s the largest of all mammalian families. As a point of comparison, the genus [*Drosophila*](http://en.wikipedia.org/wiki/Drosophila) has over a thousand species. Yes, a single insect genus is more <s>specious</s> speciose that the largest mammalian order.


But counting species isn’t the only way to determine diversity within a taxon. We can also estimate the age of the taxon using fossils and DNA. Let’s look again at the Muridae. The species that make up this family [last shared a common ancestor approximately 25 million years ago](http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15371245&amp;dopt=Citation) (mya). To go back to our invertebrate example, the species in the genus [*Drosophila* last shared a common ancestor approximately 60 mya](http://dx.doi.org/10.1093/molbev/msg236). Once again, the invertebrate genus is more diverse than the mammalian family.


[<img alt="i-182bc895847b31eba0e529cf1cc65f61-spider.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-182bc895847b31eba0e529cf1cc65f61-spider.JPG"/>](http://en.wikipedia.org/wiki/Image:Nephila_inaurata1.JPG)


Hopefully I’ve shown you that comparing taxonomic categories across different taxa is fairly meaningless. What gets named as a family in one taxon contains less diversity than a genus in another taxon. But what about the spiders? What would be the equivalent collaborator for humans to accomplish the same amount of inter-family collaboration that we see in the spider web in Texas? A group of [entomologists at Texas A&amp;M sampled spiders from the web](http://insects.tamu.edu/news_events/tawakonispiders/) and classified them based on their families and species. The three most abundant families were:


1. [Tetragnathidae](http://en.wikipedia.org/wiki/Tetragnathidae) (long-jawed orb weavers)
1. [Salticidae](http://en.wikipedia.org/wiki/Salticidae) (jumping spiders)
1. [Araneidae](http://en.wikipedia.org/wiki/Araneidae) (orb-weaver spiders)


All three of these families belong to the suborder [Araneomorphae](http://en.wikipedia.org/wiki/Araneomorphae), which was thought to have radiated in the late [Paleozoic](http://en.wikipedia.org/wiki/Paleozoic) or early [Mesozoic](http://en.wikipedia.org/wiki/Mesozoic) Era ([reference](http://dx.doi.org/10.1146/annurev.es.22.110191.003025)), or about 250 mya. Tetragnathidae have been observed as early as the [Cretaceous Period](http://en.wikipedia.org/wiki/Cretaceous), which came at the end of the Mesozoic Era, about 150 mya. We’ll use this date as a lower-bound estimate of the divergence times for the spiders found in the web in Texas. What follows is therefore a search for a taxon with the minimum amount of divergence from humans to achieve the equivalent amount of divergence between the spiders in the huge web.


[<img alt="i-7714947feb664551154ac1fb6d4d5d33-kangaroo.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7714947feb664551154ac1fb6d4d5d33-kangaroo.jpg"/>](http://en.wikipedia.org/wiki/Image:Kangaroo_and_joey03.jpg)


Mammals diverged from birds and reptiles about 300 mya. This date can be used to calibrate a molecular clock, which can then be used to estimate the divergence dates between different vertebrate lineages provided we can sequence DNA from extant representatives of those lineages. [Kumar and Hedges](http://dx.doi.org/10.1038/31927) used this approach to estimate the divergence times between humans and various other vertebrates. At about 150 mya, the human lineage (and all other [eutherian mammals](http://en.wikipedia.org/wiki/Eutheria)) diverged from the [marsupials](http://en.wikipedia.org/wiki/Marsupial). This was followed by a rapid radiation of the eutherians, giving rise to the mammalian orders we see today. Humans and gibbons (Poser’s chosen equivalent collaborator) diverged about 15 mya.


Contrary to what Poser claims, humans collaborating with gibbons is not equivalent to the various spider families working together to build I gigantic web. To accomplish such an inter-taxon collaboration, humans would have to work with kangaroos, koalas, or possums. I’m not sure how they do things in Australia, but I doubt they’ve got Tasmanian devils working construction.

#Is there anything Copy Number Variation cannot do?
Date: September 16, 2007

Carl Zimmer has [a post covering three recent papers on gene duplication](http://scienceblogs.com/loom/2007/09/10/todays_odd_bedfellows_spit_yea.php): one on [amylase variation in humans](http://dx.doi.org/10.1038/ng2123), one on [whole genome duplication in yeast](http://dx.doi.org/10.1038/msb4100170), and one on [duplications of genes in the *Drosophila arizonae* reproductive tract](http://dx.doi.org/10.1371/journal.pgen.0030148). In all three papers, results are presented showing the importance of duplicated genes in adapting to the environment.


Now, [gene duplication](http://evolgen.blogspot.com/2005/10/name-that-tune.html) isn’t [anything new](http://scienceblogs.com/evolgen/2006/02/copy_number_polymorphism.php) around [these parts](http://scienceblogs.com/evolgen/2006/12/the_6_rule.php). Those who know me know that I have a bit of an interest in gene duplication. Those who don’t, well, consider yourself informed that I have a bit of an interest in gene duplication.


Given Carl’s penchant for writing about studies with mass appeal, I’m surprised he ignored [this paper](http://dx.doi.org/10.1101/gr.6557307) on gene duplication ([press release](http://www.eurekalert.org/pub_releases/2007-07/cshl-gsp072607.php)). Why? Well, the abstract includes the following punchline:


> 
Many of the genes identified here are likely to be important to lineage-specific traits including, for example, human-specific duplications of the **AQP7** gene, which represent intriguing candidates to underlie the key physiological adaptations in thermoregulation and energy utilization that permitted human endurance running.



I guess we’re predisposed for marathon running because certain genes were duplicated in our genomes. Add that to the amylase duplications that give us the right saliva for our preferred diet. Of course, amylase duplications aren’t anything new; [*Drosophila* geneticists have been studying duplicated amylase genes for years](http://www.genetics.org/cgi/content/abstract/126/1/131).

#Publishing Original Research on Blogs — Revisited
Date: September 26, 2007

Over a year ago I [threatened to perform some original research and publish it on my blog](http://scienceblogs.com/evolgen/2006/04/publishing_original_research_o.php). I got as far as [writing an introduction](http://scienceblogs.com/evolgen/2006/04/publishing_original_research_o_1.php) to the project, but I never actually posted any data. I know, I suck.I had hoped to make the project simple enough that people could follow along. The problem was the available data were not in a form that would be accessible for most readers. So, I’ve held off until now. But the appropriate data have now been deposited in [Genbank](http://www.ncbi.nlm.nih.gov/Genbank/), so I can continue the series.


Over the next couple of days I’ll post the previously published entries, and they will be followed by the subsequent new entries. I’m hoping to walk through this slowly enough that anyone with a computer and an internet connection can mimic what I’m doing. I’ll cover retrieving sequence data from Genbank and analyzing those data using various techniques from molecular evolution. It’s [molecular evolution for the masses](http://scienceblogs.com/evolgen/2006/07/molecular_evolution_for_the_ma.php).

#Multicellular Eukaryotic Genomes are Teh Suck
Date: September 26, 2007

The human genome is one big, bloated motherfucker. It’s almost all non-protein-coding DNA. The same is true for many other eukaryotic genomes. Sure, some of it has a function. But a whole lot of it (and maybe most of it) is just [junk](http://www.google.com/search?q=junk&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search).


There are some who point to a relationship between genome size and organismal completexity and argue that those large genomes are necessary to explain the compelexity they observe. There are other that disagree — T.R. Gregory at [Genomicron](http://genomicron.blogspot.com/) being one of the more vocal objectors in the geno-blogosphere. First off, how do you measure complexity? Second of all, what species will you sample for your measures of genome size? And how will you measure genome size? By number of basepairs? By number of genes?





Anyway, a couple of weeks ago, [Gregory posted](http://genomicron.blogspot.com/2007/09/whats-wrong-with-this-figure.html) this figure from a [Science News article entitled Genome 2.0](http://www.sciencenews.org/articles/20070908/bob9.asp):


[<img alt="i-9e2a23088c9980f92d903a97e602c0af-noncoding_dna.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9e2a23088c9980f92d903a97e602c0af-noncoding_dna.jpg"/>](http://sciencenews.org/articles/20070908/bob9.asp)


The amount of non-coding DNA is plotted for various species with sequenced genomes. The bars in light blue are bacterial species, the dark gray is brewers’ yeast, the green bar is a plant, the purple bars are two invertebrates, and the orange bar is humans. The article suggests that humans have so much non-coding DNA because it serves an important function — it’s what makes us so gosh darn complex. There’s also the underlying “chain of being” message in the figure, with the “primative” species on the left end of the X-axis and the “advanced” species on the right end.


But if you thought that figure was bad, [Larry Moran](http://sandwalk.blogspot.com/2007/09/genome-size-complexity-and-c-value.html) provided an even <s>better</s> worse one:


<img alt="i-b56b08f9630c34741653bc8285e7eefb-genome_size_dap.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b56b08f9630c34741653bc8285e7eefb-genome_size_dap.jpg"/>


Here, the great chain of being is even more prominently displayed. There’s also so much [paraphyly](http://en.wikipedia.org/wiki/Paraphyletic) and [polyphyly](http://en.wikipedia.org/wiki/Polyphyly) it’ll make a cladist’s head explode. Why group unicellular eukaryotes together, excluding all other eukarotes? Why group invertebrates together, excluding vertebrates? Why group chordates, excluding vertebrates? Why exclude humans from the vertebrates? And, despite what [some people may claim](http://genomicron.blogspot.com/2007/09/discovery-wants-to-demote-fungi.html), fungi and plants are distant relatives.


Based on the position of the dog’s ass in the second bar graph, Gregory has dubbed these types of plots Dog’s Ass Plots ([DAPs](http://genomicron.blogspot.com/2007/09/dogs-ass-plots-daps.html)):


> Indeed, the human datum would accurately be placed roughly below the dog’s ass in this figure if it included a proper sampling of diversity.


Here are some other recent posts from Genomicron on the topic of the misrepresentation of genome size and non-coding DNA:


- [Genome Size and Gene Number](http://genomicron.blogspot.com/2007/09/genome-size-and-gene-number.html)
- [Junk DNA: let me say it one more time](http://genomicron.blogspot.com/2007/09/junk-dna-let-me-say-it-one-more-time.html)
- [Worst figure of all](http://genomicron.blogspot.com/2007/09/worst-figure-of-all.html)
- [On the lookout for DAPs!](http://genomicron.blogspot.com/2007/09/on-lookout-for-daps.html)


And, don’t forget, [creationists love to misuse genome size data](http://scienceblogs.com/evolgen/2006/06/the_creationists_discover_the.php) when arguing against evolution.

#Tangled Bank #89
Date: September 26, 2007

The [newest edition of Tangled Bank](http://scienceblogs.com/aardvarchaeology/2007/09/tangled_bank_89.php) has been posted by Martin at Aardvarchaeology. Go read yourself some science!

#Francisco Ayala is a Macro-phyle
Date: September 27, 2007

Or is he micro-phobe?


Nature Genetics has published a [mostly positive review](http://dx.doi.org/10.1038/ng1007-1179) of the [new Evolution textbook](http://www.amazon.com/dp/0879696842) by Nick Barton and others (the others include blogger [Jonathan Eisen](http://phylogenomics.blogspot.com/)) The review is penned by Francisco Ayala. Among the things Ayala brings up is the coverage various taxa receive:


> Surprisingly, however, five of the nine chapters of Part II are dedicated to the history of microbial evolution, and only one chapter deals with the diversification of plants and animals.


So, of the nine chapters in Part II, six deal with particular groups of organisms. Of those six, five focus on microbial evolution. That’s 83%, which seems fair given that microbes probably make up more than 83% of life on earth. Do we know so much more about animal and plant evolution that they deserve over 20% of the taxon specific material? I wonder what Jonathan Eisen thinks; after all, [he studies microbial evolution](http://128.120.136.15/mediawiki/index.php/Main_Page).

#Publishing Original Research on Blogs – Part 1
Date: September 28, 2007

**This is a repost (with some edits) of an introduction to publishing original research on blogs — a series I am [reintroducing](http://scienceblogs.com/evolgen/2007/09/publishing_original_research_o_2.php). The original entry can be found [here](http://scienceblogs.com/evolgen/2006/04/publishing_original_research_o.php).**


In April of last year, Bora [pushed the idea](http://sciencepolitics.blogspot.com/2006/04/publishing-hypotheses-and-data-on-blog.html) of [publishing original research](http://sciencepolitics.blogspot.com/2006/04/even-more-on-science-online-publishing.html) (hypotheses, data, etc) on science blogs. As a responsible researcher, I would need to obtain permission from any collaborators (including my advisor) before publishing anything we have been working on together. But what about small side projects or minor findings that I don’t expect to publish elsewhere? As it turns out, such a project has been laying dormant since I first started working on it as a class project a few years ago. I will reveal more information about this project in subsequent posts, but suffice it to say this research is far from earth shattering. My primary objective is not to present any important findings, but rather to give my readers insight into how easy it is to study molecular evolution. After reading this series (I hope) someone will be able to download some sequences from the [NCBI database](http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi) and perform their own analyses. That being the case, all of the software I use will be freely available and easy to run on a PC (sorry Mac users, but that’s my current environment).

#Sex in the Fourth Dimension
Date: September 29, 2007

Most of us are so used to the male+female=baby system of reproduction that we practice that it doesn’t even occur to us that there are other options. Sure, there’s the occasional instance of [parthenogenesis](http://scienceblogs.com/evolgen/2006/12/on_the_genetics_of_virgin_birt.php) in some megafaunal species, but that seems like the exception rather than the norm. And we do recognize that a lot of the microbes that make up the majority of life on Earth reproduce asexual. But, when it comes to sex, we’re stuck on male+female=baby.


Kurt Vonnegut thought otherwise. In his classic novel *Slaughterhouse Five*, he introduced the Tralfamadorians and their understanding of human reproduction:


> 
The Tralfamadorians tried to give Billy clues that would help him imagine sex in the invisible dimension. They told him that there could be no Earthling babies without male homosexuals. There could be babies without female homosexuals. There couldn’t be babies without women over sixty-five years old. There could be babies without men over sixty-five. There couldn’t be babies without other babies who had lived an hour or less after birth. And so on. It was gibberish to Billy.



Sure, that’s science fiction or fantasy, but there are some animals that utilize reproductive systems nearly as complex as Vonnegut’s version of humans. A very accessible review of those crazy reproductive systems can be found in [this Slate article entitled “Why are there two sexes?”](http://www.slate.com/id/2174380/). Besides taking a romp through some of the odder reproductive systems (ie, organisms with three sexes), there is also a brief treatment of why the majority of sexually reproducing organisms only have two sexes.





In addition to the examples of odd-ball sexual systems presented in the Slate article (clam shrimp with ZZ males in addition to ZW and WW hemaphrodites, or harvester ants with two types of males and two types of queens), there’s another odd sexual system found in a common model organism. You’d think that the animal models would resemble humans in many ways. Well, [*Drosophila* sex chromosomes are a bit different than human sex chromosomes](http://scienceblogs.com/evolgen/2006/12/origins_of_amniote_sex_chromos.php), but [**Caenorhabditis elegans**](http://en.wikipedia.org/wiki/C_elegans) are even stranger. These roundworms have hermaphrodites (XX) and males (XO). The hermaphrodites are capable of mating with themselves, while the males (which make up less than 1% of all progeny when a hemaphrodite gives birth) must mate with hemaphrodites in order to pass on their genes. Why males are even maintained is [up for speculation](http://dx.doi.org/10.1126/science.1087946).


The Slate article also discusses why two sex systems are so prevalent in sexually reproducing organisms. We know that systems with more than two sexes can evolve, so why aren’t they more common? Laurence Hurst has done some work trying to figure out the answer to this question (see [here](http://links.jstor.org/sici?sici=0962-8452%2819920323%29247%3A1320%3C189%3ACFATNO%3E2.0.CO%3B2-Z) and [here](http://links.jstor.org/sici?sici=0962-8452%2819960422%29263%3A1369%3C415%3AWATOTS%3E2.0.CO%3B2-7)). It basically comes down to mitochondria and other cytoplasmic goodies. In many sexually reproducing species the males only pass on their genetic material while the females contribute organelles in addition to DNA. This requires an either or system (either you pass on the other stuff or you pass on just DNA), so it must involve only two sexes. The arguments in favor of this hypothesis are further fleshed out in Nick Lane’s book [Power, Sex, Suicide](http://www.nick-lane.net/Power,%20Sex,%20Suicide.html).

#DonorsChoose.org — Week 1 Update
Date: October 9, 2007

[<img alt="Give to DonorsChoose" border="0" height="60" src="http://static.donorschoose.org/images/link/link_donorschoose_large.gif" width="468"/>](http://www.donorschoose.org/donors/viewChallenge.html?id=17153)



There are two exciting pieces of news about the [DonorsChoose.org ScienceBlogs Challenge](http://scienceblogs.com/evolgen/2007/10/donors_choose_2007.php). First of all, Seed has offered to match all donations by ScienceBlogs readers for the second year in a row. Last year, Seed matched donations up to $10,000, but this year they’re willing to match up to $15,000. So, [start donating](http://www.donorschoose.org/donors/viewChallenge.html?id=17153). We’re currently 15% of our way to our goal of $1000.


Secondly, I have an offer for evolgen readers who donate to the [evolgen challenge](http://www.donorschoose.org/donors/viewChallenge.html?id=17153). When you donate, save your confirmation email so that you can win one of the still unannounced prizes from Seed. In the meantime, if you [send me](mailto:evolgen[at]yahoo[dot]com) a confirmation email noting that you donated to the [evolgen challenge](http://www.donorschoose.org/donors/viewChallenge.html?id=17153), you’ll be entered in a drawing to win the evolgen Grand Prize of Supporting Science Education Challenge Memorial Championship — I know, it’s a catchy title. What’s the prize? For that, you’ll need read on.





**The evolgen Grand Prize of Supporting Science Education Challenge Memorial Championship**


Anyone who donates to the [evolgen DonorsChoose.org challenge](http://www.donorschoose.org/donors/viewChallenge.html?id=17153) and forwards their confirmation email to evolgen[at]yahoo[dot]com will be entered into the evolgen Grand Prize of Supporting Science Education Challenge Memorial Championship (hereafter referred to as the “Grand Prize”). There is a limit of one entry per email address. At the end of the challenge, all entrants will be entered into a lottery. The winner will be chosen by a random drawing. The amount donated will be independent of the probability of winning — all donors will be considered equally regardless of the amount of money contributed. The winning donor will be notified by email, and he/she will confirm acceptance of the prize via email.


**The Grand Prize**


The winner of the Grand Prize will be involved in naming a new a gene that will be characterized within the next year or so (not an actual length of time). This involvement will include communication regarding the gene’s evolutionary history, its hypothesized function, and any other unique characteristics of the gene. The Grand Prize winner will be permitted to suggest a name for the new gene, and, if that name is deemed appropriate, the Grand Prize winner’s suggested name will be used to name the new gene. The researchers involved in publishing the research describing the new gene reserve the right to deny the use of the suggested name if it is deemed [inappropriate](http://www.youtube.com/watch?v=DwHEpHK342w) for scientific publication. The Grand Prize winner will be allowed to suggest another name if the initial suggestion is deemed inappropriate.


**In Conclusion**


So, everyone, [get donating](http://www.donorschoose.org/donors/viewChallenge.html?id=17153), and you may get to name a gene!

#Frank Wants a Shiny Prize
Date: October 10, 2007

Please God, don’t let it happen. Please don’t let Franky Collins win [this stupid award](http://geneticalliance.org/ws_display.asp?filter=policy.honor.cosponsor). I don’t usually make pleas to you, but, given that this is a matter regarding a man of faith<sup>TM</sup>, I figured it’s a good time to plea to the sky fairy. So, do whatever you can to prevent Congress from awarding Frank the medal.


As for my more terrestrial readers, here’s the skinny. The Genetic Alliance is [petitioning the US Congress](http://geneticalliance.org/ws_display.asp?filter=policy.honor.cosponsor) to award Collins the Congressional Gold Medal. Why? Because he’s in charge of the National Human Genome Research Institute ([NHGRI](http://genome.gov/)). Now, the NHGRI has done a great job in promoting genome research, a field in which I’m actively involved. But Frank’s an embarrassment to all geneticists.


Genomics relies heavily on evolutionary biology. Take a look at the [current issue of Genome Research](http://www.genome.org/current.shtml), one of the top journals focusing on genomics. Many of those articles either employ comparative techniques or study variation within populations. The analytic techniques involved are based in population genetics, phylogenetics, and molecular evolution. What does this have to do with Frank? Well, [Frank](http://scienceblogs.com/evolgen/2006/06/francis_collins_opens_mouth_in.php) don’t [understand evolution](http://scienceblogs.com/evolgen/2007/07/junk_on_junk_1.php). Frank’s a crappy biologist.


(Via [Hsien](http://www.eyeondna.com/2007/10/10/wheres-dr-francis-collins-of-the-national-human-genome-research-institute/).)

#DonorsChoose.org Prizes!!!
Date: October 11, 2007

Remember [those prizes I promised](http://scienceblogs.com/evolgen/2007/10/donorschooseorg_week_1_update.php) if you donated to any of the [ScienceBlogs DonorsChoose.org challenges](http://www.donorschoose.org/donors/leadershipboard.html?category=17)? I [described the prize](http://scienceblogs.com/evolgen/2007/10/donorschooseorg_week_1_update.php) for one lucky donor to the [evolgen challenge](http://www.donorschoose.org/donors/viewChallenge.html?id=17153). Now Seed has revealed what they’re giving away to a few lucky contributors to any of the ScienceBlogs challenges. [Janet](http://scienceblogs.com/ethicsandscience/2007/10/blogger_challenge_progress_rep.php) has the lowdown at her blog — the lucky donors will win either a t-shirt, mug, subscription to Seed, or copy of “The Best American Science Writing 2007”. Also, one really lucky donor will win an iPod nano. Go read [Janet’s post](http://scienceblogs.com/ethicsandscience/2007/10/blogger_challenge_progress_rep.php) to learn the details.


Links:


- [Blogger Challenge progress report (day 11) plus prizes from Seed!](http://scienceblogs.com/ethicsandscience/2007/10/blogger_challenge_progress_rep.php) — Adventure in Ethics and Science
- [ScienceBlogs DonorsChoose.org Leaderboard](http://www.donorschoose.org/donors/leadershipboard.html?category=17)
- [evolgen DonorsChoose.org Challenge](http://www.donorschoose.org/donors/viewChallenge.html?id=17153)
- [The evolgen Grand Prize of Supporting Science Education Challenge Memorial Championship](http://scienceblogs.com/evolgen/2007/10/donorschooseorg_week_1_update.php)

#Publishing Original Research on Blogs – Part 5
Date: October 12, 2007

**Previous entries:**


- [Part 1 – Introduction](http://scienceblogs.com/evolgen/2007/09/publishing_original_research_o_3.php)
- [Part 2 – The Backstory](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_4.php)
- [Part 3 – Obtaining Sequences](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_5.php)
- [Part 4 – Obtaining More Sequences](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_6.php)


*This post is part of a series exploring the evolution of a duplicated gene in the genus *Drosophila*.  Links to the previous posts are above.  Part 5 of this series (Examining the Outgroups) can be found below.*


**Examining the Outgroups**


In the [previous post](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_6.php) I mentioned that the two outgroup species in our analysis, the mosquito *Anopheles gambiae* and the honeybee *Apis mellifera*, have only one copy of the aldolase gene. If that’s the case, then it’s likely that the gene was duplicated after the divergence of *Drosophila* from *Anopheles*. How can we confirm this? In this entry we will use the [NCBI’s BLAST webpage](http://www.ncbi.nlm.nih.gov/blast/Blast.cgi) to search for aldolase genes in the outgroup species.





The Basic Local Alignment Search Tool ([BLAST](http://www.ncbi.nlm.nih.gov/Class/BLAST/blast_course.short.html)) allows you to search a sequence against a database of other sequences. We’ll use BLAST to search the two aldolase genes from *D. melanogaster* against the *Anopheles gambiae* and *Apis mellifera* genomes. To do so, the BLAST program will translate the gene’s DNA sequence into a protein sequence. Each genome will also be translated into every possible protein it can encode. This is called TBLASTX, meaning a database of nucleotide sequences (each genome) will be translated and searched by a translated query (each *D. melanogaster* aldolase gene). The first thing we need to do is select TBLASTX from the [main BLAST page](http://www.ncbi.nlm.nih.gov/blast/Blast.cgi):


[<img alt="i-e3e4748ea1fd62e634cf2e3adc1f5458-blast_main_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-e3e4748ea1fd62e634cf2e3adc1f5458-blast_main_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/blast_main_lg.php)<br/>*Click to enlarge*


If you do that, you’ll be taken to the [TBLASTX page](http://www.ncbi.nlm.nih.gov/blast/Blast.cgi?PAGE=Translations&amp;PROGRAM=tblastx). We’ll start by pasting the sequence of the *D. melanogaster* Aldolase gene (available in [this zip file](http://scienceblogs.com/evolgen/2007/10/aldolase_cds.zip)) into the input box:


[<img alt="i-9fe20a5ab63926a72151d0e78204e44f-blast_input_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9fe20a5ab63926a72151d0e78204e44f-blast_input_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/blast_input_lg.php)<br/>*Click to enlarge*


Next, we’ll select the database for TBLASTX. In the “Organism” box, enter “Anopheles gambiae”. As you enter the text, multiple options will appear:


[<img alt="i-b0f11f03ff8a67497028008e5e6a304b-blast_database_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b0f11f03ff8a67497028008e5e6a304b-blast_database_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/blast_database_lg.php)<br/>*Click to enlarge*


Finally, click “BLAST” and your query will be processed. It may take a few minutes to complete the search, but eventually your result will appear. When it does, this is what you will see:


[<img alt="i-87669510a51779ecf6c4408cea590ef6-agam_aldolase_blastx_sm.GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-87669510a51779ecf6c4408cea590ef6-agam_aldolase_blastx_sm.GIF"/>](http://scienceblogs.com/evolgen/2007/10/agam_aldolase_blastx_lg.php)<br/>*Click to enlarge*


Each horizontal line indicates a sequence from *A. gambiae* that matches our query sequence. The coordinates of the match against the query sequence are given along the top, and the color of the line indicates how good the match is (red is excellent, purple is very good, green is good, blue is fair, and black is poor). The first matching sequence comes from the *A. gambiae* genome sequence, while the rest of the sequences are from cDNA clones. What are cDNA clones? Well, they’re basically sequences from mRNA, which means they aren’t from the genome project. There’s only a single hit to the gene from the *A. gambiae* genome sequence. That means there’s probably one copy of an aldolase gene in the *A. gambiae* genome.


We’ll also search the *A. gambiae* genome using the CG5432 gene from *D. melanogaster*. If we search the *A. gambiae* sequences, we get the following result:


[<img alt="i-658e4261179347b8ba02045930b30fb7-agam_CG5432_tblastx_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-658e4261179347b8ba02045930b30fb7-agam_CG5432_tblastx_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/agam_CG5432_tblastx_lg.php)<br/>*Click to enlarge*


There’s also only a single match to CG5432 in the *A. gambiae* genome, and it’s to the same region that matches Aldolase. This further supports our hypothesis that the mosquito has only one copy of an aldolase gene. Next, we can search the same *D. melanogaster* gene against the *A. mellifera* (honeybee) sequences. Here is how to format the search:


[<img alt="i-3775a4a846225147abfcf174dfc60b70-blast_database_apis_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3775a4a846225147abfcf174dfc60b70-blast_database_apis_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/blast_database_apis_lg.php)<br/>*Click to enlarge*


If you perform that search, you’ll see that there are two *A. mellifera* sequences that match Aldolase: 


[<img alt="i-8209d16de0da5dc813c68f14b7186554-amel_aldolase_tblastx_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-8209d16de0da5dc813c68f14b7186554-amel_aldolase_tblastx_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_aldolase_tblastx_lg.php)<br/>*Click to enlarge*


And if you search CG5432 against the *A. mellifera* database, you also get two sequences with excellent matches:


[<img alt="i-7b3ec8ba8ef1d7ba74bfbb3a179ae719-amel_CG5432_tblastx_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7b3ec8ba8ef1d7ba74bfbb3a179ae719-amel_CG5432_tblastx_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_CG5432_tblastx_lg.php)<br/>*Click to enlarge*


Both searches match the same two *A. mellifera* sequences ([XM_623339.2](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748948&amp;dopt=GenBank&amp;WebEnv=0FxDr9tkO-210FhZ9fL1LleaGG91sB44yJ-kpwrolnc0X9oZoEogf2M_NRmbRL-oe0t_okBr3yKvql7%4026420CFE6F2628C0_0083SID&amp;WebEnvRq=1) and [XM_001121298.1](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748958&amp;dopt=GenBank&amp;WebEnv=03BM4rpfMv88g8LZAH60BEVPNm69LbJS6VtE6ceiHWiBRnVPYdUAkgDYz2cjuXkR4Hr4VCHCKeFdcoy%4026420CFE6F2628C0_0083SID&amp;WebEnvRq=1)), both of which are predicted homologs of the *D. melanogaster* Aldolase gene.  We can download the coding sequences (CDS) for each of those mRNAs from Genbank. To do so, we’ll click on the line in the BLAST result corresponding to the gene. That will take you to another area of the webpage containing an alignment of the *D. melanogaster* sequence with that from *A. mellifera*. If you click on the name of the *A. mellifera* above the alignment, you’ll be taken to the Genbank page for that particular sequence. Alternatively, you can click on the name of the sequence above, which I’ve included with a link to the Genbank page.


[<img alt="i-cbe51080e9752b418413559bb9aa8307-amel_XP_623342_genbank_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-cbe51080e9752b418413559bb9aa8307-amel_XP_623342_genbank_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_XP_623342_genbank_lg.php)<br/>*Click to enlarge*


That’s a portion of the Genbank page for [XM_623339.1](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=110748948&amp;from=90&amp;to=1187&amp;view=gbwithparts). To ensure you’re downloading the CDS, click on the link to the CDS on the Genbank page (shown above). Then, select “FASTA” from the “Display” dropdown:


[<img alt="i-2052afa845b2ec86c2a3e1a40601a5f7-amel_XP_623342_fasta_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-2052afa845b2ec86c2a3e1a40601a5f7-amel_XP_623342_fasta_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_XP_623342_fasta_lg.php)<br/>*Click to enlarge*


You can either copy and paste the FASTA sequence into a text editor or select “Send to: File” and save the FASTA sequence to your hard drive. Now, do the same for [XM_001121298.1](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748958&amp;dopt=GenBank&amp;WebEnv=03BM4rpfMv88g8LZAH60BEVPNm69LbJS6VtE6ceiHWiBRnVPYdUAkgDYz2cjuXkR4Hr4VCHCKeFdcoy%4026420CFE6F2628C0_0083SID&amp;WebEnvRq=1). If you’re having trouble downloading the CDS for these two genes, I have uploaded the FASTA files. You can dowload them from the following links: [XM_623339](http://scienceblogs.com/evolgen/2007/10/Amel_XM_623339.fas), [XM001121298](http://scienceblogs.com/evolgen/2007/10/Amel_XM_001121298.fas).


Why is there one match to the aldolase genes in the *A. gambiae* database, but there are two matches in the *A. mellifera* database? We’ll try and figure that out in a subsequent post. But, before we do that, we’ll take a look at the evolutionary relationship of *Drosophila*, *Anopholes* (mosquito), and *Apis* (honeybee). That will allow us to formulate a hypothesis about why *A. mellifera* has two matches to aldolase, while *A. gambiae* has only one.

#Publishing Original Research on Blogs – Part 6
Date: October 16, 2007

**Previous entries:**


- [Part 1 – Introduction](http://scienceblogs.com/evolgen/2007/09/publishing_original_research_o_3.php)
- [Part 2 – The Backstory](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_4.php)
- [Part 3 – Obtaining Sequences](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_5.php)
- [Part 4 – Obtaining More Sequences](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_6.php)
- [Part 5 – Examining the Outgroups](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_7.php)


*This post is part of a series exploring the evolution of a duplicated gene in the genus *Drosophila*.  Links to the previous posts are above.  Part 6 of this series (Evolutionary Relationships) can be found below.*


**Evolutionary Relationships**


While we were probing the outgroup genomes for copies of aldolase genes using TBLASTX ([Examining the Outgroups](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_7.php)), we discovered that there are two excellent matches to aldolase genes in the honeybee, *Apis mellifera*, genome. However, there is only one match in the mosquito, *Anopheles gambiae*, genome. In this entry, we’ll take a look at the evolutionary relationships of *Drosophila*, *Anopheles*, and *Apis* and formulate hypotheses for the different numbers of matching sequences.


<br/>
<img alt="i-94ccc5aa634a08772afd01d892c15179-insect_tree.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-94ccc5aa634a08772afd01d892c15179-insect_tree.gif"/>


Both *Drosophila* and mosquitos are [dipterans](http://en.wikipedia.org/wiki/Diptera), while bees are [hymenopterans](http://en.wikipedia.org/wiki/Hymenoptera). Therefore, *Drosophila* and *Anopheles* are closer relatives to each other than either is to *Apis*. That means we expect *Drosophila* genomes to be more similar to *Anopheles* genomes than to *Apis* genomes. So, it’s surprising that *Apis mellifera* has two matches to aldolase in its genome, while *Anopheles gambiae* has only one.


To address the issue of multiple matches in the *A. mellifera* genome, we should figure out what exactly those two matches are. There are two explanations. First, there may in fact be two aldolase genes in the *A. mellifera* genome. Second, there may be a single aldolase gene in the *A. mellifera* genome, but it has two annotated transcripts. The two transcripts may arise via alternative splicing of a single gene, much like we see in [*D. melanogaster*](http://scienceblogs.com/evolgen/2007/10/publishing_original_research_o_5.php).


To test these two hypotheses, we can examine the Genbank entries for the two *A. mellifera* sequences ([XM_623339](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748948&amp;dopt=GenBank) and [XM_001121298](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748958&amp;dopt=GenBank)). The entry for XM_623339 contains the following information in its annotation:


> 
Apis mellifera similar to Aldolase CG6058-PF, isoform F (LOC550785), mRNA.



While the entry for XM_001121298 includes this:


> 
Apis mellifera similar to Aldolase CG6058-PA, isoform A (LOC725455), mRNA.



The two *A. mellifera* sequences were annotated as similar to the same *D. melanogaster* gene (CG6058, or Aldolase), but each *A. mellifera* sequence matches a different *D. melanogaster* splice-form. That suggests the two sequences from *A. mellifera* are alternative spice-forms of the the same gene.


We can also take advantage of the *A. mellifera* genome browser on the NCBI webpage. Here is a screenshot of the Genbank entry for [XM_623339](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748948&amp;dopt=GenBank):


[<img alt="i-55dddecd4dddd45d91de577f46f6a169-amel_XM_62339_genbank_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-55dddecd4dddd45d91de577f46f6a169-amel_XM_62339_genbank_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_XM_62339_genbank_lg.php)<br/>*Click to enlarge*


The red arrow points to a link that will take you to a browser for the chunk of DNA in which this sequence is located. That chunk of DNA is known as [NW_001253175](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?view=graph&amp;val=NW_001253175), and, if you also examine the Genbank entry for [XM_001121298](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&amp;db=Nucleotide&amp;list_uids=110748958&amp;dopt=GenBank), you’ll find that NW_001253175 contains both *A. mellifera* aldolase sequences. In order to find XM_623339 and XM_001121298, however, we’ll have to use their synonyms (LOC550785 and LOC725455); these synonyms are given in the Genbank entry for each sequence. When you visit the browser for [NW_001253175](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?view=graph&amp;val=NW_001253175), enter “LOC550785” into the search box, as shown below:


[<img alt="i-cee5f7b77c8a40d5a561ac2fcb6d3179-amel_NW_001253175_sm.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-cee5f7b77c8a40d5a561ac2fcb6d3179-amel_NW_001253175_sm.gif"/>](http://scienceblogs.com/evolgen/2007/10/amel_NW_001253175_lg.php)<br/>*Click to enlarge*


This will allow you to visualize the region in which LOC550785 (aka, XM_623339) is located:


<img alt="i-ecb993e059d3fdd524fca4e8c77b95ec-amel_LOC550785.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ecb993e059d3fdd524fca4e8c77b95ec-amel_LOC550785.gif"/>


This sequence spans from positions 271,541 to 291,134 in this chunk of the *A. mellifera* genome. If the two *A. mellifera* sequences are alternative splice-forms of the same gene, they should be located in the same region. On the other hand, if they are unique genes, they should be located in different regions of the genome. The other sequence (LOC725455) is found in the same chunk ([NW_001253175](http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?view=graph&amp;val=NW_001253175)). If we search for it in the browser, here is what we see:


<img alt="i-7b3db718be887e81d8f8f03e843143b8-amel_LOC725455.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7b3db718be887e81d8f8f03e843143b8-amel_LOC725455.gif"/>


The coordinates of this sequence are 266,908-270,253, and these are different than those of the other sequence. Therefore, the two *A. mellifera* sequences are located in different parts of the genome (although in close proximity to one another). They are unique genes, not alternative splice-forms of the same gene.


We conclude that *Drosophila* species and *A. mellifera* both have two aldolase genes, while *A. gambiae* has only one. This leaves an important question unresolved: did the same duplication event give rise to the duplicate aldolase genes in *A. mellifera* and *Drosophila*? If that’s the case, then *A. gambiae* would have suffered a loss of one of those genes. Conversely, an ancestral aldolase gene may have been duplicated independently in *Drosophila* and in honeybee. We will attempt to answer this question in a subsequent post by reconstructing the phylogeny of these genes.

#Scientific Communication
Date: October 16, 2007

I want to highlight two excellent items related to scientific communication:

#A Job Posting
Date: October 24, 2007

Too funny to pass up:


> 
**Seeking NIH Geneticist as early (paid) reader of Science/Adventure Novel**
I’m looking for a one or more readers to vet the science and NIH-specific details of a science/adventure novel. This is a “hard science” book along the lines of Carl Sagan’s “Contact,” but with certain spiritual elements. The ideal reader would be someone with a graduate level genetics background at the NIH, but who is open to possibilities beyond rigid, scientific reductionism — — somewhere between Richard Dawkins and Francis Collins.
Willing to consider a reading fee for the right person.


If you meet the requirements and are interested, you can find the contact information for the person who posted the ad [here](http://www.bio.net/bionet/mm/biojobs/2007-October/022224.html).

#Pseudoscience on Preprint Servers
Date: October 24, 2007

Peer-to-Peer, one of *Nature*‘s many blogs, has a [post on pseudoscience on preprint servers](http://blogs.nature.com/peer-to-peer/2007/10/open_forums_and_pseudoscience.html). The post is in response to a post from another blog ([creationists using nature precedings to pre-publish junk science](http://www.chesnok.com/daily/2007/08/09/creationists-using-nature-precedings-to-pre-publish-junk-science/)) that pointed out a potentially pseudoscientific article on *Nature*‘s preprint server, *Nature Precedings* ([The saltational model for the dawn of H. sapiens, chin, adolescence phase, complex language and modern behavior](http://precedings.nature.com/documents/579/version/1)). The article in question came off as creationist tripe to selena, who blogged about it at [Tending the Garden](http://www.chesnok.com/daily/2007/08/09/creationists-using-nature-precedings-to-pre-publish-junk-science/).





This brings up a couple of questions. Taking a narrow focus, is [the article](http://precedings.nature.com/documents/579/version/1) pseudoscience? While I know a little about the study of human origins, I do not feel comfortable to judge this article. I also lack the time to read anything more than the abstract:


> 
A new model may contribute to resolve the origin problem of H. sapiens. According to our new viewpoint, Neandertals were neither one of our direct ancestors nor a different species. Their origin was not in Europe 150-200 or 300 thousand years ago. As for the origin of H. sapiens, it was neither in Africa roughly 2 million years ago nor roughly 200 thousand years ago. In other words, both the Multiregional model and the recent African origin model seem wrong. Our own species arose in the Middle East approximately 150 thousand years ago and split into two subspecies: Moderns and Neandertals. Rapid and radial expansion of H. sapiens from the origin implies a revolution (see Figure 1). Complex language, modern behavior and even adolescence phase plus chin might be included into the revolution. This possibility seems consistent with the data and could also be tested via; 1-the origin of complex language based on the modern human form of FOXP2 gene 2-the origin of adolescence or adolescent growth spurt 3- the genetic origin of the Flores hobbits. If all of these three origins appeared in the Levant about 150 thousand years ago then our model is true in all aspects. This speciation seems an unexpected revolution or macroevolution occured in several thousand years. The last pre-sapiens hominids may be extinct due to OIS 6 namely without replacement by sapiens.



Well, the FOXP2 point seems to be moot — Neandertals have the [same protein sequence](http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/foxp2_krause_el_sidron_2007.html) as moderns — although regulatory differences aren’t well understood. Aside from that, does anyone who knows better want to take a look at the paper and figure out if it’s good science, pseudoscience, or just plain crap?


The other question that comes up is what is the role of editors in managing preprint servers? Should they remove pseudoscience? Or should it be the responsibility of commenters, acting as sort-of-peer-reviewers, to flag the dreck? But what happens when preprint servers don’t get many comments? If the comment traffic is low on all articles, then it’s hard to imagine that the pseudoscience ones will be flagged by many readers.

#Some day I sell DNA!
Date: October 25, 2007

Why is this video so damn funny?


[<img alt="i-928f890192955fe172d702736a24b9c7-sell_DNA.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-928f890192955fe172d702736a24b9c7-sell_DNA.gif"/>](http://www.youtube.com/watch?v=ZA1NoOOoaNw)


<br/>
<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/ZA1NoOOoaNw"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/ZA1NoOOoaNw" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>

#Separated at Birth?
Date: October 25, 2007

<img alt="i-0f86c5200325a3c57894c13d078f3fdc-three_white_men.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-0f86c5200325a3c57894c13d078f3fdc-three_white_men.gif"/>

#Make that 12 Drosophila Genomes
Date: November 7, 2007

What happens when I [mention a paper describing two more *Drosophila* genomes](http://scienceblogs.com/evolgen/2007/11/two_more_drosophila_genomes.php)? Well, I get a [comment](http://scienceblogs.com/evolgen/2007/11/two_more_drosophila_genomes.php#comment-629460) telling me that the 12 Genomes Papers have gone live over at Nature. They have provided a nifty [12 Genomes Portal](http://www.nature.com/nature/focus/drosophila/index.html) for us to navigate the articles published by Nature based on data from the [12 Genomes Project](http://rana.lbl.gov/drosophila/). Now, these aren’t the only papers published from the 12 Genomes Project — look for 12 Genomes Papers in the November issues of [Genetics](http://www.genetics.org/) (currently not web-available), [PLoS Genetics](http://genetics.plosjournals.org/) (also not up on the website), and the December issue of [Genome Research](http://www.genome.org/).


The 12 Genomes Papers include two main papers — one on [genome evolution across the entire genus](http://dx.doi.org/10.1038/nature06341) and another on [using comparative genomics to identify functional elements](http://dx.doi.org/10.1038/nature06340). In addition to those two papers, Nature and Nature Genetics have also published some companion papers, including a really cool one on the [demasculazation of X chromosomes](http://dx.doi.org/10.1038/nature06330). I hope to have much more to say as I read through these papers.

#Biology Department Scavenger Hunt!!
Date: November 8, 2007

Chad asked a fun question last week, and I just got around to finding it yesterday:


> 
[What items should be on the list for a scavenger hunt through an academic physics department?](http://scienceblogs.com/principles/2007/11/dorky_poll_physics_scavenger_h.php)



Let’s now ask: what items should be on a list for a scavenger hunt through a biology department? Taking some hints from Chad and his commenters, here are my ideas to prime the pump:


- A reagent bottle with a label dating it to the 1980s
- An out of use fume hood above “the line”
- A paper copy of a PLoS Journal (they [exist](https://www.odysseypress.com/plosmedicine/plos_medicine_subscription.html))
- A non-top-heated thermal cycler
- A [Project Steve](http://www.natcenscied.org/article.asp?category=18) Steve
- A journal article used to prop up something
- A lab with more desktop computers than people (laptops don’t count)
- A non-model organism without a sequenced genome
- A PI without a personal webpage
- Someone doing fieldwork in winter
- A poster on display from a meeting at least five years ago


What do you think?

#Are Transcription Factors Enriched in Every Dataset?
Date: November 13, 2007

Is it just me or does every analysis that looks for over-represented gene ontology (GO) terms turn up transcription factors? It doesn’t matter if the study is looking for genes under [positive selection](http://dx.doi.org/10.1038/nature04240) or something else. It just seems like transcription factors are enriched in every dataset.

#The Condition My Condition is In
Date: November 21, 2007

I hate vanity posts — who the fuck outside my mom cares how I’m doing, and she don’t read this — but I feel somewhat obliged to explain the lack of activity on this unread blog. This is especially important in case my Seed overlords stop by and notice a stark absence of any recent posts and a deficiency in posting regularity over the past couple of weeks. Aside from the few regular visitors, I doubt anyone has noticed the near death of evolgen.


That said, here’s what I’ve been up to instead of blogging. I’m in the middle of that clusterfuck known as the last year of grad school. Those of you who have been through it know it’s a painful combination of trying to complete your research, trying to write your dissertation, trying to publish the unpublished parts, trying to find a post-doc, and trying to find a granting agency willing to give you a fellowship to fund that illusive post-doc. I have my priorities in order, and blogging isn’t anywhere near the top of the list. Maybe if Seed were to offer me a cool $1k per unique visit, evolgen would become obligation number one. Scratch that — I’ll take 1.000 euro (note the decimal rather than a comma), or even $1k Canadian. Considering the high pay-out is unlikely, this blog will remain in a persistent vegetative state for the near future.





Hey! You looked below the fold. This may or may not be a scam. Do you know jack-shit about evolutionary genetics? Want to write my blog for me? Send me an email or drop a note in the comments.

#Of Geese and Ganders
Date: November 23, 2007

Brian at Laelaps has written a post entitled “[What’s good for the gander isn’t always good for the goose](http://scienceblogs.com/laelaps/2007/11/whats_good_for_the_gander_and.php)“, in which he describes some examples of sexual dimorphism in charismatic vertebrates. Studying the phenotypes of these traits is interesting, but what’s happening on the genomic level? That is, how do differences between males and females affect the distribution of genes on chromosomes?


[<img alt="Blogging on Peer-Reviewed Research" height="85" src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" width="70"/>](http://www.researchblogging.org/)


Many of the traits that are beneficial for males are deleterious in females (and vice versa). For example, male sheep with big horns will mate more and leave more offspring, but females with big horns will waste their energy on those horns rather than producing babies (feminism has yet to catch on amongst bighorn sheep). If a single gene controls this trait, there will be antagonism between males and females for which variants are most fit. This [sexual antagonism was modeled in an elegant way](http://links.jstor.org/sici?sici=0014-3820%28198407%2938%3A4%3C735%3ASCATEO%3E2.0.CO%3B2-5) by William Rice. Rice developed a model that integrated sexual antagonism with sex chromosomes, and showed that, depending on a couple parameters, you could predict whether sexually antagonistic genes will be located on X chromosomes or autosomes.





Here are Rice’s results in a nutshell. For these examples, we’re assuming males are the heterogametic sex (XY), and females are homogametic (XX). If beneficial mutations are recessive, you would expect male-biased genes to accumulate on the X chromosome. That’s because those beneficial mutations will manifest themselves immediately in males (they aren’t masked by a dominant allele), while they will be essentially neutral in females (she will still have a copy of the dominant allele). This will lead to more fixations of male-biased mutations on the X chromosome than on the autosomes (where beneficial mutations will be masked in both males and females).


Alternatively, if beneficial mutations are dominant, they will be under selection regardless if they are on the X chromosome or an autosome. But the X chromosome spends 2/3 of it it’s evolutionary history in females, but only 1/3 in males (assuming equal numbers of breeding males and females). Therefore, selection for male-biased genes will be stronger on the autosomes than on the X chromosome, while selection for female-biased genes will be stronger on the X chromosome. And we will expect male-based genes to accumulate on the autosomes, while female-biased genes will accumulate on the X chromosome.


That’s the gist of the model. If we know the dominance of new beneficial sex-biased alleles and the sex ratio of breeding individuals, we can predict the genomic distribution of male- and female-biased genes. The problem is, it’s damn near impossible to know the values of those parameters, so the theory won’t get us much further. That said, it’s an important first step in figuring out the distribution of sex-biased genes.


Next time, we’ll take a look at what the data show regarding the distribution of these sexually antagonistic genes.


---



Rice WR. 1984. Sex Chromosomes and the Evolution of Sexual Dimorphism. Evolution 38: 735-742 [URL](http://links.jstor.org/sici?sici=0014-3820%28198407%2938%3A4%3C735%3ASCATEO%3E2.0.CO%3B2-5)

#Is Auburn the new Birmingham?
Date: November 26, 2007

<img alt="i-7b2ef70acd9ce6073358681f7f2c9308-birminghamdogs.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-7b2ef70acd9ce6073358681f7f2c9308-birminghamdogs.jpg"/>


So, I missed the three year anniversary of evolgen (it was last Wednesday for those of you keeping score). What does that have to do with police dogs and civil rights protesters in Alabama in the 1960s? Absolutely nothing. But I’m combining two unrelated topics into a single post — neither of which have **anything** to do with the American civil rights movement. Or do they?


Police dogs attacking black people are as big a part of Alabama culture as nooses hanging from trees in Louisiana. Too soon? Whether it’s at a protest march or on a football field, there’s not denying it. Anyway, here’s a video to prove it:


<br/>
<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/uSpfyIaS6wE&amp;rel=1&amp;border=0"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/uSpfyIaS6wE&amp;rel=1&amp;border=0" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>


**Added:**More dog on Auburn violence:


<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/WezgAIBY6Ys&amp;rel=1&amp;border=0"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/WezgAIBY6Ys&amp;rel=1&amp;border=0" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>

#3QD Hates Me. They really hate me!!
Date: November 26, 2007

In a list of the most overrated things, we’re the Churchills of popular science:


> Popular science: ScienceBlogs. Politics gets more hits than science, so ScienceBlogs recruits screamers rather than interesting popularizers or important scientists.


More [here](http://3quarksdaily.blogs.com/3quarksdaily/2007/11/a-case-of-the-m.html).

#Sequencing, Mapping, and Chips!
Date: November 27, 2007

I came across two press releases yesterday, entitled:


**[Entire Yeast Genome Sequenced](http://www.sciencedaily.com/releases/2007/11/071126110438.htm)**


and


**[University of Toronto scientists map entire yeast genome](http://www.eurekalert.org/pub_releases/2007-11/uot-uot112607.php)**


Upon reading the first, I thought, hasn’t [the entire genome of **Saccharomyces cerevisiae** already been sequenced](http://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;uid=9169866&amp;cmd=showdetailview)? And haven’t [other yeast genomes been sequenced as well](http://fungal.genome.duke.edu/)? What in the world could they be referring to? Did yet another yeast species gets its genome sequence? Does that really warrant a press release (sorry [Jason](http://fungalgenomes.org/blog/))? And the second made me think that the person in charge of titling the press release decided that genome sequencing is synonymous with mapping (it’s not).





It turns out that both press releases contain the exact same text, despite their unique titles. They’re hyping [this article that presents a genome-wide map of nucleosomes](http://dx.doi.org/10.1038/ng2117) in **S. cerevisiae**. This is important because it represents an important step in our understanding of how DNA is packaged in a cell. The authors incorporate experimental and computational tools to determine which segments of the genome are wrapped into 147 base pair bundles and which make up the space in between those bundles. This wrapping marks the first step of packaging the genome up into the tiny nucleus.


The article does not report the sequence of the genome, as the first press release claims. And while they do present a map, it’s not a [map in the classical genetics sense of the term](http://scienceblogs.com/evolgen/2006/10/genome_cartographers.php). However, a [map](http://www.flickr.com/photos/thisamericanlife/sets/72157602618985796/) is just the selective representation of a particular set of features in a certain area. So the second press release has an accurate, albeit incomplete, title.


The title of this entry refers to not only the fact that one press release calling this “sequencing” and the other “mapping”, but also to one of the techniques used by the authors of the paper in mapping the nucleosomes. They hybridized genomic DNA to microarrays (also known as “gene chips” or “chips”) to determine which segments are wrapped in nucleosomes and which are not.


---



Lee et al. 2007. A high-resolution atlas of nucleosome occupancy in yeast. Nat. Genet. 39:1235-1244. [doi:10.1038/ng2117](http://dx.doi.org/10.1038/ng2117)

#Are You Looking for an Academic Job in Biology?
Date: November 28, 2007

John Hawks linked to a [wiki with information on cultural anthropology job searches](http://johnhawks.net/weblog/topics/meta/careers/job_wiki_cultural_anthropology_2007.html). If you poke around the website, you’ll find a [list of such wikis for many different academic disciplines](http://wikihost.org/wikis/academe/wiki/start). Of interest to evolgen are the [Biology Jobs Wiki](http://wikihost.org/wikis/academe/wiki/all_biology) and the [Ecology/Evolution Jobs Wiki](http://wikihost.org/wikis/academe/wiki/ecology_botany_zoology_environmental_science_plant_vertebrate_in). If you’re currently on the job market, consider contributing information to these wikis.


Do you know of any other academic job wikis for biologists? If you do, link to them in the comments.

#Liveblogging a PLoS ONE Article
Date: November 29, 2007

PLoS ONE has recently published a paper entitled “[Beyond the Gene](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001231)” by Evelyn Fox Keller and David Harel, in which the authors take a stab at the long standing question: What is a Gene? Because this is such a big picture question, the appropriate discussion of the paper would involve a synthesis of what they authors wrote, what has previously been written, and what I think about all of that. I’m not going to do that. I’m too lazy and too stupid to do that. Instead, I’m going to read the paper and live-blog it. This one’s for you, [Bora](http://scienceblogs.com/clock/).





Quick tip: If you’re planning to read an entire paper, skip the abstract. At best, it will be a waste of time. At worst, it will plant ideas in your mind that will follow you throughout the rest of the paper like annoying little preconceived notions. I’m not reading the abstract.


Fox Keller and Harel start with a quote from Lavoisier, and the entire introduction is peppered with more quotes. Here’s the next one:


> 
“Where the meaning of most four-letter words is all too clear, that of gene is not. The more expert scientists become in molecular genetics, the less easy it is to be sure about what, if anything, a gene actually is.”
[Helen Pearson](http://dx.doi.org/10.1038/441398a)



You’ve got to be shitting me! What a load of shit! That’s fucking bullshit! The meaning of most four-letter words is not fucking clear. It’s a fucking mess. And anyone who says otherwise can go fuck themselves because they’re a mother-fucking cock-sucker. It’s downright cocky to claim otherwise.


Apparently bioinformaticians only care about definitions of genes if they include something about the location of the gene in the genome. More conventional experimental geneticists (Fox Keller &amp; Harel’s words, not mine), on the other hand, recognize that inheritance can be non-genetic (i.e., via proteins). Also, those experimental geneticists commit logical fallacies:


> 
Lindquist challenges the material basis of heredity, but perhaps even more than the bioinformatics people, she remains happy with the notion of “genetic elements”, of heritable entities. This begs the question of what is a genetic element.



It does not [beg the question](http://begthequestion.info/). It may raise the question, but it does no begging.


I present item number one that the authors do not understand evolution:


> 
Today we know that much more than DNA sequence is passed on from one generation to another, but even restricting ourselves to this DNA, which many would argue is the only truly tangible part of an organism that is actually passed down, we read that, at least for higher organisms, a mere 1-2% of the genome is spanned by protein-coding sequences.



So, organisms that [smoke more weed](http://scienceblogs.com/evolgen/2006/10/do_higher_eukaryotes_smoke_mor.php) have genomes made up of a mere 1-2% protein-coding sequences? Don’t tell me we’re gonna get into a discussion about genome size, because that’s what really affects the fraction of the genome that’s protein coding. Small genomes have a higher proportion of protein-coding sequence than large genomes. That’s what’s behind that 1-2% in humans. [In *Drosophila*, it’s closer to 10-20% percent](http://www.nature.com/nature/journal/v450/n7167/fig_tab/nature06341_T2.html). Sure, there are other functional regions, but the amount of protein-coding sequence is a pretty good estimate of the fraction of the genome that’s functional.


Helen Pearson, the person with the quote about fucking shitty cocks, brings some context to this whole debate:


> 
As Pearson observes, most geneticists are not trying to find a definition of the gene on which they can agree. Instead, they tend to use “less ambiguous words such as transcripts and exons. [And even] when it is used, the word ‘gene’ is frequently preceded by ‘protein coding’ or another descriptor….. Some things,” she concludes, “are not best portrayed by a crude four-letter word.”



That pretty much sums it up. We can stop now. Anyone worth an ounce of the weed that got some organisms high realizes that, for the sake of clarity, you should qualify the word “gene” whenever you use it. There’s really no reason to continue with the article. That said, I’ll keep reading for the sake of completeness. And for fuck’s sake.


Here’s how the authors justify going on:


> 
Kapranov et al write, “These observations suggest that genomic architecture is not colinear, but is instead interleaved and modular, and that the same genomic sequences are multifunctional”. Their argument has now been strikingly confirmed by the first results of the ENCODE Project Consortium. These findings are of immense interest, and dramatically underscore the need for new ways of thinking about DNA sequences, genomic organization, and their relationship to function. If we have learned nothing else, it is that that relation is far more complex than we had ever anticipated. Yet more recently, Gingeras suggests that this increased complexity necessitates “a reconsideration of the definition of a gene and require[s] the use of an alternative term to help to define the fundamental operational unit that relates genomic sequences to phenotypes/function.”



No, what Pearson said is pretty much correct. We can keep using the term “gene” so long as we qualify it with what we mean. If we’re talking about something that encodes a protein, then we call it a “protein-coding gene”. If the terminal product is an RNA, then it’s an “RNA-gene”, which can be made more specific by what type of RNA it encodes (rRNA, tRNA, miRNA, siRNA, snRNA, qwertyRNA . . . okay, I made that last one up). And so on.


But Fox Keller and Harel aren’t down with my jive. They want to do away with the gene. Instead of the gene, they want this:


> 
Accordingly, the first thing we’d like to do is to offer as a replacement for the gene a concept that is closely related, even if of a different kind, which we shall call the **dene**. Like the gene, our notion of dene is intended to capture the essence of genetic transmission, but, rather than being confined to denoting a discrete chunk of DNA, it is far richer and more expressive. A dene is, in fact, a general kind of **statement** about the DNA — what logicians call a predicate or a property. Denes can be used to represent vastly more intricate characteristics of the DNA sequence than the simple statement that it contains a particular subsequence.



What a load of post-modernist bull-shit. Is this the molecular biology [Sokal paper](http://en.wikipedia.org/wiki/Sokal_Affair)? It sure reads like it. Fuck you Dawkins for inventing the [meme](http://en.wikipedia.org/wiki/Meme), and fuck you Fox Keller and Harel for inventing the dene. Thankfully, this paper won’t make much of an impact, and no one will be using your silly jargon (not true for Dawkins’s meme). Instead, we’re gonna stick to the well established, and perfectly acceptable, terminology that Pearson described.


But they’re not done. Here’s the bene:


> 
For symmetry, we will refer to a statement about behavior as a **bene**. As with denes, our notion of bene will also be extremely rich, making it possible to express complex modal and temporal characteristics of the organism’s behavior over time, characteristics that go far beyond simple statements about, e.g., protein synthesis or transcription.



And the genitor (which is not a character from Transformers):


> 
In fact, it makes little sense to specify a dene — i.e., to make statements about a DNA sequence — without identifying the behaviors with which that sequence has come to be associated. Accordingly, we propose to be explicit about this conjunction, and thus introduce as our main concept, the **genetic functor**, or the **genitor**. This we define as the logical relation that says: Whenever the organism is seen to have X, it does Y. Or to use our new terms, a genitor relates a particular dene to a particular bene, stating that whenever the organism’s DNA is seen to satisfy the property expressed by the dene, it’s behavior satisfies the property expressed by the bene.



Too much jargon! Make it stop! But, no, we’re just completing the Introduction, and we’ve still got the “Analysis” and “Discussion” sections to go. This whole paper seems like an introduction to the analysis of a discussion — is it really necessary to break it into three parts?


Here’s how they unite all the jargon they’ve created:


> 
Syntactically, a **genetic functor**, or **genitor**, G is defined as a triple G = (O, D, B), which groups together the organism O with a **dene** D and a **bene** B. The former is a statement about O’s DNA and the latter is a statement about O’s behavior. Both of these will be described in more detail below, but for now it suffices to say that, semantically, the dene D is a truth-valued function of O’s DNA sequence and the bene B is a truth valued function of O’s temporal life-span.



For something that’s intended to “enable current and future research to move forward more effectively”, it sure doesn’t make anything clearer. I’d describe it as elegant, yet convoluted. Actually, it’s not really very elegant. More like convoluted, yet contrived. That pretty much sums it up.


This made me laugh:


> 
It will be noticed that between genes and denes lies a difference of only one letter, yet, we argue, the latter, as we use it, designates a concept on an altogether different logical level.



I’m not sure that it was supposed to.


They introduce another letter: “S to denote the organism O’s complete DNA sequence”. Are we having fun yet? You do realize that “This is one of the manifestations of rich logical expressiveness.” You don’t? Neither do I.


Here are some more terms that will be incorporated into the model:


- E – environment
- M – internal mechanisms
- F, G – events


We’re running out of letters.


Next, the examples. Here’s a simple one:


> 
The classical polypeptide coding unit found as a continuous stretch of DNA bounded by a stop and start codon (corresponding to Seymour Benzer’s cistron) is easily captured by a dene. The corresponding bene would be the production of a polypeptide chain (including transcription and translation), and the genitor would then capture the link between the two.



And here’s where they appear to not understand molecular genetics:


> 
Alternative splicing: Here a dene refers to any set of mRNA transcripts sewn together to form a protein-coding unit. In general, the dene would not require these to be contiguous. Each such dene would be associated with the corresponding polypeptide, the genitor capturing the relation that specifies the components of the dene corresponding to the bene that specifies production of the polypeptide.



Alternative splicing isn’t a set of transcripts sewn together. It’s a single transcript with various parts removed. Depending on which parts are removed, different processed RNAs are produced. Those different RNAs may encode different proteins (I say “may” because the non-protein-coding exons may be alternatively spliced).


They also bring up different types of RNA (rRNA, tRNA, micro-RNA, siRNA, snoRNA, sbRNA, snlRNA) and claim, “In our terminology, defining the appropriate genitors that would relate such benes to particular denes in a clean and rigorous fashion is the crux of the problem.” Why not just use the names that have been given to them? I still don’t see what their point is.


But here’s one that ****really**** irks me:


> 
Other segments of DNA without either an RNA or protein product are associated with yet different kinds of behavior. E.g., parts of the DNA, or various combinations thereof, which have a direct influence on mutation rates, would be definable as denes. A particularly interesting example of the latter is provided by stretches of small sequence repeats (SSR) that can induce slippage in the processes of replication, transcription, and even translation, and are accordingly sources of localized hyper-mutability (Moxon et al, 2006). Indeed, it has been argued that simple sequence repeats may equip the cell “with adjustable ‘tuning knobs’ for efficient adaptation” (King et al 1997).



What a load of adaptationist claptrap. These hyper-mutable repeats exist because they’re not deleterious enough to be purged by purifying selection. It’s very rare to find an [coherent adaptationist](http://dx.doi.org/10.1371/journal.pgen.0010041) [explanation for tandem repeats](http://mbe.oxfordjournals.org/cgi/content/full/18/6/1014).


As if we haven’t had enough already, Fox Keller and Harel tell us that they need to formalize their language. They claim:


> 
In the good spirit of computational and systems biology, it is obvious that mathematical and algorithmic formalization of biological concepts has the advantage of — indeed is done for the purpose of — enabling computerized analysis.



Only they’re not going to do that. And if they don’t do it, I don’t think anyone will. Sure, they’ll explain how other people should do it, but they won’t do it themselves. So, we’ll move on to the Discussion.


The discussion starts out with some talk of modularity and phenotypic plasticity. This kind of stuff gives developmental biologists a hard-on. It is accompanied by more adaptationism:


> 
Indeed, evolution selects for such adaptability (and hence, for modularity), because the ability to adapt that modularity confers on the organism enhances survival.



This is not a unanimous view, and not one that can be presented sans citation. [Modularity can evolve neutrally](http://dx.doi.org/10.1534/genetics.104.027607) (see [here](http://dx.doi.org/10.1038/nrg2267) for a review). The authors are full of shit (this time the four letter word means exactly what you think it means). The requirement: small population size. That’s an assumption humans meet, with our complexified modular genome.


Fox Keller and Harel offer a closing salvo about how traits can be inherited via mechanisms other than DNA (i.e., RNA, proteins, etc.). This supposedly relates to their denes, benes, and genitors. Somehow. Whatever. I read this crap so you don’t have to (all apologies to Lenny Bruce). And if you haven’t, don’t. It’s not worth your time. It’s vacuous navel gazing. Content free scholarship. Nothing for no one’s sake. Okay, maybe it’s for shit’s sake.


---



Fox Keller E, Harel D (2007) Beyond the Gene. PLoS ONE 2(11): e1231. [doi:10.1371/journal.pone.0001231](http://dx.doi.org/10.1371/journal.pone.0001231)

#The Ethics of Being an Open Access Publisher
Date: December 2, 2007

[<img alt="i-754235349a4041125e31c837c21f50a3-biomedcentral.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-754235349a4041125e31c837c21f50a3-biomedcentral.gif"/>](http://www.biomedcentral.com/)


[BioMed Central](http://www.biomedcentral.com/) advertises itself as “The Open Access Publisher” (see their logo floating next to this text). They publish a lot of journals, but I think the Public Library of Science ([PLoS](http://www.plos.org/)) has the lead when it comes to being THE open access publisher. That’s because everything published by PLoS is Open Access — it’s free to read, distribute, and reproduce, provided there is proper attribution. BioMed Central, not so much.


That’s right, BioMed Central, The Open Access Publisher, publishes paid-access articles. In fact, it publishes entire journals that are not open access. That includes [Genome Biology](http://genomebiology.com/), a journal I would read if I had access to the articles contained within. But I do not. Despite being published by The Open Access Publisher, Genome Biology is not open access.


BioMed Central is also behind the [Faculty of 1000](http://www.f1000biology.com/home), a website that collects reviews of papers in a sort-of online journal club for leading researchers. Faculty of 1000 is, you guessed it, not open access, despite being produced by The Open Access Publisher. I have no way of assessing the utility of Faculty of 1000 because I don’t have access to the service.


I’m not calling out BioMed Central for offering paid-access content. I have no problem with that, although I’d prefer to see all scientific publications go open access. My issue is with BioMed Central labeling themselves as The Open Access Publisher despite the fact that they publish numerous paid access journals and paid access services.

#What’s the deal with “virgin birth” (parthenogenesis)?
Date: December 3, 2007

<img alt="i-fd8cf622bbcb1b58246b4e7d485de036-dice.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-fd8cf622bbcb1b58246b4e7d485de036-dice.jpg"/>


Not all animals must have sex with another individual to produce perfectly viable offspring. And neither do humans, thanks to technological breakthroughs in artificial insemination. But what about those critters that do not require masturbation and meat basters to produce babies sans contact with another individual? Remarkably, this is quite common in the animal kingdom, although different animals go about doing it in different ways.





[*Caenorhabditis elegans*](http://en.wikipedia.org/wiki/Caenorhabditis_elegans), the roundworm that has become a popular model in developmental biology, lives in populations made up almost entirely of hermaphrodites. There are no true females, and only 1% of all individuals are male. A hermaphrodite will usually mate with itself (i.e., they self-fertilize), but the occasional male does get lucky and inseminates a hermaphrodite. Why males even exist is up for [speculation](http://dx.doi.org/10.1126/science.1087946), but this is not parthenogenesis.


For a real example of parthenogenesis, we should take a look at the [hymenopterans](http://en.wikipedia.org/wiki/Hymenoptera ), an order of insects that includes bees, wasps, and ants. Many of these guys are notable for the fact that unfertilized eggs give rise to males, while fertilized eggs gives rise to females. Because the males are born from unfertilized eggs, this is parthenogenesis. The females, on the other hand, are produced the same way most sexually reproducing animals make females — via the combination of a sperm (the male gamete) with an ovum (the female gamete). Therefore, the males are haploid (they have one copy of the genome, inherited from their mother), and the females are diploid (they have two copies of the genome, one from each parent).


Males born via parthenogenesis in hymenopterans can be thought of as half-clones of their mothers, and fathers share their entire genome with their daughters. This leads to interesting relationships between relatives because the genetic relatedness is not reciprocal. For example, while males have 100% of their genome in common with their mothers and daughters (as opposed to 50% in other sexually reproducing organisms), the mothers and daughters are only 50% related to their sons and fathers. Additionally, females are more closely related to their sisters than to their daughters, which may explain why worker females are so cooperative in bee hives.


And there are the exceptional cases, like that of a [Komodo dragon](http://dx.doi.org/10.1038/4441021a) last year (see [here](http://scienceblogs.com/evolgen/2006/12/on_the_genetics_of_virgin_birt.php) for a summary of the gory details and speculation on how it happened, and [here](http://scienceblogs.com/digitalbio/2006/12/just_in_time_for_christmas_a_v.php) for a post from Sandy). Unlike the hymenopterans, parthenogenesis in reptiles is not a default state, but results from accidents in the creation of the ovum. In bees, wasps, and ants, the males develop from haploid gametes that have not been inseminated by a male. In reptiles, however, the virgin births result from a misstep in the production of the ovum causing it to be diploid. These reptiles wouldn’t survive as haploids, but, because the ovum has a complete complement of genes and chromosomes, it can develop into a healthy animal.


So, what one means by “virgin birth” determines how someone can answer the question of what’s the deal with it. There are multiple ways to give rise to an animal without mating, only one of which is parthenogenesis. Parthenogenesis can be the default state (as in hymenopterans) or it can be a quirky event that is touted as exceptional (as in reptiles).

#Geese, Ganders, and Genomes
Date: December 4, 2007

[<img alt="Blogging on Peer-Reviewed Research" height="85" src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" width="70"/>](http://www.researchblogging.org/)


I previously described [where in a genome we would expect to find sexually antagonistic genes](http://scienceblogs.com/evolgen/2007/11/of_geese_and_ganders.php). Briefly, depending on whether a gene is male-biased or female-biased and whether beneficial mutations are dominant or recessive, we can predict whether these sexually antagonistic genes will be on X chromosomes or autosomes. As I mentioned in that post, the theoretical results can only be translated into realistic predictions if we have reliable estimates of the relevant parameters. We do not have such estimates, but we can study the distribution of sex-biased genes throughout genomes. The results of such experiments can then be used to infer how selection acts upon sexually antagonistic genes.





With the publication of complete genome sequences from multiple different animals, gene content on X chromosomes on autosomes could be compared. Manyuan Long’s research group performed some of these early studies, including one where they looked at duplicated genes in the *Drosophila melanogaster* genome [[Retroposed New Genes Out of the X in Drosophila](http://dx.doi.org/10.1101/gr.6049)]. The found an excess of duplicated genes with an ancestral copy on the X chromosome and a derived copy on an autosome. When they examined the expression patterns of these genes, they found that many of the autosomal copies of genes duplicated from the X chromosome were transcribed in the testes. It appeared that male-biased genes were accumulating on the autosomes and selected against on the X chromosome.


However, to explain the pattern, Long and colleagues did not invoke sexual antagonism. Instead, they pointed out that the X chromosome is inactivated during spermatogenesis. Therefore, testes expressed genes will be selected against if they are located on the X chromosome because they won’t be transcribed during some stages of spermatogenesis. But not everyone agreed with them. Chung-I Wu and Eugene Xu argued that selection against male-biased genes may have driven the testes expressed genes off the X, which then allowed for the evolution of X inactivation [[Sexual antagonism and X inactivation — the SAXI hypothesis](http://dx.doi.org/10.1016/S0168-9525(03)00058-1)].


[<img alt="i-89a65fda4650c08003ac953a98ab0e61-human_mouse_X_sm.GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-89a65fda4650c08003ac953a98ab0e61-human_mouse_X_sm.GIF"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-821488058d91a5395854663f090b3491-human_mouse_X.gif)


We’ll get back to the conflict between X inactivation and sexual antagonism in a little bit. But first we’ll take a look at where the male genes are in other organisms. Manyuan Long’s lab also looked at duplicated genes in the human and mouse genomes [[Extensive Gene Traffic on the Mammalian X Chromosome](http://dx.doi.org/10.1126/science.1090042)]. Just like in *Drosophila*, they found lots of genes duplicated from the X chromosome to the autosomes, and the autosomal copies were expressed in the testes. But they also found an excess of genes duplicated onto the X chromosome. One model of sexual antagonism predicts that the X-linked copies will be female-biased. Long and colleagues did not find an excess of ovary expressed genes on the X chromosome, so it was unclear why an excess of genes were duplicated onto the X chromosome.


What happens when we explicitly look for male-specific genes? David Page and colleagues did that with mouse transcripts, and they found that genes expressed only in spermatogenesis tend to be found on the X chromosome [[An abundance of X-linked genes expressed in spermatogonia](http://dx.doi.org/10.1038/86927)]. While the studies on duplicated genes found that male genes accumulate on autosomes, this study discovered that the X chromosome is a disproportionate source of male genes . . . in mice.


But the same pattern does not hold in *Drosophila*. Brian Oliver’s group compared gene expression in male and female flies and found an underrepresentation of male-biased genes on the X chromosome [[Paucity of Genes on the Drosophila X Chromosome Showing Male-Biased Expression](http://dx.doi.org/10.1126/science.1079190)]. This is consistent with the studies of duplicated genes. There are two different explanations for this pattern: either sexual antagonism drives male-biased genes off the X chromosome or X inactivation during spermatogenesis prevents testis-expressed genes from accumulating on the X chromosome.


Oliver and colleagues came up with a clever way to compare those hypotheses [[Demasculinization of X chromosomes in the Drosophila genus](http://dx.doi.org/10.1038/nature06330)]. They collected gene expression data from males and females of six additional *Drosophila* species — which was possible thanks to the [sequencing of the genomes of those species](http://scienceblogs.com/evolgen/2007/11/make_that_12_drosophila_genome.php). Just like in *D. melanogaster*, all species showed a paucity of X-linked male-biased genes. That can be seen in the first panel of the figure below.


[<img alt="i-62617c272903be12f943050b65adb351-sturgill_et_al_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-62617c272903be12f943050b65adb351-sturgill_et_al_sm.JPG"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-a952803f7164a35ec7fc4a215f7fcf72-sturgill_et_al_lg.jpg)<br/>*a, Percentage of genes with female-biased (red), non-biased (grey) and male-biased (blue) expression on chromosome arms. Muller’s elements and arms are indicated (X chromosomes in bold). Significant under-representation from a random distribution of genes in an expression class (chi-squared test) is noted (*P &lt; 10-4). The top 20% of differentially expressed genes (by ranked P values) are assigned a sex-biased expression class. b, Box plots of average hybridization intensities for males by chromosome arm. Twenty-fifth to seventy fifth percentiles (boxes), medians (lines in boxes) and ranges (whiskers) are indicated. c, As in a, but with genes with predicted testis-biased expression removed. D. melanogaster, D. mel; D. simulans, D. sim, D. yakuba, D. yak; D. ananassae, D. ana; D. pseudoobscura, D. pse; D. virilis, D. vir; D. mojavensis, D. moj.*<br/>[[Click to enlarge](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-a952803f7164a35ec7fc4a215f7fcf72-sturgill_et_al_lg.jpg).]


They then did something clever with the data; they excluded the testis-biased genes (that is genes expressed in testes significantly more than somatic tissue). If X-inactivation drove the male-biased genes off the X chromosome, excluding testis-biased genes should eliminate the discrepancy between the X chromosome and autosomes. If, however, selection against male-biased genes on the X chromosome drives the demasculinization of the X chromosome, removing testis-biased genes from the dataset should have no effect on the discrepancy. It turns out that removing the testis-biased genes does not eliminate the deficiency of male-biased genes on the X chromosome. Oliver and colleagues conclude that natural selection drives male-biased genes off the X chromosome independently of the effects of X inactivation during spermatogenesis.


What are we left with? Well, there is evidence that male-biased genes are selected against on the X chromosome. This supports the hypothesis that beneficial mutations in male-biased genes are dominant. However, this does not hold across all taxa — see the mouse study where testis-specific genes were found on the X chromosome. Could it be that we can’t generalize the dominance of beneficial mutations for different classes of genes (perhaps the testis specific genes have recessive beneficial mutations)? Or maybe the dominance of beneficial mutations varies across taxa.


---



Betran E, Thornton K, Long M. 2002. Retroposed New Genes Out of the X in Drosophila. Genome Res 12: 1854-1859 [doi:10.1101/gr.6049](http://dx.doi.org/10.1101/gr.6049)


Emerson JJ, Kaessmann H, Betran E, Long M. 2004. Extensive Gene Traffic on the Mammalian X Chromosome. Science 303: 537-540 [doi:10.1126/science.1090042](http://dx.doi.org/10.1126/science.1090042)


Parisi M, Nuttall R, Naiman D, Bouffard G, Malley J, et al. 2003. Paucity of Genes on the Drosophila X Chromosome Showing Male-Biased Expression. Science 299: 697-700. [doi:10.1126/science.1079190](http://dx.doi.org/10.1126/science.1079190)


Sturgill D, Zhang Y, Parisi M, Oliver B. 2007. Demasculinization of X chromosomes in the Drosophila genus. Nature 450: 238-241 [doi:10.1038/nature06330](http://dx.doi.org/10.1038/nature06330)


Wang PJ, McCarrey JR, Yang F, Page DC. 2001.An abundance of X-linked genes expressed in spermatogonia. Nat Genet 27: 422-426 [doi:10.1038/86927](http://dx.doi.org/10.1038/86927)


Wu C-I, Xu EY. 2003. Sexual antagonism and X inactivation — the SAXI hypothesis. Trends Genet 19: 243-247 [doi:10.1016/S0168-9525(03)00058-1](http://dx.doi.org/10.1016/S0168-9525(03)00058-1)

#Genetics Links
Date: December 5, 2007

There are two recent genetics related posts on other blogs that evolgen readers might find interesting:


First, check out [Query Gene](http://llama.med.harvard.edu/~jklekota/QueryGene.html) (via [ScienceRoll](http://scienceroll.com/2007/12/03/a-google-powered-gene-sequence-web-search-engine/)). This web-ap allows you to couple a nucleotide blast search with a Google search for a term related to your blast query. Here is how the creators describe it:


> 
Query Gene is a web-based program that searches for information about genetic sequences on the web. It is distinctive because it is not limited to a single database, but instead captures genetic information on the entire Internet using Google. Query Gene works by taking a gene sequence in combination with other search terms, finds similar sequences using NCBI’s MegaBlast, retrieves the descriptions of those matching genes from NCBI’s Entrez Nucleotide database, and performs a series of Google searches using the combination of your original search terms and each gene description. The percent sequence identity is indicated alongside each match: this indicates how much of your queried sequence is contained in the sequence it matches.



That’s a pretty cool mash-up for those of you doing small-scale work with blast.


The second item is a [post from MissPrism in which she describes genetic mapping](http://capacioushandbag.blogspot.com/2007/12/as-promised-genetic-mapping.html). This actually stems from a previous post where she bitched about [science writers conflating genome sequencing with the genetic code](http://capacioushandbag.blogspot.com/2007/11/raaagh-that-does-it.html). [Who would waste their time criticizing science writers](http://scienceblogs.com/evolgen/2006/10/10_million_to_decode_nicholas.php)? [Definitely not me](http://scienceblogs.com/evolgen/2006/10/genome_cartographers.php). [Never](http://scienceblogs.com/evolgen/science/science_news/). Anyway, check out both posts from MissPrism — she does a great job explaining the genetic code, DNA sequencing, and genetic maps.

#Where does your funding come from?
Date: December 9, 2007

The Scientist blog reports that a [representative of the National Science Foundation (NSF) was at the annual meeting of the America Society for Cell Biology (ASCB)](http://www.the-scientist.com/blog/display/53957/). The NSF representative pointed out a couple of things things:


<li>
If your proposal describes research designed to find a cure for some disease, the NSF will not fund it. Well, duh! The NSF is about funding basic research. If you want to cure diseases, go ask the National Institutes of Health (NIH) for money. Research about human diseases and human health are not fundable by the NSF.
</li>
<li>
If your lab is well funded, don’t expect the NSF to fund your proposal. I’ve heard from top-notch scientists about NSF proposals that received multiple excellent reviews, but were not funded because the lab had too much money.
</li>





These are important points. First of all, both of these appear to stem from the fact that the NSF has much less money to divvy out than the NIH. The [NIH has a budget of nearly $30 billion](http://www.nih.gov/about/budget.htm), while the [NSF has around $6 million](http://www.nsf.gov/about/congress/110/highlights/cu07_0308.jsp#final). The NIH funds biomedical research only, while the NSF must fund many different types of scientific research — biology, chemistry, physics, etc.


With such a diverse range of topics it must fund and much less money than the NIH with which to fund it, it makes sense that the NSF would shy away from funding research that the NIH would fund. Why “waste” their limited budget on projects that should be funded by the much richer granting agency?


Also, the NSF receives many more proposals than they can fund, and they must come up with creative ways to decide who gets funded. Many top-notch proposals will get turned down, so the NSF ensures that labs with less funding get the money that is necessary to keep their research projects above water. A well-funded lab will not be as drastically affected by an unfunded NSF proposal, but a lab that is just scratching by will suffer if their equally rated proposal does not get funded.

#Sciencedebate 2008
Date: December 10, 2007

Are you bothered by the total disregard for science shown by some US presidential candidates? Mike Huckabee does not heart evolution. Other candidates aren’t saying much. Would you like to see where the candidates stand on various science and technology issues?


Sheril and Chris at [the Intersection](http://scienceblogs.com/intersection/) have put together a campaign for a [Science Debate amongst the presidential candidates in 2008](http://www.sciencedebate2008.com/www/index.php?id=2). They have the [support of many top notch scientists and science advocates](http://www.sciencedebate2008.com/www/index.php?id=7). You too can sign on if you agree with the following statement:


> 
Given the many urgent scientific and technological challenges facing America and the rest of the world, the increasing need for accurate scientific information in political decision making, and the vital role scientific innovation plays in spurring economic growth and competitiveness, we call for a public debate in which the U.S. presidential candidates share their views on the issues of The Environment, Health and Medicine, and Science and Technology Policy.



[Sciencedebate 2008](http://www.sciencedebate2008.com)

#Recent Accelerated Adaptive Evolution in Humans
Date: December 14, 2007

For those of you interested in recent adaptive evolution in some insignificant bipedal primate, John Hawks and pals have published a paper in PNAS describing something you’ll find interesting. Of course, if you’re interested in such things, you already know that. Here are some links related to Hawks et al. paper:


- The Hawks et al. paper presents data to suggest a recent burst in adaptive evolution along the human lineage. The reason for this burst is an increase in populations size, allowing for more beneficial mutations in the species. Eventually, the paper itself will be available [here](http://www.pnas.org/cgi/doi/10.1073/pnas.0707650104). Until then, you can find it [here](http://www.anthro.utah.edu/PDFs/accel.pnas.smallpdf.pdf).
- [This post from p-ter at GNXP](http://www.gnxp.com/blog/2007/12/notes-on-evidence-for-acceleration.php) takes a critical look at the methodology employed by Hawks et al. P-ter argues that the methodology is biased toward detecting recent events. Hawks replies to p-ter [here](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/acceleration_rarely_asked_questions_2007.html?seemore=y).
- John Hawks has a couple of other posts about the paper and the coverage surrounding it. He [announces the paper here](http://johnhawks.net/weblog/topics/evolution/selection/acceleration_embargo_ends_2007.html), describes the [press coverage here](http://johnhawks.net/weblog/site/acceleration_reception_2007.html), [summarizes the results here](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/accel_story_2007.html), and [links to a bunch of people here](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/accel_blog_links_2007.html).
- Razib has posted come stuff on this, too (he’s interested in these boring topics). He has [described the theory and results](http://scienceblogs.com/gnxp/2007/12/accelerated_adaptive_human_evo.php) and [covered some of the stuff other people have written](http://scienceblogs.com/gnxp/2007/12/accelerated_adaptive_human_evo_1.php).
- Various other bio-bloggers have put in their 2cents. The new blog [Popgen ramblings suggests that Hawks et al. did not perform the appropriate simulations to test for false positives](http://popgen-ramblings.blogspot.com/2007/12/why-simulations-are-important.html). Some other non-specialists have also weighed in. These include [T. Ryan Gregory’s complaints about the way the authors are describing the research in the press](http://genomicron.blogspot.com/2007/12/evolution-with-bullet.html), [Larry Moran’s skepticism](http://sandwalk.blogspot.com/2007/12/accelerated-human-evolution.html), and [Greg Laden’s admiration](http://scienceblogs.com/gregladen/2007/12/there_is_a_new_paper.php).


If I read the paper, I may offer my own opinions. Maybe. And that’s if I read the paper any time soon. But I figured I’d post something on it because it’s getting so much hype. I thought [Hawks didn’t like hype](http://scienceblogs.com/evolgen/2006/05/you_say_hybridization_hawks_sa_1.php).

#Why Accelerated Adaptive Evolution isn’t Faster Evolution
Date: December 21, 2007

There’s been a whole lot of hype around the [Hawks et al. paper describing a recent burst of adaptive evolution](http://scienceblogs.com/evolgen/2007/12/recent_accelerated_adaptive_ev.php) in the human genome. The problem is a lot of people are conflating accelerated adaptive evolution with accelerated evolution. Take this for example:


> 
<p>12/11: Accelerated Human Evolution<br/>
In recent years, humans have evolved at a much higher rate than previously thought, according to a new paper in PNAS. By analyzing genome variations, researchers found that the rate of human evolution was fairly stagnant until about 50,000 years ago, and then–because of larger populations, climate changes, and disease– accelerated until at least 10,000 years ago. It probably continues at a high rate today.</p>



That blurb was taken from the ScienceBlogs buzz from December 11. I’m not picking on the ScienceBlogs editors, but I just happened to have that quote handy. I’m sure you can find others floating around the internet. The point is, there’s a fair bit of confusion regarding adaptive evolution. [This is nothing new](http://scienceblogs.com/evolgen/2007/05/jonah_lehrer_doesnt_understand.php), but it warrants a blog post explaining why accelerated adaptive evolution does not mean faster evolution.





Imagine a new mutation arising in a population. That mutation, in very simple terms, will either be deleterious, neutral, or beneficial in regards to its fitness effect. If it’s deleterious, it will almost certainly be lost by the next generation — although some deleterious mutations do overcome the odds (that’s why they’re odds, not fates) and increase in frequency over many generations. If it’s neutral, the mutation has a low probability of eventually fixing (i.e., being found in every individual after many generations). The probably of fixation of a neutral mutation is proportional to the inverse of the population size; the average time to fixation, in generations, is proportional to the population size — based on a very simple model of binomial sampling. And, finally, you have beneficial mutations, which have a probability of fixation proportional to the fitness benefit they confer.


If you add up all three classes of mutations — deleterious, neutral, and beneficial — and figure out how many have fixed over the time scale you’re looking at, you get the amount of evolutionary change along the lineage in question. So, to say that there was increased evolution along the human lineage in recent history implies that there was an increase in the total number of genetic changes. However, an increase in the amount of adaptive evolution (or an increase in the number of mutations fixed by positive selection), means there was an increase in the number of beneficial changes along the human lineage in recent history.


Why does this matter? Well, it’s a pretty well accepted fact (thanks to work by [Kimura](http://en.wikipedia.org/wiki/Motoo_Kimura), [Jukes](http://www.genetics.org/cgi/content/full/154/3/955), [King](http://dx.doi.org/10.1126/science.164.3881.788), etc.) that evolution at the DNA sequence level is dominated by neutral changes. Therefore, accelerated evolution would mean that the neutral rate of evolution increased. A change in the rate of neutral fixations would, most likely, require a change in the mutation rate. There are also some other scenarios to explain a change in the rate of neutral evolution, but that would require me to delve into the nearly neutral theory, which is beyond the scope of this discussion (see  [here](http://scienceblogs.com/evolgen/2006/02/more_on_misconceptions.php) and [here](http://scienceblogs.com/evolgen/2007/03/neutral_expectations.php)).


How did they infer adaptive evolution? They didn’t count up changes and assign them to each class, if that’s what you’re wondering. Instead, they used some common statistical techniques developed over the past couple of decades to look for signatures of positive selection in publicly available data of human genetic variation. The data set, known as [HapMap](http://www.hapmap.org/), contains the genotypes of hundreds of individuals at thousands of sites in the genome that are known to be variable in human populations. Basically, which allele (or version of the site) you have at a particular site is expected to be correlated with the alleles found at nearby sites — i.e., if I have an A at site 1000, you can predict I’ll probably have a G at site 1050 — based on data gathered from many individuals. This is known as linkage disequilibrium (the alleles at each site aren’t randomly assigned).


There is a baseline level of linkage disequilibrium (LD) in any genome, and an excess of LD in a particular region needs to be explained by invoking some evolutionary force. One possibility is that recombination, the process that mixes up and randomizes the sites, is low in that particular region. Another scenario is that natural selection has led to the recent near fixation of a beneficial mutation in that region, taking the closely linked sites along with it. That leads to a large block of highly correlated alleles and, therefore, high LD in that region. The authors used this approach to identify regions of the genome with signatures of recent positive selection (i.e., high LD above the baseline level). They found many more recent events than older events, as if there were a burst of adaptive evolution in recent history.


The criticisms of the paper are based on the power to detect adaptive evolution using LD. Some people have argued that [this approach is much better at detecting recent events](http://www.gnxp.com/blog/2007/12/notes-on-evidence-for-acceleration.php), leading to a [significant bias in the number of recent events reported](http://www.gnxp.com/blog/2007/12/more-notes-on-acceleration.php). That’s because LD decays over time, causing older events to be masked. Additionally, some have argued that the authors did not [adequately consider the possibility that the apparent burst of recent adaptive evolution was due to chance](http://popgen-ramblings.blogspot.com/2007/12/why-simulations-are-important.html). There’s a lot of stochasticity in evolutionary systems (i.e., a lot of background noise, or “this ain’t physics, bitches”), and signatures of positive selection can often be explained by purely neutral processes. To test this hypothesis, it’s standard fare to [run simulations using realistic parameters](http://popgen-ramblings.blogspot.com/2007/12/re-why-simulations-are-important.html) (mostly population size and changes therein) to determine an expected distribution of your result (or test statistic) under non-adaptive scenarios. You’d be amazed to discover the range of oddities possible under neutrality (population geneticists usually test for these oddities using [coalescent simulations](http://scienceblogs.com/evolgen/2006/02/the_coalescent.php)). Anyway, some people think the authors failed to adequately consider the possibility that their result is possible (and not just possible, but likely with a p-value&gt;0.05) without invoking adaptive evolution.


Why would there be an increase in adaptive evolution? This has been the point that John Hawks has been hammering most on his blog (if you haven’t been [reading his updates](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/), you should). Here is the theory, in a nutshell. The larger a population, the more mutations you’ll find in that population (the per individual, per generation mutation rate won’t change, but you have more individuals so you have more total mutations). The more mutations, the more beneficial mutations (once again, not per individual, but in total). The more beneficial mutations, the more likely that, within that population, a beneficial mutation will be found that offers an adaptive solution to something encountered by the population in its environment. Therefore, larger populations will have more beneficial mutations upon which natural selection will act and more adaptive evolution. Furthermore, natural selection is more efficient in large populations because the stochastic factors leading to the random loss off new mutations (whether they’re beneficial or neutral) are counter-balanced by more alleles being visible to natural selection (this is, once again, the realm of the nearly neutral theory, which I’m avoiding in this post).


Why won’t the absolute rate of evolution change? As I mentioned before, evolution at the DNA sequence level is dominated by neutral changes. The rate of neutral evolution (and the amount of neutral changes) is independent of population size. That’s because the number of neutral mutations each generation is equal to the neutral mutation rate (u) multiplied by the number of haploid genomes in the population (2N, or 2 * the number of individuals, because each individual has two copies of the genome, one from mom and one from dad). The probability any single mutation fixes is equal to its frequency, which is 1/(2N) for any new mutation (it’s found in a single genome out of the 2N genomes in the population). If, in each generation, you have 2Nu neutral mutations and the probability each one fixes is 1/(2N), then the rate of fixation of neutral mutations is just u (2Nu*1/[2N]).


So, Hawks and colleagues found more evidence for adaptive evolution with signatures of recent events than those with signatures that they occurred further in the past. They did not, contrary to some reports, find that human evolution has accelerated in the recent history. I’m not saying that some aspects of human evolution have not accelerated (perhaps phenotypic changes have). But it’s misleading to say that “evolution” accelerated because that implies that all aspects of evolution have accelerated. Hawks et al. were working with DNA sequence data, so they could only say whether certain aspects of molecular evolution were accelerated. Even there, only adaptive evolution experienced a recent speed-up, not all of molecular evolution.

#Phylogeny Friday — 28 December 2007
Date: December 28, 2007



#Accelerated Evolution — yet again
Date: December 30, 2007

Can positive selection drown out neutral evolution? That’s what [John Hawks claims](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/acceleration_neutral_evolution_2007.html) in response to [my post on accelerated evolution](http://scienceblogs.com/evolgen/2007/12/why_accelerated_adaptive_evolu.php). Hawks points out that, rather than looking at the neutral fixation rate (which is equal to the mutation rate, u), we should be more interested in the average time to fixation of a neutral mutation (the product of 4 times the effective population size, 4N<sub>e</sub>). On the time scale he’s looking at (40,000 years), neutral evolution shouldn’t really matter because Hawks says that the N<sub>e</sub>=100,000, which makes 4N<sub>e</sub>&gt;40,000.


Therefore, the majority of new mutations that arose in the past 40,000 years and have fixed (or nearly fixed) in a population should be either beneficial or linked to beneficial mutations. This allows for neutral evolution, however it’s nearly all driven by hitchhiking upon the back of adaptive evolution. But this hypothesis hinges on the assumption that N<sub>e</sub>&gt;10,000; this is brought up at Popgen Ramblings in [another post critical of the Hawks et al. paper](http://popgen-ramblings.blogspot.com/2007/12/another-comment-on-hawks-et-al.html). There’s an important distinction between N<sub>e</sub> (the ****effective**** population size) and the ****census**** population size. N<sub>e</sub> is a parameter within population genetics theory that allows us to treat a population as ideal, but it is greatly influenced by the historical demography of a population (especially drastic decreases in population size). So much so that’s it’s not practical to estimate N<sub>e</sub> from current population sizes or archaeological data — N<sub>e</sub> can only be estimated from population genetic data. The post at Popgen ramblings is skeptical of Hawks et al’s estimate of N<sub>e</sub>.


Finally, Hawks paints me as an anti-adaptationist in the tradition of Stephen Jay Gould. This is reminiscent of [Larry Moran’s bit where he claims to be a pluralist](http://scienceblogs.com/evolgen/2007/07/whither_adaptation.php) in the midst of the stark-raving-mad adaptationists. Only the roles are reversed. This time I’m the damn dirty neutralist, whereas I was previously labeled as the gullible adaptationist. I’m having a hard time keeping track of which team I’m on. Maybe that makes me the true pluralist.

#One “Art”, Please
Date: January 2, 2008

[<img alt="i-ad930d61d2b012ec56210f7865f0e151-openlab07-200.png" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ad930d61d2b012ec56210f7865f0e151-openlab07-200.png"/>](http://scienceblogs.com/clock/2008/01/open_lab_2007_the_winning_entr.php)


I can’t draw for shit. And I’ve got crappy penmanship to boot. All in all, I don’t end up with aesthetically pleasing creations when I put pen or pencil to paper. So it’s ironic that my one blog post selected for the [2007 edition of the Open Lab](http://scienceblogs.com/clock/2008/01/open_lab_2007_the_winning_entr.php) is a [comic that I made](http://scienceblogs.com/evolgen/2007/05/the_lab_fridge.php). It’s called [The Lab Fridge](http://scienceblogs.com/evolgen/2007/05/the_lab_fridge.php), and it’s a play off of a [PhD Comic](http://www.phdcomics.com/comics/archive.php?comicid=860). Thankfully, I had Illustrator at my disposal in creating the comic.


I don’t think it’s my best post of the year, but it is the best (and only) comic that I drew this year. For those of you looking to make it into Open Lab next year, go for one of the niche categories where there’s less competition — get drunk and write a poem, or something. I nominated a few posts from evolgen for Open Lab, and, while I’m not surprised that the Lab Fridge comic made it, I’m kind of bummed that [this post on the Texas spiderweb](http://scienceblogs.com/evolgen/2007/09/family_values.php) didn’t.

#Francis Collins Should not be Pres. Science Advisor
Date: January 7, 2008

Matt Nisbet thinks that [Francis Collins should be the next presidential science advisor](http://scienceblogs.com/framing-science/2008/01/an_endorsement_for_francis_col.php). He does this after rejecting excellent popularizes of science, such as [Neil deGrasse Tyson](http://research.amnh.org/~tyson/) and [E.O. Wilson](http://www.eowilson.org/), on the following grounds:


> 
Most science popularizers such as Wilson or Tyson don’t have the years of government experience to understand the machinations of Federal science policy. Moreover, they have a paper trail of strong opinions on issues that might make appointment politically tough.



I’m not sure what exactly those issues upon which they have strong opinions are. Is it that they’re both atheists? If so, they are amongst the least militant/evangelical/fundamentalist/new/[insert favorite pejorative] atheists amongst visible atheist scientists. Heck, Wilson wrote a [book reaching out to evangelicals to help with conservation](http://www.amazon.com/Creation-Appeal-Save-Life-Earth/dp/0393062171). And Tyson took what I would describe as a [fairly moderate position at Beyond Belief](http://www.youtube.com/watch?v=uYBFqse7tiU) in questioning Richard Dawkins’ approach in helping the public understand science.





I think it’s reasonable to question Wilson’s and Tyson’s experience in working on the policy side of science, however the science advisor makes no policy decisions. If you think he should have more of a policy role, then it’s appropriate to request someone with policy experience. But I don’t think the policy experience is as important as experience explaining science to the general public — Wilson and Tyson are two of the best at that, and there are other options.


Now, assume we say that the science advisor should have policy experience. Does this make [Francis Collins](http://www.genome.gov/10000779) a viable candidate? I argue NO. Collins is the director of the National Human Genome Research Institute ([NHGRI](http://www.genome.gov/)), a part of the National Institutes of Health ([NIH](http://www.nih.gov/)). The importance of biology in the near future of science is undeniable, so a biologist would be a good choice (as would a chemist or physicist). He is also an Evangelical Christian who has made his beliefs well known in a [book](http://books.google.com/books?id=JcMCmBnpHGsC) and various magazine articles. Nisbet argues that Collins’ religious faith would allow him to reach to a large portion of America (i.e., the religious nutjobs that currently takes anti-science positions).


I think Collins has no academic credibility. What I mean by that is that Collins is unqualified for the position he currently holds. This brings into question using his position as head of the NHGRI as support for appointing him to science advisor. The reason Collins is unqualified for his current position is his public displays of ignorance regarding evolutionary biology. This is important in respect to NHGRI because so much of genomics research relies heavily on evolutionary theory.


Here are a few examples of Collins displaying his ignorance:


- [He thinks humans have stopped evolving](http://scienceblogs.com/evolgen/2006/06/francis_collins_opens_mouth_in.php). The linked post is from June 2006. In the year and half since that post even more evidence has been revealed that goes against Collins’ belief.
- He thinks that [the entire human genome is functional](http://scienceblogs.com/evolgen/2007/07/junk_on_junk_1.php), an absolutely absurd idea.
- He wrote heavily about what he calls “Moral Law” in his book. He believes that current research cannot explain human morality, therefore goddidit. This is a god of the gaps argument. However, in making his argument, he disregarded large swaths of research on the evolution of altruism, thereby artificially increasing the gaps. This is brought up in [a review of Collins’ book](http://www.talkreason.org/articles/Theistic.cfm) and [this Scientific American article](ttp://www.sciam.com/article.cfm?chanID=sa006&amp;articleID=0006559D-DF6D-150E-9D8283414B7F0000) (I quoted the relevant part [here](http://scienceblogs.com/evolgen/2006/09/science_faith_and_compromise.php)).


The first two bullets reveal that he does not understand the material covered within the Institute he directs. The third point indicates he’s willing to go outside his ken and, without proper research, misrepresent what is known about a particular aspect of biology.


Please note, I’m not arguing against Collins as science advisor because of his religious beliefs. Rather, I’m pointing out that (a) he shouldn’t have the position he currently has, and (b) he does not have the appropriate understanding of science expected of a science advisor. Collins allows his non-evidence-based belief system to determine what aspects of our scientific knowledge he is willing to accept (rather than the evidence, as science should be done). This goes beyond what [Ken Miller](http://www.millerandlevine.com/km/) does — additionally, Miller is a much better popularizer of science than Collins.


So, if you want a science advisor with policy experience, pick someone other than Francis Collins. And don’t choose the person because of their religious beliefs (either religious or atheist). Choose him or her because they are a well respected scientist with experience talking about science to non-scientists.

#Teaching the Importance of Stochasticity in Biology Courses
Date: January 10, 2008

There’s a large element of “chance” in all biological systems. Whether it’s a biochemical process within a cell, the movement of cells throughout an organism, or the evolution of the those organisms, stochasticity plays a large a part in biology. Unfortunately, this is often missed by most students of biology — either because they fail to grasp the concept or it is never even presented to them.


[Yann points us to an article](http://yannklimentidis.blogspot.com/2008/01/do-students-of-evolutionary-biology.html) in PLoS Biology on teaching students how to appreciate the importance of randomness (I prefer to call it stochasticity) in biology:


> 
Klymkowsky MW, Garvin-Doxas K (2008) Recognizing Student Misconceptions through Ed’s Tools and the Biology Concept Inventory. PLoS Biol 6(1): e3 [doi:10.1371/journal.pbio.0060003](http://dx.doi.org/10.1371/journal.pbio.0060003)



The article presents some approaches that can be used by instructors when teaching about various aspects of biology to enforce the importance that stochasticity plays in all of biology.

#Science on the Opinion Pages
Date: January 10, 2008

John Timmer has a must read post on the [coverage of science in the New York Times editorial pages](http://arstechnica.com/journals/science.ars/2008/01/08/an-editorial-war-on-science). The [science section of the NYTimes](http://nytimes.com/pages/science/) is probably the best in the country. [Carl Zimmer](http://query.nytimes.com/search/query?ppds=bylL&amp;v1=CARL%20ZIMMER&amp;fdq=19960101&amp;td=sysdate&amp;sort=newest&amp;ac=CARL%20ZIMMER&amp;inline=nyt-per) leads the way, and there are many other solid science writers published by the NYTimes. Even [Nicholas Wade](http://topics.nytimes.com/top/reference/timestopics/people/w/nicholas_wade/), who has a [known diction problem](http://www.google.com/search?q=nicholas+wade&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search), manages to produce some pretty good articles.


This excellence in science reporting does not extend to the opinion pages. From [Behe’s article on Intelligent Design](http://www.nytimes.com/2005/02/07/opinion/07behe.html) to [Sam Brownback writing about evolution](http://www.nytimes.com/2007/05/31/opinion/31brownback.html) to a [questionable piece](http://www.nytimes.com/2007/11/14/opinion/lweb14brain.html) on [fMRI and politics](http://www.nytimes.com/2007/11/11/opinion/11freedman.html) the track record on the opinion pages is far from stellar. These are opinion pages, but they’re often presenting opinions that are not based on facts and reality. If the times is fine presenting unsubstantiated opinions, they lose quite a bit of credibility. That may be [the direction they’re heading in](http://www.mattbors.com/archives/331.html).

#Scientists for Better PCR
Date: January 11, 2008

Let’s pretend [this](http://bio-rad.cnpg.com/lsca/videos/ScientistsForBetterPCR/) never happened:


<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/7uafUVNkuzg&amp;rel=1"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/7uafUVNkuzg&amp;rel=1" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>


Those are the Scientists for Better PCR, complete with their own wanna-be Boss. Because when you need to find out who the daddy is, you turn to PCR.

#Competitive Release and Antibiotic Resistance
Date: January 14, 2008

While much of the research in evolutionary biology is purely academic in nature — designed for the purpose of understanding the biology of a system rather than for immediate human benefit — there is some research that yields immediate practical uses. One research area that is particular fruitful in this regard involves applying evolutionary theory toward combating infectious disease agents. The metaphor of an arms race is often used to describe the evolutionary dynamics of parasites and their hosts. This true for the naturally evolved reactions in hosts, but also for human engineered treatments such as antibiotics.


[<img alt="Blogging on Peer-Reviewed Research" height="85" src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" width="70"/>](http://www.researchblogging.org/)


The evolutionary approach toward fighting infectious microbes with antibiotics has led to a few accepted paradigms in the public consciousness. For example, in industrialized western nations, doctors are advised to prescribe antibiotics for only a small, select group of patients. Those patients receive high doses of antibiotics and are told to finish the prescribed bottle of pills even if they begin to feel better. We’ll get to the theory behind this in a moment, however the basic idea is that this approach is intended to prevent the spread of antibiotic resistance amongst the pathogenic microbes.


> 
The advantage microbes gain from their innate adaptability is augmented by the widespread and sometimes inappropriate use of antibiotics. A physician, wishing to placate an insistent patient who has a virus or an as-yet undiagnosed condition, sometimes inappropriately prescribes antibiotics. ****Also, when a patient does not finish taking a prescription for antibiotics, some bacteria may remain. These bacterial survivors are more likely to develop resistance and spread.**** Hospitals also provide a fertile environment for antibiotic-resistant germs as close contact among sick patients and extensive use of antibiotics select for resistant bacteria. Scientists also believe that the practice of adding antibiotics to agricultural feed promotes drug resistance. [[The Problem of Antimicrobial Resistance](http://www.niaid.nih.gov/factsheets/antimicro.htm), NIAID Fact Sheet]



The point I have emphasized in the quote above is the focus of this essay, and you can find such advice in [many other places](http://www.google.com/search?q=antibiotic+resistance+prescription+finish). Other precautions are also employed to prevent the spread of antibiotic resistance — the unnecessary use of antibiotics (i.e., for viral infections) and the strong selective pressures in hospitals are both major public health concerns. But does evolutionary theory support the idea that patients should finish the long antibiotic prescriptions as they are currently advised? Does this approach prevent the spread of antibiotic resistance or does it encourage it? A [recent paper in PNAS addresses this issue using a mouse model of malaria](http://dx.doi.org/10.1073/pnas.0707766104), and the authors come to a conclusion that goes against the accepted paradigm.





Microbes can be classified as resistant to antibiotics or sensitive to them. Resistant individuals can tolerate the presence of an antibiotic, while sensitive ones are unable to reproduce in such conditions. The current approach of antibiotic treatments is designed to purge a host of a large swath of microbes. This results in the near total elimination of sensitive strains, which allows for an environment devoid of competitors for the resistant strains. The theory upon which this approach is based focuses on eliminating new resistant strains from arising by removing any lineages which may give rise to resistant strains. However, antibiotic resistance often arises via very simple mutational processes (sometimes even by a single mutation, and [in the case of malaria](http://www.pandasthumb.org/archives/2007/10/full-text-of-th.html), by a series of mutations that are very accessible).


In their new paper, Wargo et al. argue the removal of sensitive bacteria from a host releases the resistant individuals from competition for various intra-host resources. They support their claim with data from experiments in which mice were infected with *Plasmodium chabaudi*, a relative of the parasite which causes malaria in humans, *P. falciparum*. The mice were either infected with a strain of *P. chabaudi* resistant to an antimalarial drug, a strain sensitive to the drug, or co-infected with both strains. The amount of pathogens from each strain within an individual was inferred using qualitative PCR and primers that were specific to each strain.


[<img alt="i-943c723eadfa66cfdc962beb039a07c7-Wargo_et_al_fig2.jpeg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-943c723eadfa66cfdc962beb039a07c7-Wargo_et_al_fig2.jpeg"/>](htt://dx.doi.org/10.1073/pnas.0707766104)


In the two graphs shown above, the number of resistant individuals is plotted for four experiments (the two graphs represent two different life stages of the parasite). In each graph, the solid line represents data taken from individuals infected with only the resistant strain, while the dotted line is from individuals infected with both the sensitive and resistant strains. The two data points are for individuals that were not treated with the antimalarial drug (“0 Days drug”) and those treated with the drug (“4 Days drug”).


As you can see, in the individuals that were not treated with the drug, the mice infected with only the resistant strain have either more or the same number of resistant clones within them as the individuals treated with both the resistant and sensitive strains. That’s because, in the co-infected individuals, the two strains are competing for limited resources. This competition limits the total number of resistant individuals present. However, when the inter-strain competition is removed (via treatment with the drug), the number of resistant clones in the co-infected individuals surpasses that found in the individuals infected with only the resistant strain. Therefore, competitive release (i.e., the removal of competition with the sensitive strain) increases the amount of resistant pathogens in the host relative to hosts which were not treated with antibiotics.


This raises a concern about antibiotic treatments — you can actually increase the number of resistant pathogens found within a co-infected host by administering antibiotics. But the Wargo et al. were also interested in examining how different antibiotic regimens affected parasite density within the host. They performed a similar experiment as the one described above, but treated the mice with the antimalarial drug for either 0 days (no treatment), 1 day, or 2 days.


[<img alt="i-38f6804c1c0865995849675ffde74ecd-Wargo_et_al_fig4.jpeg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-38f6804c1c0865995849675ffde74ecd-Wargo_et_al_fig4.jpeg"/>](htt://dx.doi.org/10.1073/pnas.0707766104)


One day of treatment for co-infected mice increased the amount of resistant parasites relative to untreated individuals. However, co-infected hosts and hosts infected with only the resistant parasite had equal numbers of resistant pathogens after a single day of treatment. It was only after two days of antimalarial treatment that competitive release became apparent.


These results suggest that heavy doses of an antibiotic actually increase the number of resistant pathogens relative to more moderate treatments. This goes against the current paradigm of giving lots of antibiotics to an individual when they come down with an infection and urging them to complete the prescription regardless of whether they begin to feel better. This approach is designed to prevent resistant alleles from arising in a population, but it also gives resistant strains, once they’ve arisen, a very large selective advantage over sensitive strains.


Population genetic theory predicts that the greater the selection pressure in favor of a new beneficial mutation, the faster that new allele will increase in frequency, eventually becoming fixed in the population. The two graphs below show how fast an allele will fix in a population depending on the strength of selection. With strong selection (top graph) the resistance allele fixes faster that with weak selection (bottom graph). In the malaria example, strong selection is giving out lots of antibiotics and having patients take them for a long time.


<img alt="i-3c0e2b8309970fc4682fb95689775048-malaria_seln.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-3c0e2b8309970fc4682fb95689775048-malaria_seln.gif"/>


The current paradigm doesn’t apply the strongest selection pressure possible. For example, doctors are discouraged from prescribing antibiotics for viral infections. Additionally, other over-prescriptions of antibiotics are frowned upon. However, the large dose given to patients applies a stronger selection pressure than if more moderate doses were prescribed. These large doses are designed to prevent the appearance of mutations conferring antibiotic resistance, but these mutations arise regardless. Rather than attempting to extend the period until a resistance mutation arises, the authors argue that it would be more fruitful to prevent the dramatic increase in the resistance allele that occurs when large doses of antibiotics are prescribed.


Here’s where you’re supposed to ask: that’s great, but what about the illness experienced by the infected individuals? In other words, so what if there is a lower frequency of resistance, what if more people are sick if you give smaller doses? Well, it turns out that there isn’t a difference in mortality between the mice treated with low doses of antibiotics (1 day) and those treated with high doses (2 days). That means, in this model, you can prevent the spread of a resistance allele without causing undue suffering in the infected individuals. If you keep the dosage low, the antibiotic is effective for much longer without a cost in the health of the host.


Should this approach be implemented for human malaria and other non-viral infections? Hell no! But this approach toward antibiotic use should be examined in humans. If we can extend the life of an antibiotic without health consequences, much money will be saved and human suffering will be reduced. That’s because once the resistance allele reaches fixation, the antibiotic is no longer useful. This can’t be examined in humans using the same controlled experimentation as in mice, but I wonder if some clever epidemiologists can come up with a way to determine whether this approach is feasible.


---



Wargo AR, Huijben S, de Roode JC, Shepherd J, Read AF. 2007. Competitive release and facilitation of drug-resistant parasites after therapeutic chemotherapy in a rodent malaria model. PNAS 104: 19914-19919 [doi:10.1073/pnas.0707766104](http://dx.doi.org/10.1073/pnas.0707766104)

#Gene Chips and Conspiracy Theories
Date: January 15, 2008

What’s a gene chip? Well, it’s not a **gene** chip, that’s for sure. It’s a microarray. It has various parts of a genome arrayed on a small chip. The parts of the genome you put on the chip depends on what you’re interested in studying. You then take some DNA you’re interested in and apply it to the chip. If a DNA sequence matches a sequence on the chip, it hybridizes to the part of the chip with the matching DNA sequencing. You then use a special technology to visualize which parts of the chip have DNA hybridized to them.


This technology can be used to differentiate between different single nucleotide polymorphisms (SNPs) across an entire genome. It can also be used to quantify transcripts of various genes in a genome based on how much hybridization occurs on the array. And it can be used to count the number of copies of a particular sequence in a genome; this is a way of quantifying copy number polymorphism.


[Illumina](http://www.illumina.com/) is one of the leading makers of gene chips. They’re in cahoots with both [23andMe](https://www.23andme.com/) and [deCODEme](http://www.decodeme.com/). So, watch out, ’cause they’re out to rule the world.


<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/M87SuvQ2qKI&amp;rel=1"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/M87SuvQ2qKI&amp;rel=1" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>


If you ignore butchering of the biology at the beginning, it’s a pretty amusing video. Better watch out or they’ll discover your genetic code, man.


(Via [BBGM](http://mndoci.com/blog/2008/01/13/wallstrip-on-illumina/).)

#How Easy is it to Write About Junk DNA?
Date: January 16, 2008

[Alex is pissed about science writers neglecting important discoveries in cell biology](http://scienceblogs.com/transcript/2008/01/todays_rant_the_biggest_story.php):


> 
Why are cell biology, molecular biology, biochemistry, microbiology never covered in the media? I’ve spoken to so many science journalists – most of whom have no science training. I’ve come to the conclusion that the barrier is too high – as a result when it comes time to write about these topics, most science journalists end up writing about “genomes” and “junk DNA”. These are easy subjects – sometimes they’re discussed within the framework of evolution, but never within the context of “how a cell or an organism operates”.



Really Alex? Junk DNA is an easy subject? Let’s see how [Greg Laden covers junk DNA](http://scienceblogs.com/gregladen/2008/01/genes_are_only_part_of_the_sto.php):


> 
The “Junk DNA” story is largely a myth, as you probably already know. DNA does not have to code for one of the few tens of thousands of proteins or enzymes known for any given animal, for example, to have a function. We know that. But we actually don’t know a lot more than that, or more exactly, there is not a widely accepted dogma for the role of “non-coding DNA.” It does really seem that scientists assumed for too long that there was no function in the DNA.



Ouch! Now, Greg is no geneticist. Thankfully, [T. Ryan Gregory puts Greg in his place](http://genomicron.blogspot.com/2008/01/signs-of-function-in-non-coding-rnas-in.html). All that hype about newly discovered functions for non-coding DNA? It’s just that: hype. Don’t believe it. We know that some non-protein-coding DNA has function. Despite that, the majority of the human genome is non-functional. It’s junk. It’s a graveyard of self-replicating crap that accumulates and doesn’t get eliminated because the fitness cost isn’t deleterious enough.


As for science writers, they don’t understand junk DNA either. Alex, most science writers can cover the topic of junk DNA about as well as they can cover [signal sequences that promote nuclear export of mRNA](http://dx.doi.org/10.1371/journal.pbio.0050322).

#Lewontin Reviews Gould
Date: January 30, 2008

Everyone’s blogging about Stephen J. Gould’s **Structure of Evolutionary Theory** ([Razib](http://scienceblogs.com/gnxp/2008/01/the_structure_of_evolutionary.php), [John Lynch](http://scienceblogs.com/strangerfruit/2008/01/setting_off_on_a_long_journey.php), [Laelaps](http://scienceblogs.com/laelaps/2008/01/blogging_the_structure_of_evol.php)). I’m not. The book’s too long, and I’m too busy. But that doesn’t mean I can’t link to them, and to another review of Gould. The other is [Richard Lewontin’s review of two Gould books](http://www.nybooks.com/articles/21033): *The Richness of Life: The Essential Stephen Jay Gould* and *Punctuated Equilibrium*. The latter is a chapter in *Structure of Evolutionary Theory*. Yes, a chapter of a book was released as a book on its own.


[Lewontin’s review of the two books](http://www.nybooks.com/articles/21033) isn’t really a review of the two books. It’s more of a eulogy or an obit for Gould written a couple of years to late. Of course, it’s published in the New York Review of Books, which, from what I can gather, doesn’t really publish book reviews. Anyway, it’s an interesting piece, but don’t look to it for information about the two books it’s allegedly reviewing.


That said, one passage really caught my eye:


> 
Of the best known and most active of active public intellectuals, only two have resisted the impulse to invent and advertise theories of human nature, its evolution, and its manifestation in history and social institutions. One was Carl Sagan, who largely avoided grand theories of humanity and, with the modesty appropriate to an astronomer, stuck to explaining the universe. The other was Stephen Jay Gould.



So, how do you explain the reissue of **[The Mismeasure of Man](http://en.wikipedia.org/wiki/The_Mismeasure_of_Man)**? No matter which side of the bell curve you fall on, that was done to address theories of human nature, its evolution, and its manifestation in history and social institutions.


(Via [Lynch](http://scienceblogs.com/strangerfruit/).)

#Phylogeny Friday — 1 February 2008
Date: February 1, 2008

**<u>Paraphyly in Drosophila</u>**


Many biology students have hands-on experience working with *Drosophila melanogaster*. This little fly is one of the major workhorses of genetics. It may not be for long. That’s not to say people will stop working with the fly, but the fly may no longer be named “*Drosophila melanogaster*“. That’s because the *Drosophila* genus is paraphyletic and should be split into multiple genera. Oh, and *D. melanogaster* doesn’t hold the rights to the name *Drosophila*. Those rights belong to *D. funebris*, the first species named in the *Drosophila* genus. ([Christopher Taylor at the Catalogue of Organisms has more](http://catalogue-of-organisms.blogspot.com/2008/01/drosophila-forever.html).)


So, if *D. melanogaster* loses the rights to the genus name “Drosophila”, what will it be named? Well, the Drosophila genus can be divided into two subgenera. One of those subgenera is also named “Drosophila”. The other subgenus, Sophophora, contains *D. melanogaster*, amongst many other species. Therefore, [*D. melanogaster* should be renamed “*Sophophora melanogaster*](http://sandwalk.blogspot.com/2008/01/sophophora-new-model-organism.html) if the genus were split into multiple genera.





Here’s a phylogeny of the subset of the *Drosophila* with completely sequenced genomes:


[<img alt="i-966ae5c22c3bdd72706f213febef43ae-Dros_tree.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-966ae5c22c3bdd72706f213febef43ae-Dros_tree.gif"/>](http://rana.lbl.gov/drosophila/)


Notice *D. melanogaster* amongst the Sophophoran species. Only this tree is extremely stripped down, omitting all the genera nested within the *Drosophila* genus. This is a much better tree:


[<img alt="i-8d3973231187b4aecf0f33e474b4849f-markow_ogrady_dros_tree.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-8d3973231187b4aecf0f33e474b4849f-markow_ogrady_dros_tree.gif"/>](http://dx.doi.org/10.1146/annurev.genet.39.073003.112454)


Notice the four other genera nested within the Sophophora side of the phylogeny: Chymomyza, Scaptodrosophila, Zaprionus, and Dorsilopha. The last one in that list was named for the sake of being confused with Drosophila. And the Drosophila subgenus is intermingled with three other genera: Siphlodora, Engiscaptomyza, and Scaptomyza. It’s all one big mess.


So, will *Drosophila melanogaster* be ranamed *Sophophora melanogaster*? I doubt it. It’s up to taxonomists, and they never get anything done. This basically comes down to taxonomic precedence versus changing the name of one of the best known species. Neither will win, and we’re gonna be left with a paraphyletic taxon. Once again, taxonomy is meaningless.


---



Markow TA and O’Grady PM. 2005. Evolutionary genetics of reproductive behavior in **Drosophila** Annu Rev Genet 39: 263-291 [doi:10.1146/annurev.genet.39.073003.112454](http://dx.doi.org/10.1146/annurev.genet.39.073003.112454)

#What do you call your advisor?
Date: February 10, 2008

Not Jim or Sam or Sally or Jane. Or Dr. Acula. But how do you refer to your advisor when you’re talking to someone else? Do you call him/her “my advisor”? Do you call him/her “my boss”? Do you call him/her “that person I see once every two weeks who provides the money for me to get my PhD”?


This is a Sunday unScientific Survey. Leave your answer in the comments.

#Jonathan Eisen at the Advances in Genome Biology and Technology Meeting
Date: February 12, 2008

Jonathan Eisen’s been blogging the Advances in Genome Biology and Technology ([AGBT](http://agbt.org)) meeting in [Marco Island](http://www.marco-island-florida.com/). [Here’s what he’s written so far](http://phylogenomics.blogspot.com/search/label/Marco%20Island%20AGBT%20Meeting%202008?max-results=20):


- [Advances in Genome Biology and Technology Meeting – First Post](http://phylogenomics.blogspot.com/2008/02/advances-in-genome-biology-and.html)
- [AGBT Marco Usland Update – Long Live Sequencing](http://phylogenomics.blogspot.com/2008/02/agbt-marco-usland-update-long-live.html)
- [Marco Island Evening One – The Strange and the Good](http://phylogenomics.blogspot.com/2008/02/marco-island-evening-one-strange-and.html)
- [More notes from Marco Island/ AGBT](http://phylogenomics.blogspot.com/2008/02/more-notes-from-marco-island-agbt.html)
- [Coolest Thing at Marco Island – The Polonator](http://phylogenomics.blogspot.com/2008/02/coolest-thing-at-marco-island-polonator.html)
- [Marco Island sequencing frenzy – are we getting lost in all the data?](http://phylogenomics.blogspot.com/2008/02/marco-island-sequencing-frenzy-are-we.html)
- [Marco Island – Saving Some of the Best for Last](http://phylogenomics.blogspot.com/2008/02/marco-island-saving-some-of-best-for.html)


The second to last post is the best, in my opinion. But there’s some interesting points about genome sequencing scattered throughout.

#These Are the People at Your Departmental Seminar
Date: February 15, 2008

Do you ever sit in a boring departmental seminar and scope out the other folks in the room? You’ll pick up some odd behaviors. Like the guy picking his nose — gross! Or the secret couple that can’t be open about their relationship because it breaks some university policy sitting a bit too close to one another. Well, here are a few folks that you’ll probably see in every departmental seminar:


<li>
**The nodder:** This guy affirms every part of the talk with a nod. The background information — he gets it. The data — he gets it. The conclusions drawn from arm waving and rampant speculation — he gets it. In reality, he’s just trying to show off how smart he thinks he is. With every nod, he’s saying, “Look at how smart I am. I understand all this stuff. Do you? I bet you don’t.” At the end of the talk, I’d like to make sure he gets it. And by “it” I mean my fist. And by “get” I mean square between the eyes.
</li>





<li>
**The critic:** Unlike the nodder, this guy don’t dig the balls the speaker is throwing. He ain’t playing that game. The speaker puts up a suspect piece of data, and the critic shakes his head. Uh oh! You know this guy will be asking some hardcore questions at the end of the seminar. He’ll be busting chops like chops have never been busted before. This motherfucker may even leave the speaker curled up in a ball soaked in a pool of his own urine on the floor. That’s the speaker’s urine, not the critic’s. Peeing on a guy would just be gross.
</li>
<li>
**The pisser:** This guy finds one tiny, miniscule, insignificant point and asks a long drawn-out question about it. Unlike the critic, who asks intelligent questions that get at the underlying assumptions and theory of the work at hand, the pisser is just taking a piss. He’s stupid, uninsightful, and doesn’t understand science.
</li>
<li>
**The pre-schooler:** This dude uses the departmental seminar as his nap-time. He sits in the back, and when the lights go out, he’s nodding off faster than River Phoenix in *My Own Private Idaho*. Someone get this guy a good-night’s sleep.
</li>
<li>
**The rookie:** The pre-schooler knows seminar time is nap time. The rookie is an undergrad or first-year grad student who’s new to the whole departmental seminar thing. He’s overworked, he undersleeps, and he’s sitting in the middle of the room. He nods off throughout the seminar, and everyone’s looking at him. Wake up boy, and get your act together. And next time, sit in the back of the room with the pre-schooler.
</li>
<li>
**The grand advertiser:** No matter how unrelated, insignificant, and uninteresting this guy’s research is, he’ll find a way to link it to the topic of the seminar. And no one will give a shit. The speaker will pay lip-service to the question, but we’ve already died a little inside because the grand advertiser has wasted our time advertising the crap his lab studies.
</li>


What’s missing from the list? And if you don’t recognize one of these, it most likely describes you.

#Where are the faculty bloggers?
Date: February 24, 2008

I have an intuition, backed up by absolutely no evidence, that my particular area of interest (evolutionary genetics) has more faculty blogging about stuff related to their research than other fields. This is most likely the result of my interest in those blogs, and, hence, my increased awareness of them compared to blogs of faculty in other research areas. From a quick scan of my blogroll and the feeds I subscribe to, here’s a list of research faculty who blog about evolutionary genetics:


- [TR Gregory](http://genomicron.blogspot.com/index.html)
- [Rod Page](http://iphylo.blogspot.com/)
- [John Hawks](http://johnhawks.net/weblog)
- [John Logsdon](http://johnlogsdon.blogspot.com/index.html)
- [Jonathan Eisen](http://phylogenomics.blogspot.com/index.html)
- [Larry Moran](http://sandwalk.blogspot.com/index.html)


That’s not a lot of blogs, but it’s also not a huge field. How does that compare with faculty who blog in your research area?


**Note:** I’ve restricted the list to faculty who can be considered PIs of an active research lab. You can argue all you like about whether such a list should be restricted to this group, but that’s the group I’m interested in. I’m also not looking at totally anonymous bloggers because, well, we don’t know who they are. Also, I’m most interested in faculty blogging about stuff related to their research.

#The Eisens Continue to Rock the PLoS
Date: February 26, 2008

[Jonathan Eisen is the new Academic Editor in Chief at PLoS Biology](http://phylogenomics.blogspot.com/2008/02/plos-biology-20.html), and he’s [kicking it off with this editorial](http://dx.doi.org/10.1371/journal.pbio.0060048). In his editorial, Jonathan describes how he became an Open Access publishing advocate. The header of his article features a short biography with an interesting item:


> 
Jonathan A. Eisen is Academic Editor-in-Chief at PLoS Biology. He is also at the University of California Davis Genome Center, with joint appointments in the Section of Evolution and Ecology and the Department of Medical Microbiology and Immunology, University of California Davis, Davis, California, United States of America. E-mail: jaeisen@ucdavis.edu. Blog: http://phylogenomics.blogspot.com.



It’s a link to Jonathan’s blog! [Speak of the devil](http://scienceblogs.com/evolgen/2008/02/where_are_the_faculty_bloggers.php).


This is also further evidence that the Eisens are taking over PLoS. Jonathan’s brother, [Mike](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c493101445894113e58908b592d9da16-18_eisen_toast3_sm.JPG), is one of the [co-founders of PLoS](http://www.plos.org/about/board.html#eisen).

#Encyclopedia of Life
Date: February 28, 2008

*Poorly done and over clicked?*


Rod Page has a [post worth reading in which he’s “deliberately critical”](http://iphylo.blogspot.com/2008/02/encyclopedia-of-life-first-impressions.html) of the Encyclopedia of Life ([EOL](http://www.eol.org/)). You should be able to visit the [EOL](http://www.eol.org/) at that link. Only you can’t, as of two days following the release. You see, [they weren’t able to handle the 11.5 million hits](http://news.nationalgeographic.com/news/2008/02/080227-AP-encyclopedi.html) they received within the first few hours of going live. We’re only left to wonder whether this thing will ever get off the ground.

#Inferring Demographic History Using Multiple Loci
Date: March 2, 2008

One of the drums I beat around here pertains to inferring demographic history using molecular markers (i.e., DNA data). I’ve been known to go off on people who make claims about ancestral population sizes based on studies of a single locus or gene. You see, studying a single locus only gives you the evolutionary history of that locus. There is no way to untangle the affects of natural selection from those of demography without examining multiple loci.


The coalescent is a popular statistical technique used to study DNA sequence polymorphism. Combining bayesian analysis with coalescent theory has led to some advancements in inferring demographic history. Alexei Drummond and colleagues developed what they call a [Bayesian Skyline Plot](http://dx.doi.org/10.1093/molbev/msi103), which allows you to infer historical changes in population sizes. On his blog, [Drummond describes an extension to the Bayesian Skyline Plot which takes advantage of data from multiple loci](http://bioinf.cs.auckland.ac.nz/index.php/2008/03/02/the-coalescent-for-bottlenecks/). Here’s what they found:


> 
These results demonstrate the essential role of multi-locus data in recovering complex population dynamics. Multi-locus data from a small number of individuals can precisely recover past bottlenecks in population size which can not be characterized with a single locus. However typical data sets used today are probably too small for obtaining precise estimates of population history and providing information on past bottlenecks.



The example given is one of multiple bottlenecks. When looking at a single locus, one can only detect one bottleneck event. However, examining multiple loci allows one to detect bottleneck events that occurred prior to the most recent event. I’d be hesitant to infer ****any**** bottleneck events from a single locus for the reasons I gave above.

#Charlesworth on Lynch
Date: March 5, 2008

[Brian Charlesworth has reviewed Michael Lynch’s *The Origins of Genome Architecture*](http://dx.doi.org/10.1016/j.cub.2008.01.008) for Current Biology. Charlesworth’s review is generally positive, and he agrees that population size may be an important factor in genome evolution. However, he thinks that Lynch overplays the role relaxed selective constraint in small populations plays in the evolution of genomic complexity.


Charlesworth argues that sexual reproduction may be partly to blame for some of the features found in the bloated genomes of many eukaryotes. For example, the abundance of transposable elements may be the result of sexual reproduction, not relaxed constraint:


> 
It is true that, on average, bacteria have much large N<sub>e</sub> values than most eukaryotes for which we currently have data, but they differ in numerous other respects as well, for example lack of regular sexual reproduction. As all good comparative biologists know, it is very difficult to disentangle cause and correlation from wide comparisons. Alternatives to many of Lynch’s explanations of the patterns can be envisaged, and his arguments do not seem to rule these out. For example, as he himself describes in Chapter 7, the spread of transposable elements through the genomes of a host population is dependent on some degree of sexual exchange between members of the populations, and the correlations described by Lynch could thus at least partly be explained by lack of such exchange.



Additionally, introns may be a byproduct of sexual reproduction:


> 
Could it be that the invention of regular sexual reproduction made it easier for mobile, initially self-splicing introns to invade the genome in large numbers? This possibility is not explored by Lynch, who resorts (p. 261) to the untestable hypothesis that there was a long period of reduced N<sub>e</sub> among ancestral eukaryotes. This is getting dangerously close to the adaptationist just-so stories that he ridicules in the final chapter.



Is Lynch committing to a [frail nearly neutral hypothesis](http://scienceblogs.com/evolgen/2007/07/the_frailty_of_nearly_neutral.php)?


So, the book appears a bit biased. Oh, well. It makes some interesting arguments that warrant further inspection. Those arguments, however, should be taken with a grain of salt, as there may be adequate alternatives to the explanations proposed by Lynch.

#Stuff White ScienceBloggers Like
Date: March 6, 2008

Apparently, [ScienceBlogs is loaded with white people](http://whitescienceblogs.googlepages.com/index.html). Hell, the [whitest person I know](http://scienceblogs.com/gnxp/) blogs in this very domain. That got me thinking. Sure, we may look white. But are we really white? I mean, ****really**** white. So white that we like the [stuff white people like](http://stuffwhitepeoplelike.wordpress.com/).





We do have someone who really likes [graduate school](http://stuffwhitepeoplelike.wordpress.com/2008/03/04/81-graduate-school/):


[<img src="http://scienceblogs.com/clock/Reshaven.jpg"/>](http://scienceblogs.com/clock/)


And we’ve got a [Canadian](http://stuffwhitepeoplelike.wordpress.com/2008/02/24/75-threatening-to-move-to-canada/):


[<img src="http://scienceblogs.com/shiftingbaselines/JacquetSEED.jpg"/>](http://scienceblogs.com/shiftingbaselines/)


And someone who likes to [study abroad](http://stuffwhitepeoplelike.wordpress.com/2008/02/22/72-study-abroad/):


[<img src="http://scienceblogs.com/scientificactivist/upload/2006/06/scientificactivistprofile.gif"/>](http://scienceblogs.com/scientificactivist/)


We look pretty white, huh? Well, it gets whiter. We’ve got a [lawyer](http://stuffwhitepeoplelike.wordpress.com/2008/02/06/56-lawyers/):


[<img src="http://scienceblogs.com/denialism/images/chris_hoof.jpg"/>](http://scienceblogs.com/denialism/)


A [dog](http://stuffwhitepeoplelike.wordpress.com/2008/02/04/53-dogs/) owner:


[<img src="http://scienceblogs.com/islandofdoubt/upload/2006/12/me-fergus.jpg"/>](http://scienceblogs.com/islandofdoubt/)


Guys who like [living by the water](http://stuffwhitepeoplelike.wordpress.com/2008/02/04/51-living-by-the-water/):


[<img src="http://scienceblogs.com/deepseanews/upload/2007/01/scubacraig.jpg"/>](http://scienceblogs.com/deepseanews/)[<img src="http://scienceblogs.com/deepseanews/upload/2007/01/peter_chinchorro.jpg"/>](http://scienceblogs.com/deepseanews/)


And [marathon](http://stuffwhitepeoplelike.wordpress.com/2008/01/26/27-marathons) runner:


[<img src="http://scienceblogs.com/bushwells/02_BAA.jpg"/>](http://scienceblogs.com/bushwells/)


Man, we’re the whitest group of whities I’ve ever been blinded by when the sun reflected off their pasty white skin. White people — they suck!

#Judson on Junk and Genome Size
Date: March 6, 2008

Olivia Judson (aka, [Dr. Tatiana](http://www.drtatiana.com/book.shtml)) has a [blog at the NYTimes website](http://judson.blogs.nytimes.com). It’s usually a good read, but [she has been known to go off the deep end](http://scienceblogs.com/loom/2008/01/24/hopeless_monstersa_guest_post.php). In this week’s entry, Judson posts on how [bones are not the only fossils](http://judson.blogs.nytimes.com/2008/03/04/bones-are-not-the-only-fossils/). What other fossils does Judson write about? Genomes.


Judson’s focus is on genome size. She’s clear that differences in genome size are the result of stuff other than genes, but she doesn’t want to call that extra stuff “junk”. Here’s why:


> 
I don’t like “junk,” which suggests the DNA is useless: even if it doesn’t hold the instructions for making proteins, it may still serve a valuable purpose. For instance, stretches of non-coding DNA between parts of a gene can regulate how fast the protein gets made.



Um, Olivia, most of that stuff is useless. It’s there to be there. We know because of positive evidence that a lot of the extra stuff is well characterized junk. Don’t believe me? Read the posts at [Genomicron](http://genomicron.blogspot.com/2008/02/junk-dna-quotes-of-interest-series.html).


But that’s not Judson’s main focus, so I’ll lay off. Instead, she focuses on the marks left by genome size in fossils. For example, we can study cell sizes in fossils and use those to infer the size of the genomes of those extinct organisms. That was done in a study of [dino-genomics](http://scienceblogs.com/evolgen/2007/03/your_bones_got_a_little_genome.php), which found that the small genome sizes of birds evolved tens of millions of years before birds took flight. That threw a wrench in the hypothesis that small genomes in birds were selected for flight.


I was surprised that, in a discussion of genomic fossils, that Judson chose to focus on the literal fossils of genomes. I expected a discussion of the fossils ****in**** genomes. You know, the transposable elements and such that make up all that junk in the genome.

#Genomics and DNA Barcoding
Date: March 7, 2008

Identifying and cataloging biological diversity is challenging. One way to do go about IDing all the life forms is to sequence a known region of the genome in all those species. This is known as DNA barcoding. An article in PNAS reports on the [DNA sequence of a gene found useful for DNA barcoding in plants](http://dx.doi.org/10.1073/pnas.0709936105  ). In a [review of the paper](http://dx.doi.org/10.1073/pnas.0800476105  ), the following table is presented:

||DNA Barcoding|Genomics
|Number of species|All (or most)|One (or few)
|Number of gene regions|One (or few)|All (or most)


The gist: DNA barcoding results in the sequencing of a single gene in a bunch of species, while genome sequencing gives us the sequence of an entire genome in a single species. This may be true now, but for how long? The dropping price of sequencing will allow us to get information from many genomic regions in many species. These won’t be high quality whole genome sequences, but the age of doing DNA barcoding with a single gene won’t last for long.

#The Demise of PLoS?
Date: March 7, 2008

I really like the [PLoS](http://www.plos.org/) journals. Their mission — to make research freely available — is totally awesome. And, on top of that, the journals publish very interesting research. PLoS Biology is a top notch journal, with papers on par with those in Science and Nature. And the specialty journals, like [PLoS Genetics](http://www.plosgenetics.org/) and [PLoS Computational Biology](http://www.ploscompbiol.org/), consistently have articles that I find quite attractive.


But the website is [totally fucked](http://www.plos.org/cms/node/334). I can’t access most of the articles in PLoS Genetics. Thankfully, the articles are all mirrored on [PubMed Central](http://www.pubmedcentral.nih.gov/tocrender.fcgi?journal=331&amp;action=archive), but it’s a major pain to track those down after following a failed link to the PLoS Genetics site. If you can’t access the articles on a journal’s website, what good is the website?


Hopefully the problem will be fixed soon. It’s been going on for a the past few days. Will this be the demise of PLoS? If you can’t read the articles, the journal is useless.

#Not all next-gen sequencing technologies are created equal
Date: March 11, 2008

The Next Generation Sequencing blog has a post on [low coverage of A/T regions with Solexa sequencing](http://www.clcngs.com/2008/03/11/evaluating-the-illuminasolexa-genome-analyzer-for-whole-genome-re-sequencing/). The post is in reference to a paper in Nature Methods on genome resequencing in *C. elegans* ([doi:10.1038/nmeth.1179](http://dx.doi.org/10.1038/nmeth.1179)). Here’s how the NextGen Sequencing blog summarizes it:


> 
However, it points to a general lack of coverage in A/T rich regions (see figure 2 of the supplementary material) which leaves a number of zero size gaps in the assembly – places where reads sit shoulder to shoulder but simply do not overlap. Having found these problematic A/T rich regions, the authors went back and took a look across the genome, where they found a general correlation between A/T content and read coverage. This correlation was stronger when examining a 200 bp window than when examining a 32 bp window. 200 bp corresponds to the size of the amplicons that are amplifying during the cluster generation step prior to sequencing and 32 bp corresponds to the number of cycles in the actual sequencing by synthesis procedure. This finding made Hillier et al. conclude that failure to amplify A/T rich regions during cluster generation is the cause of the low coverage (other reasons for the bias such as hairpin formation were also explored but discarded).



This is an issue if Solexa is to become a dominant way to resequence genomes. However, there are other applications of the Solexa technology that will probably not be affected. These include using [Solexa to quantify gene expression](http://dx.doi.org/10.1038/nmeth0506-401) and to genotype known variants segregating in a population (both of these jobs are currently dominated by microarrays). The low read coverage in A/T rich regions shouldn’t affect the genotyping of known variants. Problems will arise when the goal of a resequencing project is to identify novel variants. However, [454 sequencing](http://en.wikipedia.org/wiki/454_Life_Sciences) should work well for to achieve that goal.


As a related aside, if you don’t know much about next generation sequencing, but would like to learn, check out these two reviews:


Mardisa ER. 2008. The impact of next-generation sequencing technology on genetics. Trends Genet 24: 133-141 [doi:10.1016/j.tig.2007.12.007](http://dx.doi.org/10.1016/j.tig.2007.12.007)


von Bubnoffa A. 2008. Next-Generation Sequencing: The Race Is On. Cell 132: 721-723 [doi:10.1016/j.cell.2008.02.028](http://dx.doi.org/10.1016/j.cell.2008.02.028)

#Try this line in a bar this weekend:
Date: March 13, 2008

[<img alt="i-ab6cc2981fb2292a9479f9ab6f3fe848-diesel_junkDNA_sm.PNG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ab6cc2981fb2292a9479f9ab6f3fe848-diesel_junkDNA_sm.PNG"/>](http://www.dieselsweeties.com/archive/1969)


It’s not very funny, but it’s about a topic that [comes up around here often](http://www.google.com/search?q=junk+dna&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search) (groans…). That said, this is something you’d expect to see over at [Genomicron](http://genomicron.blogspot.com/2008/02/junk-dna-quotes-of-interest-series.html), not evolgen. In fact, the guy looks a bit like a young [TR Gregory](http://www.genomesize.com/gregorylab/people/#_0).


For the full strip, go [here](http://www.dieselsweeties.com/archive/1969).

#Sleep, Sex, and Drosophila
Date: March 18, 2008

60 Minutes ran a special on the [science of sleep](http://www.cbsnews.com/stories/2008/03/14/60minutes/main3939721.shtml) this week. The special included an interview with [Scott McRobert](http://www.sju.edu/biology/smcrober.htm) about sleep deprivation and mating in Drosophila.


> So if lack of sleep impacts our appetite, our metabolism, our memory, and how we age, is there anything it doesn’t affect? How about sex? Scientist Scott McRobert at St. Joseph’s University in Philadelphia is asking that very question, studying fruit flies.


McRobert could be seen showing Lesley Stahl a fly in a vial, which he then sucked up in a pooter. He then placed that male, along with a female, in a small dish.


> Stahl watched as McRobert used a bizarre contraption to suck a male drosophila (fruit fly) out of a vial and put him into a little dish with a female.


McRobert does the play by play as the male courts the female.


> 
McRobert gave Stahl a play-by-play of the action. “Okay. So now, the female’s walking around the outside of the chamber,” McRobert explains. “And the male’s in the center. And you see he’s orienting toward her, everywhere she goes.”
“He’s following her. If you watch closely, he’ll touch her with his front legs. It’s hard to see, but he will. And he’ll sing. Here comes the song,” McRobert says.



That’s right, [Drosophila can sing](http://dx.doi.org/10.1006/anbe.1999.1167).


> Flies sing, he tells Stahl, by lifting one wing to the side and vibrating it up and down.


And then: they do it! That’s right, CBS showed footage of S-E-X on the TeeVee. Really. It was HAWHT!!!





But the special is on sleep, so they then do the same experiment with a sleep-deprived male. This male’s courtship behavior can be described as disinterested. That’s right, sleep-deprivation makes flies less horny. Rather than sing to the female, the male spends his time rubbing his hands together. It’s just sad.


McRobert, however, is tentative to relate his findings to humans:


> “If you want to take this to the level of humans. And this is something that geneticists rarely do if they’re smart. And I probably shouldn’t do it either. But the take-home lesson is ‘Get enough sleep,'” McRobert says. “I mean, the successful male drosophila is a drosophila that gets enough sleep.”


I’ve embedded the relevant video below. You can see McRobert featured at the 8 minute mark.


<embed allowfullscreen="true" flashvars="link=http://www.cbsnews.com/sections/i_video/main500251.shtml?id=3942132n&amp;releaseURL=http://release.theplatform.com/content.select?pid=GGNyasqVGrp78ngVyM_n4BI5DIxpGOUf&amp;partner=newsembed&amp;autoPlayVid=false&amp;prevImg=http://thumbnails.cbsig.net/CBS_Production_News/656/264/60_sleeppart2_31608_480x360.jpg" height="361" pluginspage="http://www.macromedia.com/go/getflashplayer" src="http://www.cbs.com/thunder/swf/rcpHolderCbs-prod.swf" type="application/x-shockwave-flash" width="370"/>


While Stahl and the 60 Minutes crew refer to Drosophila as “fruit flies”, McRobert knows better. This is from [his website](http://www.sju.edu/biology/smcrober.htm):


> My research focuses on the genetic, ecological, and evolutionary aspects of animal behavior. Animals utilized in my work include pomace flies (genus Drosophila), fish, turtles, and frogs.


[Mel Green](http://www.genetics.org/cgi/content/full/162/1/1) would be proud.

#This is the Future of Eukaryotic Genome Sequencing
Date: March 31, 2008

As I have [mentioned before](http://scienceblogs.com/evolgen/2007/05/the_future_of_eukaryote_genome.php), *de novo* sequencing of whole eukaryotic genomes may be a thing of the past (or, at least, these whole genome projects won’t be getting very much more common). Instead, I proposed that people would use the new high-throughput technologies to sequence parts of the genome they found interesting. What did I propose they’d sequence (based on discussion I’d had with various folks)? [The transcriptome](http://scienceblogs.com/evolgen/2007/09/the_future_of_eukaryotic_genom.php), or, essentially, a whole genome cDNA library. This allows for a much higher coverage of the genes in the genome, but sacrifices information regarding non-transcribed sequences and gene order.


It turns out that sacrifice is one worth making. In the April issue of Molecular Ecology, Vera et al. report that they used 454 to sequence the genome of the Glanville fritillary butterfly ([doi:10.1111/j.1365-294X.2008.03666.x](http://dx.doi.org/10.1111/j.1365-294X.2008.03666.x)). This organism is a popular model in population biology, but had no genetic resources. That’s changed thanks to this project. Here’s the punchline from the abstract:


> 
We conclude that 454 sequencing, when performed to provide sufficient coverage depth, allows de novo transcriptome assembly and a fast, cost-effective, and reliable method for development of functional genomic tools for nonmodel species. This development narrows the gap between approaches based on model organisms with rich genetic resources vs. species that are most tractable for ecological and evolutionary studies.



Molecular Ecology also has a review of the paper ([doi:10.1111/j.1365-294X.2008.03699.x](http://dx.doi.org/10.1111/j.1365-294X.2008.03699.x)).

#Phylogeny Friday — 11 April 2008
Date: April 11, 2008

**<u>Drosophila Are Not Fruit Flies Edition</u>**


As I have mentioned before, [Drosophila](http://scienceblogs.com/evolgen/drosophila/) are [not fruit flies](http://www.genetics.org/cgi/content/full/162/1/1). [Tephritids](http://en.wikipedia.org/wiki/Tephritidae) are fruit flies. Drosophila feed on rotting fruit, while true fruit flies feed on fresh fruit. That makes true fruit flies agricultural pests. Drosophila, on the other hand, are connoisseurs of the finer things in life — wine, beer, cheese, and the like.


[<img alt="ResearchBlogging.org" height="50" src="http://www.researchblogging.org/images/rbicons/ResearchBlogging-Medium-White.png" width="80"/>](http://www.researchblogging.org)


In addition to rotting fruit, Drosophila also feed on [mushrooms](http://www.jstor.org/stable/2408009) and [crabs](http://scienceblogs.com/evolgen/2007/09/that_crab_has_flies.php). Yes, crabs. Well, they don’t actually feed on the crabs, just like they don’t actually feed on the fruit. The flies and their larva are more interested in the yeast that ferment fruit or the microbes found on crabs.


Drosophilids that live on crabs can be found in two locations: the Caribbean and [Christmas Island](http://en.wikipedia.org/wiki/Christmas_Island). One species of Caribbean Drosophila, **D. endobranchia**, was last observed in the mid-1960s. It’s placement in phylogeny of Drosophila was not well resolved, and it was thought that it may be extinct. A new paper in PLoS ONE reports the rediscovery of that species, as well as the evolutionary relationship of the crabby flies ([doi:10.1371/journal.pone.0001942.g001](http://www.plosone.org/article/fetchArticle.action?articleURI=info:doi/10.1371/journal.pone.0001942))





The geographic location of **D. endobranchia** and its host, the black crab, as well as the life history of these flies, is summarized in the following figure.


[<img alt="i-14562f51bdf9c4ee9ceb33209dcbb3fb-crab_flies.png" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-14562f51bdf9c4ee9ceb33209dcbb3fb-crab_flies.png"/>](http://www.plosone.org/article/fetchArticle.action?articleURI=info:doi/10.1371/journal.pone.0001942)<br/>(A) The black crab (Gecarcinus ruricola, black morph). (B) The male holotype of **D. endobranchia** collected by H.L. Carson in 1966, now in the collections of the National Museum of Natural History, Washington, D.C. (C) Grand Cayman; Numbers refer to sites were crab flies were found. Scale bar 5 km. Image courtesy of NASA. (D) Male fly courting a female fly under the watchful eye of their host (a yellow morph black crab). (E) First instar fly larvae are found in the nephric pads (yellow arrow). The larvae feed on microorganisms, which cleanse the urine (exuded from the green gland; red arrow) of nitrogenous waste compounds. (F) Second instar is spent inside the gill chambers.


The article itself (from which the preceding image was taken) is written in a colloquial style not common in scientific publications. Here’s a taste, in which the authors describe their collection protocol:


> 
The flies strike a peculiar sight in real life. They essentially hardly move at all, are extremely reluctant in leaving their host crabs and are hard pushed to take flight. Although the flies are sluggish, the crabs on which they reside are anything but. Chasing after crabs through a pitch-black jungle (growing on a razor sharp labyrinthine limestone ground), while trying to aspirate flies from their carapaces is not trivial. Obtaining large amounts of flies in this way is simply a nightmare. The scarcity of the flies, and the nocturnal and shy nature of their hosts made it a daunting task to figure out the biology of these odd flies.



It really captures the joy (and struggle) of field work. The article goes on to describe various observation made by the researchers based on their collections. But this is Phylogeny Friday, so we’re going to skip right to the evolutionary relationships of flies that live on crabs.


The other two species of crab-living flies were pretty well placed in the Drosophila phylogeny. The Christmas Island fly, **Lissocephala poweilli** belongs to a primative genus closely related to Drosophila. The other Caribbean fly, **D. carcinophila** falls in the [**repleta** group](http://flybase.org/static_pages/allied-data/phylogeny/Drosophilidae-Tree/repletagroup.html), which probably means nothing to most people reading this blog. For some context, the *repleta* group is more closely related to [picture-wing Drosophila](http://scienceblogs.com/evolgen/2007/06/lots_of_pretty_wings_in_plos_o.php) and the cactus-feeding Drosophila than it is to *D. melanogaster*.


Stensmyr et al. sequenced five genes from **D. endobranchia** in order to resolve its phylogenetic position in the Drosophila genus. Here are the trees they constructed using three of the genes (the position of **D. endobranchia** is indicated by the black arrow):


[<img alt="i-ff16faa25a83b6894d47ca0568969226-crab_fly_phylogeny.png" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ff16faa25a83b6894d47ca0568969226-crab_fly_phylogeny.png"/>](http://www.plosone.org/article/fetchArticle.action?articleURI=info:doi/10.1371/journal.pone.0001942)


**D. endobranchia** falls within the virilis-repleta radiation, which also contains the other Caribbean species, **D. carcinophila**. However, **D. endobranchia** does not appear to be in the *repleta* group, but it is part of a closely allied group (the **canalinea** group). That means the two species of Caribbean Drosophila that inhabit crabs are closely related, but their similar life-histories are not the result of shared ancestry. Rather, they evolved into crab-flies independently, perhaps because something about Neotropical repleta flies predisposes them to host-shift onto crabs.


---



Green MM. 2002. It really is not a fruit fly. Genetics 162: 1-3 [[link](http://www.genetics.org/cgi/content/full/162/1/1)]


Kimura MT. 1980. Evolution of Food Preferences in Fungus-Feeding Drosophila: An Ecological Study. Evolution 34: 1009-1018. [[link](http://www.jstor.org/stable/2408009)]


Stensmyr MC, Stieber R, Hansson BS, Vosshall LB. 2008. The Cayman Crab Fly Revisited — Phylogeny and Biology of Drosophila endobranchia. PLoS ONE 3: e1942. [doi:10.1371/journal.pone.0001942](http://dx.doi.org/10.1371/journal.pone.0001942)

#Varmus Screws the Pooch
Date: April 13, 2008

[Harold Varmus](http://en.wikipedia.org/wiki/Harold_E._Varmus) is one of the most high profile advocates of open access to biomedical research. As one of the [cofounders of the Public Library of Science](http://www.plos.org/about/board.html#varmus) (PLoS), he has played an important role in making published results freely available to all. And he’s a [Nobel Laureate](http://nobelprize.org/nobel_prizes/medicine/laureates/1989/varmus-autobio.html), which ain’t too shaby either.


Varmus was interviewed by Ira Flatow for NPR’s [Science Friday](http://www.sciencefriday.com/) program about the [NIH’s new policy](http://www.sciencefriday.com/program/archives/200804114) requiring that research publications presenting results funded by the NIH be deposited in [PubMed Central](http://www.pubmedcentral.nih.gov/) (the NIH’s free online archive of biomedical journal articles) within a year of publication. Before I get into the problems with Varmus’ interview, I’d like to highlight one important point Varmus made: the current NIH policy is a comprimise between the ideal open access solution and what pay-access publishers want. In the ideal situation, all papers published about publicly funded research would be made freely available at the moment of publication. There would be no moving wall before they can be freely obtained. But that would ruin the business model of the pay-access journals, who make a fair bit of their profit from both individual and institutional (i.e., university) subscriptions. If a big chunk of the articles in those journals were freely available, the journals would stand to lose many of their subscribers.


Despite making that point, Varmus struggled to point out the big differences between open access and pay-access journals. He was clear that the differences lie in the business models, but he did not present the details of those differences with the clarity required when speaking to the general public. First, he made it seem as if open access journals are the only ones that require a payment from the author in order to publish in those journals. Second, he failed to clearly point out that open access and peer review are orthogonal issues, despite a perfect opportunity to do so.





Varmus made the important point that open access and pay-access journal differ primarily in their business models, not in their approach toward publishing scientific results. However, he made it seem as if open access journals are the only ones who require authors to pay to publish. This is not the case, as pay-access journals also require authors to pay publication fees. Using a few journals from my field as examples, [Genome Research](http://www.genome.org/misc/ifora_acceptedmss.shtml) charges $40 per page, [Molecular Biology and Evolution](http://www.oxfordjournals.org/our_journals/molbev/for_authors/page_charges.html) charges $50 per page, [Evolution](http://www.blackwellpublishing.com/submit.asp?ref=0014-3820) charges $55 per page, and [Genetics](http://www.genetics.org/misc/iforapdf.shtml) charges $65 per page. Those page charges may differ depending on whether you are a member of a society associated with the journal, and they often also charge additional fees for publishing color figures. Generally, a ten page article with one color figure will cost approximately $1000.


PLoS also charges authors to publish in its journals. For example, [PLoS Biology](http://journals.plos.org/plosbiology/policies.php#publication) charges $2750, [PLoS Genetics](http://www.plosgenetics.org/static/policies.action#publication) charges $2100, and [PLoS ONE](http://www.plosone.org/static/policies.action) charges $1250. These charges are fixed and do not depend on the length or contents of the article. [Nucleic Acids Research](http://www.oxfordjournals.org/our_journals/nar/for_authors/openaccess/article/submittal.pdf), an open access journal published by Oxford University Press, charges $2670 to authors. Additionally, pay access journals often give authors an option to publish their paper with free access, for an additional cost. For [Genome Research](http://www.genome.org/misc/open_access.shtml) this cost is $2000.


As you can see, the publication charges for open access journals are often slightly more than those for pay-access journals. But it’s not as if one business model requires author payments and the other does not. Additionally, all journals I have encountered allow editors to waive publication costs if the author(s) cannot afford them. Some are more draconian in their presentation of those rules — see the [page charge policy of MBE](http://www.oxfordjournals.org/our_journals/molbev/for_authors/page_charges.html) — but they all at least make that option available. This is true for both pay access and open access journals. Therefore, the primary difference in the business models between pay access and open access journals is not in the presence of author charges. There is a difference in the magnitude and application of those charges, but the primary difference is in whether the journal requires readers to pay to access the content.


What about Varmus’ defense of the peer review in open access journals? In this case, a caller phoned in to ask whether open access journals have the same peer review standards as pay-access journals (quite reminiscent of the whole [PRISM](http://www.arl.org/sparc/advocacy/07-0907prism.shtml) affair). Varmus correctly pointed out that open access journals require the same scientific rigour as pay-access journals. However, he failed to hammer home the point that open access and peer review are completely orthogonal issues. That is, whether a journal offers open access to its content is independent of the nature of peer review the articles are put through. Conflating open access with peer review is a propaganda strategy of the anti-open access lobby, [PRISM](http://www.arl.org/sparc/advocacy/07-0907prism.shtml). I think Varmus wasted a great opportunity to make this explicitly clear. 


In addition to discussing the NIH open access policy, open access business models, and peer review, Varmus also talked about publishing negative results. He pointed out the need for journals to make negative results available. This was somehow connected to Varmus plugging the new interactive features available in some PLoS journals — specifically, the ability of readers to comment on articles (a feature this is noticeably under-used), which I presume would allow readers to mention how their own negative results relate to the article.


The entire interview is available on the [Science Friday website](http://www.sciencefriday.com/program/archives/200804114). Varmus did a good job, but he could have done much better. He needs practice in hammering home the primary talking points. Some of his message got muddled in his lack of a punchy delivery. I worry that the points may have been missed by those people not familiar with the details.

#El Chupacabra de Barcelona
Date: April 14, 2008

I think I’m cursed. Or I have bad luck. Or conference organizers think I’m a morning person. Alright, so maybe I really am a morning person. But that’s besides the point. Because it sucks to give a talk on a Saturday morning. Saturday mornings should be reserved for things like Belgian waffles, homemade hash browns, made to order omelets, and mimosas. Not for 12 minute talks about my research. How much can you fit into a 12 minute talk? Not much. And you end up speeding up at the end when you realize you’ve got about 1 minute to make it through four slides. It’s either that or go over 12 minutes, cutting into the three minutes allocated for questions at the end of your talk. I’m not even going to get into the ordeal that are questions at the end of 12 minute talks.


I gave one of those 12 minute talks at the [Fly Meeting](http://www.drosophila-conf.org/genetics/gsa/dros/dros2008/) a week-or-so ago. All signs indicate that it went well. We’ll see how the manuscript is treated in review. The biggest problem was that the talk was at 9:15 on a Saturday morning. Not only did I have to deal with all the regular issues that come with a 12 minute talk a big conference (yes, the Fly Meeting is a pretty big conference, with over 1000 participants), but I had to speak to a room full of tired and hung-over scientists. I think half of the evolutionary geneticists had spent the night before partying. The only thing worse is giving a talk on ****Sunday**** morning, the morning after ****everyone**** spent the night before hanging out in a tree drinking beer (or so I’ve heard). Thankfully, they only give invited speakers the Sunday morning talks at the Fly Meeting, so insecure grad students don’t have to deal with half-empty rooms and a disinterested audience.


[<img alt="i-662dcc3a658ee1fd1e3180b9842d2a22-smbe_2008_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-662dcc3a658ee1fd1e3180b9842d2a22-smbe_2008_sm.JPG"/>](http://www.smbe2008.com/)


Okay, I’m probably making too big of a deal out of giving a talk on a Saturday morning at one meeting. But I haven’t told you when I’m scheduled to speak at the next meeting I’m going to. That said, I bet you can guess. No, it’s not 9:15 on a Saturday morning. It’s 9:30, which I guess is a slight improvement over 9:15. The talk will be given at the Society for Molecular Biology and Evolution conference ([SMBE 2008](http://www.smbe2008.com/)) in Barcelona. Being in Barcelona should make up for the fact that I need to give a talk on a Saturday morning.


Last year, we had a [blogger meet-up at SMBE in Halifax](http://network.nature.com/forums/smbe2007/181) (picture [here](http://pandasthumb.org/archives/2007/06/smbe-2007-scien.html)). I’ve set up another Nature Network Group, [SMBE 2008](http://network.nature.com/group/smbe2008), where we can coordinate a meet-up of bloggerly folks. Basically, if you’re reading this post, you’re invited. If you have a blog, even better. [Logsdon](http://johnlogsdon.blogspot.com/), I’m looking at you.

#The Natural Selection of Ideas
Date: April 19, 2008

It’s funny because some people think both groups are wrong:



Originally from [Tom the Dancing Bug](http://www.gocomics.com/tomthedancingbug/2008/04/19/).

#Watson’s Genome
Date: April 20, 2008

Last year, [Craig Venter](http://en.wikipedia.org/wiki/Craig_Venter) became the first single person to have his genome sequence published ([doi:10.1371/journal.pbio.0050254](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371%2Fjournal.pbio.0050254)). That genome was sequenced using the old-school [Sanger technique](http://en.wikipedia.org/wiki/DNA_sequencing#Chain-termination_methods). It marked the second time the complete human genome had been published (which led to some [discussion as to whether the publication deserved to be published in a high profile journal](http://genefinding.blogspot.com/2007/09/plos-biology-vanity-publisher.html) like PLoS Biology), and the first time all of the sequence came from a single individual.


[<img alt="ResearchBlogging.org" height="50" src="http://www.researchblogging.org/images/rbicons/ResearchBlogging-Medium-White.png" width="80"/>](http://www.researchblogging.org)


This past week, Nature published the second complete genome sequence of a single individual ([doi:10.1038/nature06884](http://dx.doi.org/10.1038/nature06884)). Like Venter, this individual is also a famous scientists, known both within his field and to non-experts. His name is Jim Watson, he helped discover the structure of DNA, and he’s been [eviscerated](http://www.dailymail.co.uk/pages/live/articles/news/news.html?in_article_id=488232&amp;in_page_id=1770) in the [past few months](http://www.cnn.com/2007/TECH/science/10/25/watson.resigns/index.html) because of [some comments he made about racial differences](http://entertainment.timesonline.co.uk/tol/arts_and_entertainment/books/article2630748.ece). So, we can expect the publication of his genome to be received with conflicted emotions.


What makes the sequencing of Watson’s genome different from that of Venter’s? It’s the technology. Watson’s genome was sequenced using one of the next generation sequencing technologies ([454](http://evolgen.blogspot.com/2005/04/bead-based-resequencing.html)), which allows much more sequencing bang for the buck. This isn’t a [$1000 genome](http://genomebiology.com/2006/7/7/112), but it’s a step in that direction.





Along with the article, Nature has published a [News and Views piece on the Watson genome paper](http://www.nature.com/nature/journal/v452/n7189/full/452819a.html). The article takes many steps to point out that, while we are at the cusp of an era of personal genomics, there are many limitations as to what we can do with these data. Given our current knowledge, we can’t say much about Watson’s phenotype that we didn’t already know based on non-genomic information. Personal genomics needs not only data, but also ways of assigning genomic variants to particular phenotypes. At this point, we’re probably limited to advising a couple about the risks that their child will acquire a particular genetic disease based on sequencing of the mother and father’s genomes.


The real value of Watson’s genome sequence is a proof of principle. This project was completed at a low cost (relative to the previous methods of sequencing genome), but with high quality. They were able to identify not only sequence variation between Watson and the reference genomes, but also [structural variation](http://www.google.com/search?hl=en&amp;newwindow=1&amp;safe=off&amp;q=structural+variation+%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search). To truly realize the potentials of personal genomics, we need many more of these genomes, with the phenotypic data on the individuals.


---



Levy S, Sutton G, Ng PC, Feuk L, Halpern AL, et al. 2007. The Diploid Genome Sequence of an Individual Human. PLoS Biol 5: e254 [doi:10.1371/journal.pbio.0050254](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371%2Fjournal.pbio.0050254)


Wheeler DA, Srinivasan M, Egholm M, Shen Y, Chen L, et al. 2008. The complete genome of an individual by massively parallel DNA sequencing. Nature 452: 872-876 [doi:10.1038/nature06884](http://dx.doi.org/10.1038/nature06884)

#Expelled Exaggerated?
Date: April 21, 2008

[<img border="0" class="inset" src="http://www.expelledexposed.com/banner-140.jpg"/>](http://www.expelledexposed.com)


The creationist movie that everyone* is talking about <s>came out</s> was released this weekend. [Early reports](http://www.boxofficemojo.com/daily/chart/?sortdate=2008-04-20&amp;p=.htm) have [Expelled](http://www.expelledexposed.com) coming in 9th nationally in weekend gross, with about $3 million. That’s a lot of money, and you can [color Randy Olson freakin’ impressed](http://scienceblogs.com/shiftingbaselines/2008/04/meet_ben_stein_the_new_spokesm.php). However, put in the context of what the producers were expecting, it’s [not so good](http://pandasthumb.org/archives/2008/04/expelled-the-fi.html).


That doesn’t stop Randy from [pulling at Matt Nisbet](http://cosmicvariance.com/2008/03/23/politicians-and-critics/) and touting how awesome the creationists are and how shitty the “evolutionists” are. Fucking “evolutionists”! I’m gonna go off on a rant here, but, before I do, allow me to point out the beautiful irony that is Randy Olson. The guy comes out and criticizes “evolutionists” for being really bad at getting their message out to the general public. He even goes so far as making a [movie about how shitty the “evolutionists” are at getting their message out to the general public](http://www.flockofdodos.com/). The entire point of the movie is about how “evolutionists” can’t communicate with the general public. Randy Olson is trained to communicate with the general public, and loves telling you about how good the creationists are at communicating with the general public. You’d think that would make Randy Olson the perfect person to communicate with the general public about evolution. Only he doesn’t; instead, he bitches about how bad the “evolutionists” are at communicating with the general public. Randy Olson can join Matt Nisbet as the official backseat drivers of science communication.


And that brings me to my rant. For a guy who’s an advocate of science, as well as a trained biologist, Randy Olson sure likes to use the creationist frames. Like “evolutionist”. Fuck it, Randy, why not just call ourselves baby eating satanists. When you use the label “evolutionist”, you’re playing right into the creationists’ game. You’ve fallen for their frame. This isn’t an issue between creationists and evolutionists — it’s between anti-science and science. That’s our frame, and that’s the truth. The [wedge](http://www.antievolution.org/features/wedge.html) and all that shit are an attack on not only evolution, but all of science. If you don’t get that, you don’t understand any of this shit.


*Everyone = hard-core creationists, people who read science blogs, and 38 year olds who live in their mom’s basement.





---



Your prize for clicking through below the fold is the preview for Ben Stein’s next film, Sexposed:


<object height="355" width="425"><param name="movie" value="http://www.youtube.com/v/-ThQQuHtzHM&amp;hl=en"/><param name="wmode" value="transparent"/><embed height="355" src="http://www.youtube.com/v/-ThQQuHtzHM&amp;hl=en" type="application/x-shockwave-flash" width="425" wmode="transparent"/></object>

#Who said this?
Date: April 21, 2008

Someone was asked something along these lines by a member of some legislative body:


> 
How will your research help protect this country?



That someone replied with something like this:


> 
It won’t, but it will keep this country worth protecting.



The exact wording in those quotes probably differ from what was actually said.


This isn’t a rhetorical question, nor is it an exercise in trivia. I don’t know who said it, what the exact context of the quote was, or whether this was actually said by anyone, anytime, anywhere. So, if you know of the exact quote, who said it, and where it was said, please leave a note in the comments. Thanks!

#Saving the Name of Drosophila melanogaster
Date: April 22, 2008

The [Drosophila genus is paraphyletic](http://scienceblogs.com/evolgen/2008/02/phylogeny_friday_1_february_20.php). That means there are species nested within the phylogeny of the genus that belong to other genera. Or, in other words, there are species descended from the most recent common ancestor (MRCA) of all Drosophila species that belong to different genera. If that doesn’t make sense, just [look at the tree](http://scienceblogs.com/evolgen/2008/02/phylogeny_friday_1_february_20.php).


A paraphyletic genus is a no-no in taxonomy. There are two ways to deal with the problem. First, the genera nested within the Drosophila phylogeny can be redesignated into the Drosophila genus. That’s not going to happen because the genus is too freakin’ big to begin with. The second option is to split the Drosophila genus into multiple genera. The genus is already split into two subgenera, Drosophila and Sophophora. *D. melanogaster*, the best known Drosophila species, is in the Sophophora subgenus. *D. funebris*, the type species of the genus (or the one that holds the rights to the name “Drosophila”), is in the Drosophila subgenus. So, if the genus gets split, *D. funebris* gets to keep the genus name Drosophila, while *D. melanogaster* will probably be renamed *Sophophora melanogaster*. That doesn’t sit well with some people.





There’s a movement afoot to ensure *D. melanogaster* gets to keep the genus name Drososophila, despite historical precedence saying otherwise. I, myself, really don’t care. I guess it would be a bit of an inconvenience if *D. melanogaster* was renamed *S. melanogaster*, considering that more papers have been written about *D. melanogaster* and its close relatives than about the species in Drosophila subgenus.


If this sort of thing gets you going — regardless of the side you’re on — the case has been brought to the International Commission on Zoological Nomenclature ([ICZN](http://www.iczn.org/)). The ICZN is like the League of Nations or Super Friends for taxonomists. Anyway, it’s [Case 3407](http://www.iczn.org/BZNDec2007cases.html) in the Bulletin of Zoological Nomenclature.


More information can be found [here](http://www.bio.net/bionet/mm/dros/2008-April/007251.html).

#Deborah Charlesworth on Lynch’s Origin of Genome Architecture
Date: April 23, 2008

[Brian Charlesworth wrote a review of Mike Lynch’s *The Origins of Genome Architecture*](http://scienceblogs.com/evolgen/2008/03/charlesworth_on_lynch.php), in which Charlesworth argues that sexual reproduction can explain many of the features Lynch claims evolved under nearly neutral processes ([doi:10.1016/j.cub.2008.01.008](http://dx.doi.org/10.1016/j.cub.2008.01.008)). Not to be left out of the party, Deborah Charlesworth has chimed in with her opinion, and it’s much more critical of Lynch than her husband’s ([doi:10.1017/S0016672308009282](http://journals.cambridge.org/action/displayAbstract?fromPage=online&amp;aid=1849824&amp;fulltextType=BR&amp;fileId=S0016672308009282)). The main thesis that Lynch has been presenting in both this book and some of his recent papers is that many features of eukaryotic genomes (introns, complicated *cis* regulatory regions, lots of non-coding DNA) arose via relaxed constraint due to small population sizes, not adaptation.


In her review, Deborah Charlesworth takes Lynch to task for not devoting enough of his discussion to within species polymorphism. Additionally, she argues that comparisons between distantly related taxa are limited by confounding variables. By ignoring recent studies that have identified evidence for selection on large portions of some genomes (and by excluding some important tests for selection, e.g., the [McDonald-Kreiman test](http://scienceblogs.com/evolgen/2006/03/detecting_natural_selection_pa.php)), Lynch paints a biased picture of what we know about adaptive evolution at the molecular level. However, Charlesworth does make the point that Lynch’s book is not the be all and end all of genome evolution; rather, it is designed to get the reader to consider other explanations besides adaptive evolution when explaining genomic features. 

#Phylogeny Friday — 25 April 2008
Date: April 25, 2008

**<u>Duh! That’s Obvious, Edition</u>**


Take a look at this mastodon skeleton:


<img alt="i-29b4b6a683253f7923a9445d7735b2b2-mastodon_skeleton.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-29b4b6a683253f7923a9445d7735b2b2-mastodon_skeleton.JPG"/>



Does it look like anything you recognize? Perhaps a large terrestrial mammal with big tusks. If you said “elephant” you win. The prize: nothing.


[<img alt="ResearchBlogging.org" height="50" src="http://www.researchblogging.org/images/rbicons/ResearchBlogging-Medium-White.png" width="80"/>](http://www.researchblogging.org)


That is half of the conclusion from a recent paper in Science ([doi:10.1126/science.1154284](http://dx.doi.org/10.1126/science.1154284)). Really. The other half: birds and dinosaurs are pretty closely related. Or, more specifically, birds and *Tyrannosaurus rex* — THE COOLEST MOST AWESOMEST OF ALL DINOSAURS EVER!! — are closely related. And, for this, they get a **Science** paper.


Now, the way they did this is pretty damn cool: they sequenced proteins from *T. rex* bones. But that was reported last year ([doi:10.1126/science.1137614](http://dx.doi.org/10.1126/science.1137614)) in the paper where [they screwed up the species name](http://scienceblogs.com/evolgen/2007/04/to_get_an_article_in_science_a.php) in the title (they got it right this time around). Anyway, some of the same people took those sequences and, along with some other sequences that they mined from various databases, constructed a phylogenetic tree of vertebrates.





[<img alt="i-c8e1056c6360b072474fc15cac92e5c3-mastodon_trex_phylogeny.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c8e1056c6360b072474fc15cac92e5c3-mastodon_trex_phylogeny.gif"/>](http://dx.doi.org/10.1126/science.1154284)



That right there is the tree, with dinos and birds together, and mastodons and elephants together. It’s totally unremarkable, but it got into **Science** (no, I’m not jealous). Now, this approach has the potential to resolve some long-standing conflicts in vertebrate systematics. This paper, however, does not resolve anything. It’s a sexy method that will probably be difficult to implement elsewhere.


---



Asara JM, Schweitzer MH, Freimark LM, Phillips M, and Cantley LC. 2007. Protein Sequences from Mastodon and **Tyrannosaurus Rex** Revealed by Mass Spectrometry. Science 316: 280-285 [doi:10.1126/science.1137614](http://dx.doi.org/10.1126/science.1137614)


Organ CL, Schweitzer MH, Zheng W. Freimark LM, Cantley LC, and Asara JM. 2008. Molecular Phylogenetics of Mastodon and **Tyrannosaurus rex**. Science 320: 499 [doi:10.1126/science.1154284](http://dx.doi.org/10.1126/science.1154284)

#Dichotomous Inverts
Date: May 4, 2008

Kevin has posted the newest [Circus of the Spineless](http://scienceblogs.com/deepseanews/2008/05/a_dichotomous_key_to_circus_of.php) at Deep See <s>Nudes</s> News. Go get your monthly fix of inverts.

#Well, I’m a Doctor
Date: May 14, 2008

I can officially claim myself as a member of the ivory tower elite. At least, that’s what they tell me.

#Whad’ya Know About Protists?
Date: May 20, 2008

My advisor has recently got me listening to [Whad’ya Know](http://www.notmuch.com). My first reaction: It’s like [Wait Wait…Don’t Tell Me!](http://www.npr.org/programs/waitwait/) Only not as funny, not as interesting, and not as good. I’ve been downloading the podcasts for the past couple of weeks, and I’m not sure whether I’ll keep subscribing in iTunes.


I’m only bringing this up because [last week’s episode](http://www.notmuch.com/Show/Archive.pl?s_id=508) contained a very egregious example of someone knowing just enough biology to get themselves in trouble. The sad part was that the person should have known better. Why? She teaches biology at the university level.


What happened? At the midpoint of the first hour of [the show](http://www.notmuch.com/Show/Archive.pl?s_id=508), Michael Feldman conducted the quiz. The in-house participant was Deb, a grasslands ecologist. Deb was asked about the diet of early Americans, which [turned out to be seaweed](http://www.telegraph.co.uk/earth/main.jhtml?xml=/earth/2008/05/08/sciweed108.xml). Deb then pointed out that seaweed is not a plant. Huh? Then she said it’s algae, which is a protist. Really? Deb doth protist too much, methinks.





It’s really sad to watch Deb struggle to display her out-dated understanding of eukaryotic taxonomy. It’s especially disconcerting because she’s extremely proud of her knowledge that eukaryotes (a term she does not use) are divided into four kingdoms — animals, plants, fungi, and protists. And she requires her students to know this. That is, she requires her students to gain an incorrect understanding of eukaryotic evolution.


First off, seaweed is a plant. It’s just not a land plant. The green algae, of which seaweed is a member (one of the few things Deb got right), are aquatic plants. They have no vascular system for transporting water and nutrients because they can absorb those from the surrounding water. The water also provides them with support. Algae also have no roots, no seeds, and no fruits. As plants began to colonize land, they evolved adaptations to the terrestrial environments they encountered. First, came a waxy cuticle to prevent water-loss. Then, they evolved vasculature ([xylem](http://en.wikipedia.org/wiki/Xylem) and [phloem](http://en.wikipedia.org/wiki/Phloem)) to transport water and nutrients to and from the roots and leaves. After that came pollen and seeds. The pollen allows individuals to fertilize each other without needing water to transport the male gametes. The seeds provide protection for the developing embryo of the plant. Flowering plants, the largest extant radiation of plants, also evolved flowers and fruits to aid in reproduction. See [here for more on the evolution of land plants](http://scienceblogs.com/evolgen/2007/07/phylogeny_friday_06_july_2007.php).


But what about that outdated misnomer, protist? If, as Deb claims, anything eukaryote that’s not an animal, fungus, or land plant, then nearly every eukaryote is a protist. That’s because the majority of eukaryotes don’t fall into one of those three small taxa.


[<img alt="i-5feb4cbd88250bf87fba504db8ff9f8c-Eukaryote_tree.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5feb4cbd88250bf87fba504db8ff9f8c-Eukaryote_tree.jpg"/>](http://scienceblogs.com/evolgen/2006/08/phylogeny_friday_25_august_200.php)


Land plants make up a small portion of the plant group (shown in green in the top-left of the figure). Many seaweeds are green algae, a close relative of land plants (close in the scope of this tree). Animals are also a single lineage on this tree, highlighted in a red box in the bottom-right. And fungi, the most diverse of the three non-protist taxa, are made up of a few of the lineages closely related to animals in the [Opisthokont](http://en.wikipedia.org/wiki/Opisthokont) clade. For more on this topic, see [this post on the eukaryotic phylogeny](http://scienceblogs.com/evolgen/2006/08/phylogeny_friday_25_august_200.php) and [this article](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.0020220).


Everything else that I didn’t just describe would be classified as a protist according to Deb. That’s all the [Excavates](http://en.wikipedia.org/wiki/Excavate), [Rhizaria](http://en.wikipedia.org/wiki/Rhizaria), [Chromalveolates](http://en.wikipedia.org/wiki/Chromalveolates), red algae, and green algae. Protists also make up a large portion of the [Unikonts](http://en.wikipedia.org/wiki/Unikont), the group containing animals and fungi. You’re probably asking, the whats? Exactly. Much of those organisms are single-celled amoeba-like critters. They’re not worthy of any sophisticated, evolutionarily-accurate taxonomy, and they get thrown in the meaning-less grab-bag known as “protists”.


What’s wrong with that taxonomic strategy? First, it does not reflect the evolutionary relationships of the organisms. Protist is a [paraphyletic](http://en.wikipedia.org/wiki/Paraphyletic) taxon — essentially, a group of organisms that excludes some organisms within that evolutionary group (like including reptiles to the exclusion of birds). Also, understanding the true evolutionary relationships is essential for studying the biology of these organisms. For example, the “protists” that are closely related to animals help us understand how animals evolved from single-celled organisms. Anyone knowledgeable in eukaryote systematics does not use the term “protist”, except to point out that it is incorrect.


I would hope that Deb, a woman in charge of teaching college students about evolutionary biology, would realize that protist is a meaningless classification. No contemporary introductory biology course worth anything uses that classification scheme. The taxon protist only still lingers because of its historical use. Let’s hope that it does not linger much longer.

#The Evils of Open-Access in the Developing World
Date: May 21, 2008

I recently [chastised Harold Varmus for equating open-access publishing with pay-to-publish](http://scienceblogs.com/evolgen/2008/04/varmus_screws_the_pooch.php). While open-access journals do tend to have higher author charges than pay-access journals, many journals make money from both author charges and subscriber charges. That is, they are pay to publish and pay to read.


A letter to Nature this week challenges the idea that open-access publishing is good for the developing world by assuming the same fallacy as Varmus ([doi:10.1038/453450c](http://dx.doi.org/10.1038/453450c)). Raghavendra Gadagkar, of the Indian Institute of Science, argues that the pay to publish model prevents underfunded researchers from publishing their work. This assumes that open-access publishers have a limited number of waivers for author fees. He would prefer journals publish articles free of charge and charge for access to the articles. In short, Gadagkar values authorship over readership.


Gadagkar makes a valid point — that a pay to publish model could severely limit the academic output from developing countries. However, it’s unclear whether the open-access publishers have a limited number of fee waivers to offer. Will the open-access publishing model collapse if too many authors require fee waivers? Where is the tipping point? And will well funded researchers be willing to bear the burden of paying the author charges, indirectly, of poorly funded researchers?

#The Probability of Winning the NBA Draft Lottery
Date: May 22, 2008

<img alt="i-702d555cece852fc5f16bfb63a6eb542-nba-draft-lottery.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-702d555cece852fc5f16bfb63a6eb542-nba-draft-lottery.gif"/>


On Tuesday night, the National Basketball Association (NBA) held their annual draft lottery. In the draft, each team is given the opportunity to select a few players that have declared themselves eligible for the draft (either after completing at least one year of college in the United States or being from another country and over 18 years old). The order of picks in the NBA draft is determined with a goal of awarding earlier picks to teams that performed the worst the previous season. However, rather simply giving the worst team the first pick, second worst the second, etc., the NBA takes the teams that failed to make the playoffs and assigns them a probability of earning the first pick based on their record the previous season. They have been using a probability system based on the previous season’s performance since 1990 and [those probabilities can be found here](http://www.nba.com/history/lottery_probabilities.html).


The draft lottery was instituted to prevent the worst team from automatically earning the first pick — which allowed the NBA to assuage any fears that a team would intentionally “tank” the end of the season to draft a young stud. From 1985-89, the NBA gave each non-playoff team an equal chance of winning the draft lottery and earning the first pick in the draft. This process began with the New York Knicks earning the first pick in 1985, which allowed the to draft [Patrick Ewing](http://en.wikipedia.org/wiki/Patrick_Ewing), regarded by all the experts as the best available player. Many people accused the NBA of rigging the lottery so the young superstar would be paired with its marquee franchise. But the Knicks never won a championship with Ewing, and he earned a notorious reputation as a guy whose teams improved without him in the lineup (see the [Ewing Theory](http://sports.espn.go.com/espn/print?id=1193711)).


<img alt="i-ced93f6c831a5abdd00abd86aa7d6513-NBA_lottery_show.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-ced93f6c831a5abdd00abd86aa7d6513-NBA_lottery_show.gif"/>


From 1990-93, the non-playoff teams were ranked by their records, and the team with the worst record had 11 chances of winning the lottery, the second worst had 10 chances, the third worst had 9 chances, etc, etc, all the way to the eleventh worst team who had one chance. This ended after the 1993 draft lottery, when the Orlando Magic captured the first pick despite having the best record of all the non-playoff teams, and a mere 1.5% chance of winning the lottery. From 1994 to the present, the probability of earning the first pick has been weighted even more toward the teams with the worst records. The worst team has a one in four chance of winning the first pick, and the second worst team has a one in five chance (a[ full list of the current probabilities can be found here](http://en.wikipedia.org/wiki/NBA_Draft_Lottery#Process)). After the first pick is awarded, the teams with the second and third picks in the draft are also determined using the lottery. The draft order of the remaining teams — those that were not selected in the lottery — are based on their records (the worst teams draft earlier, and the better teams later).


Because the lottery is weighted in favor of the worst teams, sports journalists and commentators are often surprised when the worst teams do not win the first three picks in the draft. Here is how [the Associated Press reported it](http://hosted.ap.org/dynamic/stories/B/BKN_DRAFT_LOTTERY?SITE=MOSTP&amp;SECTION=HOME&amp;TEMPLATE=DEFAULT):


> 
Only twice have teams with the worst record won the lottery since the current format began in 1994. Though the lottery is weighted to give teams with the poorest records the best chance to win, the longshots keep finding a way.



That’s 15 draft lotteries, and only two times has the number one pick been awarded to the worst team. Usually, the worst team has a 25% chance of winning the lottery. However, when the Toronto Raptors and Vancouver Grizzlies (now in Memphis) joined the NBA in 1996, the were denied the right to win the first pick until 1999. In 1996 and 1997, the Grizzlies had the worst record in the NBA, so we should really only consider the 13 other seasons. But in 1998, because the Grizzlies and Raptors could not win the lottery, the worst team, the Denver Nuggets, had over a one in three chance of winning (they did not win, and ended up with the third pick in the draft).


<br/>
<img alt="i-19d2d72ef3e8458bca0f03431510a2cb-NBA_Draft_logo.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-19d2d72ef3e8458bca0f03431510a2cb-NBA_Draft_logo.gif"/>


If we consider each of the 13 drafts in which the worst team was allowed to win the first pick, we can determine the expected number of draft lotteries won by the worst team. In 12 of those lotteries, the worst team had a 25% chance of winning, and in one draft, they had a 35.92% chance. Therefore, we expect the worst team to earn the first pick in between three and four draft lotteries. What’s the probability they only won two lotteries if the lotteries were fair? It’s about 19.5%, which is comparable to the probability that the worst teams won three and four lotteries (about 25% and 21.5%, respectively). Yes, the worst team has won the lottery less often than expected, but it’s not as an unlikely scenario as the journalists seem to believe.


The other aspect of this is the low probability of the best non-playoff teams winning the lottery. For example, in this year’s lottery, the Chicago Bulls had a 1.7% chance of winning the first pick, yet they’ll be selecting first in the upcoming NBA draft. Other winning teams with low probabilities of success (below 10%) include the Golden State Warriors in 1995 (9.5%), New Jersey Nets in 2000 (4.4%), Houston Rockets in 2002 (8.9%), Milwaukee Bucks in 2005 (6.3%), the Toronto Raptors in 2006 (8.8%), and the Portland Trailblazers in 2007 (5.3%).


What’s the probability that a team with lower than a 10% chance gets the first pick? It’s not 10%. In the current scheme, eight of the 14 lottery teams have less than a 10% chance of winning the lottery. If you combine the probability of winning the first pick for each of those teams, you find that there’s over a 27% chance that one of those teams wins. Therefore, it’s more likely that the number one pick goes to a team with less than a 10% chance of winning than the worst team in the league. There’s also a 12.5% chance that one of the teams with less than a 5% chance of winning the lottery gets the first pick.


How likely is it that a seven teams with less than a 10% chance of winning the lottery got the first pick in the NBA draft? We’ll use this year’s probabilities to save a fair bit of calculation. I invite someone else to try this using the probabilities from each year (I can provide the data). Anyway, when we do the math, we find that there’s less than a 6% chance that the draft lottery will be won seven times by teams with less than a 10% chance of winning in a 15 year span. Here are the probabilities that a team with less than a 10% chance of winning will win in zero through fifteen years of draft lotteries:


<img alt="i-4b0ca739ab21d41071dc9815b7cfbd72-nba_draft_10percent.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-4b0ca739ab21d41071dc9815b7cfbd72-nba_draft_10percent.gif"/>


As you can see, the most likely outcome is that four teams with less than a 10% chance of winning the draft lottery will get the first pick over 15 years.


Three teams with less than a 5% chance of winning the draft lottery have earned the first pick in the NBA draft. That’s expected in about 18% of all permutations over a 15 year span. The two most likely scenarios are that these teams would win the lottery one or two times. The probabilities of all possible number of times that a team with less than a 5% chance of winning gets the first pick are shown below:


<img alt="i-177017dfcd2c93ce1f1189cfecfd0b3f-nba_draft_5percent.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-177017dfcd2c93ce1f1189cfecfd0b3f-nba_draft_5percent.gif"/>


The last two calculations (the ones with the accompanying graphs) should be taken with a grain of salt. As I mentioned, I used only the probabilities from this year. A bit more work — using the probabilities from each of the fifteen years — may reveal that the outcomes of those drafts are not as unlikely as I proposed. However, the take home message here is that, despite what some sportswriters may claim, it’s not all that unlikely to see the worst team (i.e., the one with the highest probability of winning) not win the draft lottery. Additionally, teams that get lucky in the draft lottery, capturing a high draft pick despite a low probability of doing so, are expected very often.


**Related:** [A mathematician writes about the probability of the outcome of the 2007 NBA draft lottery](http://www.math.harvard.edu/~leingang/?p=130).

#A Neutral Theory of Memetics
Date: May 27, 2008

<img alt="i-4f4e40ca05c55fa1569d69cc450aeb64-The_Selfish_Gene3.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-4f4e40ca05c55fa1569d69cc450aeb64-The_Selfish_Gene3.jpg"/>


In his classic book from 1976, [*The Selfish Gene*](http://en.wikipedia.org/wiki/The_Selfish_Gene), Richard Dawkins introduced the concept of the [meme](http://en.wikipedia.org/wiki/Meme). Basically, a meme can be thought of as a cultural gene — an idea that is transmitted in a population.


This being Dawkins, memetics has a certain adaptationist flavor to it. *The Selfish Gene* introduces evolution from the gene’s eye view. But Dawkins is a staunch advocate of natural selection as the primary driving force behind evolution. The debate over the role of natural selection in evolution has been carried out using empirical data, theoretical modeling, and philosophical arguments. I won’t get into any details of it here.


What I am interested in is the adaptationist nature of memetics. From what I have seen, memeticists argue that memes survive or fail based on their ability to propagate through a culture. Memes that are good at infecting new individuals persists, while those that are not go extinct. The probability of extinction depends on some properties inherent to the memes and the culture they inhabit.


<br/>
<img alt="i-0b9d25b7e63d529fe135b8496da6d3ce-dennett_pss_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-0b9d25b7e63d529fe135b8496da6d3ce-dennett_pss_sm.JPG"/>


[Daniel Dennett](http://en.wikipedia.org/wiki/Daniel_Dennett) has taken a prominent role as an advocate for memetics. I spoke with him a few weeks ago and asked if there was a neutral theory for memetics. In his reply, he suggested that neutral evolution is pretty irrelevant. With such a position, it’s hard to imagine that he doesn’t spend much time thinking about neutral memes.


Memetics itself is a controversial field of study — in the sense that some people don’t think it’s worth studying at all. But, assuming the idea of the meme is a valid one, does it make sense to consider neutral dynamics of memes? That is, propagation of ideas merely because of their historical contingencies, and not because they are good at propagating themselves. For example, consider a cultural practice that is common because it happened to spread merely by chance. Perhaps it hitchhiked along with an adaptive meme (prayer with religion?). Or maybe it just got lucky.


On the other hand, there may be no need for a neutral theory. Perhaps coming along at the right point in time is the adaptive feature of that meme. So, are there neutral memes? Or does it not make sense to think of neutral memes?

#A Physicist, a Marine Biologist, and a Geneticist Walk into a Bar…
Date: May 28, 2008

[Chad](http://scienceblogs.com/principles/)‘s in town for the [DAMOP meeting](http://scienceblogs.com/principles/2008/05/division_of_atomic_molecular_a.php). What that means isn’t all that important in the grand scheme of things. But it did give us a chance to have a blogger meet-up. So, I grabbed [Professor Steve Steve](http://prof.stevesteve.org/) and Kevin (of [Deep Sea Nudes](http://scienceblogs.com/deepseanews/) and the [Unimportant 95%](http://other95.blogspot.com/)), and we met Chad for some dinner and beers.



Kevin’s posted [another version of the picture](http://scienceblogs.com/deepseanews/2008/05/physics_and_beer_have_everythi.php).

#Good-bye Frank!
Date: May 28, 2008

[Francis Collins](http://en.wikipedia.org/wiki/Francis_Collins_%28geneticist%29), head of the [National Human Genome Research Institute](http://www.genome.gov/) (NHGRI), will be [stepping down from that position in August](http://www.genome.gov/27026551). Collins has overseen the sequencing of the human genome, the [HapMap project](http://www.hapmap.org/), the [ENCODE](http://www.genome.gov/10005107) project, and many other large advances in genomics. The NHGRI has had a major impact in the field of genetics while Collins was at its helm.


[Collins also has a history of putting his foot in his mouth](http://www.google.com/search?q=francis+collins&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search) — not that it’s a bad thing to do so (lord knows my foot’s spent more time lodged in my face than on the ground). He’s been incorrect about [morality](http://scienceblogs.com/evolgen/2006/10/francis_collins_morals.php) and [human evolution](http://scienceblogs.com/evolgen/2006/06/francis_collins_opens_mouth_in.php), to name two of the more noticeable faux pas. I hope the next head of the NHGRI has more solid grasp of evolutionary genetics than Collins does. Modern genomics requires it.


So, who should take over as head of the NHGRI in August? I have some ideas.





- [Eric Lander](http://www.wi.mit.edu/research/faculty/lander.html): Founding director of the [Broad Institute](http://www.broad.mit.edu/) and a major player in human genomics. He could very well turn down the position of head of the NHGRI because he’s got it pretty good where he currently is.
- [Stephen O’Brien](http://rex.nci.nih.gov/lgd/staff/obriens.htm): Currently at the National Cancer Institute (NCI), where he studies genomic diversity. Would be a natural fit to transition to the more human-centric genomic diversity work going on at NHGRI. Already at the NIH.
- [Elaine Ostrander](http://www.genome.gov/12513335): If I’m going to nominate a cat geneticist (O’Brien), it only seems appropriate that I also nominate a dog geneticist. Ostrander is a leader in making the domestic dog a model organism for studying human genetic disease. She’s also already at NHGRI.
- [Eric Green](http://www.genome.gov/staff/Green/): He’s the Scientific Director of the NHGRI and a big player in comparative genomics.
- [William Gahl](http://www.genome.gov/10005723): He’s the Clinical Director of the NHGRI. If the NIH wants to keep a MD/PhD in charge, Gahl may be their man.
- [Alan Guttmacher](http://www.genome.gov/10005495): Deputy Director of the NHGRI. He’ll be acting director starting August 1, until they find a replacement for Collins.
- [David Haussler](http://www.cbse.ucsc.edu/staff/haussler.shtml): Heads the University of California, Santa Cruz comparative genomics group. Is a huge figure in human genomics, and has his hands in pretty much all the big science projects funded by the NHGRI.


That is by no means an exhaustive list. It’s just some of the names that came to mind or jumped out when browsing the NHGRI webpage. It’s definitely biased toward the type of research I think the NHGRI should be funding. Do you have any other suggestions? Who should be the next director? Who will be the next director? Those last two questions may have very different answers.


**See also:** Jonathan Eisen thinks [Collins may be stepping down to act as an advisor for a presidential campaign](http://phylogenomics.blogspot.com/2008/05/francis-collins-steeping-down-from.html).

#The Phylogenetics of Animal Testing
Date: May 30, 2008

**<u>Phylogeny Friday — 30 May 2008</u>**


[<img alt="i-dd57ac17f6d46f6e86aa7e2aa29d768c-animal_research_war.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-dd57ac17f6d46f6e86aa7e2aa29d768c-animal_research_war.gif"/>](http://www.amazon.com/Animal-Research-War-Michael-Conn/dp/023060014X)


Research on animals in under attack throughout the world. Animal rights activists not only stage rallies against animal testing, but they also engage in criminal behavior. They vandalize property, sabotage experiments, and terrorize researchers. How can scientists fight back?


Michael Conn and James Parker have written book documenting the animal rights issue from the scientists’ perspective ([The Animal Research War](http://www.amazon.com/Animal-Research-War-Michael-Conn/dp/023060014X)). Conn and Parker have also briefly described their position in [the FASEB Journal](http://www.fasebj.org/cgi/content/full/22/5/1294). Here is how they summarize their book:


> 
This book is a personal account of what it is like to be intimidated (and yes, one of the authors has experienced what it is like to be intimidated while traveling, at home, and at work)–but we want the book to go far beyond that and to be a thoughtful analysis of the effect of animal extremists on the world’s scientists, their institutions, and professional societies.
We trace the changing way in which the public and legal system views animals and the evolution of the animal rights movement. We profile its leadership. We show how funds given in the belief the donor is supporting homes for stray animals may actually be routed into activities that destroy the life-saving work of university researchers. We look at the arguments frequently used by animal activists. Most important, we reveal the truth behind animal research.



I’m bringing all of this up because Andrew Read has written a review of *The Animal Research War* for Nature ([Vivisectionists strike back](http://dx.doi.org/10.1038/453592a)). Read’s take on the book is largely negative. Not because he disagrees with research on live animals — he actively does research on mammals (reviewed [here](http://scienceblogs.com/evolgen/2008/01/competitive_release_and_antibi.php)) — but because Read thinks Conn and Parker have glossed over a lot of important details. Read argues that Conn and Parker’s sound-bite approach toward combating animal rights activists, while necessary, needs more open information regarding how animals are treated and the benefits of research on animals.





Read also brings up a very interesting point regarding the restrictions on performing research with certain types of animals. Specifically, how do we choose the amount of legal hurdles required to perform experiments on primates, cats, dogs, rodents, birds, and even invertebrates.


> 
Another difficult issue is how we should deal with phylogeny. In the United States, rats, mice and birds are not given the same legal cover as primates, cats and dogs. Under the UK system, one species of octopus is covered, but you can do what you like to the others. Is there a sound basis for replacing mice with zebrafish in toxicology testing? Should there be welfare protection for crustaceans? Or no protection for chickens?



There is no welfare protection for insects, including Drosophila. At my university, protection is only granted to vertebrates, although I’m hazy on the details. I’m not sure whether people working on fish, amphibians, or reptiles need to fill out IACUC forms. I know it’s a big deal for the mouse labs on campus.


But the important issue here is: are our current restrictions on animal research phylogenetically sound? If the situation is as Read describes, then they are not. Here’s a phylogeny of mammals:



Primates and rodents are in Clade III, while cats and dogs (Carnivora) are in Clade IV. Why are cats dogs treated better than mice and rats, despite the fact that mice and rats are closer relatives to humans? Probably because people have an emotional attachment to cats and dogs, while mice and rats are viewed as disgusting pests.


This is not to say that phylogenetics should be the deciding factor for the amount of legal cover given to specific animals. But the decisions should be made rationally. Other factors, such as cognitive ability, should also be taken into account. That’s the justification behind protecting one species of octopus (it’s so smart!), even though that decision is a bit murky, as Read points out.


I’m nowhere near qualified to offer an intelligent opinion on the matter, but I do find it interesting. How much weight should evolutionary relationships have in deciding which species get legal protection in animal research?

#Where I’ve Been the Past Couple Weeks
Date: June 15, 2008

<img alt="i-71a671fdd9bd6221f36656c337c23fec-Sagrada_Familia.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-71a671fdd9bd6221f36656c337c23fec-Sagrada_Familia.jpg"/>


If you check the archives of this blog (which I know all of you do on a fairly regular basis), you’ll see that I haven’t posted anything in over two weeks. Those kind of blog hiatuses can’t be good for traffic, but I have a legitimate excuse: I was out of the country. Readers with a keen eye for international architecture will recognize the picture attached to this post. Those without, may not. Either way, I figure I should tell you that it’s one of the facades of [Sagrada Familia](http://en.wikipedia.org/wiki/Sagrada_Fam%C3%ADlia), Antoni Gaudi’s  best known unfinished church.


Why post a picture of this famous tourist spot? Because I was in Barcelona for the [SMBE meeting](http://smbe2008.com/) last week. I met up with [John Logsdon](http://johnlogsdon.blogspot.com/) (among many other people), and apparently [John Dennehy](http://evilutionarybiologist.blogspot.com/2008/06/smbe-08.html) was as well. It was an excellent conference, as SMBE meetings usually are. My only complaint: too many sessions that I wanted to attend that were scheduled concurrently.


Anyway, I intend to blog about a couple talks that I attended that discussed some interesting published results. I may also blog about some other Barcelona/SMBE stuff if the spirit fills me.

#How many genes do you share with your twentieth cousin?
Date: June 18, 2008

John Hawks has an interesting [post on what it means to be human](http://johnhawks.net/weblog/2008/06/18#what-it-means-to-be-human-2008) in which he argues that our “human-ness” (humanity?) is our shared evolutionary history. I like it. But Hawks also writes the following:


> 
It is our history that connects us to our distant relatives, not our genes. Even with a close relative like a twentieth cousin, there is a decent likelihood that you will share no genes at all because of your shared kinship from your most recent common ancestor. By the fiftieth generation, it is a virtual certainty. You are a genetic stranger to your ancestors.



I could share no genes with my twentieth cousin? This kind of sloppy use of terminology is not what I’ve come to expect from John. He’s usually at his best when writing about human genetics. You see, the quoted statement could vary from true to wildly inaccurate depending on the definition of gene we’re using.





Let’s start with the molecular genetics definition of “gene”. In this case, a gene is a locus on a chromosome that performs some function. Recent work has shown that individuals vary in the particular genes they carry (as a result of [copy number polymorphisms](http://www.google.com/search?q=copy+number+polymorphism&amp;q=%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search), CNPs). That means I might have a gene you don’t have, or you might carry a gene that I’m missing. But CNPs do not lead to totally different genes between two individuals. In fact, a large number of our genes are shared with our most distant mammalian relatives, indicating they’re probably found in all humans as well. So, using this definition of “gene”, Hawks is dead wrong. But that’s not the definition he intended.


Next, let’s consider a population genetics definition, where “gene” actually means “[allele](http://en.wikipedia.org/wiki/Allele)“. In this case, “sharing a gene” with someone actually means having the same allele as another person. If we limit ourselves to the 20,000 or so protein coding genes in the human genome, what’s the likelihood that two individuals have no genes with the same sequence in common? An analysis of coding sequence polymorphisms within humans and differences from chimpanzee genes found that 92.6% of the &gt;10,000 genes analyzed had at least one protein coding polymorphism or difference ([doi:10.1038/nature04240](http://dx.doi.org/10.1038/nature04240)). That means, conservatively, a few percent of all human genes do not vary within human populations (if anyone can cite data on synonymous variation, I’d greatly appreciate it). If those genes don’t vary within the human species, you’re guaranteed to have at least some shared genes with everyone. But I that’s not what John Hawks meant either.


Finally, we’ll consider a genealogical definition of gene. In this case, we’re interested in whether the genes two people have are identical by descent because they were inherited from one of the most recent common ancestors of the two people. For every gene, full siblings have a 75% chance of sharing the same gene, according to this definition of a gene. Because we have two copies of each gene (one from the mother and one from the father), we can also determine the probability full siblings will share zero, one, or two genes. There’s a 25% chance full siblings share zero genes, a 50% chance they share one gene, and a 25% chance they share both genes. First cousins will have 25% chance of sharing a gene. The probabilities of sharing genes by descent continue to decrease as you begin to deal with more distant relatives. This is what John Hawks meant when he wrote that you probably don’t share many genes with most of your relatives.


Despite the low probability of having the same genes passed on to you as your cousins had passed on to them, you still share an evolutionary history with them. In fact, all humans share an evolutionary history — a tighter history than we share with our closest living relatives, the chimpanzees. And that’s what Hawks says makes us human. As I said before, I like that definition, but I don’t think it’s the only one we can use.


---



Bustamante et al. 2005. Natural selection on protein-coding genes in the human genome. Nature 437: 1153-1157 [doi:10.1038/nature04240](http://dx.doi.org/10.1038/nature04240)


Gillespie 2004. Population Genetics: A Concise Guide. Johns Hopkins University Press, Baltimore.

#Going for a Sweep in Game 3
Date: June 23, 2008

[Dak at Fire Joe Morgan asks](http://www.firejoemorgan.com/2008/06/everyone-hates-adam-dunn.html):


> 
I’ve been watching a fair amount of SportsCenter / BBTN today, and every two minutes someone mentions that there are “seven teams going for a sweep in an interleague series!”, as if this is some sort of big deal. There are fourteen interleague series this weekend. If every match were a coin-flip, wouldn’t we expect exactly seven teams to be going for sweeps in the third game of a series?



We’ll start by assuming each team has an equal probability of winning each game, and the results of each game are independent. After one game, you’re guaranteed that one team will have a 1-0 advantage. There is a 50% chance that the team that won the first game will win the second, meaning that there is a 50% chance the series will be 2-0 after two games. Therefore, dak is correct, you would expect half of the series to be on the verge of a sweep.


Why did the media make a big deal out of this — and, I can confirm dak’s observation: this was a big deal on SportsCenter and Baseball Tonight — if it’s actually expected? Because they don’t understand basic probability. In fact, they are overtly antagonistic toward understanding simple logic and math. That’s why websites like Fire Joe Morgan are so much fun to read. Media stupidity is such an easy target, and, even when their stupidity is pointed out them, they show no interest in correcting their errors or remedying the problem. [It’s like dealing with creationists](http://scienceblogs.com/evolgen/2006/07/joe_morgan_the_creationist.php).

#Interview with Jenny Graves in PLoS Genetics
Date: June 27, 2008

PLoS Genetics has published an interview with Jenny Graves. Graves is one of the leaders in monotreme and marsupial genetics, and has been involved in some of the recent mammalian genome projects, including the platypus genome project ([doi:10.1038/nature06936](http://dx.doi.org/10.1038/nature06936)). She is also an expert in the evolution of mammalian sex chromosomes and sex determining genes. However, I’d like to point to a quote in PLoS Genetics’ interview of Graves that deals with science education:


> 
So I’m becoming very interested in education, particularly of young children, which is where I think the rot sets in. Science is not taught well even at high school level, and at primary school level it is taught by people who are generally scared of science! Anybody who has anything to do with kids this age knows they are incredibly observant and incredibly clever at working out how what they observe relates to other things. Somehow that just gets lost. I’d love to see more attention on encouraging young kids to make their own hypotheses–crazy though they may be.



The entire interview is interesting, and it touches on many of the highlights (and lowlights) of Graves’ career.

#The Evolutionary History of Birds
Date: June 27, 2008

**<u>Phylogeny Friday — 27 June 2008</u>**


I haven’t done a [Phylogeny Friday](http://www.google.com/search?hl=en&amp;newwindow=1&amp;safe=off&amp;q=%22phylogeny+friday%22+%28site%3Aevolgen.blogspot.com+%7C+site%3Ascienceblogs.com%2Fevolgen%2F+%29&amp;btnG=Search) in about a month, but a recent paper reporting a “phylogenomic study of birds” was worth mentioning ([doi:10.1126/science.1157704](http://dx.doi.org/10.1126/science.1157704)). Now, this isn’t [phylogenomics as Jonathan Eisen defined it](http://www.genome.org/cgi/content-nw/full/8/3/163/). The bird evolution paper describes building a tree using lots of molecular markers.


I don’t have much to say about the new bird phylogeny (I’ll let [the expert handle the details](http://scienceblogs.com/grrlscientist/2008/06/early_birds_shake_up_avian_tre.php)), but I wanted to post one of the trees for Phylogeny Friday. Here is one of the phylogenies they present, attempting to reconcile their results with those that had been previously published:


[<img alt="i-1d6af69d2c3a35cc5db1aa79afdfc24c-bird_phylogeny_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-1d6af69d2c3a35cc5db1aa79afdfc24c-bird_phylogeny_sm.JPG"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5ee98ecbb12a314e15a588299eb3d60e-bird_phylogeny_lg.jpeg)<br/>Click to enlarge.


The three columns at the tips indicate three previously published taxonomic classifications. Those with black text are monophyletic taxa in this new study, while those with white text are paraphyletic. The different colored lineages indicate various clades that are supported by this new study (e.g., green indicates land birds, blue are water birds, etc.).


---



Eisen. 1998. Phylogenomics: Improving Functional Predictions for Uncharacterized Genes by Evolutionary Analysis. Genome Res. 8: 163-167 [[link]](http://www.genome.org/cgi/content-nw/full/8/3/163/)


Hackett et al. 2008. A Phylogenomic Study of Birds Reveals Their Evolutionary History. Science 320:1763-1768 [doi:10.1126/science.1157704](http://dx.doi.org/10.1126/science.1157704)

#Synteny — A Semantic Debate
Date: June 30, 2008

There’s a post up at Pharyngula describing the concept of synteny in comparative genomics ([Basics: Synteny](http://scienceblogs.com/pharyngula/2008/06/basics_synteny.php)). The definition given by PZ Myers will sound pretty familiar to those of you who have read some of the genomics literature. The problem: it’s not quite correct. It’s actually the definition that I think most comparative genomics folks would give if they were asked to define synteny. But they keep using that word, and I don’t think it means what they think it means. What’s the definition? Here it is in PZ’s own words:


> 
Synteny is the conservation of blocks of order within two sets of chromosomes that are being compared.



I disagree. While this is what many genomicists mean when they write or talk about synteny, they are wrong. Instead, I would argue that synteny merely means that genes are found on the same chromosome. Synteny says nothing about the order of genes. What give me the right to say this?





Let’s take a quick journey through the literature. In a paper comparing the genomes of various mammals, Joseph [Nadeau and colleagues wrote the following](http://www.genetics.org/cgi/content/abstract/147/1/289):


> 
**Synteny** refers to the occurrence of two or more genes on the same chromosome, whereas **conserved synteny** refers to two or more homologous genes that are syntenic in two or more species, regardless of gene order on each chromosome, i.e., synteny but not necessarily gene order is conserved (Figure 2; see also NADEAU 1989). **Conserved linkage** pertains to the conservation of both synteny and order of homologous genes between species (Figure 2; see also NADEAU 1989). A **disrupted synteny** refers to circumstances where a pair of genes are located on the same chromosome in one species but their homologues are located on different chromosomes in another species, i.e., the genes are syntenic in only one of the two species.



[<img alt="i-2d75cf4cd6e3e7af82e6061cd4b1d292-synteny_nadeau.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-2d75cf4cd6e3e7af82e6061cd4b1d292-synteny_nadeau.gif"/>](http://dx.doi.org/10.1016/0168-9525(89)90031-0)


In the Nadeau framework, if genes are found in the same order in two species, we say there is conserved linkage. Is there any precedence for this terminology? Well, here’s the relevant passage from Nadeau’s 1989 paper ([doi:10.1016/0168-9525(89)90031-0](http://dx.doi.org/10.1016/0168-9525(89)90031-0)):


> 
Conserved syntenies are homology segments composed of two or more pairs of homologous genes located on the same chromosome, regardless of gene order. These represent the first formal evidence for conservation.
Conserved linkages are the most rigorously defined segments because both synteny and gene order must be conserved. Distinctions between the three characterizations, which are illustrated in Fig. 1, are important for understanding the extent and nature of conservation and for assessing progress towards saturated maps of linkage and synteny homologies.



In the figure to the right, genes A, B, and C are used to illustrate conserved synteny and conserved linkage. In panel A, there is no conserved synteny. Panel B shows conserved synteny, but not conserved linkage. And Panel C shows conserved linkage (which implies conserved synteny).


Why does any of this matter? Well, rather than talk about things like micro- and macro-synteny, as Myers and various other do, there is only synteny. This clarifies the terminology a bit. We have a separate term for the conservation of gene order of syntenic genes — conserved linkage. A uniform and clear vocabulary would make the literature and discussion within the comparative genomics community more precise. Precision is good, right?


Is anyone in the comparative genomics community using this terminology? Yes, there are a few people, but it’s a small group that doesn’t carry much weight: Drosophila geneticists. But, hey, Drosophilists invented modern genetics, so they are a bit of an authority on the topic. The second sequenced Drosophila genome, that of *D. pseudoobscura*, provided an opportunity to compare gene order between that species and *D. melanogaster* ([doi:10.1101/gr.3059305](http://dx.doi.org/10.1101/gr.3059305)). In this paper, the conserved linkage terminology was used.


Okay, but what the heck are syntenic blocks? Yeah, that’s a bit of a oddity. It appears that syntenic block has become the term to refer to syntenic genes with conserved linkage. I’d prefer “conserved linkage blocks”, but those of us who favor clear terminology may have lost this war. So, it now seems like there are syntenic genes (those found on the same chromosome) and syntenic blocks (groups of syntenic genes found in the same order in the species being compared).


---



Ehrlich et al. 1997. Synteny Conservation and Chromosome Rearrangements During Mammalian Evolution. Genetics 147: 289-296 [[link]](http://www.genetics.org/cgi/content/abstract/147/1/289)


Nadeau 1989. Maps of linkage and synteny homologies between mouse and man. Trends Genet. 5: 82-86 [doi:10.1016/0168-9525(89)90031-0](http:/dx.doi.org/10.1016/0168-9525(89)90031-0)


Richards et al. 2005. Comparative genome sequencing of **Drosophila pseudoobscura**: Chromosomal, gene, and cis-element evolution. Genome Res. 15:1-18 [doi:10.1101/gr.3059305](http://dx.doi.org/10.1101/gr.3059305)

#Darwinism is Dead
Date: July 17, 2008

Olivia Judson says [Darwinism is dead](http://judson.blogs.nytimes.com/2008/07/15/lets-get-rid-of-darwinism/). [She’s right](http://scienceblogs.com/evolgen/2006/11/darwin_as_a_false_idol.php). Anyone who talks about “Darwinism” or “evolutionists” gets my attention. That’s not to say that any use of those terms is incorrect. But they are often used as framing devices by creationists, and those frames get carried over into the lay discussion of biology. You should read her discussion of why we should get rid of Darwinism.


On a somewhat unrelated note, Judson also writes the following:


> 
We’d want to discuss evolution beyond natural selection — the other forces that can sometimes cause (or prevent) evolutionary change. For although natural selection is the only creative force in evolution — the only one that can produce complex structures such as wings and eyes — it is not the only force that affects which genes will spread, and which will vanish.



Natural selection is not creative. It acts on variation that is created by mutation. That makes mutation the only creative force. Natural selection, like all other forces besides mutation, can produce complex structures (yes, [complexity can be produce by entirely neutral processes](http://dx.doi.org/10.1126/science.1089370)). But, without mutation, there is no new variation upon which those forces can act.

#Creationists Love Baseball
Date: July 17, 2008

<img alt="i-84ae6eb6b8766eca8ea8bd6059cd4713-reds.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-84ae6eb6b8766eca8ea8bd6059cd4713-reds.gif"/>


The [Creation](http://www.pandasthumb.org/archives/2007/05/welcome-to-the-1.html) [Museum](http://www.pandasthumb.org/archives/2007/05/media-coverage.html) is located in northern Kentucky, just across the Ohio border from Cincinnati. [Answers in Genesis](http://www.talkorigins.org/faqs/organizations/#creation), the folks behind the so-called “museum”, [claim that their “museum” is within a 6 hour drive of 2/3 of the US population](http://stevereuland.blogspot.com/2007/01/they-suck-at-geography-too.html). This is not true — Kentucky is in the middle of bumfuck nowhere (I’m an expert on cities in the middle of bumfuck nowhere), and most people in the US can’t get there in 6 hours.


But the museum is damn close to Cincinnati — it’s in what people would call the “greater Cincinnati area”, and it’s closer to the so-called Cincinnati airport (which is actually in Kentucky) than the airport is to Cincinatti. Anyway, it’s so close to Cincinnati (how close is it?) that Answers in Genesis is advertising their “museum” on during what can be considered the most important event in all of Cincinnati: the broadcast of their beloved Reds’ games ([Reds think evolution is for suckers](http://www.thermocaster.com/2008/07/15/reds-think-evolution-is-for-suckers/)). If you listen to the local baseball team on the radio, you’re going to hear about the Creation Museum.


Now, here’s where it all comes full circle. You see, one the greatest Reds of all time, Joe Morgan, also happens to be an [absolute ignoramus when it comes to baseball](http://www.firejoemorgan.com/2005/04/some-questions-and-some-answers.html) (Hey, Bob. You remember the other day, you asked me what the definition of “irony” was…). I’ve previously dubbed him the [baseball creationist](http://scienceblogs.com/evolgen/2006/07/joe_morgan_the_creationist.php). Well, it appears that the Reds are going back to their roots by advertising the Creation Museum on their broadcast — reaching out to Little Joe in the way a bunch of ignorant fucktards only know how.

#Oh Deer!
Date: July 18, 2008

Until yesterday, there was a span of about two weeks in which this blog laid dormant. I did that on purpose because I didn’t want to give you all the blogging you crave. All three of you who may crave my blogging. And I’m guessing not one of those three even noticed the silence. Ah, the joys of insignificance.


Besides being a lazy dick, I actually have a valid excuse for my silence: I was moving. Or, rather, I am moving (my move is not yet complete). Where am I moving? From grad-school to post-doc. It also happens to be a move from one city in the middle of nowhere to another city in the middle of nowhere. But I’m even more in the middle of nowhere in my new locale that I was previously. So much so, that when I looked out my window on the first day in my new home, this is what I saw:


<img alt="i-85648561af57a39eaa16956b69cc9fa0-ohdeer.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-85648561af57a39eaa16956b69cc9fa0-ohdeer.jpg"/>


And I think I saw at least one of those each day I was there. That’s not to say that there aren’t deers in the middle of nowhere town I lived in during grad school (there are plenty); but my new house is in the middle of nowhere within the middle of nowhere. It’s isolate, which is not necessarily a bad thing.


In closing, two things: First, hopefully blogging will resemble yesterday’s activity more that the previous two weeks’ worth. Second, it’s not that I don’t like living in the small towns (I do), it’s just that I was surprised to be living so close to what can pass as “nature” in this modern age. I’m sure PZ Myers can top me by showing off how small his town is, how many woodland creatures he has prancing through his yard each day, and how many fucked up people he has sending him death threats every day.

#The Great Chain of Phylogenetic Wrongness
Date: July 18, 2008

**<u>Phylogeny Friday — 18 July 2008</u>**


[<img alt="i-79041b603a4234e9696cc165f220d8ee-nature_platypus_phylogeny_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-79041b603a4234e9696cc165f220d8ee-nature_platypus_phylogeny_sm.JPG"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-255906ba65dea653575a3cf03946cb15-nature_platypus_phylogeny.jpg)


When they published the initial analysis of the complete platypus genome ([doi:10.1038/nature06936](http://dx.doi.org/10.1038/nature06936)), **Nature**, as they’re wont to do, also put out a news item announcing the major findings ([doi:10.1038/453138a](http://dx.doi.org/10.1038/453138a)). That news article included a phylogeny illustrating the evolutionary relationships of various animal species in various stages of having their complete genomes sequenced.


The problem with the illustration: they got some of the relationships wrong. This sparked a letter from [Peter Ducey](http://www.cortland.edu/BIOLOGY/faculty/ducey.htm) of SUNY Cortland ([doi:10.1038/454027d](http://dx.doi.org/10.1038/454027d)), in which he wrote the following:


> 
In your News story ‘Top billing for platypus at end of evolution tree’, the graphic depicting genome status presents a shocking new phylogeny of the Vertebrates — with Archosaurs (birds and crocodilians) and Mammals forming a monophyletic group.



In all honesty, the tree is so craptacularly drawn that it’s hard to say that Archosaurs and Mammals are monophyletic based on the illustration. It looks more like the branching order of most vertebrates (and most animals) is unresolved, but I can definitely understand how Ducey interprets the tree. Either way, **Nature** did a bad job with this illustration. Why did they suck so bad? Here’s what Ducey thinks:


> 
The bad news is that this dramatic new ‘proposal’ is completely adrift from the research Article by Wesley C. Warren and colleagues that the figurative tree is intended to illustrate, and it continues a persistent tendency in popular literature to portray all evolution as leading towards humans.



Totally! The idea of a great chain of being, starting with bacteria and progressing through animals to humans, permeates much of the popular discussion of evolution. And it’s just not true. It’s a phylogenetic fallacy.


Coincidentally, Ducey points out, **Science** repeated the same error **Nature** made. In their coverage of the platypus genome paper ([doi:10.1126/science.320.5877.730](http://dx.doi.org/10.1126/science.320.5877.730)), **Science** included this image:


<img alt="i-e7260af0c4f9e3f5d1fdef09ed3f5acb-science_platypus_tree.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-e7260af0c4f9e3f5d1fdef09ed3f5acb-science_platypus_tree.gif"/>


Ouch! At least the tree accompanying the **Nature** article was ambiguous enough that it could be interpreted to represent the true phylogeny. While the cladogram from **Science** does not perpetuate the great chain of being myth, it does represent a completely resolved phylogeny that also happens to be incorrect.

#New Gene Makes Flies Less Gay
Date: July 21, 2008

We all know that [Drosophila are the gayest bunch of gays](http://evolgen.blogspot.com/2005/06/friday-random-ten-3-june-2005.html) that ever [gayed up genetics](http://scienceblogs.com/evolgen/2006/11/gay_drosophila_fighting.php). This is especially true when you create mutations in [fruitless](http://flybase.org/reports/FBgn0004652.html) ([nee fruity](http://www.wellcome.ac.uk/News/News-archive/Browse-by-date/2003/Features/WTD004521.htm)), “the gay gene”. Male flies with mutations in *fruitless* will try to get it on with other males (e.g., [doi:10.1016/S0092-8674(00)81802-4](http://dx.doi.org/10.1016/S0092-8674(00)81802-4)).  That’s gay!


<img alt="i-fa5a4e267e81a6674e47b694971739c3-Dai_etal_2008_fig3c.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-fa5a4e267e81a6674e47b694971739c3-Dai_etal_2008_fig3c.JPG"/>


But fruitless is an old school gene that needs to be fucked up to turn the flies gay ([doi:10.1093/molbev/msj070](http://dx.doi.org/10.1093/molbev/msj070); the first author on that paper is, I shit you not, named Gailey). Drosophila really aren’t as gay as they are made to appear in the articles describing *fruitless* mutants. But the males are still kind of in to dudes, as is shown by a new paper from Manyuan Long’s group ([doi:10.1073/pnas.0800693105](http://dx.doi.org/10.1073/pnas.0800693105)).





Long’s group previously described an RNA gene that is unique to *D. melanogaster*, and not found in any of its sibling species ([doi:10.1073/pnas.072066399). The gene, named <a href="http://flybase.org/reports/FBgn0083990.html">sphinx](http://dxdoi.org/10.1073/pnas.072066399), arose when the transcript of its parent gene was reverse transcribed and inserted back into the genome (in a process known as retroposition). It has also evolved quite rapidly, suggesting it was acted on by natural selection. Additionally, there are male-specific and a female-specific transcripts of **sphinx**.</a>


So, there is probably some evolution story to unravel about the *sphinx* gene. In the new paper by Dai et al., Long and colleagues have performed a functional analysis of **sphinx**, and what they found was quite interesting. Via an analysis of flies carrying mutant forms of *sphinx*, they were able to show that the gene plays a role in preventing males from courting other males. Males carrying two mutant copies of *sphinx* spend more time courting other males than flies with a good copy of *sphinx*.


<img alt="i-9fce889c009abc1168e5404291cadea3-Dai_etal_2008_fig3a.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-9fce889c009abc1168e5404291cadea3-Dai_etal_2008_fig3a.JPG"/>


Here’s where you ask: but *sphinx* is a new gene — what about the close relatives who don’t have any copies of *sphinx*? Well, they’re gayer than *D. melanogaster*. When wild type flies are compared, males from the other species all spend more time courting other males than do *D. melanogaster* males, suggesting that the *sphinx* gene keeps *D. melanogaster* males from being as gay as their close relatives.


<img alt="i-be5c50a8f14215ba6fd47ad8b7cee9c8-Dai_etal_2008_fig4a.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-be5c50a8f14215ba6fd47ad8b7cee9c8-Dai_etal_2008_fig4a.gif"/>


This would be a great opportunity to test genetic engineering for the “correction” of teh gay. You see, if you were to insert *sphinx* into the other species, they should be less gay. Unfortunately, Dai et al. don’t do that experiment, so we’re left to wonder with gayness can be cured by transgenics.


---



Dai et al. 2008. The evolution of courtship behaviors through the origination of a new gene in Drosophila. PNAS 105: 7478-7483 [doi:10.1073/pnas.0800693105](http://dx.doi.org/10.1073/pnas.0800693105)


Gailey et al. 2006. Functional Conservation of the fruitless Male Sex-Determination Gene Across 250 Myr of Insect Evolution. MBE 23: 633-643 [doi:10.1093/molbev/msj070](http://dx.doi.org/10.1093/molbev/msj070)


Ryner et al. 1996. Control of Male Sexual Behavior and Sexual Orientation in Drosophila by the fruitless Gene. Cell 87: 1079-1089 [doi:10.1016/S0092-8674(00)81802-4](http://dx.doi.org/10.1016/S0092-8674(00)81802-4)


Wang et al. 2002. Origin of sphinx, a young chimeric RNA gene in Drosophila melanogaster. PNAS 99: 4448-4453 [doi:10.1073/pnas.072066399](http://dxdoi.org/10.1073/pnas.072066399)

#The Evolution & Medicine Review
Date: July 22, 2008

[<img alt="i-93ff69b29a4fb2c11e9db414f0a271f0-doonesbury_evol_medicine_sm.GIF" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-93ff69b29a4fb2c11e9db414f0a271f0-doonesbury_evol_medicine_sm.GIF"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-77eef08af41adf86cf8c9e74277b8ebf-doonesbury_evol_medicine.gif)


All good medicine is evidence based — that is, diagnoses and treatments are developed via the scientific method. Oftentimes, evolutionary biology is employed to understand human health and diseases. This is known as [evolutionary medicine](http://www.evolutionandmedicine.org/).


Evolutionary medicine is a growing field that takes an interdisciplinary approach toward studying human disease. Tools from population ecology, molecular evolution, comparative anatomy, and many other fields are all integrated with clinical medicine to improve our understanding of human disease and develop new treatments. This approach can be applied to infectious diseases, congenital diseases, and acquired diseases.


Despite the rapid growth of this field, there is no central society or journal that deals with these topics. Additionally, because relevant research is often spread across otherwise disparate subdisciplines, people looking for literature on evolutionary medicine must search many different journals. To remedy this problem, a group of researchers working in evolutionary medicine have started an on-line review journal, or Web Based ReView (WeView), called [The Evolution &amp; Medicine Review](http://evmedreview.com/). The creators view it as a sort-of cross between a blog and a review journal, although I’d categorize it as a single topic blog written by experts.


[The Evolution &amp; Medicine Review](http://evmedreview.com/) does not publish original research. Instead, the list of contributors produce short reviews on topics that interest them (very bloggy, not that it’s a bad thing). You can [read more about the EMR](http://evmedreview.com/?page_id=4) on their website.


(Via [EvolDir](http://evol.mcmaster.ca/~brian/evoldir/Other/Evolution_Medicine.journal).)

#A Sizzling Hypocrisy
Date: July 22, 2008

<img alt="i-069ff01f5cab943956c577eed80c7c5d-sizzle.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-069ff01f5cab943956c577eed80c7c5d-sizzle.jpg"/>


Randy Olson left a career as a marine biologist ([Titleist!](http://en.wikipedia.org/wiki/The_Marine_Biologist)) to become a film maker. His first feature project was [Flock of Dodos](http://scienceblogs.com/evolgen/2006/06/flock_of_dodos_1.php), a movie I enjoyed. His second film is [Sizzle](http://www.sizzlethemovie.com/), a movie <a href="http://scienceblogs.com/sizzle.php
">reviewed by lots of ScienceBloggers a couple weeks ago</a>. The gist: [a lot of ScienceBloggers didn’t like sizzle](http://www.google.com/cse?cx=017254414699180528062%3Auyrcvn__yd0&amp;q=sizzle&amp;sa.x=0&amp;sa.y=0&amp;sa=search). Neither did a reviewer for *Nature* ([doi:10.1038/454279a](http://dx.doi.org/10.1038/454279a)).


I did not request a review copy of the movie because I don’t like to diverge much from the main themes of evolgen: evolutionary genetics, manatees, and the douchebag who writes this blog. But some of the recent discussion surrounding *Sizzle* has caught my attention, and I think it relates to the broader issue of science outreach (that is: communicating science to the general public).





[Chris Mooney](http://scienceblogs.com/intersection/2008/07/in_reviewing_sizzle_can_scienc.php) is one of the ScienceBloggers who did like the movie. He thinks scientists didn’t like the movie because they saw Randy’s character as a reflection of themselves:


> 
Could it be that, for some of these hypercritical bloggers, Randy Olson’s documentarian character in Sizzle is really their reflection in the mirror? After all, the character is basically a caricature of someone who repeatedly demands facts, facts, facts, and can’t relate to non-scientists, have a good laugh, enjoy a good story.



After viewing the film at a public screening during Outfest, a gay and lesbian film festival, [Chris has more to say about *Sizzle*](http://scienceblogs.com/intersection/2008/07/sizzling_in_la.php):


> 
After all, here we have an innovative attempt, by a scientist-filmmaker who is one of us and part of our community, to reach new demographics with information about one of our recurrent and core topics–global warming. In short, Sizzle represents the kind of effort that we ought to be promoting vigorously, selflessly, and with enthusiasm–especially given the lack of much available funding for this kind of experiment in Hollywood from major studios.



As someone who has not seen *Sizzle*, but who has seen Randy’s other movie and read his blogging on other issues related to science outreach, I can understand the negative critiques of Olson’s new movie. In Randy’s mind, [he’s the expert on communicating science](http://scienceblogs.com/evolgen/2008/04/expelled_exaggerated.php), and he loves telling scientists that they’re doing it wrong. However, he won’t tell us how to do it right!


The problem with Chris’s argument is the following. It’s not that scientists don’t support reaching out to the general public — see, for example, scientists who blog. It’s that the scientists who saw *Sizzle* didn’t think Olson did a good job of communicating science to the general public. Perhaps the limited audience of non-scientists who saw the movie enjoyed it (according to Randy and Chris), but did it accomplish anything in increasing their understanding or awareness of the issues? If not, did it accomplish any of the film maker’s goals (if there were any goals beyond entertainment).


Alas, I may not be allowed to bring up such criticisms of the Olson/Mooney message. [According to Mooney](http://scienceblogs.com/intersection/2008/07/sizzling_in_la.php):


> 
Yet instead, what I see is far too much negativity, far too much criticism, directed at a film that is funny and damn good and ultimately very profound–all of which the audience in LA clearly appreciated. So why can’t we?



Apparently, Randy Olson and Chris Mooney are allowed to criticize scientists for doing a bad job with outreach. But scientists aren’t allowed to criticize Olson when he does a bad job of it (after all, Olson is “The Expert”). I think I smell a hypocrite. I don’t mind Randy Olson riding in the backseat of the science outreach/communication wagon (he’s not gonna take a pro-active position and drive), so long as he stops telling everyone else, “You’re doing it wrong!” If we’re doing it wrong, show us how to do it right.

#Attack of the Horny Peacock
Date: July 25, 2008

Last month I [mentioned that I had been in Barcelona](http://scienceblogs.com/evolgen/2008/06/where_ive_been_the_past_couple.php) at the Society for Molecular Biology and Evolution conference. I arrived in Spain early in the morning, and, after I got checked in to my hotel, I went with a couple friends to the [Barcelona Zoo](http://www.zoobarcelona.com/ZOO_Barcelona/English/English.asp). This zoo is famous for housing [Snowflake](http://www.zoobarcelona.com/ZOO_Barcelona/English/English.asp), the albino gorilla who lived at the zoo from the mid-1960s until he died in 2003. They still have an impressive collection of primates despite the loss of the zoo’s icon.


Additionally, the zoo has a roaming band of peafowl. The peacocks and peahens have free reign of the grounds, and you’ll often see them roosting in the trees. When we were there, the males were aggressively courting the females and letting out loud mating calls. I captured this photo of a peacock trying to impress a hen:


<img alt="i-e807074c9b5a884d6a5f3d5ca4f158ab-barcelona_peacock.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-e807074c9b5a884d6a5f3d5ca4f158ab-barcelona_peacock.jpg"/>


She would have none if it, though. Eventually, the peahen scurried into the bushes, and the peacock wandered off to find a hen that would be impressed by his massive train.

#Crocodile Fishing
Date: July 26, 2008

So, I’m just hanging out here by the side of the water waiting for my lunch. Sure, I could go in the water and get my lunch. But that’s not how I roll. I wait patiently for my prey to get within striking distance, and then I attack. So, here I am just hanging out by the side of the water.


[<img alt="i-2005fed5ecbbd85516af73afbf211226-croc_fish_1_sm.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-2005fed5ecbbd85516af73afbf211226-croc_fish_1_sm.jpg"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-605349e4c1b905b5f40816fd5b11d4b8-croc_fish_1_lg.jpg)<br/>**Click to enlarge.**


There they are. Just a bunch of cichlids waiting to be eaten. By me. Lunch would be good right now. But they won’t get close enough. So I’ll just hang out here with my jaw agape.





And I can see them. They’re not too far away. Maybe one of them will swim close enough to the shore. And then I’ll ****SNAP**** right down on them. It would beat the hell out of the freeze-dried meat the zoo-keepers serve to me.


[<img alt="i-b0fd331051f2be3f9da3c7bcac6b5485-croc_fish_2_sm.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b0fd331051f2be3f9da3c7bcac6b5485-croc_fish_2_sm.JPG"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-f77acae2b9cb9de1e8fbf402f7f2fbe8-croc_fish_2_lg.jpg)<br/>**Click to enlarge.**


Meanwhile, ten feet away and below the surface:


[<img alt="i-853e010617d4e045940004e14bef2851-croc_fish_3_sm.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-853e010617d4e045940004e14bef2851-croc_fish_3_sm.jpg"/>](http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-37b269f990099fddc3e9000256f210cc-croc_fish_3_lg.jpg)<br/>**Click to enlarge.**


Holy shit! There’s a giant scaly creature with a huge mouth and sharp teeth right over there! Must swim away fast! Must swim away faster! Faster, I say. FASTER!


*I snapped these pics at the Barcelona Zoo some time after I saw [the peacock trying to get his groove on](http://scienceblogs.com/evolgen/2008/07/attack_of_the_horny_peacock.php). That crocodile just sat there waiting for the cichilds to get close enough to score a free meal. I wonder if the two crocs take turns waiting for a fish to make a wrong turn.*

#Sex & SMBE in 2009
Date: July 28, 2008

[<img alt="i-d4953cd94ff8908d648493e7aca8d6b0-SMBE2009_small.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d4953cd94ff8908d648493e7aca8d6b0-SMBE2009_small.jpg"/>](http://ccg.biology.uiowa.edu/smbe/)


The University of Iowa is hosting next year’s meeting of the Society of Molecular Biology and Evolution, [SMBE 2009](http://www.smbe2009.org/). I usually go to the annual SMBE conference, and I was probably going to attend SMBE 2009. Now I’m definitely going. Why? Because [John Logsdon just announced](http://johnlogsdon.blogspot.com/2008/07/coitus-interruptus-in-iowa.html) that they’ll be hosting a pre-conference meeting on the [Evolution of Sex and Recombination](http://ccg.biology.uiowa.edu/sexrec/).


The Sex &amp; Recombination meeting was supposed to happen this summer, and it was scheduled for the week prior to the [Evolution 2008](http://www.cce.umn.edu/conferences/evolution/) conference in Minnesota. But mother nature interrupted, and the flooded campus was in no state to host the conference ([coitus interruptis](http://johnlogsdon.blogspot.com/2008_06_01_archive.html) as John calls it). So it’s really good to hear that they’ll be able to have the meeting in 2009. Hopefully I’ll have some sexy results to present.

#Is There Too Much Hypothesis Testing?
Date: August 1, 2008

About a month ago, we were told that [theory is dead](http://scienceblogs.com/evolgen/2008/07/the_end_of_theory.php). That was the thesis of [Chris Anderson’s article in Wired](http://www.wired.com/science/discoveries/magazine/16-07/pb_theory). Rather than testing hypotheses using the scientific method, Anderson argues that scientists are merely generating loads of data and looking for correlations and stuff. The article was a bit muddled, but that’s Anderson’s main point . . . I think.


Well, now the Times Higher Education has published an article by [Tim Birkhead](http://www.shef.ac.uk/aps/staff/acadstaff/birkhead.html) in which he argues the opposite ([In praise of fishing trips](http://www.timeshighereducation.co.uk/story.asp?storycode=403006)). Birkhead says that the scientific establishment is too attached to hypothesis testing. This means funding agencies are reluctant to approve projects that are designed to collect data without testing *a priori* hypotheses. Here’s the brunt of his argument:


> 
The scientific research councils seem to be obsessed by hypothesis testing. Many times I have heard it said by referees rejecting a proposal: “But there was no hypothesis.” The research council modus operandi now seems to be the h-d method, but taken to an almost ludicrous extreme – researchers basically have to know what they are going to find before putting in a research application (in truth, they need to have done the research, even though they will not actually say so, to write a convincing account of what they intend to do). That’s hardly the best way of doing science. The other problem, and it is a much more serious one, is that by knowing what you will find out, science becomes trivially confirmatory and inherently unlikely to discover anything truly novel.



So, which one is it? Have we abandoned hypothesis testing? Or are we so attached to hypothesis testing that we overlook novel projects that would explore underdeveloped areas of research?

#How do you really feel, Dr. Wakeley?
Date: September 15, 2008

[<img alt="i-5f3a9302f47bbd086bdb779829bb9125-coalescent_theory.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-5f3a9302f47bbd086bdb779829bb9125-coalescent_theory.jpg"/>](http://www.roberts-publishers.com/wakeley/)


I’m currently working my way through [John Wakeley’s book on Coalescent Theory](http://www.roberts-publishers.com/wakeley/index.html). (The website has a few pre-publication chapters if you want to take a peek.) In his introductory chapter, Wakeley introduces the concept of gene genealogies. He’s careful to point out that, unlike the phylogenies we construct using inter-specific data, we don’t actually use intra-specific gene genealogies to infer the relationships of the sequences we’ve sampled:


> 
Readers used to “tree thinking,” which is the subject of Section 1.1, will have little trouble seeing the close connection between gene genealogies and patterns of variation in a sample. It will be clear in what follows, however, that there is a danger in carrying over too much from the field of phylogenetics. Gene genealogies are by their nature unobservable, and they are treated as random variables in a statistical setting. This is necessary because genetic reproduction in a finite population is a random process. Sources of randomness include simple Mendelian inheritance as well as other factors, such as the effects of weather, predation, and competition on the birth and death of individuals.
Thus, while in phylogenetics the tree structure itself is significant, within species it is often the case that particular gene genealogies provide little information about population-level processes and events. These issues are manifest in recent controversies surrounding the use of nested clade analysis ([Templeton et al., 1995](http://www.genetics.org/cgi/content/abstract/140/2/767); [Templeton, 1998](http://dx.doi.org/10.1046/j.1365-294x.1998.00308.x)) or, more broadly, about differing sensibilities between the fields of intraspecific phylogeography ([Avise et al, 1987](http://dx.doi.org/10.1146/annurev.es.18.110187.002421); [Avise, 1989](http://www.jstor.org/pss/2409356), [2000](http://www.amazon.com/Phylogeography-Formation-John-C-Avise/dp/0674666380)) and coalescent theory. See [Knowles and Maddison (2002)](http://www.ncbi.nlm.nih.gov/pubmed/12453245) and [Hey and Machado (2003)](http://dx.doi.org/10.1038/nrg1112) for two perspectives on these issues.



Basically, Wakeley is calling out two high-profile researchers for their role in blurring the line between gene genealogies and phylogenetics: [Alan Templeton](http://www.biology.wustl.edu/faculty/templeton/) and [John Avise](http://www.faculty.uci.edu/profile.cfm?faculty_id=5292). Avise gets credited for creating [phylogeography](http://en.wikipedia.org/wiki/Phylogeography), while Templeton is responsible for [nested clade analysis](http://darwin.eeb.uconn.edu/eeb348/lecture-notes/nca/node3.html). Both of these approaches have classically ignored the role stochastic processes play in shaping intra-specific genetic variation. In doing so, practitioners of phylogeography often invoke just-so-stories to explain the demography history of their species of interest.





Now, this isn’t the first time a population geneticists has questioned the statistical rigor of phylogeography (Wakeley provides references for further discussion of the issue, but also see [Beaumont and Mahesh’s Panchal’s critique](http://dx.doi.org/10.1111/j.1365-294X.2008.03786.x)). In fact, in [Rasmus Nielsen’s review](http://dx.doi.org/10.1554/BR06-3.1) of a [different book on coalscent theory](http://www.amazon.com/gp/product/0198529961/002-3473940-3958414) he also [pointed out the necessity of applying coalscent theory in understanding demographic history](http://scienceblogs.com/evolgen/2006/02/the_coalescent.php):


> 
One of the important insights we have gained from coalescent theory is that the same population history may generate very different gene trees if repeated and that very different historic scenarios may sometimes generate gene trees that are surprisingly similar.



Perhaps we’re in need of a t-shirt to wear at next year’s [Evolution Meeting](http://www.evolutionsociety.org/meetings.asp):


<img alt="i-119a3412e21b679f324c1cadc3704bc8-down_with_phylogeography.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-119a3412e21b679f324c1cadc3704bc8-down_with_phylogeography.gif"/>

#What Post(s) Should I Submit to Open Lab?
Date: September 28, 2008

[<img class="inset" src="http://scit.us/openlab/openlab08-submit.150.png"/>](http://openlab.wufoo.com/forms/submission-form/)


Every year, the best science blog posts are collected in a book, the Open Laboratory (here are the [2006](http://www.lulu.com/content/631016) and [2007](http://www.lulu.com/content/1869828) editions). [Last year’s edition](http://scienceblogs.com/evolgen/2008/01/one_art_please.php) included my cartoon, [The Lab Fridge](http://scienceblogs.com/evolgen/2007/05/the_lab_fridge.php). It wasn’t my best post of the year, but it filled one of the niche categories published in the book.


Bora has been [soliciting submissions](http://scienceblogs.com/clock/2008/09/open_laboratory_2008_submissio_1.php) for this year’s edition of Open Lab. I’ve dug through my archives and found a few posts that I think are worthy of submission. Unfortunately, I’m the worst judge of my own writing, so I need some help. I’ve provided links to the short list of my best blogging of 2008 (up ’til now). Go check them out and let me know which one you like best.





- [**These Are the People at Your Departmental Seminar**](http://scienceblogs.com/evolgen/2008/02/these_are_the_people_at_your_d.php):
#Adaptationist Junk
Date: October 5, 2008

The human genome (like all mammalian genomes) is loaded with sequences that don’t perform any known function. And many of these sequences are junk. And it’s not just mammals — many animal genomes are loaded with junk, as are those of other eukaryotes. That’s not to say that some of the sequences of unknown function do, in fact, have a function that we have yet to identify. However, much of the junk comes from the remnants of transposable element (parasitic sequences that hop around the genome).


That’s not the impression you get from the introduction to a review of transposable elements (TEs) published in the most recent issue of Cell ([doi:10.1016/j.cell.2008.09.022](http://dx.doi.org/10.1016/j.cell.2008.09.022)):


> 
Following the discovery of transposons in plants and bacteria, the presence of mobile DNA in eukaryotic species gained widespread acceptance. However, the concept of “controlling elements” gave way to disparaging terms such as selfish DNA and “junk DNA.” Nevertheless, the notion of transposable elements as merely molecular parasites, benign at best and powerful mutagens at worst, that hijack cellular mechanisms for their own selfish propagation, seemed incomplete to some biologists. Given that evolution tends to dispose of that which is useless and harmful for a species, it was curious that the genome should be cluttered with so much “junk.” Now we understand that genomes have coevolved with their transposable elements, devising strategies to prevent them from running amok while coopting function from their presence. Repetitive DNA, and retrotransposons in particular, can drive genome evolution and alter gene expression. Evolution has been adept at turning some “junk” into treasure.



I don’t see how the hypothesis that transposons are selfish genetic elements conflicts with the observation that host genomes have coevolved with these parasites. That is, in fact, what we expect to happen between hosts and parasites — [red queen](http://en.wikipedia.org/wiki/Red_Queen) and all.





As for cooption, that probably applies to a minority of TEs in eukaryotic genomes. The vast majority of them are just there, incapable of being removed by natural selection. Why? Because selection can only efficiently remove features that are deleterious enough to overcome the effects of genetic drift (the random change in allele frequencies due to sampling error).


Are TEs just hangers on? No, they can dramatically change the architecture of a genome. However, that does not meaning they are being [exapted](http://en.wikipedia.org/wiki/Exaptation). It just means that they’re there, and that’s what they do. They can lead to the deletion of functional elements. They can rearrange chromosomes. They can even be used to control the expression of nearby genes. That last one is an example of exaptation. The others are merely things that happen as the result of TEs. They can be deleterious, neutral, or beneficial.


But TEs aren’t kept around because they have the potential to do lots of stuff to the host genome. That’s not how natural selection works — it can’t look into the future. The TEs are kept around because the deleterious effects of those mutations (rearrangements, insertions, deletions) aren’t bad enough to cause the TEs to be purged from the genomes in which they reside.


---



Goodier and Kazazian. 2008. Retrotransposons Revisited: The Restraint and Rehabilitation of Parasites. Cell 135: 23-35 [doi:10.1016/j.cell.2008.09.022](http://dx.doi.org/10.1016/j.cell.2008.09.022)

#Mendel’s Garden #25 — Call for Submssions
Date: October 6, 2008

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://mendels-garden.blogspot.com/)


I will be hosting Mendel’s Garden #25 at evolgen. That’s right: the carnival is back! [Mendel’s Garden](http://mendels-garden.blogspot.com/) is the original blog carnival devoted to genetics. Submit your genetics related posts to evolgen[at]yahoo[dot]com.


Because Mendel’s Garden has been dormant for the past few months, I’ll be accepting posts written from March through October. So, dig through your archives to find something good, and shoot me an email.

#Slightly Deleterious in Trans
Date: October 7, 2008

One of the hot topics in evolutionary biology concerns the relative contributions of protein coding sequence changes and non-coding changes that lead to differences in the expression of protein coding genes. A subset of this debate can be summarized as *cis* versus *trans*. Non-coding sequences that regulate gene expression are known as *cis* regulatory elements (CREs). The protein coding genes that bind to CREs and control the expression of genes act in *trans*.


<img alt="i-b871f783f41c2231e0d960d909263b94-wray_coyne_sm.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-b871f783f41c2231e0d960d909263b94-wray_coyne_sm.jpg"/>


That’s the background,and here are some more details. The poster boy for the importance of CREs is Sean Carroll, who argues that the [pleiotropic](http://en.wikipedia.org/wiki/Pleiotropy) effects of *trans* changes makes them unlikely to play a large role in the evolution of form ([doi:10.1371/journal.pbio.0030245](http://dx.doi.org/10.1371/journal.pbio.0030245)). On the other side of the debate is Jerry Coyne, who penned a strongly worded opinion piece with Hopi Hoekstra arguing that the evidence does not favor Carroll’s hypothesis ([doi:10.1111/j.1558-5646.2007.00105.x](http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x)). The past few years have seen a multiple studies examining the relative contributions of *cis* and *trans* changes using analyses of gene expression in hybrids (e.g., [doi:10.1038/nature02698](http://dx.doi.org/10.1038/nature02698)). In the end, the data reveal that both *cis* and *trans* changes play important roles.


Along similar lines as the experiments testing for *cis* and *trans* contributions to gene expression differences, Bernardo Lemos and colleagues in Dan Hartl’s lab have examined the dominance of *cis* and *trans* mutations by measuring gene expression in the parents and progeny of crosses between *Drosophila melanogaster* from various populations ([doi:10.1073/pnas.0805160105](http://dx.doi.org/10.1073/pnas.0805160105)). It’s a nifty study, and I’m interested in a conclusion they draw regarding the temporal dynamics of *cis* and *trans* changes.





Here is the relevant passage from the discussion of their paper:


> 
We argue that **cis**– and **trans**-regulation undergo distinct population genetic dynamics across short and long timescales, which lead to a relative overabundance of **trans**-regulation within population and a relative overabundance of **cis**-regulation between populations.



They support this conclusion with two observations. First, there is an excess of *trans* variation within populations. And, second, *cis* changes tend to be additive, while *trans* changes are dominant/recessive. That means recessive *trans* mutations can segregate in populations without phenotypic effects, while *cis* changes are exposed to selection from the get go (either purged by purifying selection or fixed by positive selection).


What’s interesting is how this model explains the maintenance of slightly deleterious mutations (namely, *trans* variation) by describing their dominance. While there may be a lot of *trans* variation in a population, it is unlikely to contribute to the divergence between species because, once a slightly deleterious *trans* mutation reaches an appreciable frequency, it will be purged by purifying selection in homozygotes. It’s an interesting angle, and it supports Carroll’s hypothesis — although without directly invoking the pleiotropic effects of *trans* mutations.


---



Carroll 2005. Evolution at Two Levels: On Genes and Form. PLoS Biol. 3: e245 [doi:10.1371/journal.pbio.0030245](http://dx.doi.org/10.1371/journal.pbio.0030245)


 Hoekstra and Coyne 2007. The locus of evolution: evo devo and the genetics of adaptation. Evolution 61: 995-1016 [doi:10.1111/j.1558-5646.2007.00105.x](http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x)


Lemos et al. 2008. Dominance and the evolutionary accumulation of cis- and trans-effects on gene expression. PNAS 105: 14471-14476 [doi:10.1073/pnas.0805160105](http://dx.doi.org/10.1073/pnas.0805160105)


Wittkopp et al. 2004. Evolutionary changes in cis and trans gene regulation. Nature 430: 85-88 [doi:10.1038/nature02698](http://dx.doi.org/10.1038/nature02698)

#Seminar Answer Flow Chart
Date: October 8, 2008

Inspired by the [Sarah Palin Debate Flow Chart](http://www.dailykos.com/storyonly/2008/10/3/43222/8057/718/618653), here’s the Seminar Answer Flow Chart:


<img alt="i-d21bac2034d353bee279d623e4257273-Seminar-Answer-Flowchart.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d21bac2034d353bee279d623e4257273-Seminar-Answer-Flowchart.gif"/>

#Get Your Masatoshi Nei Trading Card
Date: October 9, 2008

The Penn State Alumni Association has produced trading cards featuring the best and the brightest of the university’s faculty ([Pa. trading cards highlight brains, not brawn](http://news.yahoo.com/s/ap/20081008/ap_on_fe_st/odd_penn_st_trading_cards)). The cards are only available at University President Graham Spanier’s tailgate parties on home football weekends. The biologists featured include entomologist [Jim Tumlinson](http://www.ento.psu.edu/Personnel/Faculty/tumlinson.htm) and molecular evolution pioneer [Masatoshi Nei](http://www.bio.psu.edu/People/Faculty/Nei/).


<img alt="i-66a7af5745ab98504460ab4e7249703f-penn_st_trading_cards_nei.gif" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-66a7af5745ab98504460ab4e7249703f-penn_st_trading_cards_nei.gif"/>


I haven’t been able to find a complete list of all ten faculty members featured, and I don’t think that individual images of the ten cards are available online. If anyone can track them down, post a link in the comments.


**Update:** [Nina Fedoroff](http://146.186.172.254/lsc/fedoroff.html) has a card as well.

#They are NOT fruit flies
Date: October 27, 2008

<img alt="i-26df8e5d3bd7467c63c62d0a8b411e52-olive_fruit_fly.JPG" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-26df8e5d3bd7467c63c62d0a8b411e52-olive_fruit_fly.JPG"/>


In the recent kerfuffle over [Sarah Palin’s disparaging remarks about “fruit fly” research](http://www.youtube.com/watch?v=HCXqKEs68Xk), an important point was missed by the general public, scientists, and even Drosophila geneticists: [she wasn’t talking about Drosophila](http://scienceblogs.com/mikethemadbiologist/2008/10/about_that_fruit_fly_research.php). Now, this point has been clarified by a few people (notably [Mike the Mad Biologist](http://scienceblogs.com/mikethemadbiologist/2008/10/about_that_fruit_fly_research.php)), and I think people are starting to get it. But it was remarkable how people automatically assumed she was talking about Drosophila.


Okay, maybe it wasn’t so remarkable, given that even Drosophila researchers refer to these little insects as “fruit flies”. The problem with that nomenclature is that [Drosophila are not fruit flies](http://scienceblogs.com/evolgen/2008/04/phylogeny_friday_11_april_2008.php). The [olive fruit flies](http://www.napavalleyregister.com/articles/2008/04/09/news/local/doc47fc6ca1ad0ee555305173.txt) to which Palin was referring are true fruit flies ([Tephritids](http://en.wikipedia.org/wiki/Tephritidae)), and these guys are major agricultural pests. She was actually criticizing applied research (as opposed to basic), which makes her comments even more absurd.


But everyone out there assumes that she’s ragging on Drosophila research (which, for the most part, tends to be quite basic in nature — note that basic does not imply worthless), even Drosophila geneticists. I’ve been notified of “Sarah Palin’s attack on Drosophila” by multiple Drosophilists, only to point out to them that she wasn’t going after us. Funny thing is, there is at least one Drosophila geneticist who also works on Bactrocera (the genus to which Palin’s olive fly belongs): [Michael Eisen](http://www.michaeleisen.org/blog/?p=100). He’s also [fed about by the Drosophila/fruit-fly mix-up](http://www.michaeleisen.org/blog/?p=106). And you should be too. The least we can hope for is that people who actually study Drosophila will know that they’re not fruit flies.

#Another Call for Submissions to Mendel’s Garden #25
Date: October 28, 2008

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://mendels-garden.blogspot.com/)


I will be hosting the next edition of [Mendel’s Garden](http://mendels-garden.blogspot.com/) on Sunday, November 2. If you have written any blog posts about genetics in the past few months, send me a link (evolgen[at]yahoo[dot]com). Also, if you’ve seen any good genetics posts on other people’s blogs, let me know.


For those not in the know, Mendel’s Garden is the original genetics blog carnival. On the first Sunday of every month, the carnival plays host to the best genetics blogging of the past month. After a hiatus of a few months, the carnival is back.

#Mendel’s Garden #25
Date: November 2, 2008

After a few months off, here’s the return of [Mendel’s Garden](http://mendels-garden.blogspot.com/).


<li>
[Blast from the past](http://network.nature.com/people/ennis/blog/2008/10/06/blast-from-the-past): rENNISance woman gives us a post on viral genetics.
</li>
<li>
[Figuring out DNA looping with unbelievably advanced technology](http://scienceblogs.com/gregladen/2008/09/figuring_out_dna_looping_with.php): Greg Laden reviews a paper on the structure of nucleic acids.
</li>
<li>
[In the fly, delayed reproduction also delays aging](http://ouroboros.wordpress.com/2008/09/10/in-the-fly-delayed-reproduction-also-delays-aging/): Ouroboros describes research on senescence in Drosophila.
</li>
<li>
[Balancer Chromosomes](http://sandwalk.blogspot.com/2008/10/balancer-chromosomes.html): Larry Moran describes this marked chromosomal inversions (see also [Hermann Muller Invented the Balancer Chromosome](http://sandwalk.blogspot.com/2008/10/hermann-muller-invented-balancer.html)).
</li>
<li>
[The Genetics of Voting](http://bayblab.blogspot.com/2008/10/genetics-of-voting.html): From Bayblab, a post on the heritability of filling the ballot box.
</li>
<li>
[Science Tuesday: Back into the hornets’ nest](http://www.afreeman.org/2008/07/15/science-tuesday-back-into-the-hornets-nest/): A free man tries to understand the causes of autism.
</li>
<li>
[Freedom of the Genetic Press? Can newly created letters of life’s alphabet be patented?](http://www.tompainesghost.com/2008/09/freedom-of-genetic-press-can-newly.html): Tom Payne’s Ghost considers the patenting of novel amino-acids.
</li>
<li>
[Slightly Deleterious in Trans](http://scienceblogs.com/evolgen/2008/10/slightly_deleterious_in_trans.php): I wrote about a new chapter in the cis vs. trans debate.
</li>


That’s it for this month’s edition of [Mendel’s Garden](http://mendels-garden.blogspot.com/). If you’d like to host in December, or a subsequent month, shoot me an email (evolgen-at-yahoo-dot-com).

#I Voted . . .  Blindly
Date: November 4, 2008

**If you hold on until the end of the post, you’ll see that it’s got science content. But you’ll have to wait until the end.**


First, here’s how it starts. I had an optometrist appointment at 8am this morning. I could have voted prior to the appointment, but the polling place is in between the optometrist’s office and work, so I figured I’d vote after getting my eyes checked. Big mistake. The eye doctor dilated my pupils, which, much to my surprise, made me far-sighted (I’d never had both pupils dilated simultaneously before). I did not realize this until, when checking out of the office, I could not read the credit card receipt I signed. I thought, “Fuck, how the hell am I gonna vote if I can’t read the ballot?”


An aside: the doctor had a picture of his daughter posing next to Dick Cheney with her violin. Doc said something about her being a musician, so I imagine she won some competition that earned her a trip to the White House.


So, I ride my bike to the polling place (don’t worry, the pupil dilation didn’t affect my distance vision). Thankfully, there was a short line. When I checked in, I asked the retired people how small the writing on the ballot is. They pointed me to a sample ballot across the room. I could not read it. Ever resourceful, I asked if anyone had reading glasses. I figured one of the five retirees staffing the place would be able to lend me a pair. They did not. Luckily, a nice lady waiting in line to vote let me borrow her specs, which, while not perfect, allowed me figure out which line on the ballot corresponded to each party.





[<img alt="i-d8984c62b8254025bcd5c2258d774345-voting_machine.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-d8984c62b8254025bcd5c2258d774345-voting_machine.jpg"/>](http://flickr.com/photos/mhaithaca/242556582/)


Next issue: figuring out my county’s neolithic voting contraption. I walked into the booth, where I encountered a large red lever and a “ballot”. I’m not sure if there were instructions inside the booth because I couldn’t read much of anything. “What am I supposed to do?” I asked. No response. “What am I supposed to do?” Still no response. The one armed lady in charge of operating the voting contraption did not think that it was her obligation to tell me how to operate my side of the device. I’m pretty sure it was. The retired people let me down again. Finally, she told me that I should pull the red lever, which would close the curtain. After that, I was supposed to tick off my choice on the “ballot”. I put ballot in scare quotes because it was intimidating. The so-called ballot was a large sheet of paper in a grid, with the columns indicating the offices I’d be voting for, and the rows labeled with the party names. I could only make out the party names and the offices I was voting for; the rest of the “ballot” was a blur, literally. I voted by flipping a black switch in the square on the grid representing the candidate I supported for that office. Finally, I was able to escape the booth by flipping the red lever back to its starting position.


I could barely read the ballot, but I think I voted this morning. I blame this ordeal on my optometrist, who is obviously a Republican operative trying to suppress the vote of all those socialists in a small college town. How else would you explain the the photo of Dick Cheney?


Now for the science content: How does pupil dilation work? Is [tropicamide](http://en.wikipedia.org/wiki/Tropicamide) the primary drug used? How does it target just the nerves involved in controlling the pupil? And why does dilating your pupils make you far-sighted?

#Genes in the Post-Genomic Era
Date: November 12, 2008

You would think that geneticists would have a good definition of “gene”. After all, genes are what we study. In introductory biology courses, you may have been introduced to the concept of the gene as the unit of heredity. That’s all well and good, but when you begin to study genes at a molecular level (i.e., looking at DNA sequences), that definition ceases to be practical. The advent of DNA sequencing led to the concept of the gene as an open reading frame, and the post-genomic era has challenged the very idea of the gene.


I’ve previously discussed the definition of gene ([What is a gene?](http://scienceblogs.com/evolgen/2006/10/what_is_a_gene.php), [What is a gene?](http://scienceblogs.com/evolgen/2007/07/what_is_a_gene_1.php) — yes, two different posts with the same title), but I didn’t get into very many details. Alas, I don’t feel like spending much time laying out my opinion, suffice it to say I think “gene” is an obsolete, overly generic term that should be replaced by a more specific term whenever possible. Luckily, the New York Times has published an article by Carl Zimmer sketching out some of the possible interpretations ([Now: The Rest of the Genome ](http://www.nytimes.com/2008/11/11/science/11gene.htm)). This lets me pick and chose my favorite meaning from a variety of opinions represented in Carl’s piece.





A lot Zimmer’s article deals with the results from the pilot [ENCODE](http://www.genome.gov/10005107) project. One part of the project was a careful examination of which DNA sequences are transcribed into RNA. This led to some remarkable findings, including the discovery that a lot of transcripts consist of sequences encoded in different parts of the genome:


> 
Encode’s results reveal the genome to be full of genes that are deeply weird, at least by the traditional standard of what a gene is supposed to be. “These are not oddities — these are the rule,” said Thomas R. Gingeras of Cold Spring Harbor Laboratory and one of the leaders of Encode.
A single so-called gene, for example, can make more than one protein. In a process known as alternative splicing, a cell can select different combinations of exons to make different transcripts. Scientists identified the first cases of alternative splicing almost 30 years ago, but they were not sure how common it was. Several studies now show that almost all genes are being spliced. The Encode team estimates that the average protein-coding region produces 5.7 different transcripts. Different kinds of cells appear to produce different transcripts from the same gene.
Even weirder, cells often toss exons into transcripts from other genes. Those exons may come from distant locations, even from different chromosomes.
So, Dr. Gingeras argues, we can no longer think of genes as being single stretches of DNA at one physical location.
“I think it’s a paradigm shift in how we think the genome is organized,” Dr. Gingeras said.



Another highly touted finding from ENCODE was that the majority of the genome is transcribed. This led some people to conclude that much of the genome consists of undescribed functional elements.


> 
These discoveries left scientists wondering just how much noncoding RNA our cells make. The early results of Encode suggest the answer is a lot. Although only 1.2 percent of the human genome encodes proteins, the Encode scientists estimate that a staggering 93 percent of the genome produces RNA transcripts.
John Mattick, an Encode team member at the University of Queensland in Australia, is confident that a lot of those transcripts do important things that scientists have yet to understand. “My bet is the vast majority of it — I don’t know whether that’s 80 or 90 percent,” he said.



That would mean the human genome is chock full of genes. However, just because something is transcribed does not necessarily mean that it is functional. Many sequences may be aberrantly transcribed, representing merely background noise. That is, a lot of the potential “genes” aren’t really genes at all. Of all the people quoted in the article, I find myself agreeing with Ewan Birney and David Haussler the most:


> 
Despite the importance of noncoding RNA, Dr. Birney suspects that most of the transcripts discovered by the Encode project do not actually do much of anything. “I think it’s a hypothesis that has to be on the table,” he said.
David Haussler, another Encode team member at the University of California, Santa Cruz, agrees with Dr. Birney. “The cell will make RNA and simply throw it away,” he said.
Dr. Haussler bases his argument on evolution. If a segment of DNA encodes some essential molecule, mutations will tend to produce catastrophic damage. Natural selection will weed out most mutants. If a segment of DNA does not do much, however, it can mutate without causing any harm. Over millions of years, an essential piece of DNA will gather few mutations compared with less important ones.
Only about 4 percent of the noncoding DNA in the human genome shows signs of having experienced strong natural selection. Some of those segments may encode RNA molecules that have an important job in the cell. Some of them may contain stretches of DNA that control neighboring genes. Dr. Haussler suspects that most of the rest serve no function.



We’re still left without a concrete definition of a gene, which leads me back to my original conclusion: we should simply abandon the term when dealing with anything beyond simple classical genetics. The gene is far too general, and more specific terminology is warranted in most cases. And I haven’t even touched on the importance of epigenetics (i.e., heritable chromatin modifications, DNA methylation, etc.) and how that affects our definitions.

#Is Nested Clade Analysis Worthwhile?
Date: November 16, 2008

<img alt="i-0f31133106de1e9748bd4fd398d96881-nested_clade.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-0f31133106de1e9748bd4fd398d96881-nested_clade.jpg"/>


Population biologists often want to infer the demographic history of the species they study. This includes identifying population subdivision, expansion, and bottlenecks. Genetic data sampled from multiple individuals can often be applied to study population structure. When phylogenetic methods are used to link evolutionary relationships to geography, the approaches fall under the guise of [phylogeography](http://en.wikipedia.org/wiki/Phylogeography).


The past decade has seen the rise in popularity of a particular phylogeographical approach for intra-specific data: nested clade analysis ([Templeton et al. 1995](http://www.genetics.org/cgi/content/abstract/140/2/767); [Templeton 2004](http://dx.doi.org/10.1046/j.1365-294X.2003.02041.x)). Many of the methods used in intra-specific phylogeography have been called into question because of their lack of statistical rigor, as I have described previously ([How do you really feel, Dr. Wakely?](http://scienceblogs.com/evolgen/2008/09/how_do_you_really_feel_dr_wake.php)). Nested clade phylogeographical analysis (NCPA) is no exception. Lacey Knowles summarizes the criticisms of NCPA in the most recent issue of **Evolution** ([Why does a method that fails continue to be used?](http://www3.interscience.wiley.com/journal/121483449/abstract)).





The strongest part of Knowles’ critique focuses on one primary issue: NCPA tends to result in false positives. While NCPA does an adequate job of inferring actual demographic events (such as population subdivision), it falsely identifies extra events. Knowles points out that it is especially biased towards inferring isolation by distance when there has not been any. Interestingly, these false inferences occur with both empirical data (for which the demographic history is assumed to be well known) and simulated data (for which the demographic history is known). Alan Templeton (the creator of NCPA and its most ardent defender) argues that the simulation studies are not an adequate test of NCPA because they only offer simple evolutionary scenarios. However, Knowles points out that if NCPA fails with simple scenarios, how can it be trusted with the complicated ones that exist in nature?


NCPA is a very popular method — Remy Petit identified over 1700 citations as of about one year ago ([doi:10.1111/j.1365-294X.2007.03589.x](http://dx.doi.org/10.1111/j.1365-294X.2007.03589.x)). Additionally, Knowles points out that a six year old critique of NCPA ([Knowles and Maddison 2002](http://www.ncbi.nlm.nih.gov/pubmed/12453245)) has been cited 210 times, often by empirical studies that still used NCPA! That raises the question: why do people continue to use NCPA if it hasn’t been shown to work? It can’t be because they don’t know of the limitations of NCPA — they’re citing papers that layout those limitations.


Finally, I will relate this to previous rants on evolgen. Knowles points out that some of the criticisms of NCPA rely on the inference of historical events from simulation studies of a single locus. As I have mentioned previously, inference of historical demographic events from a single locus is not acceptable (see [here](http://www.google.com/search?q=mtdna&amp;q=(site%3Aevolgen.blogspot.com+|+site%3Ascienceblogs.com%2Fevolgen%2F+)&amp;btnG=Search) and [here](http://scienceblogs.com/evolgen/2008/03/inferring_demographic_history.php)). There is so much stochastic noise in evolutionary systems, and trying to identify demographic history using a sample size of one does not take the large variance of the system into account. Templeton and other NCPA defenders argue that simulations using a single locus are not an adequate test of NCPA. But 88% of the NCPA studies Knowles identified used only a single locus. Not only are people using a method that has never been shown to work, but they are also using the method with insufficient data. Double fail!


---



Knowles and Maddison 2002. Statistical phylogeography. Mol Ecol 11: 2623-2635 [[link](http://www.ncbi.nlm.nih.gov/pubmed/12453245)]


Knowles 2008. Why does a method that fails continue to be used? Evolution 62: 2713-2717 [[link](http://www3.interscience.wiley.com/journal/121483449/abstract)]


Petit 2008. The coup de grâce for the nested clade phylogeographic analysis? Mol Ecol 17: 516 – 518 [doi:10.1111/j.1365-294X.2007.03589.x](http://dx.doi.org/10.1111/j.1365-294X.2007.03589.x)


Templeton et al. 1995. Separating Population Structure from Population History: A Cladistic Analysis of the Geographical Distribution of Mitochondrial DNA Haplotypes in the Tiger Salamander, **Ambystoma tigrinum**. Genetics 140: 767-782 [[link](http://www.genetics.org/cgi/content/abstract/140/2/767)]


Templeton 2004. Statistical phylogeography: methods of evaluating and minimizing inference errors. Mol Ecol 13: 789 – 809 [doi:10.1046/j.1365-294X.2003.02041.x](http://dx.doi.org/10.1046/j.1365-294X.2003.02041.x)

#Genome Sequenced, Published via Press Release
Date: November 19, 2008

Back in the day, you could sequence a genome and get a *Nature* paper out of it. Pretty soon, the sexiness of genome sequencing wore off, and it took a bit more to get into a vanity journal. You had to sequence something cute and cuddly, something extinct, or a lot of genomes at once. Any other genome sequencing projects were relegated to lower tier journals.


Now, it appears that even sequencing the genome of charismatic megafauna only gets you a press release. As TR Gregory points out, the sequencing of the Kangaroo genome was announced in such a manner ([Science by press release](http://genomicron.blogspot.com/2008/11/science-by-press-release.html)). But check out the title on the press release:


[Australian First: Kangaroo Genome Mapped](http://www.sciencedaily.com/releases/2008/11/081118092253.htm)


They report that the genome was “mapped”. Not sequenced. This is the [incorrect terminology](http://www.google.com/search?q=mapped+decoded+sequenced&amp;q=(site%3Aevolgen.blogspot.com+|+site%3Ascienceblogs.com%2Fevolgen%2F+)&amp;btnG=Search). One the positive side, at least they didn’t say the genome was [decoded](http://us.penguingroup.com/nf/Book/BookDisplay/0,,9780670063581,00.html).

#Call for Mendel’s Garden #26 Submissions
Date: November 24, 2008

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://mendels-garden.blogspot.com/)


The 26th edition of Mendel’s Garden will be hosted by [A Free Man](http://www.afreeman.org/) on December 7. If you have written a blog post about any topics in Genetics in the past month or so, send a link to Chris (chris[at]afreeman[dot]org) to be included in the carnival.


We’re also looking for hosts for upcoming editions. If you would like to host the original genetics blog carnival, send me an email (evolgen-at-yahoo-dot-com). Every month from February onward is available.

#All those types of speciation
Date: December 16, 2008

Over at Wilkins’ cabana, there’s a post ([Some new work on speciation and species](http://scienceblogs.com/evolvingthoughts/2008/12/some_new_work_on_speciation_an.php)) on a paper by Nitin Phadnis and Allen Orr ([doi:10.1126/science.1163934](http://dx.doi.org/10.1126/science.1163934)). Phadnis and Orr isolated a gene responsible for both reproductive isolation and sex-ratio distortion between two populations of *Drosophila pseudoobscura*. Wilkins doesn’t like speciation genes, and he’s rails on the concept in his post.





What I’m interested in are the comments on Wilkins’ post — primarily the confusion over what the speciation gene hunters mean when they talk about different flavors of speciation. Most population geneticists talk about three types of speciation: allopatric, sympatric, and parapatric. Wilkins’ commenters get caught up in the traditional, geographic definitions of those terms. Specifically, allopatry refers to populations with non-overlapping ranges, sympatry refers to populations with completely overlapping ranges, and parapatry refers to populations with adjacent ranges.


However, when population geneticists use those terms, they do not refer to geography. Instead, they refer to measures of gene flow, using the parameter ‘m’. That parameter is the probability that an individual from one population mates with an individual from another population. If m=0, there is no interbreeding between the two populations (these are allopatric populations). If m=0.5, there is random mating between the populations (these are sympatric populations). And 0<m<0.5 are="" means="" p="" parapatric.<="" populations="" the="">
<p>Larry Moran was also hanging out in the comments of Wilkins’ post, downplaying the importance of natural selection in speciation. Orr (and Jerry Coyne) have previously argued ([Speciation](http://www.amazon.com/Speciation-Jerry-Coyne/dp/0878930892)) that natural selection is important for all flavors of speciation (ie, all values of m). I hope it’s obvious how selection against hybrids is important for m=0.5 (or close to it). However, many people think that sympatric speciation is rare, if not impossible. Therefore, natural selection may not be important when averaged over all speciation events.

But Coyne and Orr also argued that if m=0 (allopatry), natural selection may still be important for speciation. They point to laboratory experiments where populations of Drosophila were reared in separate vials for many generations. The vials subjected to the same environment did not evolve reproductive barriers, while those reared in different environments become more reproductively isolated. This goes against the tradition view of allopatric speciation (that of Ernst Mayr), in which it was driven purely by genetic drift.

#Do people still use microarrays?
Date: December 17, 2008

Larry Moran points to a couple of posts critical of microarrays ([The Problem with Microarrays](http://sandwalk.blogspot.com/2008/12/problem-with-microarrays.html)):


- [Why microarray study conclusions are so often wrong](http://www.johndcook.com/blog/2008/12/06/why-microarray-studies-are-often-wrong/)
- [Three reasons to distrust microarray results](http://reproducibleresearch.org/blog/2008/12/10/three-reasons-to-distrust-microarray-results/)


[Microarrays](http://en.wikipedia.org/wiki/DNA_microarray) are small chips that are covered with short stretches of single stranded DNA. People hybridize DNA from some source to the microarray, which lights up if the DNA hybridizes to the probes on the array.


Most biologists are familiar with microarrays being used to measure gene expression. In this case, transcribed DNA is hybridized to the array, and the intensity of the signal is used as a proxy for the transcriptional level of a large sample of genes. Other uses include identifying [copy number polymorphism](http://dx.doi.org/10.1126/science.1158078), [genotyping single nucleotide polymorphisms](http://www.affymetrix.com/products_services/arrays/specific/genome_wide_snp6/genome_wide_snp_6.affx) (SNPs), and [capturing sequences of interest](http://www.nature.com/nmeth/journal/v4/n11/abs/nmeth1111.html) for downstream analysis.





However, many of these uses are much better implemented with next generation sequencing. For example:


- Gene expression can be measured using Solexa sequencing ([doi:10.1101/gr.079558.108](http://dx.doi.org/10.1101/gr.079558.108)). This digital quantification is far more precise than microarray analysis, which relies on hybridization intensities.
- Copy number polymorphism can be identified by 454 sequencing using paired-end reads ([doi:10.1126/science.1149504](http://dx.doi.org/10.1126/science.1149504)).
- SNP genotyping can be performed with next-gen sequencing ([doi:10.1016/j.gde.2006.10.009](http://dx.doi.org/10.1016/j.gde.2006.10.009)).
- Additionally, Solexa sequencing is replacing microarrays in the high throughput identification of DNA sequences in chromatin immunoprecipitation ([ChIP-seq](http://www.nature.com/nmeth/journal/v4/n8/abs/nmeth0807-613.html))


Now, all of these techniques require a completely sequenced genome (or transcriptome). However, so do microarrays. Therefore, the up front needs aren’t very different. Also, using microarrays to capture sequences can’t be replaced by another technology. But it does rely on next-generation sequencing for downstream analysis.


Okay, so the question “Do people still use microarrays?” is a bit of hyperbole. But will microarrays be obsolete any time soon? Not if people are still using [Sanger sequencing](http://en.wikipedia.org/wiki/DNA_sequencing#Chain-termination_methods).

#The Implementation of Molecular Evolution for the Masses
Date: December 18, 2008

A couple of years ago, there was talk in the bioblogosphere about getting the general public interested in bioinformatics and molecular evolution:


- [Amateur bioinformatics?](http://nsaunders.wordpress.com/2006/02/08/amateur-bioinformatics/)
- [Lowering the Ivory Tower with Molecular Evolution](http://salamandercandy.wordpress.com/2006/07/17/lowering-the-ivory-tower-with-molecular-evolution/)
- [Molecular Evolution for the Masses](http://scienceblogs.com/evolgen/2006/07/molecular_evolution_for_the_ma.php)


The idea was inspired by the findings of armchair astronomers — people who have no professional training, but make contributions to astronomy via their stargazing hobbies. With so much data available in publicly accessible databases, there’s no reason we can’t motivate armchair biologists to start mining for interesting results.


But how do we train these new comp-bio code-monkeys? The field of bioinformatics requires both some computational skills, as well as an understanding of biology. Finding people with both skill sets (and interests) can be tricky. Well, a framework has been laid out in a recent paper in PLoS Biology for teaching the skills ([doi:10.1371/journal.pbio.0060296](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0060296)). The authors present a web-based interface through which students implement standard online tools for DNA sequence analysis ([Annotathon](http://annotathon.univ-mrs.fr/)).





The course described in the paper takes advantage of the vast amount of data deposited in sequence repositories from metagenomic projects (specifically, the Global Ocean Sampling sequences). Starting with these data, the students perform simple molecular evolutionary analysis, including gene prediction, alignment, and phylogenetic construction. Here’s how the authors summarize their course:


> 
The goal of the course is to teach students how to computationally annotate biological sequences (DNA and protein sequences). The starting point is a short stretch of DNA sequence (such as a single metagenomic sequencing read) that students are asked to study according to two major lines of inquiry: (1) prediction of gene product putative function and (2) prediction of taxonomic group of origin.



The question remains: how can we translate these courses offered at universities to the general public? Can we inspire armchair computational biologists to analyze data outside of the classroom?


---



Hingamp P, Brochier C, Talla E, Gautheret D, Thieffry D, et al. (2008) Metagenome Annotation Using a Distributed Grid of Undergraduate Students. PLoS Biol 6(11): e296 [doi:10.1371/journal.pbio.0060296](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0060296)

#Eric Lander Teaches?
Date: December 20, 2008

[John Hawks points out](http://johnhawks.net/node/1761) that [Eric Lander has been appointed to co-chair Obama’s Council of Advisers on Science and Technology](http://www.nytimes.com/aponline/2008/12/20/us/AP-Obama.html?_r=2) along with science adviser [John Holdren](http://belfercenter.ksg.harvard.edu/experts/140/john_p_holdren.html) and Nobel Laureate [Harold Varmus](http://www.mskcc.org/mskcc/html/1780.cfm). Here’s how the AP article describes Lander:


> 
[Lander](http://www.wi.mit.edu/research/faculty/lander.html), who teaches at both MIT and Harvard, founded the [Whitehead Institute](http://www.wi.mit.edu/)-MIT Center for Genome Research in 1990, which became part of the [Broad Institute](http://www.broad.mit.edu/) in 2003. A leading researcher in the Human Genome Project, he and his colleagues are using the findings to explore the molecular mechanisms behind human disease.






Wait, Eric Lander teaches? Really? It turns out he does:


> 
In addition to his research, Eric is an enthusiastic teacher. He has taught MIT’s core introductory biology course for a decade and, in 1992, won the Baker Memorial Award for Undergraduate Teaching at MIT. He has lectured to both scientific and lay audiences about the medical and social implications of genetics, and delivered a special Millennium Lecture at the White House in 2000.<br/>[[link](http://www-genome.wi.mit.edu/node/543)]



But this really points out flaw in how the general public, including journalists, understand academia. If I were to describe Eric Lander’s professional appointment (or nearly any other research professors appointment, for that matter), “teaching” would not be the first item on the list. In fact, a lot of profs don’t teach at all. Research comes first, then advising grad students and post-docs (which is a kind of teaching, but not the in classroom variety that I imagine most people picture when they say so-and-so teaches at a university) and getting grants (which could be bundled, along with writing papers, under the umbrella of “research”), followed by teaching (if they do that at all). However, most folks only saw their professors as undergrads (if they went to college at all), and in that environment they were teaching classes.


Eric Lander is a professor, not a teacher. And he’s also taking steps into politics under the upcoming administration. This has got me wondering whether he may move into the position of [director of the NHGRI](http://scienceblogs.com/evolgen/2008/05/goodbye_frank.php) (a post vacated by Francis Collins earlier this year).

#Mendel’s Garden #27 – Call for Submissions
Date: January 2, 2009

[<img alt="i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg" src="http://scienceblogs.com/evolgen/wp-content/blogs.dir/296/files/2012/04/i-c7222c65ce70c8ece48a4bec1b54c8b8-mendels_garden.jpg"/>](http://mendels-garden.blogspot.com/)


[Mendel’s Garden](http://mendels-garden.blogspot.com/) is the original genetics blog carnival. The next edition will be hosted by Jeremy at [Another Blasted Weblog](http://wp.jeremycherfas.net/). If you would like to submit a blog post to be included in the carnival, send an email to Jeremy (jcherfas at mac dot com). The carnival should be posted within the next few days, so get your submissions in ASAP.


Also, hosts are needed for future editions of Mendel’s Garden. If you would like to host, please send me an email (evolgen at yahoo dot com). A new edition is usually posted around the first Sunday of each month.

#This is a Good-bye Post
Date: January 16, 2009

This is the final post ever at evolgen. It was a fun 4+ years, the last three spent at ScienceBlogs, but it has come time for me to close up shop.


When I [first got into blogging](http://evolgen.blogspot.com/), I did it as a way to share what was on my mind to the few people who would read what I had to say (usually in topics related to evolution and genetics, but not always). It was a fun hobby, and my blog gave me a public venue to talk about articles I was reading, concepts that I found interesting, and summarize important areas of research.


However, the blog has begun to feel more like a burden. I no longer post because I feel like I have something I want to say, but rather out of obligation (to my contract with Seed, to the five readers who read this site regularly, to my own need to keep generating new content because the blog needs new content). It wasn’t fun anymore. This is reflected in the sporadic posting over the past year — small bursts of inspiration surrounded by frequent periods of ennui (what you now call [meh](http://www.theregister.co.uk/2008/11/17/meh/)).


I had a really good time blogging at the ScienceBorg, but the time has come for me to move on. I’m not sure whether I’ll ever start blogging again, but, if I do, it will be at a new venue. I’ll make sure to let y’all know what’s up via obnoxious emails and requests for more highly-trafficked folks to link to my new internet cave.

